{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Welcome to the command line!  \n",
    "Lets go over some basic commands  \n",
    "ls lists everything in the current directory -- try it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;34mDesktop\u001b[0m    \u001b[01;34mDownloads\u001b[0m                                \u001b[01;34mMusic\u001b[0m     \u001b[01;34mPublic\u001b[0m     \u001b[01;34mVideos\u001b[0m\n",
      "\u001b[01;34mDocuments\u001b[0m  Lgn_nod_double_BASH_CSHL_20191016.ipynb  \u001b[01;34mPictures\u001b[0m  \u001b[01;34mTemplates\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# just type ls and hit enter\n",
    "# or you can cut and paste this whole block of text\n",
    "# the '#' symbols make these lines into comments\n",
    "ls"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "cd lets you change the current directory -- try it!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# change directory to 'Desktop'\n",
    "cd Desktop"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What is in the \"Desktop\" directory?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "???"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# we can use 'relative paths' to quickly move up, out of the Desktop directory\n",
    "cd .. # moves up one level"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# we can use 'absolute paths' to go to a precise location\n",
    "cd /scratch # move to a folder that has a full address of \"/scratch\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# By default in CyVerse Atmosphere, \n",
    "# we need to change the ownership of the /scratch drive \n",
    "# to use as our project space\n",
    "# we can use ls to see the current ownership\n",
    "\n",
    "ls -l / #option '-l' lists more information\n",
    "# current ownership is 'root'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[sudo] password for sal269: \r\n"
     ]
    }
   ],
   "source": [
    "# We need special privileges to change the ownership\n",
    "# we can use the 'super user do', sudo command to access these privileges\n",
    "# You will be prompted for your CyVerse password\n",
    "\n",
    "sudo chown $(whoami) /scratch"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " did it work?  \n",
    " how can we check?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Enter your current iRODS password:\r\n"
     ]
    }
   ],
   "source": [
    "# To copy the data required for this tutorial\n",
    "# we will need to use a set of software tools called IRODS.\n",
    "# This seems somewhat unique to CyVerse -- and it is annoying\n",
    "\n",
    "# initialize your connection to the CyVerse datastore\n",
    "\n",
    "iinit"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "follow the prompts and enter:  \n",
    " 1) your cyverse password  \n",
    " 2) host = data.cyverse.org  \n",
    " 3) port = 1247  \n",
    " 4) zone = iplant  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/iplant/home/sal269:\n",
      "  wab2_WTparent_MA311_Alignment_SVcalling.ipynb\n",
      "  C- /iplant/home/sal269/BLIND_B73_SHSL002\n",
      "  C- /iplant/home/sal269/BLIND_BTx623_SHSL001\n",
      "  C- /iplant/home/sal269/BLIND_network\n",
      "  C- /iplant/home/sal269/BLIND_phylostrat\n",
      "  C- /iplant/home/sal269/CSHL_Cereal_Genomics_2019\n",
      "  C- /iplant/home/sal269/FAS\n",
      "  C- /iplant/home/sal269/FAS1_LSK109_190407_rerun\n",
      "  C- /iplant/home/sal269/FAS1_natrev_190422\n",
      "  C- /iplant/home/sal269/FUN_RNAseq\n",
      "  C- /iplant/home/sal269/IGI_proposal\n",
      "  C- /iplant/home/sal269/LA1987_BSA\n",
      "  C- /iplant/home/sal269/Lg2_RNAseq\n",
      "  C- /iplant/home/sal269/Lgn\n",
      "  C- /iplant/home/sal269/Lgn_nod_double_RNAseq\n",
      "  C- /iplant/home/sal269/Maize_SAM_CTs\n",
      "  C- /iplant/home/sal269/Mo17_genome\n",
      "  C- /iplant/home/sal269/PGEC_imaging\n",
      "  C- /iplant/home/sal269/PMB290\n",
      "  C- /iplant/home/sal269/RNAmappeR\n",
      "  C- /iplant/home/sal269/RNAseq_data\n",
      "  C- /iplant/home/sal269/SML_SAM_LMrnaseq\n",
      "  C- /iplant/home/sal269/Solanum RNAseq\n",
      "  C- /iplant/home/sal269/TOR_riboseq\n",
      "  C- /iplant/home/sal269/VM_output\n",
      "  C- /iplant/home/sal269/analyses\n",
      "  C- /iplant/home/sal269/at_genome\n",
      "  C- /iplant/home/sal269/coge_data\n",
      "  C- /iplant/home/sal269/dhlamo_gatk\n",
      "  C- /iplant/home/sal269/gatk\n",
      "  C- /iplant/home/sal269/larp1\n",
      "  C- /iplant/home/sal269/maize_genome\n",
      "  C- /iplant/home/sal269/multiqc\n",
      "  C- /iplant/home/sal269/old_image_data\n",
      "  C- /iplant/home/sal269/reptin_RNAseq\n",
      "  C- /iplant/home/sal269/reptin_RNAseq_190420\n",
      "  C- /iplant/home/sal269/snps\n",
      "  C- /iplant/home/sal269/sorghum_genome\n",
      "  C- /iplant/home/sal269/sorghum_panicles\n",
      "  C- /iplant/home/sal269/wab2_BSA\n",
      "  C- /iplant/home/sal269/wab2_WTparent_MA311\n",
      "  C- /iplant/home/sal269/wab2_mutpool_lsk109_190325\n",
      "  C- /iplant/home/sal269/wab2_mutpool_rad004_190226\n",
      "  C- /iplant/home/sal269/wab2_parent_lsk109_190911\n"
     ]
    }
   ],
   "source": [
    "# check that your iRODS connection is set up\n",
    "\n",
    "ils #ls -- but i for iRODS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0/62 -  0.00% of files done   0.000/6914.627 MB -  0.00% of file sizes done\n",
      "Processing Zea_mays.AGPv3.30.repaired.gtf - 68.797 MB   2019-10-15.22:23:08\n",
      "Zea_mays.AGPv3.30.repaired.gtf - 45.864/68.797 MB - 66.67% done   2019-10-15.22:23:10\n",
      "Zea_mays.AGPv3.30.repaired.gtf - 68.797/68.797 MB - 100.00% done   2019-10-15.22:23:10\n",
      "1/62 -  1.61% of files done   68.797/6914.627 MB -  0.99% of file sizes done\n",
      "Processing ZmaysAGPv3.30.fa - 2004.922 MB   2019-10-15.22:23:10\n",
      "ZmaysAGPv3.30.fa - 39.999/2004.922 MB -  2.00% done   2019-10-15.22:23:12\n",
      "ZmaysAGPv3.30.fa - 199.995/2004.922 MB -  9.98% done   2019-10-15.22:23:13\n",
      "ZmaysAGPv3.30.fa - 239.995/2004.922 MB - 11.97% done   2019-10-15.22:23:13\n",
      "ZmaysAGPv3.30.fa - 399.991/2004.922 MB - 19.95% done   2019-10-15.22:23:14\n",
      "ZmaysAGPv3.30.fa - 519.988/2004.922 MB - 25.94% done   2019-10-15.22:23:15\n",
      "ZmaysAGPv3.30.fa - 679.984/2004.922 MB - 33.92% done   2019-10-15.22:23:16\n",
      "ZmaysAGPv3.30.fa - 719.984/2004.922 MB - 35.91% done   2019-10-15.22:23:16\n",
      "ZmaysAGPv3.30.fa - 879.980/2004.922 MB - 43.89% done   2019-10-15.22:23:17\n",
      "ZmaysAGPv3.30.fa - 999.977/2004.922 MB - 49.88% done   2019-10-15.22:23:17\n",
      "ZmaysAGPv3.30.fa - 1119.974/2004.922 MB - 55.86% done   2019-10-15.22:23:18\n",
      "ZmaysAGPv3.30.fa - 1279.971/2004.922 MB - 63.84% done   2019-10-15.22:23:19\n",
      "ZmaysAGPv3.30.fa - 1439.967/2004.922 MB - 71.82% done   2019-10-15.22:23:19\n",
      "ZmaysAGPv3.30.fa - 1541.207/2004.922 MB - 76.87% done   2019-10-15.22:23:20\n",
      "ZmaysAGPv3.30.fa - 2004.922/2004.922 MB - 100.00% done   2019-10-15.22:23:23\n",
      "2/62 -  3.23% of files done   2073.719/6914.627 MB - 29.99% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.1.ht2 - 658.589 MB   2019-10-15.22:23:28\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 39.999/658.589 MB -  6.07% done   2019-10-15.22:23:30\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 279.994/658.589 MB - 42.51% done   2019-10-15.22:23:31\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 524.639/658.589 MB - 79.66% done   2019-10-15.22:23:32\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 609.288/658.589 MB - 92.51% done   2019-10-15.22:23:33\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 613.939/658.589 MB - 93.22% done   2019-10-15.22:23:33\n",
      "ZmaysAGPv3.30.hisat2.1.ht2 - 658.589/658.589 MB - 100.00% done   2019-10-15.22:23:33\n",
      "3/62 -  4.84% of files done   2732.309/6914.627 MB - 39.51% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.2.ht2 - 489.824 MB   2019-10-15.22:23:35\n",
      "ZmaysAGPv3.30.hisat2.2.ht2 - 159.996/489.824 MB - 32.66% done   2019-10-15.22:23:37\n",
      "ZmaysAGPv3.30.hisat2.2.ht2 - 199.995/489.824 MB - 40.83% done   2019-10-15.22:23:38\n",
      "ZmaysAGPv3.30.hisat2.2.ht2 - 279.994/489.824 MB - 57.16% done   2019-10-15.22:23:38\n",
      "ZmaysAGPv3.30.hisat2.2.ht2 - 282.452/489.824 MB - 57.66% done   2019-10-15.22:23:38\n",
      "ZmaysAGPv3.30.hisat2.2.ht2 - 489.824/489.824 MB - 100.00% done   2019-10-15.22:23:39\n",
      "4/62 -  6.45% of files done   3222.132/6914.627 MB - 46.60% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.3.ht2 - 1.094 MB   2019-10-15.22:23:40\n",
      "5/62 -  8.06% of files done   3223.226/6914.627 MB - 46.61% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.4.ht2 - 489.824 MB   2019-10-15.22:23:42\n",
      "ZmaysAGPv3.30.hisat2.4.ht2 - 79.998/489.824 MB - 16.33% done   2019-10-15.22:23:43\n",
      "ZmaysAGPv3.30.hisat2.4.ht2 - 239.995/489.824 MB - 49.00% done   2019-10-15.22:23:44\n",
      "ZmaysAGPv3.30.hisat2.4.ht2 - 402.450/489.824 MB - 82.16% done   2019-10-15.22:23:44\n",
      "ZmaysAGPv3.30.hisat2.4.ht2 - 404.908/489.824 MB - 82.66% done   2019-10-15.22:23:44\n",
      "ZmaysAGPv3.30.hisat2.4.ht2 - 489.824/489.824 MB - 100.00% done   2019-10-15.22:23:45\n",
      "6/62 -  9.68% of files done   3713.050/6914.627 MB - 53.70% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.5.ht2 - 865.325 MB   2019-10-15.22:23:46\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 79.998/865.325 MB -  9.24% done   2019-10-15.22:23:47\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 199.995/865.325 MB - 23.11% done   2019-10-15.22:23:48\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 359.992/865.325 MB - 41.60% done   2019-10-15.22:23:49\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 536.324/865.325 MB - 61.98% done   2019-10-15.22:23:50\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 656.321/865.325 MB - 75.85% done   2019-10-15.22:23:50\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 672.657/865.325 MB - 77.73% done   2019-10-15.22:23:51\n",
      "ZmaysAGPv3.30.hisat2.5.ht2 - 865.325/865.325 MB - 100.00% done   2019-10-15.22:23:52\n",
      "7/62 - 11.29% of files done   4578.375/6914.627 MB - 66.21% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.6.ht2 - 498.777 MB   2019-10-15.22:23:55\n",
      "ZmaysAGPv3.30.hisat2.6.ht2 - 159.996/498.777 MB - 32.08% done   2019-10-15.22:23:57\n",
      "ZmaysAGPv3.30.hisat2.6.ht2 - 404.688/498.777 MB - 81.14% done   2019-10-15.22:23:58\n",
      "ZmaysAGPv3.30.hisat2.6.ht2 - 489.383/498.777 MB - 98.12% done   2019-10-15.22:23:59\n",
      "ZmaysAGPv3.30.hisat2.6.ht2 - 498.777/498.777 MB - 100.00% done   2019-10-15.22:23:59\n",
      "8/62 - 12.90% of files done   5077.152/6914.627 MB - 73.43% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.7.ht2 - 0.000 MB   2019-10-15.22:24:00\n",
      "9/62 - 14.52% of files done   5077.152/6914.627 MB - 73.43% of file sizes done\n",
      "Processing ZmaysAGPv3.30.hisat2.8.ht2 - 0.000 MB   2019-10-15.22:24:01\n",
      "10/62 - 16.13% of files done   5077.152/6914.627 MB - 73.43% of file sizes done\n",
      "Processing WT10_S8_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:01\n",
      "WT10_S8_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:03\n",
      "11/62 - 17.74% of files done   5112.487/6914.627 MB - 73.94% of file sizes done\n",
      "Processing WT13_S12_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:03\n",
      "WT13_S12_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:05\n",
      "WT13_S12_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:06\n",
      "12/62 - 19.35% of files done   5147.824/6914.627 MB - 74.45% of file sizes done\n",
      "Processing WT18_S4_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:06\n",
      "WT18_S4_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:08\n",
      "WT18_S4_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:08\n",
      "13/62 - 20.97% of files done   5183.160/6914.627 MB - 74.96% of file sizes done\n",
      "Processing WT24_S10_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:08\n",
      "WT24_S10_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:09\n",
      "WT24_S10_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:09\n",
      "14/62 - 22.58% of files done   5218.496/6914.627 MB - 75.47% of file sizes done\n",
      "Processing WT25_S7_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:09\n",
      "WT25_S7_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:11\n",
      "WT25_S7_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:11\n",
      "15/62 - 24.19% of files done   5253.832/6914.627 MB - 75.98% of file sizes done\n",
      "Processing WT38_S5_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:11\n",
      "WT38_S5_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:13\n",
      "WT38_S5_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:13\n",
      "16/62 - 25.81% of files done   5289.168/6914.627 MB - 76.49% of file sizes done\n",
      "Processing WT41_S6_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:14\n",
      "WT41_S6_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:14\n",
      "17/62 - 27.42% of files done   5324.504/6914.627 MB - 77.00% of file sizes done\n",
      "Processing WT55_S9_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:15\n",
      "WT55_S9_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:17\n",
      "WT55_S9_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:17\n",
      "18/62 - 29.03% of files done   5359.840/6914.627 MB - 77.51% of file sizes done\n",
      "Processing WT58_S3_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:17\n",
      "WT58_S3_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:20\n",
      "WT58_S3_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:20\n",
      "19/62 - 30.65% of files done   5395.176/6914.627 MB - 78.03% of file sizes done\n",
      "Processing WT78_S2_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:20\n",
      "WT78_S2_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:21\n",
      "20/62 - 32.26% of files done   5430.512/6914.627 MB - 78.54% of file sizes done\n",
      "Processing WT93_S11_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:21\n",
      "WT93_S11_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:22\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21/62 - 33.87% of files done   5465.848/6914.627 MB - 79.05% of file sizes done\n",
      "Processing WT95_S13_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:22\n",
      "WT95_S13_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:22\n",
      "WT95_S13_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:22\n",
      "22/62 - 35.48% of files done   5501.185/6914.627 MB - 79.56% of file sizes done\n",
      "Processing WT97_S1_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:23\n",
      "WT97_S1_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:23\n",
      "WT97_S1_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:23\n",
      "23/62 - 37.10% of files done   5536.521/6914.627 MB - 80.07% of file sizes done\n",
      "Processing dbl19_S46_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:23\n",
      "dbl19_S46_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:25\n",
      "dbl19_S46_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:25\n",
      "24/62 - 38.71% of files done   5571.857/6914.627 MB - 80.58% of file sizes done\n",
      "Processing dbl28_S48_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:26\n",
      "dbl28_S48_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:29\n",
      "25/62 - 40.32% of files done   5607.193/6914.627 MB - 81.09% of file sizes done\n",
      "Processing dbl31_S41_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:29\n",
      "dbl31_S41_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:29\n",
      "dbl31_S41_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:29\n",
      "26/62 - 41.94% of files done   5642.529/6914.627 MB - 81.60% of file sizes done\n",
      "Processing dbl50_S43_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:30\n",
      "dbl50_S43_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:32\n",
      "dbl50_S43_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:33\n",
      "27/62 - 43.55% of files done   5677.865/6914.627 MB - 82.11% of file sizes done\n",
      "Processing dbl54_S44_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:33\n",
      "dbl54_S44_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:34\n",
      "dbl54_S44_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:34\n",
      "28/62 - 45.16% of files done   5713.201/6914.627 MB - 82.62% of file sizes done\n",
      "Processing dbl56_S50_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:34\n",
      "dbl56_S50_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:35\n",
      "dbl56_S50_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:35\n",
      "29/62 - 46.77% of files done   5748.537/6914.627 MB - 83.14% of file sizes done\n",
      "Processing dbl57_S52_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:35\n",
      "dbl57_S52_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:35\n",
      "dbl57_S52_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:35\n",
      "30/62 - 48.39% of files done   5783.873/6914.627 MB - 83.65% of file sizes done\n",
      "Processing dbl65_S42_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:36\n",
      "dbl65_S42_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:37\n",
      "31/62 - 50.00% of files done   5819.209/6914.627 MB - 84.16% of file sizes done\n",
      "Processing dbl66_S49_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:38\n",
      "dbl66_S49_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:38\n",
      "dbl66_S49_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:38\n",
      "32/62 - 51.61% of files done   5854.545/6914.627 MB - 84.67% of file sizes done\n",
      "Processing dbl70_S47_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:39\n",
      "dbl70_S47_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:41\n",
      "33/62 - 53.23% of files done   5889.881/6914.627 MB - 85.18% of file sizes done\n",
      "Processing dbl88_S51_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:42\n",
      "dbl88_S51_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:42\n",
      "34/62 - 54.84% of files done   5925.217/6914.627 MB - 85.69% of file sizes done\n",
      "Processing dbl89_S40_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:43\n",
      "dbl89_S40_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:43\n",
      "dbl89_S40_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:43\n",
      "35/62 - 56.45% of files done   5960.553/6914.627 MB - 86.20% of file sizes done\n",
      "Processing dbl9_S45_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:43\n",
      "dbl9_S45_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:44\n",
      "36/62 - 58.06% of files done   5995.889/6914.627 MB - 86.71% of file sizes done\n",
      "Processing lgn17_S35_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:44\n",
      "lgn17_S35_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:45\n",
      "37/62 - 59.68% of files done   6031.226/6914.627 MB - 87.22% of file sizes done\n",
      "Processing lgn21_S36_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:45\n",
      "lgn21_S36_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:46\n",
      "lgn21_S36_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:46\n",
      "38/62 - 61.29% of files done   6066.562/6914.627 MB - 87.74% of file sizes done\n",
      "Processing lgn28_S37_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:46\n",
      "lgn28_S37_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:46\n",
      "39/62 - 62.90% of files done   6101.897/6914.627 MB - 88.25% of file sizes done\n",
      "Processing lgn33_S38_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:47\n",
      "lgn33_S38_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:49\n",
      "lgn33_S38_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:49\n",
      "40/62 - 64.52% of files done   6137.234/6914.627 MB - 88.76% of file sizes done\n",
      "Processing lgn37_S33_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:50\n",
      "lgn37_S33_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:50\n",
      "lgn37_S33_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:50\n",
      "41/62 - 66.13% of files done   6172.570/6914.627 MB - 89.27% of file sizes done\n",
      "Processing lgn39_S27_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:50\n",
      "lgn39_S27_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:52\n",
      "lgn39_S27_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:52\n",
      "42/62 - 67.74% of files done   6207.906/6914.627 MB - 89.78% of file sizes done\n",
      "Processing lgn3_S39_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:53\n",
      "lgn3_S39_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:24:54\n",
      "lgn3_S39_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:54\n",
      "43/62 - 69.35% of files done   6243.241/6914.627 MB - 90.29% of file sizes done\n",
      "Processing lgn42_S34_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:54\n",
      "lgn42_S34_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:55\n",
      "44/62 - 70.97% of files done   6278.578/6914.627 MB - 90.80% of file sizes done\n",
      "Processing lgn46_S30_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:55\n",
      "lgn46_S30_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:24:59\n",
      "45/62 - 72.58% of files done   6313.914/6914.627 MB - 91.31% of file sizes done\n",
      "Processing lgn59_S28_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:24:59\n",
      "lgn59_S28_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:00\n",
      "lgn59_S28_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:00\n",
      "46/62 - 74.19% of files done   6349.250/6914.627 MB - 91.82% of file sizes done\n",
      "Processing lgn72_S31_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:01\n",
      "lgn72_S31_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:02\n",
      "lgn72_S31_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:02\n",
      "47/62 - 75.81% of files done   6384.586/6914.627 MB - 92.33% of file sizes done\n",
      "Processing lgn77_S29_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:02\n",
      "lgn77_S29_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:03\n",
      "48/62 - 77.42% of files done   6419.922/6914.627 MB - 92.85% of file sizes done\n",
      "Processing lgn8_S32_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:04\n",
      "lgn8_S32_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:05\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "lgn8_S32_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:05\n",
      "49/62 - 79.03% of files done   6455.258/6914.627 MB - 93.36% of file sizes done\n",
      "Processing nod11_S24_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:05\n",
      "nod11_S24_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:06\n",
      "nod11_S24_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:06\n",
      "50/62 - 80.65% of files done   6490.594/6914.627 MB - 93.87% of file sizes done\n",
      "Processing nod14_S18_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:06\n",
      "nod14_S18_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:06\n",
      "nod14_S18_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:07\n",
      "51/62 - 82.26% of files done   6525.930/6914.627 MB - 94.38% of file sizes done\n",
      "Processing nod16_S15_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:07\n",
      "nod16_S15_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:08\n",
      "52/62 - 83.87% of files done   6561.266/6914.627 MB - 94.89% of file sizes done\n",
      "Processing nod2_S26_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:08\n",
      "nod2_S26_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:10\n",
      "nod2_S26_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:11\n",
      "53/62 - 85.48% of files done   6596.602/6914.627 MB - 95.40% of file sizes done\n",
      "Processing nod30_S17_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:11\n",
      "nod30_S17_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:11\n",
      "nod30_S17_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:12\n",
      "54/62 - 87.10% of files done   6631.938/6914.627 MB - 95.91% of file sizes done\n",
      "Processing nod36_S21_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:12\n",
      "nod36_S21_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:12\n",
      "nod36_S21_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:12\n",
      "55/62 - 88.71% of files done   6667.274/6914.627 MB - 96.42% of file sizes done\n",
      "Processing nod43_S22_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:13\n",
      "nod43_S22_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:13\n",
      "nod43_S22_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:13\n",
      "56/62 - 90.32% of files done   6702.611/6914.627 MB - 96.93% of file sizes done\n",
      "Processing nod47_S23_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:13\n",
      "nod47_S23_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:15\n",
      "nod47_S23_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:15\n",
      "57/62 - 91.94% of files done   6737.947/6914.627 MB - 97.44% of file sizes done\n",
      "Processing nod48_S16_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:15\n",
      "nod48_S16_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:16\n",
      "58/62 - 93.55% of files done   6773.283/6914.627 MB - 97.96% of file sizes done\n",
      "Processing nod60_S19_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:16\n",
      "nod60_S19_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:17\n",
      "nod60_S19_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:17\n",
      "59/62 - 95.16% of files done   6808.619/6914.627 MB - 98.47% of file sizes done\n",
      "Processing nod61_S25_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:17\n",
      "nod61_S25_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:17\n",
      "60/62 - 96.77% of files done   6843.955/6914.627 MB - 98.98% of file sizes done\n",
      "Processing nod62_S14_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:18\n",
      "nod62_S14_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:18\n",
      "nod62_S14_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:18\n",
      "61/62 - 98.39% of files done   6879.291/6914.627 MB - 99.49% of file sizes done\n",
      "Processing nod6_S20_L001_R1_001.fastq.gz - 35.336 MB   2019-10-15.22:25:19\n",
      "nod6_S20_L001_R1_001.fastq.gz - 17.668/35.336 MB - 50.00% done   2019-10-15.22:25:19\n",
      "nod6_S20_L001_R1_001.fastq.gz - 35.336/35.336 MB - 100.00% done   2019-10-15.22:25:19\n"
     ]
    }
   ],
   "source": [
    "# when your iRODS connection is set up,\n",
    "# get the data needed for the course\n",
    "\n",
    "# iRODS command iget\n",
    "# options are -r for directories and contents\n",
    "#             -P for a progress update\n",
    "#             -K for checksums to see that file was transferred completely\n",
    "#             -N 4 to use 4 CPUs\n",
    "iget -rPKf -N 4 /iplant/home/sal269/CSHL_Cereal_Genomics_2019/course_materials"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "# phew, that was a lot\n",
    "# change to the 'course_materials' directory\n",
    "\n",
    "cd course_materials"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;34mmaize_genome\u001b[0m  \u001b[01;34mraw_reads\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "# what is in there?\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT10_S8_L001_R1_001.fastq    lgn17_S35_L001_R1_001.fastq\n",
      "WT13_S12_L001_R1_001.fastq   lgn21_S36_L001_R1_001.fastq\n",
      "WT18_S4_L001_R1_001.fastq    lgn28_S37_L001_R1_001.fastq\n",
      "WT24_S10_L001_R1_001.fastq   lgn33_S38_L001_R1_001.fastq\n",
      "WT25_S7_L001_R1_001.fastq    lgn37_S33_L001_R1_001.fastq\n",
      "WT38_S5_L001_R1_001.fastq    lgn39_S27_L001_R1_001.fastq\n",
      "WT41_S6_L001_R1_001.fastq    lgn3_S39_L001_R1_001.fastq\n",
      "WT55_S9_L001_R1_001.fastq    lgn42_S34_L001_R1_001.fastq\n",
      "WT58_S3_L001_R1_001.fastq    lgn46_S30_L001_R1_001.fastq\n",
      "WT78_S2_L001_R1_001.fastq    lgn59_S28_L001_R1_001.fastq\n",
      "WT93_S11_L001_R1_001.fastq   lgn72_S31_L001_R1_001.fastq\n",
      "WT95_S13_L001_R1_001.fastq   lgn77_S29_L001_R1_001.fastq\n",
      "WT97_S1_L001_R1_001.fastq    lgn8_S32_L001_R1_001.fastq\n",
      "dbl19_S46_L001_R1_001.fastq  nod11_S24_L001_R1_001.fastq\n",
      "dbl28_S48_L001_R1_001.fastq  nod14_S18_L001_R1_001.fastq\n",
      "dbl31_S41_L001_R1_001.fastq  nod16_S15_L001_R1_001.fastq\n",
      "dbl50_S43_L001_R1_001.fastq  nod2_S26_L001_R1_001.fastq\n",
      "dbl54_S44_L001_R1_001.fastq  nod30_S17_L001_R1_001.fastq\n",
      "dbl56_S50_L001_R1_001.fastq  nod36_S21_L001_R1_001.fastq\n",
      "dbl57_S52_L001_R1_001.fastq  nod43_S22_L001_R1_001.fastq\n",
      "dbl65_S42_L001_R1_001.fastq  nod47_S23_L001_R1_001.fastq\n",
      "dbl66_S49_L001_R1_001.fastq  nod48_S16_L001_R1_001.fastq\n",
      "dbl70_S47_L001_R1_001.fastq  nod60_S19_L001_R1_001.fastq\n",
      "dbl88_S51_L001_R1_001.fastq  nod61_S25_L001_R1_001.fastq\n",
      "dbl89_S40_L001_R1_001.fastq  nod62_S14_L001_R1_001.fastq\n",
      "dbl9_S45_L001_R1_001.fastq   nod6_S20_L001_R1_001.fastq\n"
     ]
    }
   ],
   "source": [
    "# your RNAseq reads are in the raw_reads folder\n",
    "# how many are there?\n",
    "\n",
    "ls ./raw_reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     52      52    1585\n"
     ]
    }
   ],
   "source": [
    "# not easy to tell from just listing them\n",
    "# we can use a special unix character called a pipe \"|\"\n",
    "# to pass our ls output to another command wc, which can count words\n",
    "\n",
    "ls ./raw_reads | wc\n",
    "# output is #lines #words #bytes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "how many files are in your 'maize_genome' folder?  \n",
    "???"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">scaffold_68 dna:scaffold scaffold:AGPv3:scaffold_68:1:105:1\n",
      "TGAACAATGAGACATATATCTTGAAGAATATGCTTGAGTTATGGAGGGAATAACGTACTT\n",
      "TGACAGATGTCATCGTCGAGGGGGAGCGAATGCTTATATTGATCA\n",
      ">scaffold_140 dna:scaffold scaffold:AGPv3:scaffold_140:1:109:1\n",
      "ACTTTCTTAAACCTTCCTCCCTTTGATAAAGGTTTGGTATTTTTCGGTTTCTATTTTTGT\n",
      "TGTTCATTTTTTGTCATTAGGCAGATTAAAGGATATCTAAGTAAGATAG\n",
      ">scaffold_427 dna:scaffold scaffold:AGPv3:scaffold_427:1:110:1\n",
      "ATCCTGTTATTTTTTTACTAGGAATTCTGCTCCCTACGTAAAAGTTTTTTCGTTTAGGGG\n",
      "AAAAACCAAAAATAAAAAGAGAATGGAAGAATTCTTCTTATTCCATAAGA\n",
      ">scaffold_236 dna:scaffold scaffold:AGPv3:scaffold_236:1:114:1\n"
     ]
    }
   ],
   "source": [
    "# we can use a command called 'head' to look at the first few lines of a large file\n",
    "\n",
    "# this is the fasta format\n",
    "head ./maize_genome/ZmaysAGPv3.30.fa\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\tgramene\texon\t4854\t5188\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t5342\t5407\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t5857\t5975\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t6108\t6265\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t6362\t6517\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t6639\t6797\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t6918\t7120\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t7594\t7903\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\texon\t9193\t9652\t.\t-\t.\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n",
      "1\tgramene\tCDS\t5127\t5188\t.\t-\t2\ttranscript_id \"transcript:GRMZM2G059865_T01\"; gene_id \"gene:GRMZM2G059865\";\n"
     ]
    }
   ],
   "source": [
    "# this is the gtf format\n",
    "head ./maize_genome/Zea_mays.AGPv3.30.repaired.gtf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@K00364:134:H2YYHBBXY:1:1212:2037:33193 1:N:0:AGCACTTC+AATCGCTG\n",
      "GCACTCAACACTACTCCCGTGGTTGCAGAATGGGCAGGAAAAGACAGTGTCAAGCTTGTCCATCCGCTTCTTAGGCGGTGGCTTAGCAGCTGACTTTCTCTTCCCCATGGTTCAGCTTCTTTTCTTCAGATCGGAAGAGCACACGTCTGAA\n",
      "+\n",
      "AAFFAJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJFFFJJ-7FJFJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJJAJJJJFFJJJJJAA)JJJJJFJJJJJJ<7\n",
      "@K00364:134:H2YYHBBXY:1:2209:9668:15574 1:N:0:AGCACTTC+AATCGCTG\n",
      "GTCCCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACCGGAATCCTATGATGTTATCCCATGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACGACCCGGCCA\n",
      "+\n",
      "AAFFFJJJJJJJJJJJJJFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFFJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJAFJJFFJFFAFFJFJJ7AJJJJJJ7\n",
      "@K00364:134:H2YYHBBXY:1:1110:2341:1894 1:N:0:AGCACTTC+AATCGCTG\n",
      "GGAGGTCCAGGACGGCGCCGCCAACGGCGCCGGAGAGGTACCCGACGCCCGTATAGAAAGTGATGTTCTCACCCCACGAGCGTCGTTGCGCCAGGGCCTCCTCCTGGAAGAGGAACTCCGGCGAAGTGGGGAGGTCGTAGAGCGTCCGGTA\n"
     ]
    }
   ],
   "source": [
    "# this is the fastq format\n",
    "head ./raw_reads/WT10_S8_L001_R1_001.fastq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Using the tools that introduced so far,  \n",
    "can you determine how many reads are in WT10_S8_L001_R1_001.fastq?\n",
    "\n",
    "Try it out!"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# RNAseq alignment and read counting  \n",
    "We will move on to installing and using command line tools  \n",
    "There's lots more to learn about the unix command line  \n",
    "Here's a cool book if you want to know more http://vincebuffalo.org/book/  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Command line tools often have complicated dependances,  \n",
    "sometimes the software developers aren't even aware!  \n",
    "\n",
    "To reduce the complexity, we will use a package manager called conda.  \n",
    "By making environments with explicit lists of softwares,  \n",
    "conda makes it easy to install what we will need.  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2019-10-15 23:11:48--  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
      "Resolving repo.continuum.io (repo.continuum.io)... 104.18.200.79, 104.18.201.79, 2606:4700::6812:c94f, ...\n",
      "Connecting to repo.continuum.io (repo.continuum.io)|104.18.200.79|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 75257002 (72M) [application/x-sh]\n",
      "Saving to: ‘Miniconda3-latest-Linux-x86_64.sh’\n",
      "\n",
      "Miniconda3-latest-L 100%[===================>]  71.77M   120MB/s    in 0.6s    \n",
      "\n",
      "2019-10-15 23:11:49 (120 MB/s) - ‘Miniconda3-latest-Linux-x86_64.sh’ saved [75257002/75257002]\n",
      "\n",
      "\n",
      "Welcome to Miniconda3 4.7.10\n",
      "\n",
      "In order to continue the installation process, please review the license\n",
      "agreement.\n",
      "Please, press ENTER to continue\n",
      ">>> \n"
     ]
    }
   ],
   "source": [
    "# But first, we need to install conda\n",
    "\n",
    "# use wget to download miniconda from the internet\n",
    "wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
    "\n",
    "#run the installation script\n",
    "bash Miniconda3-latest-Linux-x86_64.sh\n",
    "# press enter, scroll down and say 'yes'\n",
    "# accept the defaults and say yes again"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# we need to reload our command line to activate conda\n",
    "source ~/.bashrc "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# now when you run a command, the output is labeled with the conda environemnt\n",
    "# currently (base)\n",
    "\n",
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: 'defaults' already in 'channels' list, moving to the top\n"
     ]
    }
   ],
   "source": [
    "# lets add some information so conda will know how to install all the tools we will use\n",
    "conda config --add channels defaults\n",
    "conda config --add channels bioconda\n",
    "conda config --add channels conda-forge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting package metadata (current_repodata.json): done\n",
      "Solving environment: done\n",
      "\n",
      "\n",
      "==> WARNING: A newer version of conda exists. <==\n",
      "  current version: 4.7.10\n",
      "  latest version: 4.7.12\n",
      "\n",
      "Please update conda by running\n",
      "\n",
      "    $ conda update -n base -c defaults conda\n",
      "\n",
      "\n",
      "\n",
      "## Package Plan ##\n",
      "\n",
      "  environment location: /home/sal269/miniconda3/envs/alignment\n",
      "\n",
      "  added / updated specs:\n",
      "    - fastp\n",
      "    - hisat2\n",
      "    - python=3.6\n",
      "    - samtools\n",
      "    - subread\n",
      "\n",
      "\n",
      "The following packages will be downloaded:\n",
      "\n",
      "    package                    |            build\n",
      "    ---------------------------|-----------------\n",
      "    bzip2-1.0.8                |       h516909a_1         397 KB  conda-forge\n",
      "    ca-certificates-2019.9.11  |       hecc5488_0         144 KB  conda-forge\n",
      "    certifi-2019.9.11          |           py36_0         147 KB  conda-forge\n",
      "    curl-7.65.3                |       hf8cf82a_0         132 KB  conda-forge\n",
      "    fastp-0.20.0               |       hdbcaa40_0         2.3 MB  bioconda\n",
      "    hisat2-2.1.0               |   py36h2d50403_1        16.2 MB  bioconda\n",
      "    htslib-1.9                 |       h4da6232_3         1.2 MB  bioconda\n",
      "    krb5-1.16.3                |    h05b26f9_1001         1.4 MB  conda-forge\n",
      "    libcurl-7.65.3             |       hda55be3_0         575 KB  conda-forge\n",
      "    libdeflate-1.3             |       h516909a_0          60 KB  conda-forge\n",
      "    libedit-3.1.20170329       |    hf8c457e_1001         172 KB  conda-forge\n",
      "    libffi-3.2.1               |    he1b5a44_1006          46 KB  conda-forge\n",
      "    libssh2-1.8.2              |       h22169c7_2         257 KB  conda-forge\n",
      "    ncurses-6.1                |    hf484d3e_1002         1.3 MB  conda-forge\n",
      "    openssl-1.1.1c             |       h516909a_0         2.1 MB  conda-forge\n",
      "    perl-5.26.2                |    h516909a_1006        15.4 MB  conda-forge\n",
      "    pip-19.3                   |           py36_0         1.9 MB  conda-forge\n",
      "    python-3.6.7               |    h357f687_1005        34.6 MB  conda-forge\n",
      "    readline-8.0               |       hf8c457e_0         441 KB  conda-forge\n",
      "    samtools-1.9               |      h10a08f8_12         299 KB  bioconda\n",
      "    setuptools-41.4.0          |           py36_0         624 KB  conda-forge\n",
      "    sqlite-3.30.1              |       hcee41ef_0         2.0 MB  conda-forge\n",
      "    subread-1.6.4              |       h84994c4_1        19.7 MB  bioconda\n",
      "    tk-8.6.9                   |    hed695b0_1003         3.2 MB  conda-forge\n",
      "    wheel-0.33.6               |           py36_0          35 KB  conda-forge\n",
      "    xz-5.2.4                   |    h14c3975_1001         366 KB  conda-forge\n",
      "    zlib-1.2.11                |    h516909a_1006         105 KB  conda-forge\n",
      "    ------------------------------------------------------------\n",
      "                                           Total:       105.1 MB\n",
      "\n",
      "The following NEW packages will be INSTALLED:\n",
      "\n",
      "  _libgcc_mutex      pkgs/main/linux-64::_libgcc_mutex-0.1-main\n",
      "  bzip2              conda-forge/linux-64::bzip2-1.0.8-h516909a_1\n",
      "  ca-certificates    conda-forge/linux-64::ca-certificates-2019.9.11-hecc5488_0\n",
      "  certifi            conda-forge/linux-64::certifi-2019.9.11-py36_0\n",
      "  curl               conda-forge/linux-64::curl-7.65.3-hf8cf82a_0\n",
      "  fastp              bioconda/linux-64::fastp-0.20.0-hdbcaa40_0\n",
      "  hisat2             bioconda/linux-64::hisat2-2.1.0-py36h2d50403_1\n",
      "  htslib             bioconda/linux-64::htslib-1.9-h4da6232_3\n",
      "  krb5               conda-forge/linux-64::krb5-1.16.3-h05b26f9_1001\n",
      "  libcurl            conda-forge/linux-64::libcurl-7.65.3-hda55be3_0\n",
      "  libdeflate         conda-forge/linux-64::libdeflate-1.3-h516909a_0\n",
      "  libedit            conda-forge/linux-64::libedit-3.1.20170329-hf8c457e_1001\n",
      "  libffi             conda-forge/linux-64::libffi-3.2.1-he1b5a44_1006\n",
      "  libgcc-ng          pkgs/main/linux-64::libgcc-ng-9.1.0-hdf63c60_0\n",
      "  libssh2            conda-forge/linux-64::libssh2-1.8.2-h22169c7_2\n",
      "  libstdcxx-ng       pkgs/main/linux-64::libstdcxx-ng-9.1.0-hdf63c60_0\n",
      "  ncurses            conda-forge/linux-64::ncurses-6.1-hf484d3e_1002\n",
      "  openssl            conda-forge/linux-64::openssl-1.1.1c-h516909a_0\n",
      "  perl               conda-forge/linux-64::perl-5.26.2-h516909a_1006\n",
      "  pip                conda-forge/linux-64::pip-19.3-py36_0\n",
      "  python             conda-forge/linux-64::python-3.6.7-h357f687_1005\n",
      "  readline           conda-forge/linux-64::readline-8.0-hf8c457e_0\n",
      "  samtools           bioconda/linux-64::samtools-1.9-h10a08f8_12\n",
      "  setuptools         conda-forge/linux-64::setuptools-41.4.0-py36_0\n",
      "  sqlite             conda-forge/linux-64::sqlite-3.30.1-hcee41ef_0\n",
      "  subread            bioconda/linux-64::subread-1.6.4-h84994c4_1\n",
      "  tk                 conda-forge/linux-64::tk-8.6.9-hed695b0_1003\n",
      "  wheel              conda-forge/linux-64::wheel-0.33.6-py36_0\n",
      "  xz                 conda-forge/linux-64::xz-5.2.4-h14c3975_1001\n",
      "  zlib               conda-forge/linux-64::zlib-1.2.11-h516909a_1006\n",
      "\n",
      "\n",
      "\n",
      "Downloading and Extracting Packages\n",
      "fastp-0.20.0         | 2.3 MB    | ##################################### | 100% \n",
      "samtools-1.9         | 299 KB    | ##################################### | 100% \n",
      "perl-5.26.2          | 15.4 MB   | ##################################### | 100% \n",
      "ncurses-6.1          | 1.3 MB    | ##################################### | 100% \n",
      "sqlite-3.30.1        | 2.0 MB    | ##################################### | 100% \n",
      "ca-certificates-2019 | 144 KB    | ##################################### | 100% \n",
      "bzip2-1.0.8          | 397 KB    | ##################################### | 100% \n",
      "wheel-0.33.6         | 35 KB     | ##################################### | 100% \n",
      "pip-19.3             | 1.9 MB    | ##################################### | 100% \n",
      "libdeflate-1.3       | 60 KB     | ##################################### | 100% \n",
      "libffi-3.2.1         | 46 KB     | ##################################### | 100% \n",
      "python-3.6.7         | 34.6 MB   | ##################################### | 100% \n",
      "libssh2-1.8.2        | 257 KB    | ##################################### | 100% \n",
      "subread-1.6.4        | 19.7 MB   | ##################################### | 100% \n",
      "openssl-1.1.1c       | 2.1 MB    | ##################################### | 100% \n",
      "setuptools-41.4.0    | 624 KB    | ##################################### | 100% \n",
      "readline-8.0         | 441 KB    | ##################################### | 100% \n",
      "curl-7.65.3          | 132 KB    | ##################################### | 100% \n",
      "hisat2-2.1.0         | 16.2 MB   | ##################################### | 100% \n",
      "htslib-1.9           | 1.2 MB    | ##################################### | 100% \n",
      "libcurl-7.65.3       | 575 KB    | ##################################### | 100% \n",
      "certifi-2019.9.11    | 147 KB    | ##################################### | 100% \n",
      "libedit-3.1.20170329 | 172 KB    | ##################################### | 100% \n",
      "zlib-1.2.11          | 105 KB    | ##################################### | 100% \n",
      "xz-5.2.4             | 366 KB    | ##################################### | 100% \n",
      "krb5-1.16.3          | 1.4 MB    | ##################################### | 100% \n",
      "tk-8.6.9             | 3.2 MB    | ##################################### | 100% \n",
      "Preparing transaction: done\n",
      "Verifying transaction: done\n",
      "Executing transaction: done\n",
      "#\n",
      "# To activate this environment, use\n",
      "#\n",
      "#     $ conda activate alignment\n",
      "#\n",
      "# To deactivate an active environment, use\n",
      "#\n",
      "#     $ conda deactivate\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Create new conda environment for RNAseq alignment and counting\n",
    "# the environment is named 'alignment'\n",
    "# and contains the 3.6 release of python\n",
    "# the most up-to-date version of fastp for quality control\n",
    "#                             samtools for general read handling\n",
    "#                               hisat2 for spliced alignment\n",
    "#                          and subread for read counting\n",
    "\n",
    "conda create -y --name alignment python=3.6 fastp samtools hisat2 subread\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Intro to RNAseq  \n",
    "### and our double mutant analysis  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# activate your new conda environment\n",
    "conda activate alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14788818(97.9392%)\n",
      "Q30 bases: 14251175(94.3786%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99862\n",
      "total bases: 14943303\n",
      "Q20 bases: 14646089(98.0111%)\n",
      "Q30 bases: 14120535(94.4941%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99862\n",
      "reads failed due to low quality: 135\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 5711\n",
      "bases trimmed due to adapters: 136070\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.12208%\n",
      "\n",
      "JSON report: ./fastp_report/dbl19_S46.json\n",
      "HTML report: ./fastp_report/dbl19_S46.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl19_S46_L001_R1_001.fastq -o ./trimmed_reads/dbl19_S46.fastq.gz -h ./fastp_report/dbl19_S46.html -j ./fastp_report/dbl19_S46.json -R dbl19_S46 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14796697(97.9914%)\n",
      "Q30 bases: 14273584(94.527%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99873\n",
      "total bases: 14565276\n",
      "Q20 bases: 14290586(98.1141%)\n",
      "Q30 bases: 13799404(94.7418%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99873\n",
      "reads failed due to low quality: 122\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 15118\n",
      "bases trimmed due to adapters: 515579\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.45058%\n",
      "\n",
      "JSON report: ./fastp_report/WT58_S3.json\n",
      "HTML report: ./fastp_report/WT58_S3.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT58_S3_L001_R1_001.fastq -o ./trimmed_reads/WT58_S3.fastq.gz -h ./fastp_report/WT58_S3.html -j ./fastp_report/WT58_S3.json -R WT58_S3 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14767604(97.7987%)\n",
      "Q30 bases: 14200651(94.044%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99844\n",
      "total bases: 14925091\n",
      "Q20 bases: 14608550(97.8791%)\n",
      "Q30 bases: 14055278(94.1721%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99844\n",
      "reads failed due to low quality: 154\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 5813\n",
      "bases trimmed due to adapters: 151435\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.7197%\n",
      "\n",
      "JSON report: ./fastp_report/nod36_S21.json\n",
      "HTML report: ./fastp_report/nod36_S21.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod36_S21_L001_R1_001.fastq -o ./trimmed_reads/nod36_S21.fastq.gz -h ./fastp_report/nod36_S21.html -j ./fastp_report/nod36_S21.json -R nod36_S21 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14795059(97.9805%)\n",
      "Q30 bases: 14268948(94.4963%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99877\n",
      "total bases: 14729497\n",
      "Q20 bases: 14445288(98.0705%)\n",
      "Q30 bases: 13941416(94.6496%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99877\n",
      "reads failed due to low quality: 121\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 11234\n",
      "bases trimmed due to adapters: 351930\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.79017%\n",
      "\n",
      "JSON report: ./fastp_report/dbl89_S40.json\n",
      "HTML report: ./fastp_report/dbl89_S40.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl89_S40_L001_R1_001.fastq -o ./trimmed_reads/dbl89_S40.fastq.gz -h ./fastp_report/dbl89_S40.html -j ./fastp_report/dbl89_S40.json -R dbl89_S40 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14794710(97.9782%)\n",
      "Q30 bases: 14263178(94.4581%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99851\n",
      "total bases: 14913176\n",
      "Q20 bases: 14623821(98.0597%)\n",
      "Q30 bases: 14106093(94.5881%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99851\n",
      "reads failed due to low quality: 143\n",
      "reads failed due to too many N: 5\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 6425\n",
      "bases trimmed due to adapters: 164404\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.47803%\n",
      "\n",
      "JSON report: ./fastp_report/WT18_S4.json\n",
      "HTML report: ./fastp_report/WT18_S4.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT18_S4_L001_R1_001.fastq -o ./trimmed_reads/WT18_S4.fastq.gz -h ./fastp_report/WT18_S4.html -j ./fastp_report/WT18_S4.json -R WT18_S4 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14792515(97.9637%)\n",
      "Q30 bases: 14254463(94.4004%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99880\n",
      "total bases: 14890888\n",
      "Q20 bases: 14597822(98.0319%)\n",
      "Q30 bases: 14073963(94.5139%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99880\n",
      "reads failed due to low quality: 117\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7591\n",
      "bases trimmed due to adapters: 191021\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 3.98193%\n",
      "\n",
      "JSON report: ./fastp_report/dbl50_S43.json\n",
      "HTML report: ./fastp_report/dbl50_S43.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl50_S43_L001_R1_001.fastq -o ./trimmed_reads/dbl50_S43.fastq.gz -h ./fastp_report/dbl50_S43.html -j ./fastp_report/dbl50_S43.json -R dbl50_S43 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14756844(97.7274%)\n",
      "Q30 bases: 14172011(93.8544%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99847\n",
      "total bases: 14807667\n",
      "Q20 bases: 14485164(97.8221%)\n",
      "Q30 bases: 13920001(94.0054%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99847\n",
      "reads failed due to low quality: 144\n",
      "reads failed due to too many N: 8\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 8503\n",
      "bases trimmed due to adapters: 269337\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.43956%\n",
      "\n",
      "JSON report: ./fastp_report/lgn37_S33.json\n",
      "HTML report: ./fastp_report/lgn37_S33.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn37_S33_L001_R1_001.fastq -o ./trimmed_reads/lgn37_S33.fastq.gz -h ./fastp_report/lgn37_S33.html -j ./fastp_report/lgn37_S33.json -R lgn37_S33 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14776119(97.8551%)\n",
      "Q30 bases: 14219978(94.172%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99854\n",
      "total bases: 14972097\n",
      "Q20 bases: 14662122(97.9296%)\n",
      "Q30 bases: 14117003(94.2887%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99854\n",
      "reads failed due to low quality: 145\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 4592\n",
      "bases trimmed due to adapters: 105886\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.61087%\n",
      "\n",
      "JSON report: ./fastp_report/dbl65_S42.json\n",
      "HTML report: ./fastp_report/dbl65_S42.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl65_S42_L001_R1_001.fastq -o ./trimmed_reads/dbl65_S42.fastq.gz -h ./fastp_report/dbl65_S42.html -j ./fastp_report/dbl65_S42.json -R dbl65_S42 fastp report \n",
      "fastp v0.20.0, time used: 4 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14791061(97.954%)\n",
      "Q30 bases: 14253694(94.3953%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99863\n",
      "total bases: 14922746\n",
      "Q20 bases: 14628447(98.0278%)\n",
      "Q30 bases: 14104437(94.5164%)\n",
      "\n",
      "Filtering result:\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reads passed filter: 99863\n",
      "reads failed due to low quality: 134\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 6746\n",
      "bases trimmed due to adapters: 156596\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.88328%\n",
      "\n",
      "JSON report: ./fastp_report/nod48_S16.json\n",
      "HTML report: ./fastp_report/nod48_S16.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod48_S16_L001_R1_001.fastq -o ./trimmed_reads/nod48_S16.fastq.gz -h ./fastp_report/nod48_S16.html -j ./fastp_report/nod48_S16.json -R nod48_S16 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14769928(97.8141%)\n",
      "Q30 bases: 14198363(94.0289%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99869\n",
      "total bases: 14889712\n",
      "Q20 bases: 14574734(97.8846%)\n",
      "Q30 bases: 14017356(94.1412%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99869\n",
      "reads failed due to low quality: 122\n",
      "reads failed due to too many N: 8\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 7019\n",
      "bases trimmed due to adapters: 190554\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.40035%\n",
      "\n",
      "JSON report: ./fastp_report/WT38_S5.json\n",
      "HTML report: ./fastp_report/WT38_S5.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT38_S5_L001_R1_001.fastq -o ./trimmed_reads/WT38_S5.fastq.gz -h ./fastp_report/WT38_S5.html -j ./fastp_report/WT38_S5.json -R WT38_S5 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14782314(97.8961%)\n",
      "Q30 bases: 14233872(94.2641%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99844\n",
      "total bases: 14925450\n",
      "Q20 bases: 14623781(97.9788%)\n",
      "Q30 bases: 14088715(94.3939%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99844\n",
      "reads failed due to low quality: 148\n",
      "reads failed due to too many N: 6\n",
      "reads failed due to too short: 2\n",
      "reads with adapter trimmed: 6018\n",
      "bases trimmed due to adapters: 151087\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.36066%\n",
      "\n",
      "JSON report: ./fastp_report/nod14_S18.json\n",
      "HTML report: ./fastp_report/nod14_S18.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod14_S18_L001_R1_001.fastq -o ./trimmed_reads/nod14_S18.fastq.gz -h ./fastp_report/nod14_S18.html -j ./fastp_report/nod14_S18.json -R nod14_S18 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14783470(97.9038%)\n",
      "Q30 bases: 14241636(94.3155%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99871\n",
      "total bases: 14692076\n",
      "Q20 bases: 14398421(98.0013%)\n",
      "Q30 bases: 13880827(94.4783%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99871\n",
      "reads failed due to low quality: 124\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 3\n",
      "reads with adapter trimmed: 11967\n",
      "bases trimmed due to adapters: 388541\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.89053%\n",
      "\n",
      "JSON report: ./fastp_report/nod62_S14.json\n",
      "HTML report: ./fastp_report/nod62_S14.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod62_S14_L001_R1_001.fastq -o ./trimmed_reads/nod62_S14.fastq.gz -h ./fastp_report/nod62_S14.html -j ./fastp_report/nod62_S14.json -R nod62_S14 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14774910(97.8471%)\n",
      "Q30 bases: 14214552(94.1361%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99860\n",
      "total bases: 14902862\n",
      "Q20 bases: 14593356(97.9232%)\n",
      "Q30 bases: 14047269(94.2589%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99860\n",
      "reads failed due to low quality: 138\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 6339\n",
      "bases trimmed due to adapters: 175998\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.372%\n",
      "\n",
      "JSON report: ./fastp_report/lgn3_S39.json\n",
      "HTML report: ./fastp_report/lgn3_S39.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn3_S39_L001_R1_001.fastq -o ./trimmed_reads/lgn3_S39.fastq.gz -h ./fastp_report/lgn3_S39.html -j ./fastp_report/lgn3_S39.json -R lgn3_S39 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14791777(97.9588%)\n",
      "Q30 bases: 14260667(94.4415%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99869\n",
      "total bases: 14795957\n",
      "Q20 bases: 14506259(98.042%)\n",
      "Q30 bases: 13994194(94.5812%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99869\n",
      "reads failed due to low quality: 126\n",
      "reads failed due to too many N: 5\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 9918\n",
      "bases trimmed due to adapters: 284262\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.15892%\n",
      "\n",
      "JSON report: ./fastp_report/WT93_S11.json\n",
      "HTML report: ./fastp_report/WT93_S11.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT93_S11_L001_R1_001.fastq -o ./trimmed_reads/WT93_S11.fastq.gz -h ./fastp_report/WT93_S11.html -j ./fastp_report/WT93_S11.json -R WT93_S11 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14767128(97.7955%)\n",
      "Q30 bases: 14196505(94.0166%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99832\n",
      "total bases: 14938305\n",
      "Q20 bases: 14621928(97.8821%)\n",
      "Q30 bases: 14064648(94.1516%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99832\n",
      "reads failed due to low quality: 163\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 3\n",
      "reads with adapter trimmed: 5353\n",
      "bases trimmed due to adapters: 136534\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.93%\n",
      "\n",
      "JSON report: ./fastp_report/nod30_S17.json\n",
      "HTML report: ./fastp_report/nod30_S17.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod30_S17_L001_R1_001.fastq -o ./trimmed_reads/nod30_S17.fastq.gz -h ./fastp_report/nod30_S17.html -j ./fastp_report/nod30_S17.json -R nod30_S17 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14778293(97.8695%)\n",
      "Q30 bases: 14221706(94.1835%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99857\n",
      "total bases: 14962890\n",
      "Q20 bases: 14654885(97.9415%)\n",
      "Q30 bases: 14109687(94.2979%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99857\n",
      "reads failed due to low quality: 139\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5069\n",
      "bases trimmed due to adapters: 115517\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.46319%\n",
      "\n",
      "JSON report: ./fastp_report/lgn46_S30.json\n",
      "HTML report: ./fastp_report/lgn46_S30.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn46_S30_L001_R1_001.fastq -o ./trimmed_reads/lgn46_S30.fastq.gz -h ./fastp_report/lgn46_S30.html -j ./fastp_report/lgn46_S30.json -R lgn46_S30 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14789134(97.9413%)\n",
      "Q30 bases: 14247595(94.3549%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99882\n",
      "total bases: 14911929\n",
      "Q20 bases: 14614645(98.0064%)\n",
      "Q30 bases: 14086204(94.4627%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99882\n",
      "reads failed due to low quality: 115\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7246\n",
      "bases trimmed due to adapters: 170390\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.56382%\n",
      "\n",
      "JSON report: ./fastp_report/nod16_S15.json\n",
      "HTML report: ./fastp_report/nod16_S15.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod16_S15_L001_R1_001.fastq -o ./trimmed_reads/nod16_S15.fastq.gz -h ./fastp_report/nod16_S15.html -j ./fastp_report/nod16_S15.json -R nod16_S15 fastp report \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14786167(97.9216%)\n",
      "Q30 bases: 14248450(94.3606%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99838\n",
      "total bases: 14955721\n",
      "Q20 bases: 14657496(98.0059%)\n",
      "Q30 bases: 14131992(94.4922%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99838\n",
      "reads failed due to low quality: 157\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 5069\n",
      "bases trimmed due to adapters: 119883\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.5817%\n",
      "\n",
      "JSON report: ./fastp_report/dbl28_S48.json\n",
      "HTML report: ./fastp_report/dbl28_S48.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl28_S48_L001_R1_001.fastq -o ./trimmed_reads/dbl28_S48.fastq.gz -h ./fastp_report/dbl28_S48.html -j ./fastp_report/dbl28_S48.json -R dbl28_S48 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14785307(97.9159%)\n",
      "Q30 bases: 14244778(94.3363%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99863\n",
      "total bases: 14919347\n",
      "Q20 bases: 14619881(97.9928%)\n",
      "Q30 bases: 14093115(94.462%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99863\n",
      "reads failed due to low quality: 134\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 6544\n",
      "bases trimmed due to adapters: 159973\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.67291%\n",
      "\n",
      "JSON report: ./fastp_report/lgn39_S27.json\n",
      "HTML report: ./fastp_report/lgn39_S27.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn39_S27_L001_R1_001.fastq -o ./trimmed_reads/lgn39_S27.fastq.gz -h ./fastp_report/lgn39_S27.html -j ./fastp_report/lgn39_S27.json -R lgn39_S27 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14776936(97.8605%)\n",
      "Q30 bases: 14222762(94.1905%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99843\n",
      "total bases: 14802985\n",
      "Q20 bases: 14500916(97.9594%)\n",
      "Q30 bases: 13966888(94.3518%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99843\n",
      "reads failed due to low quality: 154\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 2\n",
      "reads with adapter trimmed: 8954\n",
      "bases trimmed due to adapters: 273404\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.26948%\n",
      "\n",
      "JSON report: ./fastp_report/nod61_S25.json\n",
      "HTML report: ./fastp_report/nod61_S25.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod61_S25_L001_R1_001.fastq -o ./trimmed_reads/nod61_S25.fastq.gz -h ./fastp_report/nod61_S25.html -j ./fastp_report/nod61_S25.json -R nod61_S25 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14774551(97.8447%)\n",
      "Q30 bases: 14215403(94.1417%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99848\n",
      "total bases: 14881895\n",
      "Q20 bases: 14572938(97.9239%)\n",
      "Q30 bases: 14029164(94.27%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99848\n",
      "reads failed due to low quality: 148\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7210\n",
      "bases trimmed due to adapters: 195163\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.14418%\n",
      "\n",
      "JSON report: ./fastp_report/nod2_S26.json\n",
      "HTML report: ./fastp_report/nod2_S26.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod2_S26_L001_R1_001.fastq -o ./trimmed_reads/nod2_S26.fastq.gz -h ./fastp_report/nod2_S26.html -j ./fastp_report/nod2_S26.json -R nod2_S26 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      "No adapter detected for read1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14754928(97.7148%)\n",
      "Q30 bases: 14170640(93.8453%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99814\n",
      "total bases: 15071914\n",
      "Q20 bases: 14739636(97.7954%)\n",
      "Q30 bases: 14161553(93.9599%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99814\n",
      "reads failed due to low quality: 183\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 0\n",
      "bases trimmed due to adapters: 0\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 7.70394%\n",
      "\n",
      "JSON report: ./fastp_report/WT24_S10.json\n",
      "HTML report: ./fastp_report/WT24_S10.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT24_S10_L001_R1_001.fastq -o ./trimmed_reads/WT24_S10.fastq.gz -h ./fastp_report/WT24_S10.html -j ./fastp_report/WT24_S10.json -R WT24_S10 fastp report \n",
      "fastp v0.20.0, time used: 5 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14773606(97.8385%)\n",
      "Q30 bases: 14221130(94.1797%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99831\n",
      "total bases: 14832596\n",
      "Q20 bases: 14527048(97.94%)\n",
      "Q30 bases: 13993589(94.3435%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99831\n",
      "reads failed due to low quality: 167\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 8722\n",
      "bases trimmed due to adapters: 241932\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.09671%\n",
      "\n",
      "JSON report: ./fastp_report/WT13_S12.json\n",
      "HTML report: ./fastp_report/WT13_S12.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT13_S12_L001_R1_001.fastq -o ./trimmed_reads/WT13_S12.fastq.gz -h ./fastp_report/WT13_S12.html -j ./fastp_report/WT13_S12.json -R WT13_S12 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14782182(97.8952%)\n",
      "Q30 bases: 14232644(94.2559%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99841\n",
      "total bases: 14965663\n",
      "Q20 bases: 14662246(97.9726%)\n",
      "Q30 bases: 14124409(94.3788%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99841\n",
      "reads failed due to low quality: 158\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 4744\n",
      "bases trimmed due to adapters: 110328\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.06317%\n",
      "\n",
      "JSON report: ./fastp_report/dbl56_S50.json\n",
      "HTML report: ./fastp_report/dbl56_S50.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl56_S50_L001_R1_001.fastq -o ./trimmed_reads/dbl56_S50.fastq.gz -h ./fastp_report/dbl56_S50.html -j ./fastp_report/dbl56_S50.json -R dbl56_S50 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14791519(97.9571%)\n",
      "Q30 bases: 14253866(94.3965%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99870\n",
      "total bases: 14845329\n",
      "Q20 bases: 14553379(98.0334%)\n",
      "Q30 bases: 14032684(94.5259%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99870\n",
      "reads failed due to low quality: 125\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 8629\n",
      "bases trimmed due to adapters: 235070\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.91702%\n",
      "\n",
      "JSON report: ./fastp_report/WT41_S6.json\n",
      "HTML report: ./fastp_report/WT41_S6.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT41_S6_L001_R1_001.fastq -o ./trimmed_reads/WT41_S6.fastq.gz -h ./fastp_report/WT41_S6.html -j ./fastp_report/WT41_S6.json -R WT41_S6 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14772146(97.8288%)\n",
      "Q30 bases: 14205662(94.0772%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99858\n",
      "total bases: 14881657\n",
      "Q20 bases: 14570412(97.9085%)\n",
      "Q30 bases: 14019414(94.206%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99858\n",
      "reads failed due to low quality: 136\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 2\n",
      "reads with adapter trimmed: 7322\n",
      "bases trimmed due to adapters: 197095\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.00053%\n",
      "\n",
      "JSON report: ./fastp_report/lgn77_S29.json\n",
      "HTML report: ./fastp_report/lgn77_S29.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn77_S29_L001_R1_001.fastq -o ./trimmed_reads/lgn77_S29.fastq.gz -h ./fastp_report/lgn77_S29.html -j ./fastp_report/lgn77_S29.json -R lgn77_S29 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14791045(97.9539%)\n",
      "Q30 bases: 14253048(94.391%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99873\n",
      "total bases: 14946421\n",
      "Q20 bases: 14651256(98.0252%)\n",
      "Q30 bases: 14125243(94.5059%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99873\n",
      "reads failed due to low quality: 124\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5835\n",
      "bases trimmed due to adapters: 134402\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.26453%\n",
      "\n",
      "JSON report: ./fastp_report/lgn8_S32.json\n",
      "HTML report: ./fastp_report/lgn8_S32.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn8_S32_L001_R1_001.fastq -o ./trimmed_reads/lgn8_S32.fastq.gz -h ./fastp_report/lgn8_S32.html -j ./fastp_report/lgn8_S32.json -R lgn8_S32 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14771182(97.8224%)\n",
      "Q30 bases: 14210093(94.1066%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99833\n",
      "total bases: 14958850\n",
      "Q20 bases: 14645526(97.9054%)\n",
      "Q30 bases: 14096730(94.2367%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99833\n",
      "reads failed due to low quality: 163\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 2\n",
      "reads with adapter trimmed: 5128\n",
      "bases trimmed due to adapters: 116124\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.26845%\n",
      "\n",
      "JSON report: ./fastp_report/dbl9_S45.json\n",
      "HTML report: ./fastp_report/dbl9_S45.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl9_S45_L001_R1_001.fastq -o ./trimmed_reads/dbl9_S45.fastq.gz -h ./fastp_report/dbl9_S45.html -j ./fastp_report/dbl9_S45.json -R dbl9_S45 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14789148(97.9414%)\n",
      "Q30 bases: 14248177(94.3588%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99852\n",
      "total bases: 14937001\n",
      "Q20 bases: 14640663(98.0161%)\n",
      "Q30 bases: 14112087(94.4774%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99852\n",
      "reads failed due to low quality: 146\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5829\n",
      "bases trimmed due to adapters: 140662\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.02577%\n",
      "\n",
      "JSON report: ./fastp_report/nod47_S23.json\n",
      "HTML report: ./fastp_report/nod47_S23.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod47_S23_L001_R1_001.fastq -o ./trimmed_reads/nod47_S23.fastq.gz -h ./fastp_report/nod47_S23.html -j ./fastp_report/nod47_S23.json -R nod47_S23 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14790916(97.9531%)\n",
      "Q30 bases: 14259673(94.4349%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99867\n",
      "total bases: 14936340\n",
      "Q20 bases: 14640903(98.022%)\n",
      "Q30 bases: 14121876(94.5471%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99867\n",
      "reads failed due to low quality: 130\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 6105\n",
      "bases trimmed due to adapters: 143577\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.46207%\n",
      "\n",
      "JSON report: ./fastp_report/dbl57_S52.json\n",
      "HTML report: ./fastp_report/dbl57_S52.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl57_S52_L001_R1_001.fastq -o ./trimmed_reads/dbl57_S52.fastq.gz -h ./fastp_report/dbl57_S52.html -j ./fastp_report/dbl57_S52.json -R dbl57_S52 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14754873(97.7144%)\n",
      "Q30 bases: 14171703(93.8523%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99833\n",
      "total bases: 14944017\n",
      "Q20 bases: 14615036(97.7986%)\n",
      "Q30 bases: 14044616(93.9815%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99833\n",
      "reads failed due to low quality: 164\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5157\n",
      "bases trimmed due to adapters: 130791\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.84864%\n",
      "\n",
      "JSON report: ./fastp_report/dbl31_S41.json\n",
      "HTML report: ./fastp_report/dbl31_S41.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl31_S41_L001_R1_001.fastq -o ./trimmed_reads/dbl31_S41.fastq.gz -h ./fastp_report/dbl31_S41.html -j ./fastp_report/dbl31_S41.json -R dbl31_S41 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14768243(97.8029%)\n",
      "Q30 bases: 14198859(94.0322%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99827\n",
      "total bases: 14924947\n",
      "Q20 bases: 14610731(97.8947%)\n",
      "Q30 bases: 14056265(94.1797%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99827\n",
      "reads failed due to low quality: 167\n",
      "reads failed due to too many N: 6\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5846\n",
      "bases trimmed due to adapters: 149001\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.63753%\n",
      "\n",
      "JSON report: ./fastp_report/dbl66_S49.json\n",
      "HTML report: ./fastp_report/dbl66_S49.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl66_S49_L001_R1_001.fastq -o ./trimmed_reads/dbl66_S49.fastq.gz -h ./fastp_report/dbl66_S49.html -j ./fastp_report/dbl66_S49.json -R dbl66_S49 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14780373(97.8833%)\n",
      "Q30 bases: 14228692(94.2297%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99848\n",
      "total bases: 14913832\n",
      "Q20 bases: 14609916(97.9622%)\n",
      "Q30 bases: 14071889(94.3546%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99848\n",
      "reads failed due to low quality: 148\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 6131\n",
      "bases trimmed due to adapters: 163372\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.59779%\n",
      "\n",
      "JSON report: ./fastp_report/nod43_S22.json\n",
      "HTML report: ./fastp_report/nod43_S22.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod43_S22_L001_R1_001.fastq -o ./trimmed_reads/nod43_S22.fastq.gz -h ./fastp_report/nod43_S22.html -j ./fastp_report/nod43_S22.json -R nod43_S22 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14775971(97.8541%)\n",
      "Q30 bases: 14223640(94.1963%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99848\n",
      "total bases: 14857220\n",
      "Q20 bases: 14550944(97.9385%)\n",
      "Q30 bases: 14015563(94.335%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99848\n",
      "reads failed due to low quality: 151\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 8127\n",
      "bases trimmed due to adapters: 219878\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.36831%\n",
      "\n",
      "JSON report: ./fastp_report/WT95_S13.json\n",
      "HTML report: ./fastp_report/WT95_S13.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT95_S13_L001_R1_001.fastq -o ./trimmed_reads/WT95_S13.fastq.gz -h ./fastp_report/WT95_S13.html -j ./fastp_report/WT95_S13.json -R WT95_S13 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total bases: 15100000\n",
      "Q20 bases: 14772735(97.8327%)\n",
      "Q30 bases: 14208054(94.0931%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99857\n",
      "total bases: 14872740\n",
      "Q20 bases: 14562288(97.9126%)\n",
      "Q30 bases: 14013195(94.2207%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99857\n",
      "reads failed due to low quality: 137\n",
      "reads failed due to too many N: 6\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7223\n",
      "bases trimmed due to adapters: 205667\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.5036%\n",
      "\n",
      "JSON report: ./fastp_report/lgn33_S38.json\n",
      "HTML report: ./fastp_report/lgn33_S38.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn33_S38_L001_R1_001.fastq -o ./trimmed_reads/lgn33_S38.fastq.gz -h ./fastp_report/lgn33_S38.html -j ./fastp_report/lgn33_S38.json -R lgn33_S38 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14789464(97.9435%)\n",
      "Q30 bases: 14254254(94.399%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99857\n",
      "total bases: 14879625\n",
      "Q20 bases: 14585082(98.0205%)\n",
      "Q30 bases: 14064704(94.5232%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99857\n",
      "reads failed due to low quality: 139\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 8005\n",
      "bases trimmed due to adapters: 198794\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 3.69932%\n",
      "\n",
      "JSON report: ./fastp_report/lgn42_S34.json\n",
      "HTML report: ./fastp_report/lgn42_S34.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn42_S34_L001_R1_001.fastq -o ./trimmed_reads/lgn42_S34.fastq.gz -h ./fastp_report/lgn42_S34.html -j ./fastp_report/lgn42_S34.json -R lgn42_S34 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14782495(97.8973%)\n",
      "Q30 bases: 14234485(94.2681%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99849\n",
      "total bases: 14899540\n",
      "Q20 bases: 14598830(97.9817%)\n",
      "Q30 bases: 14065886(94.4048%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99849\n",
      "reads failed due to low quality: 150\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7004\n",
      "bases trimmed due to adapters: 177693\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.24515%\n",
      "\n",
      "JSON report: ./fastp_report/nod6_S20.json\n",
      "HTML report: ./fastp_report/nod6_S20.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod6_S20_L001_R1_001.fastq -o ./trimmed_reads/nod6_S20.fastq.gz -h ./fastp_report/nod6_S20.html -j ./fastp_report/nod6_S20.json -R nod6_S20 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14769526(97.8114%)\n",
      "Q30 bases: 14203937(94.0658%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99859\n",
      "total bases: 14875373\n",
      "Q20 bases: 14561828(97.8922%)\n",
      "Q30 bases: 14011906(94.1953%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99859\n",
      "reads failed due to low quality: 137\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 7383\n",
      "bases trimmed due to adapters: 203399\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.31258%\n",
      "\n",
      "JSON report: ./fastp_report/lgn28_S37.json\n",
      "HTML report: ./fastp_report/lgn28_S37.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn28_S37_L001_R1_001.fastq -o ./trimmed_reads/lgn28_S37.fastq.gz -h ./fastp_report/lgn28_S37.html -j ./fastp_report/lgn28_S37.json -R lgn28_S37 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14788452(97.9368%)\n",
      "Q30 bases: 14250747(94.3758%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99864\n",
      "total bases: 14940141\n",
      "Q20 bases: 14643047(98.0114%)\n",
      "Q30 bases: 14117656(94.4948%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99864\n",
      "reads failed due to low quality: 133\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 5823\n",
      "bases trimmed due to adapters: 139323\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 5.30192%\n",
      "\n",
      "JSON report: ./fastp_report/dbl54_S44.json\n",
      "HTML report: ./fastp_report/dbl54_S44.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl54_S44_L001_R1_001.fastq -o ./trimmed_reads/dbl54_S44.fastq.gz -h ./fastp_report/dbl54_S44.html -j ./fastp_report/dbl54_S44.json -R dbl54_S44 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14775777(97.8528%)\n",
      "Q30 bases: 14220811(94.1776%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99848\n",
      "total bases: 14770076\n",
      "Q20 bases: 14467834(97.9537%)\n",
      "Q30 bases: 13934061(94.3398%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99848\n",
      "reads failed due to low quality: 148\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 2\n",
      "reads with adapter trimmed: 9613\n",
      "bases trimmed due to adapters: 307036\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.65264%\n",
      "\n",
      "JSON report: ./fastp_report/WT78_S2.json\n",
      "HTML report: ./fastp_report/WT78_S2.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT78_S2_L001_R1_001.fastq -o ./trimmed_reads/WT78_S2.fastq.gz -h ./fastp_report/WT78_S2.html -j ./fastp_report/WT78_S2.json -R WT78_S2 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      "No adapter detected for read1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14766621(97.7922%)\n",
      "Q30 bases: 14196031(94.0135%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99826\n",
      "total bases: 15073726\n",
      "Q20 bases: 14752255(97.8673%)\n",
      "Q30 bases: 14188108(94.1248%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99826\n",
      "reads failed due to low quality: 173\n",
      "reads failed due to too many N: 1\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 0\n",
      "bases trimmed due to adapters: 0\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 6.12616%\n",
      "\n",
      "JSON report: ./fastp_report/WT97_S1.json\n",
      "HTML report: ./fastp_report/WT97_S1.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT97_S1_L001_R1_001.fastq -o ./trimmed_reads/WT97_S1.fastq.gz -h ./fastp_report/WT97_S1.html -j ./fastp_report/WT97_S1.json -R WT97_S1 fastp report \n",
      "fastp v0.20.0, time used: 6 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14779895(97.8801%)\n",
      "Q30 bases: 14227662(94.2229%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99840\n",
      "total bases: 14916239\n",
      "Q20 bases: 14612569(97.9642%)\n",
      "Q30 bases: 14074658(94.358%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99840\n",
      "reads failed due to low quality: 155\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 6413\n",
      "bases trimmed due to adapters: 159801\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.57168%\n",
      "\n",
      "JSON report: ./fastp_report/dbl70_S47.json\n",
      "HTML report: ./fastp_report/dbl70_S47.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl70_S47_L001_R1_001.fastq -o ./trimmed_reads/dbl70_S47.fastq.gz -h ./fastp_report/dbl70_S47.html -j ./fastp_report/dbl70_S47.json -R dbl70_S47 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14788882(97.9396%)\n",
      "Q30 bases: 14249819(94.3697%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99861\n",
      "total bases: 14893855\n",
      "Q20 bases: 14598962(98.02%)\n",
      "Q30 bases: 14075140(94.503%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99861\n",
      "reads failed due to low quality: 138\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 7367\n",
      "bases trimmed due to adapters: 185260\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.78383%\n",
      "\n",
      "JSON report: ./fastp_report/lgn59_S28.json\n",
      "HTML report: ./fastp_report/lgn59_S28.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn59_S28_L001_R1_001.fastq -o ./trimmed_reads/lgn59_S28.fastq.gz -h ./fastp_report/lgn59_S28.html -j ./fastp_report/lgn59_S28.json -R lgn59_S28 fastp report \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14787314(97.9292%)\n",
      "Q30 bases: 14244588(94.335%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99855\n",
      "total bases: 14924464\n",
      "Q20 bases: 14626305(98.0022%)\n",
      "Q30 bases: 14096471(94.4521%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99855\n",
      "reads failed due to low quality: 138\n",
      "reads failed due to too many N: 7\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 6243\n",
      "bases trimmed due to adapters: 153726\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.59656%\n",
      "\n",
      "JSON report: ./fastp_report/WT55_S9.json\n",
      "HTML report: ./fastp_report/WT55_S9.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT55_S9_L001_R1_001.fastq -o ./trimmed_reads/WT55_S9.fastq.gz -h ./fastp_report/WT55_S9.html -j ./fastp_report/WT55_S9.json -R WT55_S9 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14773763(97.8395%)\n",
      "Q30 bases: 14215706(94.1437%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99850\n",
      "total bases: 14938506\n",
      "Q20 bases: 14627512(97.9182%)\n",
      "Q30 bases: 14082458(94.2695%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99850\n",
      "reads failed due to low quality: 148\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 6019\n",
      "bases trimmed due to adapters: 138901\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.47778%\n",
      "\n",
      "JSON report: ./fastp_report/dbl88_S51.json\n",
      "HTML report: ./fastp_report/dbl88_S51.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/dbl88_S51_L001_R1_001.fastq -o ./trimmed_reads/dbl88_S51.fastq.gz -h ./fastp_report/dbl88_S51.html -j ./fastp_report/dbl88_S51.json -R dbl88_S51 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14783014(97.9008%)\n",
      "Q30 bases: 14232086(94.2522%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99858\n",
      "total bases: 14878907\n",
      "Q20 bases: 14578409(97.9804%)\n",
      "Q30 bases: 14043074(94.3824%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99858\n",
      "reads failed due to low quality: 137\n",
      "reads failed due to too many N: 5\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 7359\n",
      "bases trimmed due to adapters: 199681\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.20808%\n",
      "\n",
      "JSON report: ./fastp_report/lgn72_S31.json\n",
      "HTML report: ./fastp_report/lgn72_S31.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn72_S31_L001_R1_001.fastq -o ./trimmed_reads/lgn72_S31.fastq.gz -h ./fastp_report/lgn72_S31.html -j ./fastp_report/lgn72_S31.json -R lgn72_S31 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14780406(97.8835%)\n",
      "Q30 bases: 14236838(94.2837%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99824\n",
      "total bases: 14714227\n",
      "Q20 bases: 14419537(97.9972%)\n",
      "Q30 bases: 13901144(94.4742%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99824\n",
      "reads failed due to low quality: 173\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 11234\n",
      "bases trimmed due to adapters: 359274\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 3.78051%\n",
      "\n",
      "JSON report: ./fastp_report/lgn21_S36.json\n",
      "HTML report: ./fastp_report/lgn21_S36.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn21_S36_L001_R1_001.fastq -o ./trimmed_reads/lgn21_S36.fastq.gz -h ./fastp_report/lgn21_S36.html -j ./fastp_report/lgn21_S36.json -R lgn21_S36 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14788748(97.9387%)\n",
      "Q30 bases: 14248800(94.3629%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99844\n",
      "total bases: 14830749\n",
      "Q20 bases: 14538039(98.0263%)\n",
      "Q30 bases: 14016305(94.5084%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99844\n",
      "reads failed due to low quality: 150\n",
      "reads failed due to too many N: 5\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 8940\n",
      "bases trimmed due to adapters: 245878\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.87695%\n",
      "\n",
      "JSON report: ./fastp_report/WT10_S8.json\n",
      "HTML report: ./fastp_report/WT10_S8.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT10_S8_L001_R1_001.fastq -o ./trimmed_reads/WT10_S8.fastq.gz -h ./fastp_report/WT10_S8.html -j ./fastp_report/WT10_S8.json -R WT10_S8 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14789346(97.9427%)\n",
      "Q30 bases: 14254177(94.3985%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99863\n",
      "total bases: 14677970\n",
      "Q20 bases: 14392129(98.0526%)\n",
      "Q30 bases: 13883815(94.5895%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99863\n",
      "reads failed due to low quality: 133\n",
      "reads failed due to too many N: 3\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 12560\n",
      "bases trimmed due to adapters: 401409\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.27933%\n",
      "\n",
      "JSON report: ./fastp_report/nod60_S19.json\n",
      "HTML report: ./fastp_report/nod60_S19.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod60_S19_L001_R1_001.fastq -o ./trimmed_reads/nod60_S19.fastq.gz -h ./fastp_report/nod60_S19.html -j ./fastp_report/nod60_S19.json -R nod60_S19 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14776661(97.8587%)\n",
      "Q30 bases: 14223261(94.1938%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99853\n",
      "total bases: 14965431\n",
      "Q20 bases: 14655603(97.9297%)\n",
      "Q30 bases: 14113200(94.3053%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99853\n",
      "reads failed due to low quality: 138\n",
      "reads failed due to too many N: 9\n",
      "reads failed due to too short: 0\n",
      "reads with adapter trimmed: 4967\n",
      "bases trimmed due to adapters: 112381\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 7.02376%\n",
      "\n",
      "JSON report: ./fastp_report/nod11_S24.json\n",
      "HTML report: ./fastp_report/nod11_S24.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/nod11_S24_L001_R1_001.fastq -o ./trimmed_reads/nod11_S24.fastq.gz -h ./fastp_report/nod11_S24.html -j ./fastp_report/nod11_S24.json -R nod11_S24 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14792756(97.9653%)\n",
      "Q30 bases: 14266845(94.4824%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99843\n",
      "total bases: 14639437\n",
      "Q20 bases: 14359263(98.0862%)\n",
      "Q30 bases: 13861879(94.6886%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99843\n",
      "reads failed due to low quality: 154\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 1\n",
      "reads with adapter trimmed: 13591\n",
      "bases trimmed due to adapters: 437032\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.57496%\n",
      "\n",
      "JSON report: ./fastp_report/lgn17_S35.json\n",
      "HTML report: ./fastp_report/lgn17_S35.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/lgn17_S35_L001_R1_001.fastq -o ./trimmed_reads/lgn17_S35.fastq.gz -h ./fastp_report/lgn17_S35.html -j ./fastp_report/lgn17_S35.json -R lgn17_S35 fastp report \n",
      "fastp v0.20.0, time used: 3 seconds\n",
      "Detecting adapter sequence for read1...\n",
      ">Illumina TruSeq Adapter Read 1\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 100000\n",
      "total bases: 15100000\n",
      "Q20 bases: 14796206(97.9881%)\n",
      "Q30 bases: 14270090(94.5039%)\n",
      "\n",
      "Read1 after filtering:\n",
      "total reads: 99879\n",
      "total bases: 14654812\n",
      "Q20 bases: 14374700(98.0886%)\n",
      "Q30 bases: 13875196(94.6801%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 99879\n",
      "reads failed due to low quality: 116\n",
      "reads failed due to too many N: 5\n",
      "reads failed due to too short: 0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reads with adapter trimmed: 13233\n",
      "bases trimmed due to adapters: 426988\n",
      "\n",
      "Duplication rate (may be overestimated since this is SE data): 4.23233%\n",
      "\n",
      "JSON report: ./fastp_report/WT25_S7.json\n",
      "HTML report: ./fastp_report/WT25_S7.html\n",
      "\n",
      "fastp -w 4 -i ./raw_reads/WT25_S7_L001_R1_001.fastq -o ./trimmed_reads/WT25_S7.fastq.gz -h ./fastp_report/WT25_S7.html -j ./fastp_report/WT25_S7.json -R WT25_S7 fastp report \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#Use fastp to look at your sequence files and verify they are ok\n",
    "#If Illumina adapter sequences are detected at the end of your read, fastp will remove them\n",
    "#for single end fastp -i in.R1.fq.gz -o out.R1.fq.gz \n",
    "\n",
    "#run fastp in command line\n",
    "mkdir ./fastp_report   #reports go here\n",
    "mkdir ./trimmed_reads  #trimmed reads go here\n",
    "\n",
    "# this is a more complicated command\n",
    "# here, we use the command 'find' to identify all files with same ending name\n",
    "# and process them one by one -- 'do'-ing the middle stuff\n",
    "# until there are no more files and we are 'done'\n",
    "\n",
    "# in the middle, we are looking at the raw reads with fastp\n",
    "find ./raw_reads -name \"*_L001_R1_001.fastq\" | while read f; do\n",
    "STEM=$(basename \"${f}\" _L001_R1_001.fastq)\n",
    "fastp -w 4 -i ./raw_reads/${STEM}_L001_R1_001.fastq \\\n",
    "    -o ./trimmed_reads/${STEM}.fastq.gz \\\n",
    "    -h ./fastp_report/${STEM}.html \\\n",
    "    -j ./fastp_report/${STEM}.json \\\n",
    "    -R \"${STEM} fastp report\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) 50/104 - 48.08% of files done   7.269/15.130 MB - 48.04% of file sizes done\n",
      "Processing WT97_S1.html - 0.000 MB   2019-10-15.23:48:11\n",
      "100/104 - 96.15% of files done   14.377/15.130 MB - 95.02% of file sizes done\n",
      "Processing dbl54_S44.json - 0.000 MB   2019-10-15.23:49:11\n",
      "104/104 - 100.00% of files done   15.130/15.130 MB - 100.00% of file sizes done\n",
      "Processing nod2_S26.json - 0.000 MB   2019-10-15.23:49:17\n",
      "(alignment) 0/52 -  0.00% of files done   0.000/469.709 MB -  0.00% of file sizes done\n",
      "Processing lgn3_S39.fastq.gz - 9.139 MB   2019-10-15.23:49:17\n",
      "1/52 -  1.92% of files done   9.139/469.709 MB -  1.95% of file sizes done\n",
      "Processing lgn42_S34.fastq.gz - 9.049 MB   2019-10-15.23:49:19\n",
      "2/52 -  3.85% of files done   18.188/469.709 MB -  3.87% of file sizes done\n",
      "Processing WT25_S7.fastq.gz - 8.873 MB   2019-10-15.23:49:20\n",
      "3/52 -  5.77% of files done   27.061/469.709 MB -  5.76% of file sizes done\n",
      "Processing nod30_S17.fastq.gz - 9.052 MB   2019-10-15.23:49:22\n",
      "4/52 -  7.69% of files done   36.113/469.709 MB -  7.69% of file sizes done\n",
      "Processing dbl50_S43.fastq.gz - 9.101 MB   2019-10-15.23:49:24\n",
      "5/52 -  9.62% of files done   45.214/469.709 MB -  9.63% of file sizes done\n",
      "Processing WT38_S5.fastq.gz - 9.151 MB   2019-10-15.23:49:26\n",
      "6/52 - 11.54% of files done   54.366/469.709 MB - 11.57% of file sizes done\n",
      "Processing nod11_S24.fastq.gz - 9.030 MB   2019-10-15.23:49:27\n",
      "7/52 - 13.46% of files done   63.396/469.709 MB - 13.50% of file sizes done\n",
      "Processing lgn72_S31.fastq.gz - 9.079 MB   2019-10-15.23:49:29\n",
      "8/52 - 15.38% of files done   72.475/469.709 MB - 15.43% of file sizes done\n",
      "Processing dbl66_S49.fastq.gz - 9.123 MB   2019-10-15.23:49:31\n",
      "9/52 - 17.31% of files done   81.598/469.709 MB - 17.37% of file sizes done\n",
      "Processing WT93_S11.fastq.gz - 8.937 MB   2019-10-15.23:49:33\n",
      "10/52 - 19.23% of files done   90.536/469.709 MB - 19.27% of file sizes done\n",
      "Processing lgn37_S33.fastq.gz - 9.143 MB   2019-10-15.23:49:36\n",
      "11/52 - 21.15% of files done   99.678/469.709 MB - 21.22% of file sizes done\n",
      "Processing dbl56_S50.fastq.gz - 9.114 MB   2019-10-15.23:49:38\n",
      "12/52 - 23.08% of files done   108.793/469.709 MB - 23.16% of file sizes done\n",
      "Processing lgn59_S28.fastq.gz - 9.039 MB   2019-10-15.23:49:38\n",
      "13/52 - 25.00% of files done   117.832/469.709 MB - 25.09% of file sizes done\n",
      "Processing nod48_S16.fastq.gz - 8.996 MB   2019-10-15.23:49:40\n",
      "14/52 - 26.92% of files done   126.828/469.709 MB - 27.00% of file sizes done\n",
      "Processing WT55_S9.fastq.gz - 9.097 MB   2019-10-15.23:49:41\n",
      "15/52 - 28.85% of files done   135.925/469.709 MB - 28.94% of file sizes done\n",
      "Processing WT97_S1.fastq.gz - 9.161 MB   2019-10-15.23:49:42\n",
      "16/52 - 30.77% of files done   145.086/469.709 MB - 30.89% of file sizes done\n",
      "Processing WT13_S12.fastq.gz - 8.991 MB   2019-10-15.23:49:44\n",
      "17/52 - 32.69% of files done   154.078/469.709 MB - 32.80% of file sizes done\n",
      "Processing dbl88_S51.fastq.gz - 9.121 MB   2019-10-15.23:49:44\n",
      "18/52 - 34.62% of files done   163.198/469.709 MB - 34.74% of file sizes done\n",
      "Processing lgn33_S38.fastq.gz - 9.092 MB   2019-10-15.23:49:45\n",
      "19/52 - 36.54% of files done   172.290/469.709 MB - 36.68% of file sizes done\n",
      "Processing nod16_S15.fastq.gz - 9.095 MB   2019-10-15.23:49:46\n",
      "20/52 - 38.46% of files done   181.384/469.709 MB - 38.62% of file sizes done\n",
      "Processing dbl89_S40.fastq.gz - 8.837 MB   2019-10-15.23:49:47\n",
      "21/52 - 40.38% of files done   190.221/469.709 MB - 40.50% of file sizes done\n",
      "Processing lgn28_S37.fastq.gz - 8.976 MB   2019-10-15.23:49:47\n",
      "22/52 - 42.31% of files done   199.197/469.709 MB - 42.41% of file sizes done\n",
      "Processing WT78_S2.fastq.gz - 9.042 MB   2019-10-15.23:49:48\n",
      "23/52 - 44.23% of files done   208.240/469.709 MB - 44.33% of file sizes done\n",
      "Processing WT58_S3.fastq.gz - 8.752 MB   2019-10-15.23:49:49\n",
      "24/52 - 46.15% of files done   216.992/469.709 MB - 46.20% of file sizes done\n",
      "Processing dbl57_S52.fastq.gz - 9.045 MB   2019-10-15.23:49:51\n",
      "25/52 - 48.08% of files done   226.038/469.709 MB - 48.12% of file sizes done\n",
      "Processing nod14_S18.fastq.gz - 9.049 MB   2019-10-15.23:49:52\n",
      "26/52 - 50.00% of files done   235.087/469.709 MB - 50.05% of file sizes done\n",
      "Processing WT18_S4.fastq.gz - 9.028 MB   2019-10-15.23:49:52\n",
      "27/52 - 51.92% of files done   244.114/469.709 MB - 51.97% of file sizes done\n",
      "Processing nod36_S21.fastq.gz - 9.135 MB   2019-10-15.23:49:53\n",
      "28/52 - 53.85% of files done   253.249/469.709 MB - 53.92% of file sizes done\n",
      "Processing dbl28_S48.fastq.gz - 9.101 MB   2019-10-15.23:49:54\n",
      "29/52 - 55.77% of files done   262.351/469.709 MB - 55.85% of file sizes done\n",
      "Processing nod61_S25.fastq.gz - 8.952 MB   2019-10-15.23:49:56\n",
      "30/52 - 57.69% of files done   271.303/469.709 MB - 57.76% of file sizes done\n",
      "Processing nod62_S14.fastq.gz - 8.924 MB   2019-10-15.23:49:56\n",
      "31/52 - 59.62% of files done   280.227/469.709 MB - 59.66% of file sizes done\n",
      "Processing nod6_S20.fastq.gz - 9.097 MB   2019-10-15.23:49:57\n",
      "32/52 - 61.54% of files done   289.324/469.709 MB - 61.60% of file sizes done\n",
      "Processing lgn21_S36.fastq.gz - 8.966 MB   2019-10-15.23:49:58\n",
      "33/52 - 63.46% of files done   298.290/469.709 MB - 63.51% of file sizes done\n",
      "Processing WT24_S10.fastq.gz - 8.999 MB   2019-10-15.23:49:58\n",
      "34/52 - 65.38% of files done   307.289/469.709 MB - 65.42% of file sizes done\n",
      "Processing dbl9_S45.fastq.gz - 9.126 MB   2019-10-15.23:49:59\n",
      "35/52 - 67.31% of files done   316.415/469.709 MB - 67.36% of file sizes done\n",
      "Processing lgn8_S32.fastq.gz - 8.984 MB   2019-10-15.23:50:00\n",
      "36/52 - 69.23% of files done   325.399/469.709 MB - 69.28% of file sizes done\n",
      "Processing dbl65_S42.fastq.gz - 9.088 MB   2019-10-15.23:50:00\n",
      "37/52 - 71.15% of files done   334.487/469.709 MB - 71.21% of file sizes done\n",
      "Processing nod2_S26.fastq.gz - 8.899 MB   2019-10-15.23:50:01\n",
      "38/52 - 73.08% of files done   343.386/469.709 MB - 73.11% of file sizes done\n",
      "Processing dbl19_S46.fastq.gz - 9.073 MB   2019-10-15.23:50:02\n",
      "39/52 - 75.00% of files done   352.459/469.709 MB - 75.04% of file sizes done\n",
      "Processing lgn17_S35.fastq.gz - 8.815 MB   2019-10-15.23:50:02\n",
      "40/52 - 76.92% of files done   361.274/469.709 MB - 76.91% of file sizes done\n",
      "Processing lgn46_S30.fastq.gz - 9.182 MB   2019-10-15.23:50:03\n",
      "41/52 - 78.85% of files done   370.456/469.709 MB - 78.87% of file sizes done\n",
      "Processing lgn77_S29.fastq.gz - 9.110 MB   2019-10-15.23:50:04\n",
      "42/52 - 80.77% of files done   379.566/469.709 MB - 80.81% of file sizes done\n",
      "Processing WT10_S8.fastq.gz - 9.036 MB   2019-10-15.23:50:04\n",
      "43/52 - 82.69% of files done   388.602/469.709 MB - 82.73% of file sizes done\n",
      "Processing WT41_S6.fastq.gz - 9.008 MB   2019-10-15.23:50:05\n",
      "44/52 - 84.62% of files done   397.609/469.709 MB - 84.65% of file sizes done\n",
      "Processing nod43_S22.fastq.gz - 9.114 MB   2019-10-15.23:50:07\n",
      "45/52 - 86.54% of files done   406.723/469.709 MB - 86.59% of file sizes done\n",
      "Processing dbl54_S44.fastq.gz - 8.994 MB   2019-10-15.23:50:08\n",
      "46/52 - 88.46% of files done   415.717/469.709 MB - 88.51% of file sizes done\n",
      "Processing dbl70_S47.fastq.gz - 9.095 MB   2019-10-15.23:50:08\n",
      "47/52 - 90.38% of files done   424.812/469.709 MB - 90.44% of file sizes done\n",
      "Processing lgn39_S27.fastq.gz - 9.035 MB   2019-10-15.23:50:10\n",
      "48/52 - 92.31% of files done   433.847/469.709 MB - 92.37% of file sizes done\n",
      "Processing nod47_S23.fastq.gz - 9.064 MB   2019-10-15.23:50:11\n",
      "49/52 - 94.23% of files done   442.912/469.709 MB - 94.29% of file sizes done\n",
      "Processing WT95_S13.fastq.gz - 8.810 MB   2019-10-15.23:50:12\n",
      "50/52 - 96.15% of files done   451.722/469.709 MB - 96.17% of file sizes done\n",
      "Processing dbl31_S41.fastq.gz - 9.093 MB   2019-10-15.23:50:12\n",
      "51/52 - 98.08% of files done   460.816/469.709 MB - 98.11% of file sizes done\n",
      "Processing nod60_S19.fastq.gz - 8.893 MB   2019-10-15.23:50:13\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# save your work\n",
    "# put fastp results into the datastore\n",
    "icd \n",
    "imkdir -p ./CSHL_Cereal_Genomics_2019/finished_analysis\n",
    "iput -rKPf -N 4 ./fastp_report\n",
    "iput -rKPf -N 4 ./trimmed_reads"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) total 5199164\n",
      "-rw-r--r-- 1 sal269 iplant-everyone   72140097 Oct 15 22:23 Zea_mays.AGPv3.30.repaired.gtf\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 2102361699 Oct 15 22:23 ZmaysAGPv3.30.fa\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone  690596865 Oct 15 22:23 \u001b[0m\u001b[01;32mZmaysAGPv3.30.hisat2.1.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone  513629196 Oct 15 22:23 \u001b[01;32mZmaysAGPv3.30.hisat2.2.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone    1146824 Oct 15 22:23 \u001b[01;32mZmaysAGPv3.30.hisat2.3.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone  513629192 Oct 15 22:23 \u001b[01;32mZmaysAGPv3.30.hisat2.4.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone  907379983 Oct 15 22:23 \u001b[01;32mZmaysAGPv3.30.hisat2.5.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone  523017430 Oct 15 22:23 \u001b[01;32mZmaysAGPv3.30.hisat2.6.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone          8 Oct 15 22:24 \u001b[01;32mZmaysAGPv3.30.hisat2.7.ht2\u001b[0m\n",
      "-rwxr-xr-x 1 sal269 iplant-everyone          8 Oct 15 22:24 \u001b[01;32mZmaysAGPv3.30.hisat2.8.ht2\u001b[0m\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#check that we have some genome index files\n",
    "\n",
    "ls -l ./maize_genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) @HD\tVN:1.0\tSO:unsorted\n",
      "@SQ\tSN:scaffold_68\tLN:105\n",
      "@SQ\tSN:scaffold_140\tLN:109\n",
      "@SQ\tSN:scaffold_427\tLN:110\n",
      "@SQ\tSN:scaffold_236\tLN:114\n",
      "@SQ\tSN:scaffold_462\tLN:115\n",
      "@SQ\tSN:scaffold_47\tLN:115\n",
      "@SQ\tSN:scaffold_119\tLN:116\n",
      "@SQ\tSN:scaffold_36\tLN:116\n",
      "@SQ\tSN:scaffold_382\tLN:121\n",
      "@SQ\tSN:scaffold_494\tLN:124\n",
      "@SQ\tSN:scaffold_453\tLN:124\n",
      "@SQ\tSN:scaffold_77\tLN:129\n",
      "@SQ\tSN:scaffold_328\tLN:131\n",
      "@SQ\tSN:scaffold_61\tLN:138\n",
      "@SQ\tSN:scaffold_275\tLN:141\n",
      "@SQ\tSN:scaffold_124\tLN:143\n",
      "@SQ\tSN:scaffold_239\tLN:147\n",
      "@SQ\tSN:scaffold_221\tLN:155\n",
      "@SQ\tSN:scaffold_235\tLN:159\n",
      "@SQ\tSN:scaffold_194\tLN:159\n",
      "@SQ\tSN:scaffold_375\tLN:164\n",
      "@SQ\tSN:scaffold_196\tLN:166\n",
      "@SQ\tSN:scaffold_482\tLN:167\n",
      "@SQ\tSN:scaffold_407\tLN:171\n",
      "@SQ\tSN:scaffold_312\tLN:172\n",
      "@SQ\tSN:scaffold_489\tLN:172\n",
      "@SQ\tSN:scaffold_454\tLN:174\n",
      "@SQ\tSN:scaffold_84\tLN:177\n",
      "@SQ\tSN:scaffold_227\tLN:178\n",
      "@SQ\tSN:scaffold_165\tLN:182\n",
      "@SQ\tSN:scaffold_37\tLN:183\n",
      "@SQ\tSN:scaffold_302\tLN:184\n",
      "@SQ\tSN:scaffold_458\tLN:193\n",
      "@SQ\tSN:scaffold_33\tLN:197\n",
      "@SQ\tSN:scaffold_100\tLN:201\n",
      "@SQ\tSN:scaffold_187\tLN:202\n",
      "@SQ\tSN:scaffold_296\tLN:205\n",
      "@SQ\tSN:scaffold_263\tLN:205\n",
      "@SQ\tSN:scaffold_274\tLN:206\n",
      "@SQ\tSN:scaffold_436\tLN:208\n",
      "@SQ\tSN:scaffold_415\tLN:210\n",
      "@SQ\tSN:scaffold_160\tLN:218\n",
      "@SQ\tSN:scaffold_411\tLN:220\n",
      "@SQ\tSN:scaffold_186\tLN:222\n",
      "@SQ\tSN:scaffold_38\tLN:223\n",
      "@SQ\tSN:scaffold_21\tLN:225\n",
      "@SQ\tSN:scaffold_306\tLN:226\n",
      "@SQ\tSN:scaffold_30\tLN:226\n",
      "@SQ\tSN:scaffold_11\tLN:231\n",
      "@SQ\tSN:scaffold_473\tLN:236\n",
      "@SQ\tSN:scaffold_486\tLN:238\n",
      "@SQ\tSN:scaffold_278\tLN:239\n",
      "@SQ\tSN:scaffold_149\tLN:239\n",
      "@SQ\tSN:scaffold_164\tLN:242\n",
      "@SQ\tSN:scaffold_82\tLN:254\n",
      "@SQ\tSN:scaffold_117\tLN:255\n",
      "@SQ\tSN:scaffold_298\tLN:260\n",
      "@SQ\tSN:scaffold_132\tLN:260\n",
      "@SQ\tSN:scaffold_459\tLN:261\n",
      "@SQ\tSN:scaffold_118\tLN:262\n",
      "@SQ\tSN:scaffold_493\tLN:270\n",
      "@SQ\tSN:scaffold_310\tLN:271\n",
      "@SQ\tSN:scaffold_10\tLN:276\n",
      "@SQ\tSN:scaffold_169\tLN:282\n",
      "@SQ\tSN:scaffold_327\tLN:287\n",
      "@SQ\tSN:scaffold_403\tLN:289\n",
      "@SQ\tSN:scaffold_209\tLN:293\n",
      "@SQ\tSN:scaffold_390\tLN:294\n",
      "@SQ\tSN:scaffold_326\tLN:296\n",
      "@SQ\tSN:scaffold_487\tLN:305\n",
      "@SQ\tSN:scaffold_76\tLN:305\n",
      "@SQ\tSN:scaffold_338\tLN:309\n",
      "@SQ\tSN:scaffold_158\tLN:311\n",
      "@SQ\tSN:scaffold_86\tLN:312\n",
      "@SQ\tSN:scaffold_259\tLN:312\n",
      "@SQ\tSN:scaffold_204\tLN:313\n",
      "@SQ\tSN:scaffold_189\tLN:318\n",
      "@SQ\tSN:scaffold_362\tLN:328\n",
      "@SQ\tSN:scaffold_414\tLN:328\n",
      "@SQ\tSN:scaffold_48\tLN:329\n",
      "@SQ\tSN:scaffold_40\tLN:350\n",
      "@SQ\tSN:scaffold_25\tLN:350\n",
      "@SQ\tSN:scaffold_294\tLN:351\n",
      "@SQ\tSN:scaffold_395\tLN:356\n",
      "@SQ\tSN:scaffold_126\tLN:356\n",
      "@SQ\tSN:scaffold_295\tLN:364\n",
      "@SQ\tSN:scaffold_255\tLN:387\n",
      "@SQ\tSN:scaffold_339\tLN:389\n",
      "@SQ\tSN:scaffold_461\tLN:390\n",
      "@SQ\tSN:scaffold_476\tLN:393\n",
      "@SQ\tSN:scaffold_356\tLN:397\n",
      "@SQ\tSN:scaffold_213\tLN:400\n",
      "@SQ\tSN:scaffold_90\tLN:402\n",
      "@SQ\tSN:scaffold_426\tLN:410\n",
      "@SQ\tSN:scaffold_148\tLN:412\n",
      "@SQ\tSN:scaffold_191\tLN:413\n",
      "@SQ\tSN:scaffold_443\tLN:415\n",
      "@SQ\tSN:scaffold_231\tLN:416\n",
      "@SQ\tSN:scaffold_1\tLN:417\n",
      "@SQ\tSN:scaffold_444\tLN:418\n",
      "@SQ\tSN:scaffold_24\tLN:419\n",
      "@SQ\tSN:scaffold_156\tLN:420\n",
      "@SQ\tSN:scaffold_471\tLN:437\n",
      "@SQ\tSN:scaffold_70\tLN:439\n",
      "@SQ\tSN:scaffold_265\tLN:442\n",
      "@SQ\tSN:scaffold_183\tLN:445\n",
      "@SQ\tSN:scaffold_372\tLN:445\n",
      "@SQ\tSN:scaffold_420\tLN:445\n",
      "@SQ\tSN:scaffold_438\tLN:445\n",
      "@SQ\tSN:scaffold_346\tLN:446\n",
      "@SQ\tSN:scaffold_488\tLN:451\n",
      "@SQ\tSN:scaffold_297\tLN:451\n",
      "@SQ\tSN:scaffold_365\tLN:452\n",
      "@SQ\tSN:scaffold_341\tLN:458\n",
      "@SQ\tSN:scaffold_15\tLN:459\n",
      "@SQ\tSN:scaffold_325\tLN:467\n",
      "@SQ\tSN:scaffold_218\tLN:469\n",
      "@SQ\tSN:scaffold_399\tLN:473\n",
      "@SQ\tSN:scaffold_320\tLN:474\n",
      "@SQ\tSN:scaffold_28\tLN:477\n",
      "@SQ\tSN:scaffold_18\tLN:478\n",
      "@SQ\tSN:scaffold_168\tLN:479\n",
      "@SQ\tSN:scaffold_32\tLN:485\n",
      "@SQ\tSN:scaffold_113\tLN:489\n",
      "@SQ\tSN:scaffold_111\tLN:490\n",
      "@SQ\tSN:scaffold_343\tLN:490\n",
      "@SQ\tSN:scaffold_283\tLN:494\n",
      "@SQ\tSN:scaffold_91\tLN:496\n",
      "@SQ\tSN:scaffold_29\tLN:503\n",
      "@SQ\tSN:scaffold_136\tLN:505\n",
      "@SQ\tSN:scaffold_342\tLN:510\n",
      "@SQ\tSN:scaffold_307\tLN:514\n",
      "@SQ\tSN:scaffold_409\tLN:526\n",
      "@SQ\tSN:scaffold_439\tLN:527\n",
      "@SQ\tSN:scaffold_45\tLN:527\n",
      "@SQ\tSN:scaffold_65\tLN:530\n",
      "@SQ\tSN:scaffold_16\tLN:546\n",
      "@SQ\tSN:scaffold_305\tLN:548\n",
      "@SQ\tSN:scaffold_22\tLN:549\n",
      "@SQ\tSN:scaffold_321\tLN:551\n",
      "@SQ\tSN:scaffold_271\tLN:553\n",
      "@SQ\tSN:scaffold_234\tLN:560\n",
      "@SQ\tSN:scaffold_102\tLN:565\n",
      "@SQ\tSN:scaffold_421\tLN:566\n",
      "@SQ\tSN:scaffold_347\tLN:572\n",
      "@SQ\tSN:scaffold_75\tLN:573\n",
      "@SQ\tSN:scaffold_178\tLN:575\n",
      "@SQ\tSN:scaffold_418\tLN:579\n",
      "@SQ\tSN:scaffold_95\tLN:587\n",
      "@SQ\tSN:scaffold_9\tLN:591\n",
      "@SQ\tSN:scaffold_54\tLN:595\n",
      "@SQ\tSN:scaffold_237\tLN:604\n",
      "@SQ\tSN:scaffold_173\tLN:605\n",
      "@SQ\tSN:scaffold_50\tLN:606\n",
      "@SQ\tSN:scaffold_129\tLN:614\n",
      "@SQ\tSN:scaffold_155\tLN:616\n",
      "@SQ\tSN:scaffold_128\tLN:616\n",
      "@SQ\tSN:scaffold_116\tLN:617\n",
      "@SQ\tSN:scaffold_122\tLN:618\n",
      "@SQ\tSN:scaffold_350\tLN:619\n",
      "@SQ\tSN:scaffold_447\tLN:620\n",
      "@SQ\tSN:scaffold_279\tLN:626\n",
      "@SQ\tSN:scaffold_430\tLN:628\n",
      "@SQ\tSN:scaffold_87\tLN:629\n",
      "@SQ\tSN:scaffold_330\tLN:635\n",
      "@SQ\tSN:scaffold_185\tLN:638\n",
      "@SQ\tSN:scaffold_83\tLN:641\n",
      "@SQ\tSN:scaffold_103\tLN:641\n",
      "@SQ\tSN:scaffold_355\tLN:643\n",
      "@SQ\tSN:scaffold_388\tLN:647\n",
      "@SQ\tSN:scaffold_120\tLN:652\n",
      "@SQ\tSN:scaffold_273\tLN:653\n",
      "@SQ\tSN:scaffold_470\tLN:654\n",
      "@SQ\tSN:scaffold_361\tLN:656\n",
      "@SQ\tSN:scaffold_264\tLN:662\n",
      "@SQ\tSN:scaffold_269\tLN:666\n",
      "@SQ\tSN:scaffold_205\tLN:667\n",
      "@SQ\tSN:scaffold_220\tLN:675\n",
      "@SQ\tSN:scaffold_110\tLN:683\n",
      "@SQ\tSN:scaffold_370\tLN:684\n",
      "@SQ\tSN:scaffold_385\tLN:685\n",
      "@SQ\tSN:scaffold_354\tLN:689\n",
      "@SQ\tSN:scaffold_190\tLN:691\n",
      "@SQ\tSN:scaffold_13\tLN:693\n",
      "@SQ\tSN:scaffold_208\tLN:693\n",
      "@SQ\tSN:scaffold_62\tLN:699\n",
      "@SQ\tSN:scaffold_464\tLN:701\n",
      "@SQ\tSN:scaffold_366\tLN:708\n",
      "@SQ\tSN:scaffold_251\tLN:709\n",
      "@SQ\tSN:scaffold_329\tLN:711\n",
      "@SQ\tSN:scaffold_292\tLN:719\n",
      "@SQ\tSN:scaffold_410\tLN:720\n",
      "@SQ\tSN:scaffold_428\tLN:730\n",
      "@SQ\tSN:scaffold_167\tLN:731\n",
      "@SQ\tSN:scaffold_51\tLN:743\n",
      "@SQ\tSN:scaffold_417\tLN:750\n",
      "@SQ\tSN:scaffold_92\tLN:762\n",
      "@SQ\tSN:scaffold_195\tLN:765\n",
      "@SQ\tSN:scaffold_293\tLN:772\n",
      "@SQ\tSN:scaffold_308\tLN:780\n",
      "@SQ\tSN:scaffold_468\tLN:783\n",
      "@SQ\tSN:scaffold_176\tLN:785\n",
      "@SQ\tSN:scaffold_93\tLN:792\n",
      "@SQ\tSN:scaffold_8\tLN:793\n",
      "@SQ\tSN:scaffold_261\tLN:795\n",
      "@SQ\tSN:scaffold_184\tLN:798\n",
      "@SQ\tSN:scaffold_474\tLN:802\n",
      "@SQ\tSN:scaffold_144\tLN:805\n",
      "@SQ\tSN:scaffold_246\tLN:814\n",
      "@SQ\tSN:scaffold_81\tLN:814\n",
      "@SQ\tSN:scaffold_241\tLN:816\n",
      "@SQ\tSN:scaffold_233\tLN:816\n",
      "@SQ\tSN:scaffold_398\tLN:822\n",
      "@SQ\tSN:scaffold_301\tLN:836\n",
      "@SQ\tSN:scaffold_367\tLN:841\n",
      "@SQ\tSN:scaffold_80\tLN:849\n",
      "@SQ\tSN:scaffold_200\tLN:865\n",
      "@SQ\tSN:scaffold_175\tLN:865\n",
      "@SQ\tSN:scaffold_478\tLN:866\n",
      "@SQ\tSN:scaffold_7\tLN:872\n",
      "@SQ\tSN:scaffold_373\tLN:874\n",
      "@SQ\tSN:scaffold_331\tLN:885\n",
      "@SQ\tSN:scaffold_180\tLN:892\n",
      "@SQ\tSN:scaffold_53\tLN:893\n",
      "@SQ\tSN:scaffold_313\tLN:897\n",
      "@SQ\tSN:scaffold_240\tLN:898\n",
      "@SQ\tSN:scaffold_266\tLN:899\n",
      "@SQ\tSN:scaffold_425\tLN:901\n",
      "@SQ\tSN:scaffold_317\tLN:907\n",
      "@SQ\tSN:scaffold_348\tLN:913\n",
      "@SQ\tSN:scaffold_369\tLN:924\n",
      "@SQ\tSN:scaffold_224\tLN:932\n",
      "@SQ\tSN:scaffold_96\tLN:934\n",
      "@SQ\tSN:scaffold_104\tLN:934\n",
      "@SQ\tSN:scaffold_188\tLN:949\n",
      "@SQ\tSN:scaffold_106\tLN:949\n",
      "@SQ\tSN:scaffold_56\tLN:958\n",
      "@SQ\tSN:scaffold_422\tLN:969\n",
      "@SQ\tSN:scaffold_222\tLN:972\n",
      "@SQ\tSN:scaffold_172\tLN:975\n",
      "@SQ\tSN:scaffold_371\tLN:981\n",
      "@SQ\tSN:scaffold_66\tLN:986\n",
      "@SQ\tSN:scaffold_380\tLN:990\n",
      "@SQ\tSN:scaffold_437\tLN:995\n",
      "@SQ\tSN:scaffold_423\tLN:997\n",
      "@SQ\tSN:scaffold_455\tLN:999\n",
      "@SQ\tSN:scaffold_401\tLN:999\n",
      "@SQ\tSN:scaffold_88\tLN:1000\n",
      "@SQ\tSN:scaffold_449\tLN:1000\n",
      "@SQ\tSN:scaffold_139\tLN:1002\n",
      "@SQ\tSN:scaffold_41\tLN:1003\n",
      "@SQ\tSN:scaffold_247\tLN:1003\n",
      "@SQ\tSN:scaffold_416\tLN:1027\n",
      "@SQ\tSN:scaffold_465\tLN:1032\n",
      "@SQ\tSN:scaffold_193\tLN:1034\n",
      "@SQ\tSN:scaffold_393\tLN:1037\n",
      "@SQ\tSN:scaffold_311\tLN:1040\n",
      "@SQ\tSN:scaffold_202\tLN:1040\n",
      "@SQ\tSN:scaffold_337\tLN:1041\n",
      "@SQ\tSN:scaffold_477\tLN:1047\n",
      "@SQ\tSN:scaffold_197\tLN:1055\n",
      "@SQ\tSN:scaffold_378\tLN:1059\n",
      "@SQ\tSN:scaffold_4\tLN:1065\n",
      "@SQ\tSN:scaffold_446\tLN:1066\n",
      "@SQ\tSN:scaffold_318\tLN:1066\n",
      "@SQ\tSN:scaffold_451\tLN:1068\n",
      "@SQ\tSN:scaffold_349\tLN:1071\n",
      "@SQ\tSN:scaffold_206\tLN:1071\n",
      "@SQ\tSN:scaffold_64\tLN:1092\n",
      "@SQ\tSN:scaffold_402\tLN:1098\n",
      "@SQ\tSN:scaffold_406\tLN:1109\n",
      "@SQ\tSN:scaffold_141\tLN:1110\n",
      "@SQ\tSN:scaffold_485\tLN:1116\n",
      "@SQ\tSN:scaffold_5\tLN:1118\n",
      "@SQ\tSN:scaffold_59\tLN:1133\n",
      "@SQ\tSN:scaffold_228\tLN:1136\n",
      "@SQ\tSN:scaffold_52\tLN:1141\n",
      "@SQ\tSN:scaffold_258\tLN:1143\n",
      "@SQ\tSN:scaffold_60\tLN:1145\n",
      "@SQ\tSN:scaffold_413\tLN:1149\n",
      "@SQ\tSN:scaffold_363\tLN:1156\n",
      "@SQ\tSN:scaffold_123\tLN:1164\n",
      "@SQ\tSN:scaffold_377\tLN:1167\n",
      "@SQ\tSN:scaffold_85\tLN:1170\n",
      "@SQ\tSN:scaffold_67\tLN:1170\n",
      "@SQ\tSN:scaffold_353\tLN:1170\n",
      "@SQ\tSN:scaffold_322\tLN:1171\n",
      "@SQ\tSN:scaffold_244\tLN:1178\n",
      "@SQ\tSN:scaffold_254\tLN:1190\n",
      "@SQ\tSN:scaffold_27\tLN:1190\n",
      "@SQ\tSN:scaffold_424\tLN:1192\n",
      "@SQ\tSN:scaffold_394\tLN:1209\n",
      "@SQ\tSN:scaffold_20\tLN:1215\n",
      "@SQ\tSN:scaffold_14\tLN:1230\n",
      "@SQ\tSN:scaffold_6\tLN:1240\n",
      "@SQ\tSN:scaffold_492\tLN:1242\n",
      "@SQ\tSN:scaffold_290\tLN:1242\n",
      "@SQ\tSN:scaffold_232\tLN:1247\n",
      "@SQ\tSN:scaffold_483\tLN:1248\n",
      "@SQ\tSN:scaffold_257\tLN:1249\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@SQ\tSN:scaffold_57\tLN:1253\n",
      "@SQ\tSN:scaffold_374\tLN:1268\n",
      "@SQ\tSN:scaffold_386\tLN:1269\n",
      "@SQ\tSN:scaffold_23\tLN:1270\n",
      "@SQ\tSN:scaffold_73\tLN:1270\n",
      "@SQ\tSN:scaffold_285\tLN:1285\n",
      "@SQ\tSN:scaffold_219\tLN:1291\n",
      "@SQ\tSN:scaffold_154\tLN:1292\n",
      "@SQ\tSN:scaffold_400\tLN:1292\n",
      "@SQ\tSN:scaffold_490\tLN:1293\n",
      "@SQ\tSN:scaffold_404\tLN:1295\n",
      "@SQ\tSN:scaffold_174\tLN:1308\n",
      "@SQ\tSN:scaffold_72\tLN:1311\n",
      "@SQ\tSN:scaffold_225\tLN:1312\n",
      "@SQ\tSN:scaffold_358\tLN:1318\n",
      "@SQ\tSN:scaffold_452\tLN:1325\n",
      "@SQ\tSN:scaffold_472\tLN:1327\n",
      "@SQ\tSN:scaffold_138\tLN:1328\n",
      "@SQ\tSN:scaffold_98\tLN:1334\n",
      "@SQ\tSN:scaffold_491\tLN:1335\n",
      "@SQ\tSN:scaffold_479\tLN:1336\n",
      "@SQ\tSN:scaffold_163\tLN:1341\n",
      "@SQ\tSN:scaffold_359\tLN:1346\n",
      "@SQ\tSN:scaffold_215\tLN:1351\n",
      "@SQ\tSN:scaffold_211\tLN:1357\n",
      "@SQ\tSN:scaffold_159\tLN:1359\n",
      "@SQ\tSN:scaffold_392\tLN:1362\n",
      "@SQ\tSN:scaffold_253\tLN:1370\n",
      "@SQ\tSN:scaffold_284\tLN:1374\n",
      "@SQ\tSN:scaffold_314\tLN:1394\n",
      "@SQ\tSN:scaffold_352\tLN:1394\n",
      "@SQ\tSN:scaffold_34\tLN:1395\n",
      "@SQ\tSN:scaffold_182\tLN:1399\n",
      "@SQ\tSN:scaffold_105\tLN:1403\n",
      "@SQ\tSN:scaffold_3\tLN:1415\n",
      "@SQ\tSN:scaffold_131\tLN:1420\n",
      "@SQ\tSN:scaffold_384\tLN:1420\n",
      "@SQ\tSN:scaffold_336\tLN:1422\n",
      "@SQ\tSN:scaffold_161\tLN:1431\n",
      "@SQ\tSN:scaffold_112\tLN:1436\n",
      "@SQ\tSN:scaffold_381\tLN:1440\n",
      "@SQ\tSN:scaffold_216\tLN:1443\n",
      "@SQ\tSN:scaffold_463\tLN:1449\n",
      "@SQ\tSN:scaffold_303\tLN:1461\n",
      "@SQ\tSN:scaffold_262\tLN:1469\n",
      "@SQ\tSN:scaffold_42\tLN:1471\n",
      "@SQ\tSN:scaffold_309\tLN:1471\n",
      "@SQ\tSN:scaffold_357\tLN:1473\n",
      "@SQ\tSN:scaffold_198\tLN:1480\n",
      "@SQ\tSN:scaffold_323\tLN:1498\n",
      "@SQ\tSN:scaffold_248\tLN:1505\n",
      "@SQ\tSN:scaffold_192\tLN:1521\n",
      "@SQ\tSN:scaffold_270\tLN:1524\n",
      "@SQ\tSN:scaffold_412\tLN:1544\n",
      "@SQ\tSN:scaffold_108\tLN:1550\n",
      "@SQ\tSN:scaffold_199\tLN:1552\n",
      "@SQ\tSN:scaffold_484\tLN:1556\n",
      "@SQ\tSN:scaffold_177\tLN:1567\n",
      "@SQ\tSN:scaffold_166\tLN:1574\n",
      "@SQ\tSN:scaffold_450\tLN:1578\n",
      "@SQ\tSN:scaffold_286\tLN:1584\n",
      "@SQ\tSN:scaffold_419\tLN:1608\n",
      "@SQ\tSN:scaffold_58\tLN:1642\n",
      "@SQ\tSN:scaffold_429\tLN:1657\n",
      "@SQ\tSN:scaffold_324\tLN:1662\n",
      "@SQ\tSN:scaffold_125\tLN:1667\n",
      "@SQ\tSN:scaffold_107\tLN:1671\n",
      "@SQ\tSN:scaffold_109\tLN:1671\n",
      "@SQ\tSN:scaffold_207\tLN:1675\n",
      "@SQ\tSN:scaffold_397\tLN:1677\n",
      "@SQ\tSN:scaffold_121\tLN:1683\n",
      "@SQ\tSN:scaffold_287\tLN:1700\n",
      "@SQ\tSN:scaffold_69\tLN:1723\n",
      "@SQ\tSN:scaffold_396\tLN:1726\n",
      "@SQ\tSN:scaffold_201\tLN:1727\n",
      "@SQ\tSN:scaffold_249\tLN:1738\n",
      "@SQ\tSN:scaffold_223\tLN:1741\n",
      "@SQ\tSN:scaffold_457\tLN:1741\n",
      "@SQ\tSN:scaffold_272\tLN:1743\n",
      "@SQ\tSN:scaffold_153\tLN:1746\n",
      "@SQ\tSN:scaffold_115\tLN:1753\n",
      "@SQ\tSN:scaffold_267\tLN:1753\n",
      "@SQ\tSN:scaffold_152\tLN:1758\n",
      "@SQ\tSN:scaffold_304\tLN:1767\n",
      "@SQ\tSN:scaffold_217\tLN:1779\n",
      "@SQ\tSN:scaffold_242\tLN:1795\n",
      "@SQ\tSN:scaffold_127\tLN:1820\n",
      "@SQ\tSN:scaffold_379\tLN:1823\n",
      "@SQ\tSN:scaffold_334\tLN:1864\n",
      "@SQ\tSN:scaffold_448\tLN:1870\n",
      "@SQ\tSN:scaffold_433\tLN:1871\n",
      "@SQ\tSN:scaffold_268\tLN:1932\n",
      "@SQ\tSN:scaffold_151\tLN:1976\n",
      "@SQ\tSN:scaffold_146\tLN:1981\n",
      "@SQ\tSN:scaffold_315\tLN:1982\n",
      "@SQ\tSN:scaffold_179\tLN:1987\n",
      "@SQ\tSN:scaffold_368\tLN:1994\n",
      "@SQ\tSN:scaffold_405\tLN:2002\n",
      "@SQ\tSN:scaffold_89\tLN:2015\n",
      "@SQ\tSN:scaffold_143\tLN:2025\n",
      "@SQ\tSN:scaffold_480\tLN:2050\n",
      "@SQ\tSN:scaffold_212\tLN:2059\n",
      "@SQ\tSN:scaffold_280\tLN:2062\n",
      "@SQ\tSN:scaffold_335\tLN:2077\n",
      "@SQ\tSN:scaffold_435\tLN:2080\n",
      "@SQ\tSN:scaffold_383\tLN:2081\n",
      "@SQ\tSN:scaffold_456\tLN:2090\n",
      "@SQ\tSN:scaffold_31\tLN:2091\n",
      "@SQ\tSN:scaffold_469\tLN:2102\n",
      "@SQ\tSN:scaffold_150\tLN:2111\n",
      "@SQ\tSN:scaffold_475\tLN:2119\n",
      "@SQ\tSN:scaffold_391\tLN:2129\n",
      "@SQ\tSN:scaffold_238\tLN:2138\n",
      "@SQ\tSN:scaffold_340\tLN:2150\n",
      "@SQ\tSN:scaffold_442\tLN:2164\n",
      "@SQ\tSN:scaffold_511\tLN:2173\n",
      "@SQ\tSN:scaffold_2\tLN:2184\n",
      "@SQ\tSN:scaffold_245\tLN:2185\n",
      "@SQ\tSN:scaffold_291\tLN:2196\n",
      "@SQ\tSN:scaffold_55\tLN:2199\n",
      "@SQ\tSN:scaffold_99\tLN:2202\n",
      "@SQ\tSN:scaffold_46\tLN:2217\n",
      "@SQ\tSN:scaffold_510\tLN:2226\n",
      "@SQ\tSN:scaffold_316\tLN:2228\n",
      "@SQ\tSN:scaffold_157\tLN:2270\n",
      "@SQ\tSN:scaffold_243\tLN:2283\n",
      "@SQ\tSN:scaffold_387\tLN:2292\n",
      "@SQ\tSN:scaffold_79\tLN:2295\n",
      "@SQ\tSN:scaffold_466\tLN:2305\n",
      "@SQ\tSN:scaffold_171\tLN:2318\n",
      "@SQ\tSN:scaffold_19\tLN:2354\n",
      "@SQ\tSN:scaffold_467\tLN:2380\n",
      "@SQ\tSN:scaffold_17\tLN:2385\n",
      "@SQ\tSN:scaffold_481\tLN:2402\n",
      "@SQ\tSN:scaffold_63\tLN:2424\n",
      "@SQ\tSN:scaffold_300\tLN:2427\n",
      "@SQ\tSN:scaffold_408\tLN:2505\n",
      "@SQ\tSN:scaffold_101\tLN:2509\n",
      "@SQ\tSN:scaffold_434\tLN:2556\n",
      "@SQ\tSN:scaffold_39\tLN:2601\n",
      "@SQ\tSN:scaffold_250\tLN:2612\n",
      "@SQ\tSN:scaffold_226\tLN:2614\n",
      "@SQ\tSN:scaffold_130\tLN:2614\n",
      "@SQ\tSN:scaffold_256\tLN:2622\n",
      "@SQ\tSN:scaffold_289\tLN:2630\n",
      "@SQ\tSN:scaffold_376\tLN:2655\n",
      "@SQ\tSN:scaffold_26\tLN:2676\n",
      "@SQ\tSN:scaffold_44\tLN:2678\n",
      "@SQ\tSN:scaffold_276\tLN:2770\n",
      "@SQ\tSN:scaffold_137\tLN:2782\n",
      "@SQ\tSN:scaffold_71\tLN:2816\n",
      "@SQ\tSN:scaffold_12\tLN:2816\n",
      "@SQ\tSN:scaffold_133\tLN:2841\n",
      "@SQ\tSN:scaffold_162\tLN:2850\n",
      "@SQ\tSN:scaffold_35\tLN:2857\n",
      "@SQ\tSN:scaffold_181\tLN:2978\n",
      "@SQ\tSN:scaffold_97\tLN:2991\n",
      "@SQ\tSN:scaffold_389\tLN:3019\n",
      "@SQ\tSN:scaffold_288\tLN:3058\n",
      "@SQ\tSN:scaffold_282\tLN:3075\n",
      "@SQ\tSN:scaffold_230\tLN:3080\n",
      "@SQ\tSN:scaffold_351\tLN:3123\n",
      "@SQ\tSN:scaffold_277\tLN:3140\n",
      "@SQ\tSN:scaffold_147\tLN:3167\n",
      "@SQ\tSN:scaffold_214\tLN:3193\n",
      "@SQ\tSN:scaffold_78\tLN:3234\n",
      "@SQ\tSN:scaffold_319\tLN:3247\n",
      "@SQ\tSN:scaffold_135\tLN:3269\n",
      "@SQ\tSN:scaffold_432\tLN:3288\n",
      "@SQ\tSN:scaffold_74\tLN:3294\n",
      "@SQ\tSN:scaffold_360\tLN:3364\n",
      "@SQ\tSN:scaffold_145\tLN:3415\n",
      "@SQ\tSN:scaffold_364\tLN:3462\n",
      "@SQ\tSN:scaffold_170\tLN:3596\n",
      "@SQ\tSN:scaffold_49\tLN:3724\n",
      "@SQ\tSN:scaffold_333\tLN:3727\n",
      "@SQ\tSN:scaffold_260\tLN:3787\n",
      "@SQ\tSN:scaffold_94\tLN:4037\n",
      "@SQ\tSN:scaffold_229\tLN:4087\n",
      "@SQ\tSN:scaffold_344\tLN:4135\n",
      "@SQ\tSN:scaffold_203\tLN:4183\n",
      "@SQ\tSN:scaffold_431\tLN:4225\n",
      "@SQ\tSN:scaffold_441\tLN:4328\n",
      "@SQ\tSN:scaffold_43\tLN:4380\n",
      "@SQ\tSN:scaffold_440\tLN:4503\n",
      "@SQ\tSN:scaffold_460\tLN:4552\n",
      "@SQ\tSN:scaffold_134\tLN:4675\n",
      "@SQ\tSN:scaffold_299\tLN:4758\n",
      "@SQ\tSN:scaffold_281\tLN:4796\n",
      "@SQ\tSN:scaffold_142\tLN:4960\n",
      "@SQ\tSN:scaffold_332\tLN:5096\n",
      "@SQ\tSN:scaffold_114\tLN:5301\n",
      "@SQ\tSN:scaffold_445\tLN:5741\n",
      "@SQ\tSN:scaffold_252\tLN:5892\n",
      "@SQ\tSN:scaffold_210\tLN:5900\n",
      "@SQ\tSN:scaffold_345\tLN:7284\n",
      "@SQ\tSN:scaffold_503\tLN:89590\n",
      "@SQ\tSN:scaffold_499\tLN:137295\n",
      "@SQ\tSN:Pt\tLN:140384\n",
      "@SQ\tSN:scaffold_505\tLN:158257\n",
      "@SQ\tSN:scaffold_504\tLN:164640\n",
      "@SQ\tSN:scaffold_506\tLN:172036\n",
      "@SQ\tSN:scaffold_497\tLN:196539\n",
      "@SQ\tSN:scaffold_500\tLN:234755\n",
      "@SQ\tSN:scaffold_501\tLN:323012\n",
      "@SQ\tSN:scaffold_495\tLN:380467\n",
      "@SQ\tSN:scaffold_498\tLN:461125\n",
      "@SQ\tSN:scaffold_509\tLN:487110\n",
      "@SQ\tSN:Mt\tLN:569630\n",
      "@SQ\tSN:scaffold_502\tLN:754969\n",
      "@SQ\tSN:scaffold_496\tLN:811006\n",
      "@SQ\tSN:scaffold_507\tLN:846390\n",
      "@SQ\tSN:scaffold_508\tLN:1357938\n",
      "@SQ\tSN:10\tLN:149632204\n",
      "@SQ\tSN:9\tLN:157038028\n",
      "@SQ\tSN:6\tLN:169407836\n",
      "@SQ\tSN:8\tLN:175377492\n",
      "@SQ\tSN:7\tLN:176826311\n",
      "@SQ\tSN:5\tLN:217959525\n",
      "@SQ\tSN:3\tLN:232245527\n",
      "@SQ\tSN:2\tLN:237917468\n",
      "@SQ\tSN:4\tLN:242062272\n",
      "@SQ\tSN:1\tLN:301476924\n",
      "@PG\tID:hisat2\tPN:hisat2\tVN:2.1.0\tCL:\"/home/sal269/miniconda3/envs/alignment/bin/hisat2-align-s --wrapper basic-0 --min-intronlen 60 --max-intronlen 50000 -p 4 -x ./maize_genome/ZmaysAGPv3.30.hisat2 -U /tmp/17424.unp\"\n",
      "K00364:134:H2YYHBBXY:1:2209:9668:15574\t4\t*\t0\t0\t*\t*\t0\t0\tGTCCCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACCGGAATCCTATGATGTTATCCCATGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACGACCCGGCCA\tAAFFFJJJJJJJJJJJJJFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFFJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJAFJJFFJFFAFFJFJJ7AJJJJJJ7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1125:30868:6097\t4\t*\t0\t0\t*\t*\t0\t0\tCAACTACGAGCTTTTTAACTGCAACAACTTAAATATACGCTATTGGAGCTGGAATTACCGCGGCTGCTGGCACCAGACTTGCCCTCCAATGGATACTCGTTAAGAGATTTAGATTGTACTCATTCCAATTACCAGAC\t-AFFF-JJFJJJJJAJJJJJJFJJJFJJJJAFJJFJJJJJJFF-FFJJ7FA<7F<FA<AAAJF77777FJ-F7AF7F<FFJJJ-7AFFAJJ7AAA--<7<JFA<AJ-7--<-A<-7F-A-7FJ7-7<-F-7A<AFF-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1118:28250:23399\t4\t*\t0\t0\t*\t*\t0\t0\tCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACCGGAATCCTATGATGTTATCCCATGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACGACCCGGCCAGTTA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJ<JJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1110:2341:1894\t16\t1\t223006137\t60\t151M\t*\t0\t0\tTACCGGACGCTCTACGACCTCCCCACTTCGCCGGAGTTCCTCTTCCAGGAGGAGGCCCTGGCGCAACGACGCTCGTGGGGTGAGAACATCACTTTCTATACGGGCGTCGGGTACCTCTCCGGCGCCGTTGGCGGCGCCGTCCTGGACCTCC\tAJ<FFJAAJFAFFJJJ<FFJJAJF7JJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFFJJJJJJJJJJJJFJFJJFJJJJJJJJJJJJJJJJJJJJJJAFAJJJJJJJJJJJJJJJJJJJJJJJJAJJF<FAA\tAS:i:-12\tZS:i:-14\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:87C57G5\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1212:2037:33193\t0\t3\t160831450\t1\t127M\t*\t0\t0\tGCACTCAACACTACTCCCGTGGTTGCAGAATGGGCAGGAAAAGACAGTGTCAAGCTTGTCCATCCGCTTCTTAGGCGGTGGCTTAGCAGCTGACTTTCTCTTCCCCATGGTTCAGCTTCTTTTCTTC\tAAFFAJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJFFFJJ-7FJFJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJJAJJJJ\tAS:i:-3\tZS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:75A51\tYT:Z:UU\tNH:i:2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1212:2037:33193\t272\t4\t163044580\t1\t127M\t*\t0\t0\tGAAGAAAAGAAGCTGAACCATGGGGAAGAGAAAGTCAGCTGCTAAGCCACCGCCTAAGAAGCGGATGGACAAGCTTGACACTGTCTTTTCCTGCCCATTCTGCAACCACGGGAGTAGTGTTGAGTGC\tJJJJAJJJJFJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJJFJF7-JJFFFJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJAFFAA\tAS:i:-3\tZS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:51T75\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1120:13687:26793\t0\t8\t154931974\t1\t151M\t*\t0\t0\tGGAGCTGAAGAACGCCGTACAGCAGTGCCTCAGCAGTTGGTGGGCACCCAGGGACGTAGATGTCCACAGGGACTATACGGTCACATCCACGTACAACAGAGTAGGAGTAATGGTAGTATCCACCACCGTTGGCACAGCTGCCCATTGAAAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1120:13687:26793\t256\t8\t154962758\t1\t151M\t*\t0\t0\tGGAGCTGAAGAACGCCGTACAGCAGTGCCTCAGCAGTTGGTGGGCACCCAGGGACGTAGATGTCCACAGGGACTATACGGTCACATCCACGTACAACAGAGTAGGAGTAATGGTAGTATCCACCACCGTTGGCACAGCTGCCCATTGAAAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2203:16467:36745\t16\t5\t166437533\t60\t149M2S\t*\t0\t0\tAGGATGTGGGCTTATCTGAGCTCCCAGCAACTGTGGCAGCTGTGAAGAATCCAAGTGGAAAAATCGTGTATGATGAGTATAACCATGAGCGCTACCCACCTGGAGACCCCAGCAAGCGTGCATTTGCCTACTTTGTGCTGAGTGGTGGGGT\tJJJJJJJFJJJF<JJJFJJJJJJJJJJJJJAJJJJJFFFJAJFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:31050:2897\t16\t1\t298410759\t60\t36M97N115M\t*\t0\t0\tCGCCGTCCGCACCGGCGGCAAGGGCACCATGCGCAGGAAGAAGAAGGCTGTACACAAGACGGTCACCACGGACGACAAGAGGCTGCAGGGCACTCTCAAGAGAGTAGGGGTCAACACCATCCCTGCAATCGAGGAGGTGAACATATTCAAG\tFAF-<FF<F<F<<F-JA77<<F-<FJAAJAAFFFJJJJJFJJAJJJJJFJJJJJJJJJJF<JFAJJF<7-FJJFFJAJA<-JJJA-FA-JJJJFJJJJFFFFJJJJJJFJJJJJFFJJJJAJJJJJJJJFAJJJJFJJJJFAJJJAFAAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1227:3447:23470\t16\t5\t4039869\t60\t1S150M\t*\t0\t0\tCCGGCCGCCGCGAAGAAGGCGCCGGTAGCCAAGAAGGCCGCTCCGGCGAAGAAGGCGGCTGCCCCTGCCCGAAAAGCGCCGTCTCGGAAGGCGAAGAAGTAGAGGAGGGGTAGGCGTCGGAGAAATGTAGGGTTTGGTTCTCGTATTACGC\t)AFJFJJJFFA<7FFF-FFJFF<JJJA7F-7JFJFJJJJJJJJJJFJJJJJJJJJJJJJFJJJJFJJJJJJJJFFJJJJJJ<F7<AJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJFJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1225:4097:12990\t0\t8\t118900022\t60\t59M1257N92M\t*\t0\t0\tCTGGAGTATGGCTTCTTGGTGGCGCGGCGCCTCTTCTTTACCGCTTCAGCATGGATATCCTTCTTGTGCTGCTTCCTGTACATTGCTGTCCAGGTAAGCTTTGCAGGCTTCAGGCGGTTGTGGAAGTAGCGCTTGCATTTCGAGTTGGCAA\tAAAFFJFJJJFJJJJFFJJFFJJJJJFJFJJAFAFJJJJJJFJFFFJJJJJFFFJ7F7JFJFFJFJJJJJAJJJJJAJJJJJFJJJJJFJJJFJJJJFFJJJJJJJJJJJJJJJJAA7AFJFFJJJ<JFFFJFJJJJFJFJFJJJFJAJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2110:19522:28604\t16\t3\t219377485\t60\t27M115N124M\t*\t0\t0\tGCCGTCCTGGAGTACCTCGTCGCTGAGGTCCTGGAGCTCGCCGGCAACGCTGCCAAGGACAACAAGAAGACGCGCATCGTCCCGCGCCACGTGCTGCTGGCGATCCGCAACGACGAGGAGCTCGGGAAGCTGCTGACCGGCGTGACCATCG\t<JJJJJJJJJFJJFJFFA7-JJJFJJF7-JJJJJJJJFJJJJJJ<A7JJJFJJJJJJJJJJJJJJJJJJJJJJJAJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:19C131\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1117:19390:38574\t4\t*\t0\t0\t*\t*\t0\t0\tCACGCCGCCCGTTGGGAAGGGAGCTTCGAGGCGGCCCGCCGCGGCGCGTCGGCCGGGCGGGCTGAGCCAATGGCACGGGCCCTTGGGGCGCGAACGCCCTAACGTGGGTCGGGGCGGGCGGCGAGCAGATGCGCCGGTTGCTAGCTTGGAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJAJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJFFJJJJJJJJJJJJJJJJJJJJA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2226:17056:14168\t16\t9\t34142757\t60\t103M118N48M\t*\t0\t0\tCATTAGGTATGGTGTTGACGGTGGAGCTACCCTTGTGACGGGTGGTGATAGGTTGGGTGACAAGGGTTTCTACATCCAGCCAACGATTTTCTCAGATGTCCAGGACGGCATGAAGATTGCTCAGGAGGAGATATTTGGGCCTGTGCAGTCG\tJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1123:11079:25615\t4\t*\t0\t0\t*\t*\t0\t0\tTCGGCCAGCGGTGCGGCCCGTGAGGGCCTCCCGCTCGTCAGCTTCCTTGCGCATCTCAGGTTTCTGAACCCGTCGACTCGCACGCATGTCAGACTCC\tAAFFFJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJFJJJJJJJJJJJJJFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2118:4219:48579\t4\t*\t0\t0\t*\t*\t0\t0\tGGACCATTCAATCGGTAGGAGCGACGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTACTAGGCATTCCTCGTTGAAGACCAACAATTGCAATGATCTATCCCCATCACGATGAAATTTCCCTAGAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJFJJJJJJJJFJJJJJ-FFJJJJ7FAJJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1120:15656:7205\t16\t8\t168604907\t60\t150M1S\t*\t0\t0\tCTCGCCGCCACTCACCAGCAAATTCCCAAGCCGCCCCTTTCCCCATCTCTTTCGCGACCCATTTGATCGATTCCTGATGGCGCCCAAGGCCGAGAAGAAGCCCGCGGAGAAAAAGCCGGCGGAGGAGAAGGCCGAGAAGAAGCCAAGGGCC\t-FA<FA<)-AF7<77JFJFAAAF<JJF-FJFJJFJFJJFFJJJJFJJAFJJJJF7FFJF<FJJA7-FFJFAFFAFJFJAJ<JJFFJFFJJJF-FJFFF<FJFJJJJFJJJJJJFJFJJJFJFJJJJJJJJJJFJJJJJJJFJJJJJFFFAA\tAS:i:-17\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:39C16C54G38\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1217:30107:40367\t16\t6\t83946152\t60\t150M1S\t*\t0\t0\tAGTACCTGATGGACTACGAGGACAAGAAGTTCCAGAACGTGTCCAAGGAGGGACTGAAGCTGGGCAAGGACTCGAAGCTCAAGGAGCTCAAGGAGTCGTTCAAGGAGCTGACTGACTGGTGGAAGAAGGCCCTGGAGAGCGAGAACGTGGT\tFA<JFJAFF<7JJJFJJFF7AJJJA-JF-<AFJFFJJJJJFF-FJF-JJJFJJJJJJJJJFJFAJFJFJJAA<JJFFFJJJJJJAFFJJJJFJJJFJJFFJJJJJJJFJJFFAJJJJJJJFFFJJJJJFFJJJJJFFJJJJJJJFFFFA<A\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2105:1996:26723\t4\t*\t0\t0\t*\t*\t0\t0\tCCTGCGGTAACTTTTCTGACACCTCTAGCTTCAAACTCCGAAGGTCTAAAGGATCGATAGGCCACGCTTTCACGGTTCGTATTCGTACTGGAAATCAGAATCAAACGAGCTTTTACCCTTTTGTTCCACACGAGCTTTCTGTTCTCGTTGA\tAAAFFJJFJJFJFJJJJJJJ7-7<FAFFJFJJFFJJFJJJJJJ<-AFJJJ7FFFJFA<JJJJJJJJFFFJFJJF<FAJJFJFFFAAJJFJJJJFFFJJJJJJJFJFJFFJ<-AFFJJJ<J<FF--FFAFAJFJ7)7<<F7F<<J7AAA<A7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1213:13788:14414\t16\t10\t132577647\t60\t151M\t*\t0\t0\tTTAGTGCCAATCAGTATTGTAGTTCGACTTGAATACTCAAACTAGCACGTTGAGCAAACCGTAGAGCTCAACACCAGGGGTTTCCCCGTCTTCTGTATAATATCTCTGGTGATGATTGTATATTAGCAATGAACCATGCAGATTTGGTCGC\tJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-24\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:37T41A5T38T26\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2110:25631:13482\t4\t*\t0\t0\t*\t*\t0\t0\tGGTCCGTGGTGATGCTGTCCCCAAGGTTGAGCAGGCAGTAGGCCTCCGTCACCGTGGGTGGGGCTGCAGGCGGCGTCATGGACATGCCCTCCAGGTACGTAGGCTTGCGGATGTAGGTGGACCTGTCGTCCCATGGGTACAGCGCCTCCTT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJ<FJJJJJJJJJJAFJJFJJFFFJJJJJAFJJJJJJJJJJJJJJJJAF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2120:18781:35655\t0\t1\t94689210\t60\t58M803N61M\t*\t0\t0\tCCACAGGGCAATCTCCTTATTAGCACTCTCAATGCTGTCACTTCCATGAATGACATTCCTGCCAATGTCGACAGCAAAGTCGCCACGGATGGTGCCGGGCTCAGAAGCCAGGGGGTTGG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:119\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1213:7923:41950\t0\t6\t9207388\t60\t1S106M82N44M\t*\t0\t0\tGGCAGGGCAGACTCGGACGAGCCCACCTTTTCACCCTTCTTGATCAGCTCCACAGGGGTGATAATTTCCACGGTACCCTTGTTAATCTTGGTGGGGATGTTAAGCACCTGGAAGAAAGAGGTCTGGGACGGATCCAGCCCAGTGTTGCCAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJ\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2201:14925:46311\t0\t5\t161944060\t60\t1S150M\t*\t0\t0\tCGTCTGGTCTGCTTATGCTTAAGAGCTTAATCTTTTGAATTATGCCATCTAAAAGATCAACTGATATTGCTATAGCTTGAGGAATGACACCTCCATGAATTTCCTTATCTTGATCAGAGCTTTTCGACTCCACAATAGCCATCCTAGGAGT\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJAJJFJJJJJJJJJJJFJJJFFJJJJFJJJJJJJJJJJJJJJJJFJJFJFJJJJJJJJJJJJJJJJJJJJJJFJJAFJJJJJJJJJF<FAJ)<F7\tAS:i:-7\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:70G79\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2112:23226:25210\t16\t10\t75711427\t60\t151M\t*\t0\t0\tTGACCATCCACACCCCGAAGGACGAGGACGAGCCGCGCCCTCCGGTCTTGGCACCTCCTGAGGTGTAAGGTCCCAACTGTGTCCTTGAAATGGCAGTTTGATCGGGTCCTAGATAGGTTTCTATCTTACGATAAGGACAATGCTAAGGCAG\t7JJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2210:14143:5147\t4\t*\t0\t0\t*\t*\t0\t0\tCTCCTCTTCGGCCTTCAAAGTTCTCATTTGAATATTTGCTACTACCACCAAGATCTGCACCGACGGCCGCTCCGCCCGGGCTCGCGCCCCGGGTTTTGCGGCGGCCGCCGCGCCCTCCTACTCATCGGGGCATGTCGCTCGCCCAGCTGGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJ-<FFFJFFFFJJJJFAAJJJJJJAJJJJJJJ))<A7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1215:9587:45291\t16\t6\t8679815\t60\t151M\t*\t0\t0\tTTGAACGTCTTGCCTGCACTTCAGTAGGCCATTTTTCATGGTTCTAGTTGTATTTGAATCCAGCGTTTGCCACTAAGATAAATCAGACATTCGTTTTGTGAACCTATTATGATGGAGCTTTATTTGCTGATAATATCTACCTACCTATTTG\tF<AJFJF<FJAJFAJJJJAJJJJJJJJJJFJJFJJJJJJ<JJJJJ<JJJJJJJJJJJFJFJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJJJJJAFFAA\tAS:i:-18\tZS:i:-28\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:31C57G49G11\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2201:20172:21957\t16\t10\t24645538\t60\t44M157N106M1S\t*\t0\t0\tCATCCGCTGCAGAGAGTCGGCGTCGGCAGCCCAACACCGTCCTGCATATTGTCATGTCATCTAGAAAGTATCCATCCGGTTCTGAGAAAAGGAAGATGAAGAAACGAGCATATGAGTTCGCTTCGAGACATGTTAGAAGACAATTTTAAGA\t<)FJJJJJJJFAAJJJJJJJJFJJJJJJJJFJJJF<JJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-7\tZS:i:-19\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:33G116\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2111:11911:14537\t0\t1\t94689235\t60\t33M803N90M\t*\t0\t0\tCTCTCAATGCTGTCACTTCCATGAATGACATTCCTGCCAATGTCGACAGCAAAGTCGCCACGGATGGTGCCGGGCTCAGAAGCCAGGGGGTTGGTGGCGCCAATGATCTTGCGGCCAGTTGTG\tAAAFFJJJJJJJFJJJJJJJJJJJ<JJJJJJJJFJJJAJFFFFJJJJJJJJJJJJFJJJJJJFJAFJ-<FJJFAJJA<FFJJJJFJFJJ<F7JJA7<FJJAJJJJJAAAJJFJA<FAFFFJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:123\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2115:21592:38838\t0\t2\t206723319\t60\t73M109N78M\t*\t0\t0\tTCTGGCTTCGTTGGGCTCGCCGGACGGCTATCTTCATCGTACTTTGCCAGCCGTGGTTTCTTCCTCTTTGGCGCTATAGCGGAAAGGGAGGTGGAATCAGAGGGTCGGGCGCCAGAAGAGGGGGCCGGCGAAGAGTTCCCACACTTGGACT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJJJFJJJFJJJJJJJJJJJJJJJFJJJFJJFJJJFJJJJJJJJAFJJJJ7JJFJJJJFFJJ<F\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:44C82A23\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1224:23115:20551\t0\t7\t2589861\t60\t69M491N53M78N29M\t*\t0\t0\tGGCACACTGCTCAATGTAAAGCATCCTGTCCTTTGTTGTTCCGAATGATTTCTCTCTGACTAATCCAGCCTTCCCAAGCTTTTCAGGACTCAACTCTTGGAATCTCGGAACAATTCGTCCTCCTGTTGCAATGGCAATCAATTCCAATTCA\tAAAFFJJJ7JFJJFJJJJJJJJJJJJF<JJJJJJFFJJJJJJJJFJ<JJJJJJJJJJJ<FJFJJJJJJJJFFJAJJFFJJJJJJJJJJJJJJJJJFJJJJJJJFFFFJJJJJJJAJJJJJJJJJ-FJ<FJJJJFFFJJJJ7AJJ<FJJJJJ\tAS:i:-4\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:27A123\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2123:18741:26512\t0\t2\t608657\t60\t151M\t*\t0\t0\tGGGCTGCTGCTCCTCCCCGGACACCATCACGTACATGAGGCACTGCGGGCACCCCACCGCCACCATCGGCTTCGGCGCCACCGCCTGCTGTACCTCGTCCTCCGCGTCCGACGACACGCACGAGCTCGCGGGCGACTCGGCCGCCGACCCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJ-<FJJJJJJJJJJJJJJJJJJJJJJJ77FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJ-AFJJJJJJJJJJJJJJJJAJ)\tAS:i:-7\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:58A70C21\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2210:19400:18546\t0\t9\t147173089\t60\t37M122N114M\t*\t0\t0\tCTCAGTAAGCTCTGTGGTGATGGATGCTTGGCGTGTCCTGTTGTAAGTGAGAGTGAGACGATCAAGCATATCGCCAGCATTTCTGCTAGAGCTGTCCATGGCAGACATACGGGCTCCAAGCTCACTGCATGCATTCTCTAGGGCACCATTG\tAAFFFJJJJJJJJFJJJJJJJJJJJJJJJFJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJFJJJFFJJJJJJJJJFJJJJJJJJ-FFAFJJFFJJJJF-AFFAJJJJJJJJFJJJJF\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1125:1560:41510\t0\t3\t196114066\t60\t26M209N67M329N58M\t*\t0\t0\tGTAGCGACCGCTTTTGCCAGCATTGTCTTTCCTGTCCCTGGGGGGCCAAAAAGTAAGATGCCTTTCCATGGAGAAAGGAGACCAGTGAAGTATTTTGGGTACTTTATGGGCATGACAACAGCCTCTTTTAGAAGGCGTTTTGCATTTTCCA\tAAF<FFJJJJJJJJFJJJJJJJAJAJJJJFJJJFJFJJAFJJJA<FJJJJJJJJJJFJJJJJJJJJJFFFJJJJJJJFJJJJJJJJJJJAAJFFJ-FJF-AJAJ-FJF-7FJJ7JFFJJJJFJ-FAAFJFFJJJ77<<A7<77A<A<<AAJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2126:20101:45185\t0\t8\t29723097\t60\t12M105N117M91N22M\t*\t0\t0\tGTTCGTAGCAATCTGGTTGTACAGATGGTAGAACAGTCCAACCAAGAATAGGTCACTAACAAACTTCGTCAAGCCTACTTTTGCAATGGCATCACTGAATCCATGCTGCATTAGCCGGGGCCCTTCAAAGATCACTGCAGGCGGAATGCAA\tAAFFFJJJJJJJJJJJ<JJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:66T84\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2213:14042:38556\t0\t4\t175673422\t60\t146M5S\t*\t0\t0\tTTAAATTCTTAATAGCAAACCAGGTTACAGAAGTGTGCTTTCTTCAACGTCTCTTCCTTCTGCCCCCCTTCTGCTTACCCTTCCCCTTGTGCTTGTCCTTCCTCATCTTCGGTTCCGACATGTCCTTGCCTGATGGTTTCGGTGGCACCAG\tAAFFFJJJJJJJ-AFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJFJJFJFAJJJJFJFFJJJJJFJ<)\tAS:i:-5\tZS:i:-18\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:146\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2216:25317:23241\t4\t*\t0\t0\t*\t*\t0\t0\tGTCCCTCTAAGAAGCTAACTACGGAGGGATGGCTCCGCATAGCTAGTTAGCAGGCTGAGGTCTCGTTCGTTAACGGAATTAACCAGACAAATCGCTCCACCAACTAAGAACGGCCATGCACCACCACCCATAGAATCAAGAAAGAGCTCTC\tAAFFFJJJJFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2128:14133:12691\t0\t2\t9509001\t60\t55M409N96M\t*\t0\t0\tACCCAGCACCCCCTGTTACTAACACATGGGTCACCCCAGTTTCATGGCGAGCGAACACGCTGGTACCGCTGAAACTAGGGGACTGTTTCACCGCGATTATGCACATGACCGTGAGGGCAGCAGCCACGGCAATCTTCCCCGTGAGATTAGG\tAAFFA-A<AFJAFFJJFFJ<JJJJAF-A-A7AJAJFF<<A<FFFJ<J<FFF-7-<FJAJJ---7-FJFJJJJJ7F-7AF7<<FAF<A-F--77AA7AAAFA7-<JFJ<J-7-7AF-A<AJJJJFA<<<-AF<FF<<7<FFJ-AAFJ-<AAF\tAS:i:-10\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:51A38A1A58\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1121:1874:35128\t4\t*\t0\t0\t*\t*\t0\t0\tCAACAACTTAAATATACGCTATTGGAGCTGGAATTACCGCGGCTGCTGGCACCAGACTTGCCCTCCAATGGATCCTCGTTAAGGGATTTAGATTGTACTCATTCCAATTACCAGACACTAACGCGCCCGGTATTGTTATTTATTGTCACTA\tAAFFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<FJJJJJJJJJJJJJJFJJJFJJJJJJJJFJJJAFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJFFFJJJJFJFJFJJJJJFJJFAFFJJFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2215:3224:25615\t16\t6\t2394367\t60\t151M\t*\t0\t0\tTCTCACCAAGCATTTCTTCTATAGCTACACATATCCAATTATGCAGAGCCTGCAACAGAGTGTTACTTCTGCAGGGATGAAAGAAATGCCTTATGAAAATTTGTTTGTCTGGAACACTTTCCTGACAGAGCCAGTACGTTCAAGGTGCCAC\tFJJJAJA-<A7JJJJJJJJFFFFFJJJJAFFJJJF-FJJJJJJJJJJJJJJJJFFFJFFJJJJJJJJJJJ<JFJAJJJJJJFJJJJJJF-AJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJAJJJJJFFJJJJJJJFJJJJJJFFAAA\tAS:i:-9\tZS:i:-20\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:10G109T30\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1109:21724:31540\t0\t6\t10121333\t1\t100M\t*\t0\t0\tGCCGGCGTCTCCGGACTTCCTAACATCGCCGTCTGCCGCCACGTCCCGGCTCGGGAAATCTTAACCCGATTCCCTTTCGGGTGACGCGCGTGATCGCGCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:100\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1109:21724:31540\t256\t2\t10186646\t1\t100M\t*\t0\t0\tGCCGGCGTCTCCGGACTTCCTAACATCGCCGTCTGCCGCCACGTCCCGGCTCGGGAAATCTTAACCCGATTCCCTTTCGGGTGACGCGCGTGATCGCGCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:100\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2212:5274:10282\t16\t4\t182743569\t60\t95M83N56M\t*\t0\t0\tCAGTTAATCAGAATGATCTAATGGTTATTCCAAAGGTACCTAGTGGTGTTTCTGGTACTGCTTTAGCTGCTTCTCTTCTTCGTGATGGTTGGAGTCTTGCTAAGATCTATGAGAAATACCAAGAAGCTACTGATGCTTTTCTTCATGAGAG\tAJJFJJJJJJJJJJJJFJJJJJJJJJJJFJJJJJJJJFJJJJJFAJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJAJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:2341:15961\t0\t1\t32583027\t60\t151M\t*\t0\t0\tTACTCCTCCAACGGGCTGAGGCCGCTGCTGTGCTCCGTGCTGGAGATGGACTCCGAGTCCTCCACGTCGAGAAGCGCGAGACCCAGGTGCGACCTCACGTTCGCGGCGGGGTAGGCCTCGTCGTCCACGCAGTCCGGCTCCGTCGCGCCCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<FJJJJJJJJJFJFFJJJJFJFJJJJJJJJ-\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:80G32C37\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2209:18994:32824\t16\t5\t77131202\t60\t151M\t*\t0\t0\tATGAGACGGCAATCCCTCTGCGGTACACCCCAAGGAAGTTTGTGATTCTACCAAAGAAAAAATACATTGCTGTCATTGAGAGTGACAAAGGTGCATTCAGCGCAGAAGAGCGAGAAGCTGCTAAGAAAGAGTGCCTTGATGCTTCTGGTGC\tFJJJJJFJJAJFAJFJJJJJFJJFFJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1123:24048:36130\t0\t5\t50119844\t60\t3S148M\t*\t0\t0\tGGTGAAGAGCAGGAAGAAGAGGAGACGCGAGAAGCGGAAGCCGAGGAAGAGCGAGATGAGGCAGCAGAGCGCGTGGAGCAGGAACCAGAAGGTGGAGGAGGCGGGCGCCCGCGGCCCCGCGCCGCCGCCGTCGGACATAGCCCCGTCTGCC\tAAAFFJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJ7FJJJJJJJFJJJJJJJJFFJJJJFJJJJJJJJFJJJJJJJJJJJ\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:148\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2228:15087:31048\t0\t4\t31643174\t60\t151M\t*\t0\t0\tCCGGCGACGACGAAGGCGGCGGGAGTGGCGCGACCTGAAGCCAGGTCACGAGCAGGAAGGCGCCGTAGCCCACCATGGCCAGCCCAACCGCCATGACGACGAGGTTGAGGAGCTTGAGGACGCACTCAAAGAAGCCCCGGCACGCCATGGC\tAA-FFJJFJJFAJJA7AFAJJJAJJJFA--7AFJAJFJFFJJFJJAFJJJFFFFAFJF<FA7FJJAJ<JJJJFAJJFJJAJJJJJJAJFJJJJJJJJJJJJJJJ-FFJFJJJFJJJJJJJJFJFF7JJJJJJJJJAJFFJJJFAJJJJJAJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:25C125\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1222:13220:15785\t16\t5\t96438780\t60\t12M312N139M\t*\t0\t0\tTATTGGCTGAAGGTACCACCAGTAATGGCCATTTAGACATAGCTACTGAAAATCCACCTGCAATTGATAACTCTCAGACAAGTCATCATAGCTCTGAATGGGTTGACCTATTTGTCAAAGAGATGATGAGCTCCTCTGACATAGATGATGC\tJJJJJFFJJJJJJFJF<-FFF-F7AF7<FJFFJF7-AJJJA<JAJAJJJF-JAAJAJJFFAAFJJJFA<JJJFJJJ<JJJFJFJFJJJJJJF<J<J<FJ<<JJJJFJ7<JFJJJ7FJJJJJFJAFF-<7JFFF<FJJJAJJJAJAJFFAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1109:11911:19179\t16\t3\t151090421\t60\t113M427N37M1S\t*\t0\t0\tAAAGCCTCACAAACAGTAGAAGGTGCAAGCAGCATTGTTCAAGATATTGAAGCACTGTCAACCATTCTCACCCCTATCAAAATTTGGCTTACTAATACTCCAAGTTCAACCAGGGAACTCAACATTGATGGAGCAAATCAGTCGCTCAGAC\tJFJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2214:25063:32578\t0\t9\t122481396\t60\t151M\t*\t0\t0\tGCCAGACTTCAAATCGCGAGATCCACTTGCGAACGATTCCGTTTTCTGTTCTGAATGGCACGCGAAGGATATGGCAGTGCTCGGTGCCAAGGACCTTCTCAAGGCGCTGGCCACAGGTGGTGCCAGTTGCATCAGGGAGAAACCTGGTGAC\tAAFFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFJFJFJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJ7JJFJJJJJJJJJJJJ<JJJJJJJJJJJFJJJJJAFFFJJJJJJJJJ7AJJJJJJFJJJ\tAS:i:-4\tZS:i:-19\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:139C11\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2222:26819:13640\t16\t3\t46179524\t60\t1S150M\t*\t0\t0\tACGGAAGGAGCATAGGGAGGAGTGCAACGAGGGCGACGGGTTCTACGGCGGCATAAGGAAGCGCCGCGGCAGCGATGAGATCTCCAAGTGGAGGCAGCTGTGGCCAACAGACGTCCTGGAGTAAGCAGGAACCGCATCCCTTCTCACACCG\t--<F7JFA-<-F7JAA)77--7-FJF<AFF<F<<AA<JJAA7-FJAAJJFJJFJFAJFFFFA-FAA-JFFA7FF7AA7-<7-7JJJF<JJJJA7--A<JJF<FF-JJFFA7A-7-F7-FFF-FA7A7JJJFJJJJJJJJJJ<FJJJFFAAA\tAS:i:-24\tXN:i:0\tXM:i:6\tXO:i:0\tXG:i:0\tNM:i:6\tMD:Z:21G39T8A0G5C51G20\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1216:5923:9789\t0\t4\t154880186\t60\t151M\t*\t0\t0\tGCGGTCCGCCCGCGGCTCCTACGCCCTAGACGCAGCAACAGGGGATGAAGGGGAGGACGGGGCGGCGGGTGCCGGGCATTCAGTGAAGGGCTCGGAGAACGGGGCGTACGACGGCCGCATGCGGAGGCAGCGTCGGCTGCGCAGCGACGAC\tAAAAFJAJFJJJJFJFA<FJJJFJJJJJFJJJJJJJFJJJJFFJJJJFJFJJJJJJFFFJJJJF<JJJJAJJJJAAAJAFJJJAJFJ<7FA7FJJAJJJJF<AJJJ<<JJJJ<JFAJJJJFJAF-FFFAJJF<JFFJ<-JJA<JJFFJJJJ\tAS:i:-24\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:29G7G1G37G73\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2105:9313:28569\t4\t*\t0\t0\t*\t*\t0\t0\tGCGTACCTCATAACGCAAAGTCGAGCACAACTACTTGCTAAGACCACATAAACATATGTAAAAAAGAGAGAGAGAAAAGGCCAGATGTTTGGTATTGACATGGACGTGCTCAAATATTCATCAAAAGGAAAGGAAAATTCGACAAAATCTG\tA<AFFJJFJJJJJJJJFJJJJ-<FJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJAJJJJJJFJJJAJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJFJJJJJJFJJJJJJJ<FJFJJJJJJJAA<JJJJJJFAAA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1110:15402:29131\t16\t1\t45406451\t60\t150M1S\t*\t0\t0\tCTATAGTTCGAGAGGAAGGCCTCTTTGGTCTTTGGTCTGGAGCACTACCAACCGTCATGCGCAACGGCACAAACCAAGCGGCAATGTTCTCAGCCAAGAACACGTTCGATATTGTTCTGTGGAAGAAGCATGAAGGGGACGGGAAGGTGCC\t<FJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-7\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:106T43\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2114:30645:36411\t16\t5\t167792333\t60\t151M\t*\t0\t0\tATGTGCCAGGGCGGCGACTTCACCCGCGGCAACGGCACCGGCGGCGAGTCCATCTACGGCGAGAACTTCCCCGACGAGAAGTTCGTGCGCAAGCACACCGCCCCCGGTGTGCTCTCCATGGCCAACGCCGGGCCCAACACCAACGGCTCCC\t-)JA-)<)JAJJAFF7-A<-FFJFJFJA7JJJAA-J<JA-FF-JJJJJJJFFJJJ-JA-F<7AA7--7JAJFJJAAJA--JFAA-<FA--77JJFJJAFFF7<7JJFF<FF7-FFJFA7--J7J7JJF7--JJA<F<J7<-FFJ<FFAAAA\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:65G85\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2215:19938:7697\t4\t*\t0\t0\t*\t*\t0\t0\tCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCA\tAAFFFFFJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJ<JJJJJJJJJJJFJJJJJJJJJJJFJJJJJJJJ<JFJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJFJFJJFJJJJJFJJJJJJJJJJJJJAFJJJJJJJJJJJJFJJJJJ)\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2115:3945:34002\t0\t8\t135664624\t60\t3S82M132N63M3S\t*\t0\t0\tCCTGCGGTGTGGCAACCTTTGCTTTCTTATCAGAAAGAGGCGTTTTCAGAGCATTTTCAGCACCTCTCTTCTTGCCTGCCTCTGGCTTATTAGGAGTTGGGGTTTCTTCCTCGTCAGCATCACTGGTATCATCCTCATCACTTGAATCATC\tAA-AFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJ-AFJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJA\tAS:i:-14\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:85T27T31\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1115:28168:5745\t0\t7\t102036266\t60\t1S150M\t*\t0\t0\tCAACCGCAAAAAATTGGGATAACTCGTGGTAACAAGGATGAAAGCATGACTGGTCATGATCTAGTTCTTTTATTTCTTCTACTGGGCAGTACACATACTTTAGGCTTTACTGAAACTCTGAAGTCTGTATTTCTCATGGAGACCTGAATGC\tAAFFFJJJJJJJJJJJJJAFJJJJJJFFJJJJJJJJJJFJFJJJJJJJJJJJFJJJFJJFJFFFAAJFA-J<JJJJJFJJJJJJJJJJJJJJJJJFFJJJFJJJJAFJFJFJJFJJFJFFAAJFJFFFFJFJAFFJ<FFJJJJJJJJ<<A<\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:112G37\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1125:16762:2861\t0\t2\t32273430\t60\t151M\t*\t0\t0\tTTCAATCTTCTTCATCGCTGATTCTGTGGTAAGGGGGTACTTGAGAATTTGGTACTGATCAAGCTTGTTCCTTCCGGTAGTGCTGATTCGTGGGTACTTGGGGTCCCTAGCCTTCTTCAGGGTCGTGGGGCGGTGAAATGTCACAGACGTG\tAAFFFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJFJJJJFJJJAJJJJJJJJJJJJFJJJJJJJFFJJFJJJJAJJAJJJJA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2124:8663:11688\t4\t*\t0\t0\t*\t*\t0\t0\tTCTCAAACGATGGCATTTGATGCACTGGCGATCCTAGGCCAAAGGTCAGCACACTCCTTGCCGATAGGTCCAGTGATTGCAGAACCTTTCATCTCACCCTTGGGATTCACA\tAAFFFJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1116:30188:29712\t16\t8\t95226823\t60\t123M712N27M1S\t*\t0\t0\tGCTGATAGGGAGAGGAGACGGAGGACCAAGAGCGGCGTCGATCTAAAAGCGTTGGCGGCGGCCATGGCCACCGCGGACCGGACCCGGACTTTCATGAAGGACGTCAAACGCGTCATCATCAAGGTGGGCACTGCAGTCGTCACGAGGGGTA\t-JJFFFJJJJFAFJ<AF777FAFJFFFJJJFFJJJJJJJFAFJFFFJJFFJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJFJF7JJJJAJFJ<JJJJJJJFAJJJJFAJFJJJJJJJJJJJJJJJJJA7JJJJJJF-FAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2201:3671:38451\t0\t1\t106506312\t60\t151M\t*\t0\t0\tAGCTGGTCGACCTGCCAGTCCCACGGCATCCGGCACCGGCCACCCGTGCGCCACGACGACCACGTCTCCATGACCCGCAGCGCCTCCATCCGTGCCCGCGCCACCTCGCCTCGCTCCACGCGCACCTTGTACACGTCCTCCATGGGACGCC\tAAFFFJJJFJJJJJJJJJJJJJJJ7AAJJFJAFJJJJJJJJJJJJJJFJJJJJJJJJJJJJAFAFJJJJJFJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAFJJJJJJJFJJF-AFJJJ<FJJJJAFAJJF<FA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1117:15168:23382\t4\t*\t0\t0\t*\t*\t0\t0\tGGGCAAGGAAGCAAACAAGCAGGGAATGGTAACTACGGTAGTATGAATTAGGGGGTGTTTGGTTTCTAAGGACTAATGTTTAGTCACTATGTTTTATTCCATTTTAGTTCCAAAATTGTCAAATATAGAAACTAAAATTTTATTATAGAAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJ-FJFJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJFJJJJJJFJJJJJJJFFJFJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1112:25976:33844\t0\t4\t239985165\t60\t151M\t*\t0\t0\tGAGAAAAGATATCAACCTTGTCAAGATGCTCATACTTATCATTCAGCATTTCCGGAGGCATATAGCGTGAATCTCCATCTTCAATTGCTAGACTCCGGTTAACAAGTGTAGCACAGCCAAAATCCCCGAGCTTATAAATACCATTTCTGAC\tAAAFFJJJJJJJJJJJJJJJJJJJFJJAJJJJJJJJJJJJJJJJFJJJJJJJJFA<FFJFJJJJJJJJJFFJJJFFAFFJJJJJJJJJJJ<AJJJFJJJJJFFJJFJFFJJJFJJJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1104:15392:3407\t16\t4\t160982558\t60\t117M\t*\t0\t0\tTTTCCATCGCTCCTACTCCTATCATCCAGAAAATCTGAGCGGTATTGATGGCGCCCAAGGCGGAGAAGAAGCCGGCGGCGAAGAAGGTGGCGGAGGAGGAGCCCTCGGAGAAGGCGG\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:117\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2104:21288:48966\t16\t9\t16121292\t60\t151M\t*\t0\t0\tTCGAAGAGTCAATGAAATACAAGAAACTGACAAATGCACAGCGGTCTGGTCTCAATCAAATCCCCAATCGAAGATTTACCCTGTGGTGGTCACCAACCATCAATCGTGCAAATGTCTATGTTGGTTTCCAGGTCCAGCTAGATCTTACGGG\tFJJJAFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJFJJJJJJJFJJJJJJJJJJJJJJJFJFJJFJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1111:13717:47841\t16\t4\t221024621\t60\t4S116M\t*\t0\t0\tCAATACTGTACATGAGGGTCGCATCTACCAGCTGAAGTTGTTTTGTGACAAGGACTATCCGGAGAAGCCACCATCTGTCAGATTCCACTCAAGAATCAACATGACATGCGTTAATCACGA\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-8\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:116\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2215:6390:19179\t16\t2\t190095135\t60\t113M\t*\t0\t0\tCAAGGATCTGGTCAAGAAGCACTCTGAGTTCATCAGCTACCCCATCTCCCTGTGGACTGAAAAGACAACAGAGAAGGAAATTTCTGACGATGAAGACGAGGAAGAGAAGAAAG\tJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:90C22\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1212:29254:10440\t16\t10\t22272036\t60\t128M1S\t*\t0\t0\tGCGGAAGGTGAAGCGGGAGTACGACGAGTTCAAGGTCAGGGTGAACTCGCTGCCAGAGGCCATCCGCCGGCGCTCCGACGCATACAACGCCGGCGAGGAGCTGCGCGCCAGGAGAAGGCAGCAGGAGGT\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJFJJJJJJJJAJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFFAA\tAS:i:-14\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:54C26C46\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1223:30015:17315\t16\t7\t119943314\t60\t151M\t*\t0\t0\tCGCCCCATCCTCTCACCCTCGCGAGTCCCGACCTCCCCCAGCCGCCGCACAGATGGCACCCAAGGCGGAGAAGAAGCCGGCGGCCAAGAAGCCCGCGGAGGAGGAGCCCGCGACAGAGAAGGTGGAGAAGGCTCCGGCGGGGAAGAAGCCC\t7-7))7--7-FA7FAA-7---7<<A-JJFJAA<7FA7-77-FFJJJJJJJJA7---JF<JA<AFJJJJJJJJFJFFFA-AJF<J<AA<F<-JJFJFJ<FA-AAFJJJJJJJJJJFAAJFFAJF-FJFJJFFJ7JJJJFJJFJJJAJFFAA<\tAS:i:-8\tZS:i:-11\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:4G2G29G113\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1210:5873:25597\t0\t2\t235393953\t60\t103M980N8M\t*\t0\t0\tGCTTCAACAACATCACTGGTGCTTCCACCAGATGGGGGCTTGAAAGGTGCTGGACCTGAAGTTCCTTCAGTTCCAGATCGCTCCCAAGGCTTTGGCGGTGGACCTGACATG\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:95T15\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1205:11363:28674\t0\t4\t23548832\t60\t151M\t*\t0\t0\tTGACGATTAAGCAGATGATATATTAGAGTTGATGCAGGGAGGGGCAGTTATGTACAGCGCTTAGTGGGACTTCGTTTGCTAAGCATACATTAGGTAGGTGCGACCAGCTAAAACTTGCCACATCAGAAGGAGAAACCAAAACTGCAGGAAG\tAAFFFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2101:18406:19285\t16\t10\t98731820\t60\t4S147M\t*\t0\t0\tCCAGAGGTGAAAGATAAGGCTGTGGAGCTGGCGAAAGCCATGGAATCTGAAGACGGGGTGACAGGTGCTGTTAGGGCATTCCTTAGACACCTGCCATCAAAAGCTGAGGAACAGAGTCTGCCGCAGTCATCGGGCTTCCTGGAATTCTTAG\tJJJJJJJJJJJJJJJJJJJJJJJJJAJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-8\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:147\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2203:13149:9192\t16\t3\t202507599\t60\t150M1S\t*\t0\t0\tCCATCCATTGAAGATCAAGCTCAGTTCCCAAGCCTTGGTGGGAAGTGAAGGGCGCCCAGCTGCTGCTGTCACCGAGCGTTGTGTTGTCTGCTGTTGTCATATTTAAATTTTGTCCGACGTTAAATTTCTGTGCCACGGTTTAAACGGAACG\tFJA<JJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFFJJFFFAA\tAS:i:-1\tZS:i:-7\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1120:14874:3987\t0\t2\t234105120\t60\t151M\t*\t0\t0\tGGTCCAGATCACAAAGCGCCCGAGGTGTCCACCGGGGGCGAGGTCGAGCAGGTTGAGGCGCTCGACGTTGGCGACGTCCACGCCGGGGAGGTTGCGGAAGGCCTTGACGACCTTGGAGCCCTCGGTGCCGTAGACGACGAGCGGGCCCTTG\tAAFFFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJ7FFJJJJJJJJFFJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJFJJJJJJAFJJFJJJJJJJJFJJJJJJJJJF<FJFJFJJJJFFJJJAFJJJJJ--7AA\tAS:i:-17\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:27G92T2T27\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1227:7395:40121\t4\t*\t0\t0\t*\t*\t0\t0\tGCATCGTTTATGGTTGAGACTAGGACGGTATCTGATCGTCTTCGAGCCCCCAACTTTCGTTCTTGATTAATGAAAACATCCTTGGCAAATGCTTTCGCAGTTGTTCGTCTTTCATAAATCCAAGAATTTCACCTCTGACTATGAAATACGA\tAAFFFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1211:22181:34547\t16\t4\t193790022\t60\t151M\t*\t0\t0\tACTGCTATGGAAGTCAGTCCTTTAGTTTTGCTCTGGCAGTGGATTGCTGTTGGGTCGTGTTTTCAGTCTTTATTGGTTCCGATCGGACGCTGTTGCGGCAAACCACCCCTGTAGATACTCTTCACATGCTGAAGAAAACTACCAGCAACCG\tAJJJFJJJAFF<FFAA-FJJJJJJJJFJFAF-FJAJJJJJFAJJ<JJJJF7<JAAJJJJJJJJJJJJJJJJJJJJ<FJJJJFJJJJAFJF7JAJJJJJJJJ<JJJJJJFJJJFJJJJFJJJJJJFJJJJJJJJJJJJJJFJJJJJJFFFAA\tAS:i:-8\tZS:i:-10\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:16T130G3\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2118:14387:2158\t4\t*\t0\t0\t*\t*\t0\t0\tCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGACGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGCATCG\tAAFFFJFJJJFFFJFFJJAF<FAJAJAJJJJJFJJJJFFFJ7-FA<FJAJFAJJAFJJF<FJJJJJ7A<F-FFFJJAFJJJJJJFFJFJ--FJFJJJJJJ<AAJFJJJJF-A-FJJJJJJ7FJFF-7-FF-AFFJJJJFA<FAFJJ-AJAA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2227:17919:23083\t16\t4\t183872321\t60\t19M1309N51M\t*\t0\t0\tCATCGGCAAGCCCGAGGCCTACGTGATGGTTGTTCTCAAGGGTTCAGTGCCTATGGCATTTGGAGGTACC\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:45G24\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1103:3873:4690\t4\t*\t0\t0\t*\t*\t0\t0\tCGCCCCTCCCGGAAGTCGGGGTTTGTTGCACGTATTAGCTCTAGAATTACTACGGTTATCCGAGTAGCACGTACCATCAAACAAACTATAACTGATTTAATGAGCCATTCGCAGTTTCACAGTTCGAATTAGTTCATACTTGCACATGCAT\tAAFFFJJJJJJJJJJJJJJJJ<FJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJ<JJJJJFJJ7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1222:4594:11003\t0\t2\t206607823\t60\t151M\t*\t0\t0\tGCGGGGTCCCTTTGCTGCACTGGTTGTGCGCGTACGGCTTGCACCCGTGCTTGATGACGCCGTGGCCCTTGGGGAGCGGGCCGCACAGGCGGTTGTAGCTGAGGTCCAGCTTCTGCAGCGTGGACAGGCGCTCCAGCGACAGCGGCACGGT\tAAFFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ7AJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJJJ)\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:4624:27795\t16\t7\t49286875\t60\t151M\t*\t0\t0\tTCAGTTTCCAACATGGTATCATTCGCCTAGGTTTTTCTTCCAAACATCTATAGCCGCCCGCCGCCAGCCGCCAGGGGGTTACCCCACCCTGCAGCCGTCGCTGGCCCGCCCCCAGCCTCCTCCCGCCGCCGGCCCCCCACCCCCAGCTGCC\tAJFJJJ<J<<FJFFJJJJF-JJJJJFJFJJJJJF77-JFJJJAFJJJJA-JJJJJJJJJJJJJJJJJJJJJFJJFJJJA7-JJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAFAA\tAS:i:-17\tZS:i:-18\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:36T43T20C45C3\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1214:18548:41546\t0\t5\t203092899\t60\t151M\t*\t0\t0\tTGGTAGTGCGGCATGGTGGAGAAGAGGTGGTGCGCGACGTGCGTGTCCGTGATGTTGTGGAACACGCGGTTGAGGATGCCGTAGTCGCGGTCCATGGTGGCCAGCGCGTCGCGCAGCCAGTCCCACTCGCTCGAGTCGTAGTGGGGGAGCG\t<AFFFJJ-FJJJJA<AFFJJJJJJJJJJFFJJFJJJJJJJJJJJFJJJJ<JJFJJJFJJJJJJJJJJFAAJJFJJ<FFJJ<<JJAJFJJJJFFFJJJJF<FJJJJAJFAJFJJJJAJFJ<FFJFJJFF<J-A<7A77FAAF-7FJJJJJ<A\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:108C42\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2122:30005:41282\t0\t8\t28166732\t60\t151M\t*\t0\t0\tGTCGGAGGTGTTGGTGACCCGGATCCCGCGGTCGCGGCACTTGGCGAGGTCGACGCGGTCGAGGCCGACGGAGAAGGAGGCAACGATCTCGAGCGCCGGGAGCGCGTCGATGAGCGCGGCGTCGGCGCCGTAGCTGGCGTTGCCGACGACG\t<AFFFAFJFJJJAJJJJJJJJJJJJJJJFAJJ-FFFJJFFFJFAJJFJJJFFFJJJJJAJFFAJFJJJJFJJJJAJFJJJJJF<FAJAJJJJJFJ<7AJJJJJJFJJJJJJJJFFJJJJJFFFJJJJ7A7JFJFAFJJFAJJ<FJJJJJJ<\tAS:i:-10\tZS:i:-11\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:5G56T88\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1125:27762:13799\t16\t9\t125944785\t60\t69M253N82M\t*\t0\t0\tCCGCTGTGGCAGGGCCGCATCTTCCACGCGCTCGCCGTGCTCTACGGCATCGTCTCCGCCGTCGCGCTGGTCCAATTGATCAGAATCGAGTGCAGGGTGCCAGAGTATGGGTGGACGACGCAGAAAGTGTTCCATTTCCTAAACTGCCTCG\t)<<FAFJA7A7)--JJAFJAFJF-7FJ7<A7FJFFFJJFF<JFFA7-AAJFAAAAAJJJ<AJJFJF-A<JJF<FA<-7-JFF7JJ--<JJA7-7A-JAA<FA-JFJA-AJF<-JJF<JFFFFJFJJJJJJJFF<JFJA--F7<JFAAFAAA\tAS:i:-8\tZS:i:-9\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:13A131T5\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2223:6796:14432\t4\t*\t0\t0\t*\t*\t0\t0\tGCCACCAAATATGATCGCCAATACGAATTGCCAGTCAAACCAATCTAGCCATTTATGGAAACAGAGGAATACATCGTACTTCCGACCCACAACAAATGTCCATACAAACCAAAAATTAAAACACAGCGGATGATATGCTAAGCACAGCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJAFFJJJJFJFJJFJJJJJJJJJJJ-JFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFJJJJJJJJJFJJJJJJJJJJJJJJJJJJFAFJJAJJJAA<AFAJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1216:11332:26476\t16\t3\t128938316\t60\t151M\t*\t0\t0\tGGAAGACTAGGTCCTTCTATGACATGCTTGACAAGGGCCAGATCAGCGTTGTCGTCGAGGAGTAAAGAGTTCTTCCCAGTTATTTTCAAAGAACTCTCTCGAGAGCATGACAAAGTGATTGCTTGCTGTGGAGATTTGTCCTGGAAATAAG\tJJJJJJJJJJJA<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2119:8197:16471\t16\t1\t37323091\t60\t151M\t*\t0\t0\tGTCCTGGGGTATGATGGTTTCACCGATCCCGAGCGTGCCCTGGAACAGTGGAAAGGATGGTCGGTTTACCCAACGCATAGTTTTCTGAATAATTGGCGTCGAGGGAGGAGGAGCACTGCCGACACGCATGCTAATGCTGAAAGGATGCGAG\tFAJJJJJJJFJJJFJJJJJJJJJJJJJFJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:71G79\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1201:25875:15170\t16\t2\t212804136\t60\t151M\t*\t0\t0\tTGTTTTCAAGGCCTATTTTGTTACAGCTTATAAGTACCTTATGCTGCCCTGGCTGTGTTGCATCAGTCGACCCTCCCTACGTAGACTTATGAGTAATTGTTGCAATTTTGTATGGACATGGTACTGGCTTGTCGGGATTTTTTGAGTACCA\t-JJAJA<)FAJFFJJJJJJJFJJJFJFJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJFF<JJ<A-FFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tZS:i:-15\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:80T70\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2207:11079:1279\t0\t3\t223185059\t60\t151M\t*\t0\t0\tCTCGACGGTGGGGGGCTGAACGACGTACTGCTTGTTCTTCTCCATTACCTCCTTGTAGTAGGAGCAGCCGTGCTTCGAGGCGAACTCCGCCGCACGCGTTACCTGTGTGCGCAGCTGCGCCAGCTTCGCCGCCATCGCCGCCGGTCTTAAA\tAAFFFJJJFFJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJFJJJJJFJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1228:23226:31505\t0\t3\t208908019\t60\t151M\t*\t0\t0\tTTGCCCGCTCTTAAATTTTGTTCCTCATTAGCACTCTTCTCATGATCCACTGAACGAAACTTATCTCCACCACGTGAGTCCAAACTCTTTTCCTTGTCTACATGGTGTTCACTCAAATCGGGTTCATTTTCTCCCTTCCTCCGTTTACCTG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJ<\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2224:26900:19689\t16\t8\t141176311\t60\t151M\t*\t0\t0\tCCTGTACCTGGGCCCGCTGTCCGGCCAGCCCCCAAGCTACCTGACCGGCGAGTTCCCCGGCGACTACGGCTGGGACACCGCGGGGCTGTCCGCCGACCCGGAGACTTTCGCCAAGAACCGCGAGCTGGAGGTGATCCACTGCCGCTGGGCC\tFFAA7AAJFJJJJJJJJJJJJJJJFJJJJFFFFJJJJJJJJJFJJJJJF-JJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFAJJJJJJJJJJJFJFJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2220:22059:9156\t0\t5\t155284758\t60\t116M1058N35M\t*\t0\t0\tGTTTTGTTGGAAACCCCCATTTGAAAGTCTTTTACCAACTCAACCTTGATGCCGCTGTAATCCGCGGCAATTAGTGTTTTGAATGCATTTTTGTTACTACTGCCAGCATGCAGTACAAGCGCCATATCGATGAAAAGCTAGCTTCACTCGA\tAAAFFJJJFFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJ7JFJJJJJJJFJJAFJFAJJFJFAJJJJJJFFJJJJJJJJJJJFJJJJJJ<JJJFJJJJJJJAJFJAJFJJ-<FFJJJJJFAJJJFJFFAJ)<\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:19644:22379\t16\t5\t167792245\t60\t146M5S\t*\t0\t0\tGCTGTGCACGGGCGAGAAGGGCATGGGCAAGTCCGGGAAGCCGCTCCACTACAAGGGCTCCACCTTCCACCGCGTCATCCCCGAGTTCATGTGCCAGGGCGGCGACTTCACCCGCGGCAACGGCACCGGCGGCGAGTCCATCTACGCTGAG\tAJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJJJFJJJJJFJJJJFJFFJJJJJFJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF<JJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJFF<FAA\tAS:i:-14\tZS:i:-23\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:22G123\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2103:14012:38715\t0\t4\t111657784\t1\t1S150M\t*\t0\t0\tCCGACGGTGAAGATTAAGGAAAGGAGAGCGAGGGGAAGAGGGGGCAGCCCTCGGCCCGATCATCCAATTCGCTCCGACGATCCAGGCATGGTTCAGTAGTCAAAGCAACTTCGTCACTTTCGTGTACCCATCGGACGGCAGCCCTTTCGGG\tAA-FFFJ-7-AAAAF-<FFF<7<7AA-<AF<7<<AAF<<77-7-7FAF<JFJAAJF7AJ-7-<FF<F-<AAAFJJJAJFJAAJ7<F<FJJFAAJJJJAJFJFJ7FJFFF<FJJFFFJF77F<AFJF<77A-F7--F<AAAF<F7FAJ7FFJ\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:10T139\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2103:14012:38715\t256\t1\t234659212\t1\t1S150M\t*\t0\t0\tCCGACGGTGAAGATTAAGGAAAGGAGAGCGAGGGGAAGAGGGGGCAGCCCTCGGCCCGATCATCCAATTCGCTCCGACGATCCAGGCATGGTTCAGTAGTCAAAGCAACTTCGTCACTTTCGTGTACCCATCGGACGGCAGCCCTTTCGGG\tAA-FFFJ-7-AAAAF-<FFF<7<7AA-<AF<7<<AAF<<77-7-7FAF<JFJAAJF7AJ-7-<FF<F-<AAAFJJJAJFJAAJ7<F<FJJFAAJJJJAJFJFJ7FJFFF<FJJFFFJF77F<AFJF<77A-F7--F<AAAF<F7FAJ7FFJ\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:10T139\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1101:14925:1437\t0\t5\t8272310\t60\t151M\t*\t0\t0\tTGTCCTTGAACACTATGGAAGCTGCAGCCAGGGCAGCAGCCATTTCAGACGCTAGATCTGAGCAAGAGTGACACTCGGTGACTGGCCTTTTGTAGTCGATGTCCTCTGGTCTCATCCAGCAGTAATGGTCATTAGGCTGACTGCTGCCTTT\tAAAFFJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JFFJJJJJJJJJFJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJ<FJJJJJJJJJJJJJJJJJJJJF-7FJJJJJAFAJJJFFJJFJJFF7\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1116:30330:17474\t0\t5\t14084811\t60\t116M147N35M\t*\t0\t0\tCGGCCGTCGAGCTCCTTGCCGTTCATGTTCTCGATAGCGTCGAGCATGGAGTTCTCGGAGGAGAAGGTAACGAAGCCGAAGCCGCGAGACCTCCCCGTCTCCCGGTCGGTGATGACCTTGGAGTCGAGGATCTCGCCGTAGGAGGCGAAGG\tAAFFFJJFJJFJJJJJJJJJJFAAJJFAJFF<JJJJJJJ<<JJFJJFFJJJJFA-<FFJJJ-A7FFJJJJJJJJJJJ7AJJAJJ<JJJJJJJF<J7AJ7FJJFFJJJJJFJJJJJJJAJFJJJJ<JAJJJJ7J77<JJJJFJJJAAJJAFF\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:86G64\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2225:27438:40244\t0\t6\t12304190\t1\t151M\t*\t0\t0\tGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCAGATCGGCGGGCAGAGCCCGCGTCAGCCTTTTATCTAATAAATGCGCCCCTCCCGGAAGTCGGGGTTTGTTGCACGTA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJ7FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJ)AJJJJJJJJJJF\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2225:27438:40244\t272\t6\t12020280\t1\t151M\t*\t0\t0\tTACGTGCAACAAACCCCGACTTCCGGGAGGGGCGCATTTATTAGATAAAAGGCTGACGCGGGCTCTGCCCGCCGATCTGATGATTCATGATAACTTGACGGATCGCACGGCCTTCGTGCCGGCGACGCATCATTCAAATTTCTGCCCTATC\tFJJJJJJJJJJA)JJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF7JJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1127:2301:35655\t0\t3\t147050301\t60\t151M\t*\t0\t0\tTGTGTCACCACGTCTGACAGGAGAAGGATCTGGTCGGCGACGTATAGGTGGAGAATCAGGACGGCGCCTAGGTGATGGGTCAGGTGGACGCCTGGGTGATTCAGCCCTTCGATTTGGAGATCGTTTCCGTAGAGGTGAAGCTGGTTTCCTT\tAAFFFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJAFJ<FFFJJJFFJJJJJJFFFJJJFJJJJJJJJJJFJJJA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:7050:7451\t0\t6\t41397720\t60\t110M\t*\t0\t0\tCCCCGCCAGCGACAGCGACGACGATGTCAGCTTCTCCAGGGAACCGTCCTAGCTGGCGCAGGCACGAGGCAAGTACGCTTAAGAGAACGGGCGAGAAAAGCTGAGAGACC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:110\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:25347:35673\t4\t*\t0\t0\t*\t*\t0\t0\tGTTCGTCTTTCATAAATCCAAGAATTTCACCTCTGACTATGAAATACGAATGCCCCCGACTGTCCCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACCGGAATCCTATGATGTTATCCCATGCTAATGTATCCAGAGCGA\tAAAFFJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJAAFJFJ<JF7\tYT:Z:UU\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2107:16620:47172\t0\t2\t6643068\t60\t1S112M1932N38M\t*\t0\t0\tCCCTCGTCCCCATCGTCACCACCATCATCATCTTCATCGCCAGCATCGACAGGTTCGAGCTCATCACTCTTCTTTCCCCTCATCTTGAAGTCTGGCACACCACAGATAGGACCCTCGGCTGCGACGGCGAGCAGCAGCTTCTCCGCGGCGT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF<<<77JJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJ7\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1111:2199:38117\t0\t5\t1822351\t60\t9S142M\t*\t0\t0\tGGCACACAGACAAAAGTACTAGTGCTTCACTTCACATCATCACCTCTTGTGTGGAACTGAAGGGGAACATGTGATGCTGGCGACGAAATCTCGGAACTAGCGCAGCAGGAATCGAACCGCAGCCACGGCATCACCCTTGTGCTCACGGAGC\tAAAFFJJJJJJJJJAFJJJJJJJJJJJJJJJJFFFJJJJJJJJFJJJJJJJJJJJJFAJFJJJJJFJJJJ-AAJJJJFFJJJJJJJJJJA<JJJFJJJFJFJJJJJJJJJJJFF7FJJFJF<FAFJAFF<<FAAFAJFJJFFFFJJJA<<F\tAS:i:-16\tZS:i:-20\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:61A80\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2112:16254:32437\t4\t*\t0\t0\t*\t*\t0\t0\tGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGACTGATGCGTCGCCG\tAAFFFJJFJJFJJJJFJJJJJJJJJ7FJJFJJFJJJJJJFJJFJFJFFJAFFFJJFJJJJJJFAFJJJJFJAFJAAAAAJFJ-<F<<JFJJJ<7A7AJJJJFAJFAJFFFFJJAF7FFJJJJFF<A7FAFAFJ--<A-7AFJJ7AA7AFJ7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2108:13129:19038\t4\t*\t0\t0\t*\t*\t0\t0\tGCAGGGTCGTACTTAAGCGCGGCCACGCGCAGTCCAGCCCGTCGGGTGTGGGCAGCACCTAGCGGCCTGCGGCTGCTGGCTAGGAAGGGCGAGGTGGACATGGCGGCGAGGAGTGAGGGCGCCATTGGAGCGGAGACCAGGGGGCGGAGGC\tAAFFFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFF7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1128:6461:26687\t16\t8\t38418616\t60\t151M\t*\t0\t0\tGATGGTGAAGAAGGCTTTGGCAAATGGCAATTGCCGCAGCAAGCCATGTCGGCGCCACTGACCGGCTTAGTGATTGGTATAATTTGGTGTGGCAGCAGCCAGGATTAATGCGCTGGTCTTTTATCTTTACTACTAGTTTGGTCTCGCCCGG\tAFJJJJFJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-5\tZS:i:-8\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:146T4\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2211:21978:34196\t0\t7\t5141970\t60\t58M115N93M\t*\t0\t0\tCTTGAACATTAGGAAGCCATCTAGGCTCCCATCCACAAACGCTAATAAGCTTCTGTGCTTGAGAGTAACAGCAGGTTTCATCTTGTTGGGCGATATCAATTCCAGTTGTACTGTCTGTTCTGTAGCCCAGCTCCCCAGACAAAACTGTGAC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJFFJF7JJJ<FJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1123:27519:19249\t0\t1\t67537009\t60\t71M80N80M\t*\t0\t0\tCTGAAAGACTCATTCAATACAGATTCTTCAGCACCATCTTGATTGGATAGTATGACCTTTGAATTGAGAATCTCCACGAATTATTCAATTAAATTCCTAAGGAACTTATTCTCGAGCATTTCAGAAGGGTTGATAGGTTGGTCAGCCTTTT\tAAFFFJJJJJFFJJ7JJJ<-FFF-F<JFJ-JJFJFFJJJJF<J<7<F--FJFJ-7FFFAF-<F<A-JJJFJFFFAF77AFJF-AFJFFFJFJJJJJAJJJJJ-77A-FJJJJ-A-77JJAJ7-<7AJ7-<7<FJJJ<--7F7F7<F))777\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:82C68\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:6482:10546\t16\t1\t103574411\t60\t151M\t*\t0\t0\tAGAATTCAAATTTGGTGGTCGCAAAAGTGTGGAGACTTTAATTCATATAAAGTTAGGCTTAACATTAGTGCAAACAGTTGTATTTTAGTTTAGATTTAGAGTACACTTATGTATGCATTGTTTGACAATGTTTATTTATGATATATTGAAT\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-5\tZS:i:-8\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:147A3\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2217:10784:28973\t4\t*\t0\t0\t*\t*\t0\t0\tTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCAGAGCCCGCGTCAGCC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJFFFJJJJJJFJFFFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2112:12763:30679\t16\t2\t234028039\t60\t151M\t*\t0\t0\tAAGACATTCAACCAGGTTGAGATCAATCCTGCGATGATCGGCCACTACCTTGCTGAGTTCTCCATCTCCTATAAGCCGGTCAAGCATGGCAGGCCCGGTATCGGTGCCACCCACTCCTCGCGGTTTATTCCTCTGAAATGAGCTGTTACCC\tJJJJJJJJJJJJAJFAJAJJFFFFJA7JF<A7JJJJJJJJJJJJJJJJF<-FFJJJJFJJFJJJFJJJJFJJJFJJJJJJJJJJJJAJJJJJJJAJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-8\tZS:i:-14\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:26G4A119\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1111:4472:41352\t4\t*\t0\t0\t*\t*\t0\t0\tGTAAATTATAATCCCAAACAAACAACCACTAAATATAAACGTTCATACAAATTGCATTCGTAGATTGATTGATGAATATATGTTTATATTTTGAGATGTGGTGGGTATATGACAAGCTCAGGTTCAATGCACCTGATTTCATTTCTTTTCT\tAAFFFJFJJJJJJJJJJJJJJJJJJFJJFJJJFJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJFJAJJF<JJJJJJJFFFJJJJJ7AJJJJFJJJJJ<FAFJJFJJJJJJJFJJJAFAFJJJFFJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2106:25591:43656\t16\t1\t16824680\t60\t151M\t*\t0\t0\tGCTGGAGGTGGGCGGCCACGCGCTCAAGACCAACATCACCACGCTGGGGCCCCTGGTGCTGCCCTGCGGGGGGCAGCTGCTCTTCTTCGCGGGCGTGCTGTTCCGCCACCTGTTCCCGTCCAAGGCCTCCACGCCGGCGCCGCCGACCACC\tJJJJJJJJJJJJFJF<AAFJJJJJFJJJJFFFFFFJAAFF<FJ7AJJJJ7--7-JJ<J777--------7777<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-17\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:65T1T0C0C1A79\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:30259:10739\t16\t8\t76949022\t60\t4S143M4S\t*\t0\t0\tAACAGGACACTGTGGATGCTGTTAATCCTGAGGAGACTACTGCCGTGGTGGAGGATGAGAAGAAACCTGAAGATGCACCACAGTCTGAGGTTGAGAAGGACAAGGAGGGCGCGGAGAATGAAGAAGAAGAAAAGGAACCTGAGGATACGGA\t--AA-<A-<)-<A7-A-JJJJA<FF<FA-F--AFAFFAFA-AFAA7-JA-AFFJJ<7F<JFFJJAJJAFJJJJ7FJJFFJFJJAJJJA<FJJFF<F<-<JJJFA--77JJJF<FJFJJJJJJJJJFJJJJJAF-FF7AFJAJFFJJF-AAA\tAS:i:-8\tZS:i:-12\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:143\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2228:29346:48896\t4\t*\t0\t0\t*\t*\t0\t0\tNTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCAGAGCCCGCGTCAGCCTTTTATTTAATAAATGAGCCCCTCCCGGAAGTGGGGGTTTGTTGCAC\t#AA<FFJJJJJJJJJJJJJJJJJFJJJFJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFFJJJJJFJJFFJJFAFJJJJJJJJ<A7FFFJJJAAFF<AJJ<-AJJ--7F--A<JFA-7FF-AAFJFFJJJFAA7))A)7A--A-<F-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1218:8379:39717\t16\t6\t20571890\t60\t1S150M\t*\t0\t0\tGGGACCTTGGGCTGGGCCGGCCGGTCCGCCTCACGGCGAGAACCGACCGGCTCGACCCTTCTGCCGGCGATGCGCTCCTGGCCTTAACTGGCCGGGTCGTGCCTCCGGCGCCGTTACTTTGAAGAAATTAGAGTGCTCAAAGCAAGCCATC\t)A7-FFJJJJF7-<JAJJJJFFJJAJJFF<JJJFFJJJJJJAJJJFJFJJAAFJFJJJJJJJJFJJJFJFJFFFFJJJJJJ<JJJJ<JJJJJJJJAJJJJJJJJJJJJJJ7FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFFFAA\tAS:i:-1\tZS:i:-6\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2221:29802:27250\t16\t6\t124987365\t60\t151M\t*\t0\t0\tCGCCCCGCCGGCGCCCGTCAACCTCACCGACATCCTCTCCAAGAAGTACTGCAAGAGCTTCGCGGGCCTCCTCGCCGCCGACCCCAAGGTCTTCGACACCCTCAACGGCACCAAGGACACAGCGCTCACCGTCTTCTGCCCCGTCGACGCG\tJJJJJJJF77J7JF7<<7FJFJF<JJFFFJJAAAJAJJJJJJJJJJAFAAJFFF<7FAJAA77JJJJA7FJJJJFF-FJJJ<F-A<7F-FJJ-F7-JAJFJFF<7JJJFJJJJJJJJJJJFJJJJJJJJFFJJJ7AAJJJJFJJJJAFAAA\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:120C30\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1112:28899:30011\t4\t*\t0\t0\t*\t*\t0\t0\tCCCGTGTCAGGATTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTG\tAAFFFJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJAFAJJJJJJJJFFFJJJ<FAFJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFFJJFFFJJFFJJFAFJFFJAFJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1207:24495:8629\t4\t*\t0\t0\t*\t*\t0\t0\tGCCCCGTATGCCTCCAAATATCGATAACGCCGAAGGTAGTAGACAACAAGCAGACTACCAATTGGATCCATGGGCGCTTCATGAGCAGACTCGAAGAGGGACTTGTGAAGATGTTTCTCTTCCTCAGAATCCCAGGGCAGATCAATCATTT\tAAFFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1101:19177:1455\t0\t4\t229352731\t60\t125M\t*\t0\t0\tACAGCATTACCATGTATTAGGTAGATTTACGGCATACAACAGGCAAAGTTCGTCCATCTGTCCAAAGCAAAATGCGCAGCAACCACACTACAAAAAACCCTCGGACGAGTGTATCAAACTATATA\tAAAAFFFJFFJJFJ-FAJJJJJJJFFJFJJF<JAJJJJJJFJJJFJFF--FJ-AJFJJ<7AFFFJFJJJ<F7AJFJFFF7JAJJFJFAJ<FJF<F<JJFJ7FFJF<FFJFFFFFJ--7A<FAFJJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:43T81\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1108:11079:41757\t4\t*\t0\t0\t*\t*\t0\t0\tGCTCGCCCAGATGGCCGGGTGTGGGTCGCGCGCTTCAGCGCCATCCATTTTCGGGGCTAGTTGATTCGGCAGGTGAGCTGTTACACACTCCTTAGCGGATTTCGAC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-FJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2107:20303:25140\t0\t2\t234103826\t60\t9M1D137M5S\t*\t0\t0\tTCTCAATACAAGTTCAGCGACACAACTAATCAAAACTTAACTAGGTCTCAAAGGCCGCTTCACTGCGTCACTCCGAGCCACTTGGTGAAGTTCTCAAACTCGGTGTAGTCACTGTCCGATATCATGGTCTTGTACCACGCCTTGCCCGCAG\tAA<FFJJJJFJJ7JJ<FJFFJJJFFJFJJJJJJJJJJJJJJJJJJJAFJJJJJJJJJJJJJJJJ<JAJJJJJJJJJJFJJJJFJJFJJJJJJJJJJJJJJJF<7JFFJJJJJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJF\tAS:i:-19\tZS:i:-24\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:1\tNM:i:2\tMD:Z:9^A110G26\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2212:31903:48825\t4\t*\t0\t0\t*\t*\t0\t0\tCACGCCGCAAACTCCATTGGAGTGGTGCCTTCTGGTTCTGAATGATTGCCTCTTCCCTCCTGACAGCCGGTGGACGGGCGCCCTGCCGTCAAAACTGTAGCCAATCTGTTGTCGGTCTTTGGTCTGTAATGTTATCCTGCTGTCT\tAAAAAAFFJJJJJJFAJJJJ-FJ-FJAJJJJJJJJJJJFJJF-FAJFFAFJJJJJJFA<AA<<A<F7FAJAJJAJ-7AAJJJJJJJ7A<JF7FFJJJF<JJFJJFJFJFJJJAAAFAFFFFJJ<AAJ-<<<JFFJJAF<AFFAJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2201:7436:13675\t4\t*\t0\t0\t*\t*\t0\t0\tCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACGACCCGGCCAGTTAAGGCCAGGAGCGCATCGCCGGCAGAAGGGTCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1219:13210:40209\t0\t6\t161427779\t60\t23M76N63M92N65M\t*\t0\t0\tGCTCTCTTCTTGCCTACCTCAGGCTTCTTAGGAGTTGGAGTGTCTTCCTCCTCATCGTCACTGGTATCATCTTCATCACTTGAATCATCATCGCCTTCTTCAGAAGATAAACCCTCATCAGAATCGTCCGTCTCATCCTCATCAGAATCAT\tAAFFFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ7JFFFJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJAFJAFJJJJJJJFJJJFJJJJFJJJJJJJJJJJJJJFJJJAJJJJJJJFF\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:103G3C43\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1126:22698:3723\t0\t7\t173870509\t60\t151M\t*\t0\t0\tCAGCATTACCTGAAGATAAAGAACTAAAACCAGCAGATGTAGAAGCACTTGAAAATCCACCAAATCCTCCAGAACCTATGCCTCCTAACTGGGGAGGTTGTCCAAAAGAACCCAGCACTGATCCAAGTGTCTGTTGAGCAGCAAACTGTGC\tAAFFFJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2102:29234:34495\t0\t9\t122479513\t60\t151M\t*\t0\t0\tGCTCCTTCTCGCTCATCGATCGATCAGGTCTGTGGCCGCCGTGGCGCATCACTTGCTGGAGGGCTCTCCCTCCACGGCCAGCGGCACGGTGCTCGCCATGGTGCGGTACTTGAGCGCGTACAGCATCTCCAGGTACCGCCGGGTCTCGCGC\tAAFAFJJJJJJJJJJJFJJJAJFJJJJJFFJJJFJJJJJJ<FJJJJAJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJF-JFFJJJJJJJJJJFJJAFJ<FAFJJJFJJJJ7JJJJAJJJ-\tAS:i:-12\tZS:i:-13\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:5C36C108\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2214:13930:49124\t0\t5\t186727830\t60\t1S150M\t*\t0\t0\tNCACTCCTCGCCGGGGGACTTGAGCCTCTCGCCGGTCAGGAACACGCAGCCGAGCTCCTCGCGCACGAAGCGGTACAGCGGGAACGACCGGCTGTCCGCGATCCGGTTCGCCACGGGGGCGGTGCCCTCGGCGACCGCCACGCGGGCGGCC\t#AAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJAFJJFFJFJJJJ\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1223:3041:30433\t0\t6\t127950124\t60\t151M\t*\t0\t0\tCTTTTAGCATGTTCCACAGAGGAGCATTCGGGTCTGATGTTGGCGCTGCCCGAATAGGTTTTCGAGCACTGTCATGGATCCCTACTAAGGTGCTGTTCAATGCATCCACGTTCCTTCTTCCATCACGTGGCAGAGCAGCCTGCGCCTTTGG\tAA<A<FJJFFF-<FFJAJF<A<<-AAJJJAFF<FJJJFJAJ<-AFAFAAJ7<JFJJJJF<J-A7<AJJJ7A<JJFFJFAJJF7AFJFAAJJFJJJJ-FJJJFJJFF-FFA<FJJJJJJ<J7J<7FJ7<<FJFAA7-JA7AJAAFJJJA---\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1114:27986:31593\t0\t8\t134264439\t60\t77M147N74M\t*\t0\t0\tATCGTACATAGCTCCCAGTTCCCTTCGCCGTCAATGAGCTTCAGCTCCAAGGCATGGAAGGGTATCAAAGCAGGCCTTTGAGAGGAAGTGATCACAGTTATGCCCATGCTAGTTGCAAGTCTGTACAGATGCTCTTCGACATCAACACTCG\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<A<F7FFFJJJJJJJJJJJJ7JFJJJFJJJJJJJJJJJJJJJJJFJFJJJJJFJJJFFJ<FJJJJJJFFJJJJJJJFFJJJ<AAFFJ7FFJJJJ<7\tAS:i:-5\tZS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:4C146\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1218:30533:9104\t16\t2\t198244952\t60\t66M684N69M\t*\t0\t0\tGCAAAATTTGATGGAATGCCAAGGTTCACCGCTCGAGTTGCTGTTATTGAGGCGCTGAAGGAAAAGGGGTTGTATAAGGACACAAAGAAGAATGAAATGAGCTTGGGTGTTTGTTCAAGAACTAATGATGTAGTG\tFFJJJJJJJJFFAJJFJFJFJJFJFJJJJFAJJJFJFJJJJJJJJJJFAFJJJFJJFF<<AJFAFJJJJJJJJJJJFFFJJJJJJJJJJJJJJJJJF<-FJJJJJFFJJJJJFJJJJJJJJJJJJJJJJJFFAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:135\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2109:28026:30573\t16\t2\t234467310\t60\t47M119N89M468N15M\t*\t0\t0\tCAAGCCACCAACTGTTCGGTTTGTCTCTAGGATGTTTCATCCAAACATATATGCTGATGGAAGCATCTGCTTGGATATCCTACAGAACCAGTGGAGTCCTATATATGATGTTGCTGCAATCTTGACTTCTATTCAGTCCCTGCTGTGTGAT\tAFAFAJFFJ<7JFFF<<FFJFJJFJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJFJJJJJJF<JJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:21C129\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1128:17746:3020\t16\t8\t167108586\t60\t20M15188N131M\t*\t0\t0\tAGCTTCCCAGCTTCTCCAAGAAAAGGAAGCTCTACTGGGATCAAAAAAAGAAAGGCTGCTTGAACATGCCAACAAGGAAATCTGAGAGGCTTGCTAAGAGAATGAAACTCATGGCATCATTGCTCCTCACGCAACGGAAAAAGATTGGAGT\t<FJJJAJJJJJJJJFJJJJFJJJFAFJJFF<FJFJJJJFJJJJJJJJJJJFJJJJJFJFJAJJJJFJJJJJJJJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFJJJJJJJFFAAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1124:8521:6554\t16\t2\t196702783\t60\t56M85N95M\t*\t0\t0\tTCTTTACCTTCAGTAACATACAAGATGCAAAGTGTTCAGGATGGCAACACAGAGCAATGTGTAATTTGCCGTGTGGAATTGGAGGAGGGTGAATCTTTGATTGCACTTCCCTGCAAGCACTTATACCATCCTGAATGCATAAACCAGTGGC\tJJJFFF--JJJJ<F-FFJJJFJJJJFJFJFJJJJJJJJJJJJJJFJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFJJJJJJJJJFJJJ<JJJJJFJJJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJFAAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1207:27387:8928\t0\t1\t250153577\t60\t151M\t*\t0\t0\tGTAGCATTCCAGACATTCAAGTCCCCCTATATATCGAAACCATCACTCTAGCAAATCCCACAAACAGAAGTTAAAAGGTACCCAGCTATGCCTGGGTAGCATTTTTTATGTTGCAGCAAAAAAGGATACATTCCCCGCTTGCGAGGTCGAT\tAAFFFJJJJJFJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJFJJJJJJFJJJFFJJFFFJFJJJJJJJJFJJAJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJFFAFFA<)JJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2205:10399:29255\t0\t8\t5915945\t60\t30M87N84M92N37M\t*\t0\t0\tATCAAAGAATGGGTGAACAAGTGCCTCCATAGCAGTGCATCTCAGATTTGGGGAGTACTGGAGTAGCCGAGAGACCAGATCAACAGCTTCTGGCGGCATCCTTTTGTGGAATACCTTGTGCCATGGGTGTGCTTTGATTTGTGGGAACTTA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJFFFJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJFJAJJJJJJFFFAFJJA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2220:28980:29835\t16\t8\t65487241\t60\t151M\t*\t0\t0\tGGATCATTACAATACTGCTCGTGGTGTTCAGAAGGTTCTTCAGAACTACAAAAATCTTCAGGATATTATTGCTATCTTGGGTATGGATGAGCTCAGTGAGGATGACAAGTTGACGGTCGCTCGTGCAAGAAAGATTCAGCGTTTCCCGAGC\tJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJF7FAJJJJJJJFJJAJJJJFJJAJJJJFFJF-<JJJJJJJJJJFJJJJJJJJJJJJ7JJAJJJJJJJJFJFJJJJJJJJJJFFFJJJJJFJJJJJJJJJJJJJJJJAJJJJJJJJJFFFAA\tAS:i:-5\tZS:i:-10\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:146T4\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1217:26453:21694\t16\t1\t211670859\t60\t94M374N57M\t*\t0\t0\tTGCTGCTCCATTCAAGCAGTGGTACCTCACTCACTATGGAGTGGACATCGGTAGGAAGAAGAAGGTGCCTGTTGCTAAGAAGGATGCGACTGAGGGACAAGAGGGTGAGGCTATAGCTGAGGAAACAAAGAAGAGTAACCATGTGCAGAGG\t<7JJFFJAFFF7JJFFFFJJFJJJJJFF<FFJJJJJJJJFFJJJJJJJJJJJFJJJJJJJJJJJFFJJJFFJJJAJFJFJJF<FAFFFFJA<FJFFJJJJJFJAFFJFFFAFJJJJJFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1205:25723:21342\t16\t7\t7960054\t60\t15M2926N15M1I17M1D103M\t*\t0\t0\tAGTAGTTCCTTACCTGGATAAGCTTCTTGCAGATACACCTATGGGAGATTTTTTTGTATCGCGAGAAAGGATCATAAATCACACCTATGGAAGACGAAGACCTGTGAATTGCACCAATGCTTTCACCATTCTGTCTCATATCAGGGTATCT\tAAJJJJJJJAJJJJFJJJJJJJJJFFFJJJJJJJJJFJJJFJJJJJJJJJJJJJFJF<FFJJJJJJJJJJJJJJFFJJJJFFJJJFJJFFJJ<FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJFFFAA\tAS:i:-16\tXN:i:0\tXM:i:0\tXO:i:2\tXG:i:2\tNM:i:2\tMD:Z:47^T103\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2215:23500:43796\t16\t7\t7459145\t60\t151M\t*\t0\t0\tCCCGCGGCGGGGACAGGGAGGGGCAGGTCACGCCGCGGTCCGCGGACTTCAACGCCTGGTACACGGACGTCATCGCCGCTGCCGAGCTCGCGGACTACGGGCCCGTGCGGGGGACCATGGTCATTCGCCCCTACGGATACGCCATCTGGGA\tJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:109A41\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2224:24748:28305\t16\t5\t74175175\t60\t130M2732N20M1S\t*\t0\t0\tGGCGGCCGCGGCGCACGGGGCGAGGCGCGGGTTCAGCGTCCAGGTTCCCGTCCCCGTCCCCGTCGAGCGGCATGGGCCCGGGCCCGGGCCCGTGCTGCTGCCCTGCCCGCAGGGGGACGGCGGCGTTCAGGGTTTTAGGTGGTATACAAGA\tJFJJJJFJJFJA-A-FFF<JAJFJJJJJJJJJJJFFJJJJJJJJFJJJJJJJJJJJJJJJJAJAAJJJJJ<A<JJJJFJJFJJAJJJJJJJJJJJJJJJJJJJFJJFJJAJJJJJJJJJJJAJJJJJJJJJFFJJJJJJJJJJJJJFFFAA\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2120:3437:22116\t4\t*\t0\t0\t*\t*\t0\t0\tACCATATATATCAACGTACGTCTACTAGCTTGGACAACTTATTTAGCCTGCAGATGGATCGATGGGCGTGATCAGAAGCACTGGAAGCCCGAGGGGACGCGCCTGCCGCAGTAGTTGACGAGGAGGCTGAGGTTGATGGGCAGGTTGAGGT\tAAFAAF-FFJJ<FF<F<<AFFA<7FJAFF-FFFFJ<F-<7-<--<JA-7<7FF-<AFFJAFJJFJAJ<---7-<-<FAJFAJAFJ-<77-7F<F<-7-A-<-FAJJFJA-AFFFFJ-FJJJFAJJF--7--AAAA-7AA<JAJJF7---A-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1108:29041:40842\t4\t*\t0\t0\t*\t*\t0\t0\tGCGAAGGAGAGGGCGGAAGATCCGGCGATGGAGGCCATGTGCGTCAGACGGGAGGGGAGAGGGAGAGACGGAGGGGAGGCGGCGGGCACGAAGAGCGTGAGAGCGACG\tAAAAFJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJ-AAFJJJJJJAJFJ-AFJFJJJJJJJJJJJJJJJJJJFJJFJJJAJJJFFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1224:12499:33598\t0\t9\t94818542\t60\t22M128N129M\t*\t0\t0\tGTGGGAACCGGGGTAACATCCTCAATGCGTCCAATTTTCATCCCAGAACGAGCAAGAGCCCTGAGAGCAGACTGAGCACCAGGTCCAGGGGTCTTGGTCTTGTTGCCTCCAGTGGCACGAAGCTTAATGTGCAGCGCTGTGATACCGAGCT\tA-FFFJFFJJJJJJJFAJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJ7JJJJFJFJFJJJFFJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJAJ7\tAS:i:-11\tZS:i:-26\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:113T23G13\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1207:14458:47049\t0\t4\t232945440\t0\t151M\t*\t0\t0\tTCAATCGCTTTTAGAAAACAGTGTACTAATCAGGTCTGAAAACATGGCATAACGCTCTGAACTTGAGACGACTAACCAAAAAAAGTGTACTCATCAGAACACACATTTCGGTGCAAGACAATCCATGAGAATGCTCTAAGCTACAGCGCTC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-12\tZS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:71T25G53\tYT:Z:UU\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1207:14458:47049\t272\t4\t232846018\t0\t151M\t*\t0\t0\tGAGCGCTGTAGCTTAGAGCATTCTCATGGATTGTCTTGCACCGAAATGTGTGTTCTGATGAGTACACTTTTTTTGGTTAGTCGTCTCAAGTTCAGAGCGTTATGCCATGTTTTCAGACCTGATTAGTACACTGTTTTCTAAAAGCGATTGA\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-12\tZS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:53C78C18\tYT:Z:UU\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1207:14458:47049\t272\t4\t233050399\t0\t151M\t*\t0\t0\tGAGCGCTGTAGCTTAGAGCATTCTCATGGATTGTCTTGCACCGAAATGTGTGTTCTGATGAGTACACTTTTTTTGGTTAGTCGTCTCAAGTTCAGAGCGTTATGCCATGTTTTCAGACCTGATTAGTACACTGTTTTCTAAAAGCGATTGA\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-12\tZS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:53C62T34\tYT:Z:UU\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:2221:3133:25949\t0\t4\t198738409\t60\t151M\t*\t0\t0\tGTCTTCTGCAGGTAGTCCAGGAACGTGTGGAACGGCCCGGCCACGCTGAGGAGGTCCGCGAGGTCCACGTGGTGCGGCGCCGCAGGTGCGGGCGCTGGCGCCGGCGCCAGGACGACTGGGGATGGCGGAGCGGGAGGGCTCTTGGGCTTCT\tAAAFFJJJFJJJJJJJ-FJFFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJAJFFFFFJJF<A<FJFJJFJAAFA<JJJJJJJJAFJJJJJJJJJJJJJJJFFJJFJJAJJFJFJ<FJJJJ<FJJFJJFFAA<-AFFFFJ<F7-<A7<<A\tAS:i:-11\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:83C39C27\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1114:29285:23751\t16\t10\t79766204\t60\t151M\t*\t0\t0\tCAACCAGCCCGACGTACTCGCCTACCAGCCCATCGTACTCACCCACATCTCCGAGCTACAGCCCAACATCACCAAGCTACAGCCCTACATCACCAAGCTACAGCCCTACATCTCCTAGCTATAGTCCTACATCACCAAGCTACAGCCCCAC\tA-<)<F-AAAAFJ<AFJJJJJJJAJFJJFJFAFJJJ7F-JJJJAAJAFJFJJJJJJJJJJJJFJJJJJFJFJJJJJJA7AJJJJJJJF<FFJJJJAFJJAJJJJJFFAFA<A-JF<JFJJF<FAJJJJJJAJFFJFF-JFFJJJJFFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1104:26068:43392\t0\t8\t168609423\t60\t151M\t*\t0\t0\tTCTTAAGCAAAATAACTGAAACTTTCTATGGAACATAGGTATACTAGCTCATAAGCCGTCCGTTTCCGAGCTCTTGGTGATTTTAGGTCCTGCCCACTGTTGGCTAGCACGTGACTTGGCCCATGGATCAAGAGGGACTCGTTGAGGAGGC\t<A-A<<7-AFJJFFJF<JF<FF7JFAA7AJFJ<J7FJJFAFFAFFFJJ-FJJF-7FAA<<FJ<<AFJAAFAAA<-<<-A-7-7FF-FAJAJJ<<7AFA-A--7F7-A<-7A<FJF7F<<7777-F<A---7FAF<FF))---7A7AJ-A)-\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1118:5081:30415\t4\t*\t0\t0\t*\t*\t0\t0\tTCTTGATTAATGAAAACATCCTTGGCAAATGCTTTCGCAGTTGTTCGTCTTTCATAAATCCAAGAATTTCACCTCTGACTATGAAATACGAATGCCCCCGACTGTCCCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJ-AJJFFJJJJJJJJJJJJAJJJJJJJJJJJJJJFJJFJJJJA<AJJJJFJJJJJJJJF<FJJJJJJJJJJJJJJJJ<AF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1105:14022:17034\t16\t4\t231271421\t60\t151M\t*\t0\t0\tAGTGAGTTTGCATTACTTCATGACCTTTTGAACTGCCTCTGCAAGACTGGACTCAGATTACGCTTGATTGGTTGCCGGCCTCACCTTCGCCTGGCTTGCGCGAGCCTGCGTCTATAGAAATGCGCCGGACTCACGTCTCCGTCGATGCAGG\t7JFF<AFJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2209:20435:21465\t4\t*\t0\t0\t*\t*\t0\t0\tCGCCGATCGAGAATATAGTACAAATTAAAGGCTGCCAATTGTTGTCATTAAGGAACTAGCTAGCTAGAAACAAAAGTTGACACACACACACACACACACACACACACACACACACACACACACACACACACACACACACACACTAATACCT\tAAFFFJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJFJJJFJ7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1220:15919:46627\t16\t6\t20571891\t60\t151M\t*\t0\t0\tGACCTTGGGCTGGGCCGGCCGGTCCGCCTCACGGCGAGAACCGACCGGCTCGACCCTTCTGCCGGCGATGCGCTCCTGGCCTTAACTGGCCGGGTCGTGCCTCCGGCGCCGTTACTTTGAAGAAATTAGAGTGCTCAAAGCAAGCCATCGC\tFFFFAJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJFFJAJJJJF<JJJFJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJAJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1110:12185:35057\t0\t7\t112263451\t0\t1S122M1I27M\t*\t0\t0\tCGGCTAGAGATCAACTGATTAAGGCGCTGCAAGGTTAAATAAATTCAACATAGCATAAAGGCGGTTCTTTAACCATCAGACAAAACATGGTATTACAATGACAAATAACACAAGCAAAAAAAATTTAAACAAGAGCTTCCTAGCTTGCAGA\tAAFFFJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJ<AJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJFJJJJJJJJJJFJJJJFJJJJJJJJJJJF<FFAA\tAS:i:-21\tZS:i:-21\tXN:i:0\tXM:i:2\tXO:i:1\tXG:i:1\tNM:i:3\tMD:Z:46C44A57\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1110:12185:35057\t256\t2\t118786727\t0\t1S122M1I27M\t*\t0\t0\tCGGCTAGAGATCAACTGATTAAGGCGCTGCAAGGTTAAATAAATTCAACATAGCATAAAGGCGGTTCTTTAACCATCAGACAAAACATGGTATTACAATGACAAATAACACAAGCAAAAAAAATTTAAACAAGAGCTTCCTAGCTTGCAGA\tAAFFFJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJ<AJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJFJJJJJJJJJJFJJJJFJJJJJJJJJJJF<FFAA\tAS:i:-21\tZS:i:-21\tXN:i:0\tXM:i:2\tXO:i:1\tXG:i:1\tNM:i:3\tMD:Z:46C44A57\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1113:17665:40649\t16\t4\t30962829\t60\t151M\t*\t0\t0\tGATTTGGAAGAGGAAAATAAATCTGATATGTCAAAGATTGGGAAGAAGACAGATAGCAAGAAGAAGAGGAGAAAAGAGAGAAATGATGGTGTTGACCTAAGACAAAATACACTAGTTGGAGAGGGCAAGAATGGTAAGAATAACAAAGAGA\t7A<AA--AAFF-A7-AFAJFJJ<JFJJJJF-AJJFFA77J<JJJJFFJJAJJA7FJJJJJFJJFFFFAA<AA-<AA<7-7-<AAAJJJJF-JFAFJ<JJFFJJJJJJJFFFJJJJJJJJJJ<J7A<JJJJJA<JJJJJJJJJJ<A<FFFAA\tAS:i:-3\tZS:i:-15\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:14C136\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1223:24302:21025\t4\t*\t0\t0\t*\t*\t0\t0\tGGAGTGGACGGCCTGCACCATCGCCGGCAAGCCATCGAGCTTCTGCTGGAGCGCGGTGGAGTGGCCCATCGGCGGCACACCAGCGGCGGAGGCGG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1226:16528:17016\t16\t3\t223394426\t60\t150M1S\t*\t0\t0\tCGCTGAAGGACTACCTATCAGCAGCTAATGGCATAAGCAGTGGCAGTGAGGTAGAGAACCCGCAGTCTGATGATGAGATGGAAGAATCAAACTGATGCATAGAAACGAGATCACCCGCTAATGCTTACTTTGGACAGTTGCTTTTGAGAGC\tJJJJJJJJFFAFJJJJJJJJFJJFFAJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFJJJJJJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2201:2534:40561\t16\t7\t113009840\t60\t151M\t*\t0\t0\tAACGCACAAAGAAGGATCTTGAGGCAGAGCAGAACAGAATAGAGGAAATTGCTGACCTTCTATGCTGGGAAAAAGTCTCAGAGAAGGGCGAGATGGCGATATGGAGGAAAAGGGTGAATACCGAGTCTTGCCCTTCTAGGCAAGAAGAATC\tAJFFAFFJJJJJJJFJ<FJJJ<<JJJJJJJJJJJJJJJJJJJJJJJFJJJJJJAJJJJJFFJJJJJJJJJJJJJJFJJJJJJJJFJJJJFAJFJJJJJFJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:14407:16120\t16\t8\t3576734\t60\t151M\t*\t0\t0\tAGCCGACGTGGACGGTGATGGCCAGATCAACTACGAGGAGTTCGTGAAGGTGATGATGGCCAAGTGAGATGATGTGCCTGTTTTGAAGCTAGTTCCTCTCTTGATGGTAGTAGCGAGTCTAGTGTTGGAGACTGACTGCATTTGGTCCGCC\tAFAAF<JJJJJ<FAJJJFJJJFAF<7A<FFF7<FA7-FFJJFFJFFJJJJAJFFF-JJJ-JF<F-JAFAFA-JA7AAFJJJJFFF--JFJJA7F<AJJJF7JJJFJFFFA<<FA-7F-JJJJJJFJJJ<FFFFJFJJJJJJFJJJAF-AA<\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:114T36\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1120:5629:28833\t4\t*\t0\t0\t*\t*\t0\t0\tCCCCGGAACCCAAAGACTTTGATTTCTCATAAGGTGCCAGCGGGGTCCTATTAGTAACACCCGCTGATCCCTGGTCGGCATCGTTTATGGTTGAGACTAGGACGGTATCTGATCGTCTTCGAGCCCCCAACTTTCGTTCTTGATTAATGAA\tAAFFFJJJJJJJFJJJJJJJJFJJJJFFJJJJJJJFJJJJJJJFJAFJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJ<JJJJFAAFJJJFFJJJJJJJJJFJJJJJJJJAFJJFJJJJJJJJJJJJJFJ<JJJJJJAJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1101:18426:1490\t4\t*\t0\t0\t*\t*\t0\t0\tTCACGATGAAATTTCCCAAGATTACCCGGGCCTGTCGGCCAAGGCTATAGACTCGTTGGATACATCAGTGTAGCGCGCGTGCGGCCCAGAACATCTAAGGGCATCACAGACCTGTTATTGCCTCAAACTTCCGTGGCCTAAACGGCCATAG\tAAFFFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJ-FJJJJJJJFJJJJJJJJJJFJJJJJJJJJAAJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJ<JJJJJJFFJJJJFJJJJJJFJJJJFJFJAF7AAF7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1124:24962:36763\t0\t4\t843453\t60\t151M\t*\t0\t0\tCGTTCATTATTGTCCCTAGGGATGGTTGCATCGATCATGTCAAACTGTGCCGAGAAGAAGAAGAGTGCCTCTCGGAACCGTGTCAAAAAGAACGGTGCACCATATGTTCCATTTACGACAGTGTGGACGAACACATTGGGCCGCATCTCGC\tAAAFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJJJFFJJFJJJJJJJFAJJJFJJJJJJJJJJJFJFJJJJJJJJJJJJJJJFJFJJJJAFFJJJJJJJJJJJJJJJJJJ7AJFJAJJFJFJJFFJJJJJJAJJFAJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1219:4198:33914\t0\t3\t132877930\t60\t134M\t*\t0\t0\tGGCGTTGCGGCGGCCGGAGTGGGCGGTGGGGACGCCTCAGAATCAGACATCGGTTGGGCGCGGAGGCGGCGACGGCGTCCGCAGCCAGGGTTAGGGTTTTGCTGCGTCGGGGAGGAGCGTGCGTGTGAGAGACT\tAAFFFJJJJJJFJJJJJJJJJJJJFJFJAJJJJJAJFF-AFJJJJFJJF-JJFFJJJFJJFJJJJJJJFJ--<AJJJJ<JJJJJJ--FAF-7AJJ<AFFJJJFFFF<-7AJJ-AJ7AJJAAAJJJ7-<<-A-FF\tAS:i:-6\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:85T21A26\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1208:22353:22327\t16\t2\t2810616\t60\t151M\t*\t0\t0\tGAATGTGCCGGACCCCAAATCGTTATCACCTAAATAACCAGTCAAGTCGCAAGTATGAATGGACTTTTGAGGACCATTGTGCCCTGTTGAACTCATGTATTACCTTGTGAAGTCCGTTGTGTCTGGTTTCTACCTGACCGTTTCGTTATCT\tJAFJJJJJJFJAFAFAFFJJFJJJJJJFJJJJJFJA7JJJJJJJJJJFJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFAFJJJJJFJJFAF<A\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1104:31527:23135\t4\t*\t0\t0\t*\t*\t0\t0\tAGGTCTTTACCACAACTTATAACATCTTGCTTGAACAAAGTCCATGACAATATTTCGTACAGGACAAAGTGGCAAGTCATAACCAACAGCTATGAAATATAACATGAGCACTTGGACTTGAACAGTCAACTGGAACAGACGAAGGACGACG\tAA<FFJJJJJJJFJFJFJJAJFJJJJJ<AJ7FFFFJJFJFJJJJJA-F7F-FFFJAF-FFAJJJF7FAJAJFF<7A<-<FFJ-AJFJAJJJ-FA7<7A<7-AF<<<FAJA<FFAFFFFA-<-<FAAF-F7<FF--AAA-<AF<-AAAFF))\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1128:13311:25333\t4\t*\t0\t0\t*\t*\t0\t0\tAGCATCCCATTTAGTCTGATCACGGAAATCGCATCTGGGTGCTCCTCCTTCAGATCATCAAGAACCATGTCGACCACCGCTGCTTTTCCACACCCGCGGGGAACCAAGAGGAGGACAGAATTGTTGCAGGCCTCGGCGACCGAGCTGGCAA\tA--F-F-A<FAJJJFJAAFAA-<FF-FAJFJF-FF<FAF7A-<F7F<F-<<FJJJAFF--FJJFFJFFJJJAFFA77<FF77F<FFJJJ<JJFJJJJAJF7F-7F7AA77-<A---77F<A<F<-7AFF7AFJJJ-----))F77)))-)-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1118:7537:36147\t4\t*\t0\t0\t*\t*\t0\t0\tGGAGCGACGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTACTAGGCATTCCTCGTTGAAGACCAACAATTGCAATGATCTATCCCCATCACGATAAAATTTCCCAAGATTACCCGGGCCTGTCGGC\tAAAFFJF7<<-FAJJJJ7A7A7<AFFJJ<JJA-7F-J<<A<<<7-F7--JAJFFFJFJJAJFAFJFA<JJFJAJF-7AJJ<J--7AJ--77<AFJFJ7FFAA-FAJJ-FAJFFFAAFFFFJFA<FFJFJJ-7A<<AJJJ))FF--F7<FJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2210:21775:39049\t16\t9\t22666812\t60\t1S55M98N93M2S\t*\t0\t0\tGCAGCACAGGATCCGCATCACGCTCTCCTCCAAGAGCGTCAAGAACCTCGAGAAAGTCTGCGGGGATCTGGTGAAGGGCGCCAAGGGTAAGGAGCTCAAGGTCAAGGGCCCCGTGCGGATGCCTACCAAGGTGCTCCACATCACCACCATG\t-7JJFJAJJJ<JJJJJJJJJJJJJJJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:148\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2110:11190:24401\t4\t*\t0\t0\t*\t*\t0\t0\tCGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTGCTAGGCATTCCTCGTTGAAGACCAAAAATTGCAATGATCTATCCCCGTCACGATGAAATTTCCCGAGATTACCAGGGCCTGTCGGCCAAGGCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ7FJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJFJJJJJJ<AJFJJJJJJJJJJJJJJJJJ77FJJJJFJFJJJJJJJJ7FFJJJJJJ--7FJJJJJJJFJJJF)-<7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1217:3914:15979\t16\t4\t69868208\t60\t151M\t*\t0\t0\tCCTGTGAGAATGCTTTTCTGTGTCAGCGTGGTTGTTTTTGCTCTAGTAAGTTGACGGATAATGATGGATATGCAATTATTTTTTTGGTGTTCCATGGCTCTAATACAGCCAGACTTCACTAGGAACTGTGTCGCTGTTTATGTTGGCAGTT\tJAFFJFFJJJJJJFFFJJJJFFFJJJJJJFFJJJJFJJJJJJFJJJJJFJJFFJJJJJJJJJJJAFFJFJJJF<JF<JJJJJFJJJJJJJJJJJJJJJFJJJJJFJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJFAJFFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1101:27072:1490\t16\t7\t156065148\t60\t151M\t*\t0\t0\tTGCCAGTTTCCTGCAGGCTTGAGCAAAGCTGAGTCTTGTATTTAGATCCAGAGAACCCCTGGTCATTTTTCAGAATGGCTGTTGAGAAAGAAAAACCTTGAAAATTCTTGTTCAGTGTATCCTATTGTTATTTGATTTCTGAAGCCGATGT\tAFA-AFJF<J<A<JJJJFFAJJF7J<-J<JFJJFFJFJJJAFJJFAJJJJJJJJJJJJAJAFJJJFJJJJFJJJFJJJJJJJJJJJJJJF-JJFFJJJJJJ<JJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJFJFJAFFAA\tAS:i:-4\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:23A127\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1213:31294:18230\t16\t1\t215380813\t60\t151M\t*\t0\t0\tCGGGTCGCGGGCGCGGGCGCCCGCCCAAGGCCAAGAGCCCGCTGGAAGCCGCCGTGAAGCAGGCCACCGCGGGGATGCCCAAGCCCCGCGGCCGCCCGCCCAAGAAGGCCAAGACCGAGGGCGCCGCGTCCCCGTCCCCGTCCCCTGCCCC\t----)F)---JA-AA--FAF<A<FF<<A-7FFFFA7-JJJJFJF<F7FJF<FJF---AAA7-FFJ77A<JJFA-<A-<AAJA-JA<-<A-7JFFFJFJF<JFJA-A-FJ<7JFJJFF-F-FA-FJJJJA<JJJA<JJFFFFJFFAA-FAAA\tAS:i:-13\tZS:i:-17\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:4G5C107C32\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2119:7994:25720\t0\t5\t63830273\t60\t126M\t*\t0\t0\tGCGCCCTTGGATTTGGCCCCCTTCATGGCGGATTCGAGCGGCGACACGGAGCGCAGCGTTCAGATCCGCGCTTGGCGGAGGGAATCCCGGTCGCGGGCGGACGAATCGGGGTCGAATCGAGGGCGA\tAAFFFFJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJFFJJJJJJF<JJJJJJJJJJJJFJFFJJJJJJJJJFJJJJJJJJJJJFJJJFJJJJJFJFJJJJAJJJJJJJJJJ<JJJJJJJFJJJJJF\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:126\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2118:13798:13869\t4\t*\t0\t0\t*\t*\t0\t0\tCAATTATGTTACGAGGCAATATTCTGACAAAACTTCACTTCAGTGCCGAGGAATTACAAAAAATCACTTCAGTGCCAAGGGTGGAACAAAACCTAACACGACCATAATTATAACATTGCAGCTAGACCACCCAGAGCTCAGAACCGTAATA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-<FJJJJJJJJJJJJJJJJJJJJJJJ-7FFJJJ-7FFJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2221:11525:23575\t16\t4\t18245370\t60\t151M\t*\t0\t0\tGTCGGCGGCGGCCGCGGCACCGCCGCCGCCGCCATCGTCGCCGCCTTCCCGCACATCCAGAGGTGCACCGTCATGGACCTCCCCCACGTCGTCGCCGAGGCTCCCGCCGGCACCGCCGGCCTGTCCTTCCACGGCGGCGACATGTTCGAGC\t7)<JFJJAFJFJJJJJJJF<JJFAJJJJJJFJJF<JFAJJJJJFJJJJJJJJJJJJJJFJ7JJJJJAJJJJJJ7FAAJJJJJJJJAJJ<JJAJJJJJJJJJJJJAJJJJJJJJJJJJJJFFFJJJJJJJJJJFFJJJJJJJJJJJJFFAAA\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:33G117\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2207:25844:37660\t0\t1\t10485420\t60\t151M\t*\t0\t0\tGCAGTACAGCAGGAGCATTATATGCACTAGTAAGTTTCATTCAGCCAATTAGACATGAGCAATAACAACAAGAAGAAACTTACCGGACCGAATGCCCTAGAATCCCAGCTACGAGTAGAGTTATCACCTTCAACCCAGCAATGCCCTTCTG\tAAFFFJJJJJAJAJJJJJAJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJAJJJJJJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJAJFJJJJJFJJFJJJJFA<AAFJ7JFAFFJJJJJJJJJFJJ<JF<<A\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1205:10551:17456\t0\tMt\t84957\t60\t148M3S\t*\t0\t0\tTCACTGAAATGTTCTAACATAGAGTAGAATCGAACATTGAAAAGATTAGTTGACAACAGTCAATCAGGCGCCAAATTACTAAACAATAGGGGTCTTTCTGTGCAAGTCAGCTGAACACCGTAATTGATGAGTGAGTAGGTGGGTTAGGAGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJFJJJFFJJJJJJJJJA\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:148\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2213:3752:5147\t0\t4\t187006043\t60\t91M\t*\t0\t0\tATATAAATTTATCAGGCTGGACTTTTTGCTCAAGCATCTCTTCAAGAAGGTTCTTAGCTGCAGAAAGCATCCGTTTCTTGCATAGGCCACT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:91\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2203:12347:26512\t16\t9\t28020872\t60\t150M1S\t*\t0\t0\tGAGGTCATCAGCCTCTGCAGCAAAACTGAACTATCATGTGCTCGCGGCTTTGGTCTCTTTGGTGGTAGCCTTGGTGTTGGGCCATTGACAAGAACTGCCTCCAGCTTGTGGTCAGCTGCTGTTTGCAGGTTGATTAGGCGTTTGACAGAAG\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1115:24972:32138\t16\t2\t104806028\t60\t118M1I32M\t*\t0\t0\tGGTACGAGCAGCAGCAGAACGCCCTGCTCGACTGCTCCAGCAGCCTCGATTTGGATGACTGTGTGGAGTGGCAAGTTCAGTAGGGAAGACGATCGTTTTAAAATTCTTGGTGGTACATAATCGATCTTGTTGAATCTTTGTGTGAAGCAAC\tJJJJJJJJJJFAFFFFJJJJJJJ<JJFJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJFJJ<JJJJJJJJJJJJJFJJJJJJJJJJJJJFFJJFJJFJJJFJJJJJJJJJFJJJJJFFFAA\tAS:i:-26\tXN:i:0\tXM:i:3\tXO:i:1\tXG:i:1\tNM:i:4\tMD:Z:49C60C24C14\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2221:12073:12708\t16\t2\t25233085\t60\t84M81N29M4850N38M\t*\t0\t0\tGGGCCCCTAAGCTCAATGAAAGGATCCTCTCTTCTCTGTCCAGGAGGTCCGTAGCTGCTCATCCATGGCATGATCTTGAGATCGGTCCTGATGCTCCTGCTGTTTTCAATGTTGTTGTTGAGATCACAAAGGGAAGCAAAGTTAAATATGA\tJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFF-<JJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2218:29102:35848\t4\t*\t0\t0\t*\t*\t0\t0\tCTCGTACGGGTCGGTGGGGATCACGGCCAGGATCTCGTCGGGAAGGTGGAAGTCTACGCCCGGCGCGGCCGCCGACGCCTCGTACCTCGCCATCGCCGGCGACGGGGCACGGGCGGGTACCTACCTGGGTGGGTCACTTCTTGGTGACGCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJJJFJJJJFJJJJJJJFF<FJAJJJJJJFJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJJJ<FJJJA<AFJAJJJ<A<A\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2104:22861:25246\t0\t4\t178091770\t60\t2S149M\t*\t0\t0\tGTCTCGTACACAGAACCAAAAGATCCACTGCCGAGATGGTGTCCCTTGGTCCAAGACGTGATAGTTCTCTTAAACCGCCTATTTGGCGACGGCGAGATCAAATACTCGGTGCTCTGAGAAGGGGACTCCGCTACAGCTTGAACCACGGCAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFFJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJFJJFJJJJJJJJJJJJF\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1116:4624:2826\t16\t1\t151367563\t60\t151M\t*\t0\t0\tTTAAAGGACATGTGTTTCAGAACGGATGTGTTGTTGCCTCGTACTATGGTGAAACATTGTCTCAGTTTTTGTGGTCGCTGTCAGGATTTTGCATTATGCCTAAAGATGTTTACTTTGGATATTTTTGTGTTAAAATGGTACTGTATTGGAT\tFA7JFAAJAJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJJJJAFFFJFJJJJ<JJJFJJJJJFJJFJFJJJJJAA-JFFJJJAJJJJJJJJJJJJJJJFFJFJJJJJJ<JJJJJJJJJJJFJJJJJJJJJAFAJJJJJJFJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1124:11109:29501\t16\t1\t269103065\t60\t23M97N127M1S\t*\t0\t0\tCATCGCTCAGCACCTTCCAGCAGATGTGGATTGCCAAGGCTGAGTACGACGAGTCTGGCCCGTCCATCGTGCACAGGAAATGCTTCTAATTCTTTGGGCCCAAGAGATGCAAAGCCGAGAGGAGCCATTATCGCCAGCCTCCCGCCCCGTA\tFFJJJJFJJJJJFJAJFJFJJJJJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1227:14438:16243\t0\t9\t73931671\t1\t1S150M\t*\t0\t0\tCGCGCTTTTGAGCTTTCTTCAGGATTTGGCCCGAACTGTTCGGCAGATTCCCACGCGTTACGCACCCGTTCGCCACTTTGTTCTCAACTATTCCGATTCCAGCAAACGGAGGCCGTTAGGTCAAAGCCGTTGCGCGCGCCCTGCAGGCAGG\tAAFFFJJFFJAFJAJJJFJJFFJFJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJFFJJJJJJJ<JJJFJJFJJJJJJ-<AJJJJAFFJJFFJJJJ-JJJ\tAS:i:-1\tZS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1227:14438:16243\t256\tMt\t421306\t1\t1S150M\t*\t0\t0\tCGCGCTTTTGAGCTTTCTTCAGGATTTGGCCCGAACTGTTCGGCAGATTCCCACGCGTTACGCACCCGTTCGCCACTTTGTTCTCAACTATTCCGATTCCAGCAAACGGAGGCCGTTAGGTCAAAGCCGTTGCGCGCGCCCTGCAGGCAGG\tAAFFFJJFFJAFJAJJJFJJFFJFJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJFFJJJJJJJ<JJJFJJFJJJJJJ-<AJJJJAFFJJFFJJJJ-JJJ\tAS:i:-1\tZS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2228:10196:46416\t16\t7\t20176819\t60\t149M2S\t*\t0\t0\tGAGGAAGATGGGGCTCGCGTTCGGGAAGCTCTTCAGCCGGCTCTTCGCCAAGAAGGAGATGCGGATCCTCATGGTCGGCCTCGACGCCGCCGGTAAAACCACCATCCTCTACAAGCTCAAGCTCGGCGAGATCGTCACCACCATCCCCAAC\tJJJJJJJF-JJJJF<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJFJJJJAJJJJFFFJJJJJJJJJJJJFFJJJJFJJFJJJJJJJJJJJJFJJJJJFJJFFJFJJJ<JJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2125:16630:6501\t0\t7\t9263016\t60\t151M\t*\t0\t0\tTGCCTCCTAGGTTGACTGGATCGCAGACGCTAAACCTGCGATCTAGCCTTGGCTGCGGGAACATCAAGTCCCCGTCCATTCATAAGCTACGGCTGAGATTGTGGAACGACCGGCGGTGGAAGAATGAGAAGTTCTCCTTCCTTCTCGCAGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-29\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:31G49T30A20G4G12\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1220:7486:46856\t0\t6\t161922365\t60\t50M335N101M\t*\t0\t0\tGCTAATCTTGTCAAGCTCCTTAGTGGCATAAAGAAGGGCAACTCCACCACCTGACACAATGCCCTCCTCCACAGCAGCCCTTGCAGCATTTAGAGCATCTGTCACCCTATCTTTTTTCTCACCAACTTCAGCTTCACTAGCTCCACCAATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:21G31G97\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1216:25641:8787\t4\t*\t0\t0\t*\t*\t0\t0\tGCGACGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTACTAGGCATTCCTCGTTGAAGACCAACAATTGCAATGATCTATCCCCATCACGATGAAATTTCCCAAGATTACCCGGGCCTGTCGGCCAA\tAAFFFJJJJJJJJJJJJJJJJJFFJJJAFAFJFJFJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJAAAJJJJJFFJJFJJJJJ<JJJJJJJJJJFJJJJJJJFFFJJJJJJJJ<JFJJJJJJJJFJAAFJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1228:31690:33158\t16\t8\t150980463\t60\t150M1S\t*\t0\t0\tGGAGTGTAAATTAATTTAAGTTACACTTCAATCTATCTCAATACATGTGGATTTGGGAGTTTTAGACCCGTATCGTAAGTTGTATAGCTTCACCTGGTTGCATCGTGGATGACCTGTTATTGTTTATTACTGTTATCCAAATATCTTGCCC\t-7-AJ<-A<7JJFJAA<FA7A<-7AF-AFF77-7-AJJFJAJJJFJFFJA--J<-<-JJJJJFJA-JJA-<JJFA<<-<F<JJJJJJFJJF7FFJ<F-A<JFFFFAJJJFJJJJJJJJJJJJFFJF-<<7JJJJJJJJJJJJJFJJFAA-A\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:8491:31435\t4\t*\t0\t0\t*\t*\t0\t0\tGTCCTCGGGCATGATGGTGACGCGCTTGGCGTGGATGGCGCAGAGGTTGGTGTCCTCGAACAAGCCGACGGGGTAGGCCTCGGCGGCCTCCTGCAGCGCGGCGACGGCGGAGGACTGGAAGCGGAGGTCGGGCTTGAAGTCCTGCGCGATC\tAAFFFJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-<JJJJJJ-7FJJJFJJJJJJJJJJJJJJJJJJJJJJJJ-FFJJJJJJJJJJJJJJJJJJJAFJJ7JFA-AAJJJJJFJJJA<JJJJAA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1215:3254:45994\t4\t*\t0\t0\t*\t*\t0\t0\tCTCCCGTGTCAGGATTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJF7JJJJJ7AJJFFJJ<<FJ-<<A-<7FFJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1221:15574:44166\t16\t2\t189435391\t60\t151M\t*\t0\t0\tTCCTCAAATCCAACTAAGCACTCTATGGATGGGGCATTTGCTGCACGTCTTGTCCGGTCAACGCACTAAGATTATAGGAGTTGCCACAAGGCCACAACTAGGTGTGAAGTTGAACAAGGAATCACATGTATTGCCCTAAGTGCACCCAGTC\tAJFAAJJJAJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2104:23470:26687\t16\t7\t171674989\t60\t124M1S\t*\t0\t0\tGGCTCAGCTCGTCAAGTTTGTGCTGGGATGCCAGTTTATCCCTTCTAGTCTATTACAGAAATGTTGTTCGGAGCTGAGTCTATTATAAACATCTTGGTCTGAGTTTATTGTTGAGTGGTGCCAGA\t<<AFJJJFFFJJJJFJJJFAFJ<JFJJJJJ<<-AJJJJJJJJJA7-<JJJJFF<7AAFJJFJFJFFJJ7AA-<F--AJAAJJJJJFJJFAF-7<JFFJJFFFJJJJJAF7<<FAJJFAJFFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:124\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2206:10744:36323\t16\t1\t174373626\t0\t64M3277N87M\t*\t0\t0\tGTGGGTTCACCAGCCGATTGTTTGCAAGGAAGGGCCACGGTTTTTTGAAAATGATATATTGGAGTTCTTAACACGCACGAAATGGTGGGCAGTTCCTCTTATCTGGTTGCCTGTTGTATGTTGGTGCCTGAGTACTTCTATCAAGATGAGC\tJJJJJJJJAAJ<FJJJJJJA7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-17\tZS:i:-17\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:39A72A35G2\tYT:Z:UU\tXS:A:+\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2206:10744:36323\t272\t1\t174376826\t0\t64M77N87M\t*\t0\t0\tGTGGGTTCACCAGCCGATTGTTTGCAAGGAAGGGCCACGGTTTTTTGAAAATGATATATTGGAGTTCTTAACACGCACGAAATGGTGGGCAGTTCCTCTTATCTGGTTGCCTGTTGTATGTTGGTGCCTGAGTACTTCTATCAAGATGAGC\tJJJJJJJJAAJ<FJJJJJJA7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-17\tZS:i:-17\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:39A72A35G2\tYT:Z:UU\tXS:A:+\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2201:19532:33229\t16\t10\t95619843\t60\t83M13749N68M\t*\t0\t0\tAAAGGGGAAGAAGCCATGGCTCCCATCGCATCTCCAACTGGAGGAGCCGTCCTTGTCAGAGAACTCGCAAATTTGAGCCCCACGCAAGTGTTTGTCGGGGACGTTGTGTTGTTAAAAGACCCAGAGAAATCTGATGATTTAATTATTCGAC\tJJJJJJJJJJJJJJJJJJJFJFJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF<<-FAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFAAA\tAS:i:-4\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:60A90\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2120:16092:19460\t16\t6\t82660873\t60\t150M1S\t*\t0\t0\tGCCAAGCGGTCCACGGCGGAAGTTCCCCCGCGCTCGCATCCGACGATGCCATCGCCACAGTAGACCAGTTCGAGGCTGCCGTCGGGATGTCCTCCTCTCAGACCATCCTCAGGGTCCTGTGGGGCAACCGGGCCGCCGTGGAGGAGCGGGC\tJJJJ<-7)JFAF-JJFFJ<FJJFJJFA<<<JJFF<FJFJ7FJFJFAFFAF<FF<J<FFFJF7-<FJJJJFF-JF7F<-A<-F<FFFJJFFAFFA7F<77FFFJFF<F<JA-<JFJAJJJA7JF-AJJJJJA<JA7FAAJJJJJFJJF<A-A\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1121:25195:37800\t0\t8\t166053224\t60\t151M\t*\t0\t0\tGATTCGACAGTAACCCAAACTACAGAACCCAGCACATCTCTACATCTTCTGATCGGGCACTGATGACAGAATATATGATGCTTAGGCCCTCTCCCCACGGATGCGGCGGGCAAGCTGGATGTCCTTTGGCATGATGGTGACGCGCTTGGCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAFJJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFJJJJFJJJJJFFJFAFJJFFJJJJ<FJJ\tAS:i:-30\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:34T36A6C17G29G24\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1214:13362:24155\t16\t8\t173836190\t60\t13M291N138M\t*\t0\t0\tGAGGAGAAAGCTGCCAAGGAGCAGAGGAAGTCCGAAGCACGGAAGTAACTGCAACAGTCATGGACCAAGAGTCCACGAAGTGACGAAGGCACGATCTGCGTCGATTGGATATTTGTTTCTAATGCACCAGCTACCTTTTCATAGACCCGGC\tJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1111:12317:27936\t16\t8\t65265489\t60\t151M\t*\t0\t0\tCATTGTCTCCGGATCCTGGGACCGCACCGTCAAGGTCTGGAACCTTACCAACTGCAAGCTGCGCTGCACTCTGGATGGGCACGGTGGCTATGTTAACGCCGGCGCTGTGAGCCCCGACGGGTCGCTGTGCGCCTCCGGCGGGAAGGACGGT\t<-FJFFJJJJJJAJJJJFAAAAJF<JAFJJJFJAJJJJFJJFF<JFJFFJFFFFFJJJJJJJJJFJFJJFJJJJ<JJJJJJJJJJJJFJJFJJ7J<JJF<F-JJFFJFJJJJJJFA<FA-JJJJFJJFFJF7-FJJJJ<JJJJJJJFFFA-\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:101T49\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2116:22962:25316\t16\t6\t20571884\t60\t151M\t*\t0\t0\tGTAGTTGGACCTTGGGCTGGGCCGGCCGGTCCGCCTCACGGCGAGAACCGACCGGCTCGACCCTTCTGCCGGCGATGCGCTCCTGGCCTTAACTGGCCGGGTCGTGCCTCCGGCGCCGTTACTTTGAAGAAATTAGAGTGCTCAAAGCAAG\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:-22\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:23937:35339\t16\t5\t14605545\t60\t151M\t*\t0\t0\tCAGCTCAGAGTCCAGGTCCCGTCCGGCAGGGCCGTCGAGCTGCTCTCCTCCCCGATCGAGTTCGGCGCGAAGAACAGGGACGCGCAGCTGGCGGCGCTGTCGCCCGCCAGGAGGCTCCTGCCGGAGGTCGTGAGCTCGCCGTCCGCCCGGG\t7JFJJJFJ<AFJJJA7)-AJJJJJJJJFJFFA7F-FFAJA777--AA7JJF7---J7A-FFFAJJJFJF<FJF-AFJJJA<JF<A-JJJJJFAFFJFFJJFAAJFJJAJJJJJJJJJJJJJJJJJJJJJJJJJFFFFJJFFJFJFJAFFAA\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:68C82\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2122:15473:10581\t4\t*\t0\t0\t*\t*\t0\t0\tGTCGTTCGTTAACGGAATTAACCAGACAAATCGCTCCTCCAACTAAGAACGGCCATGCACCACCACCCATAGAATCAAGAAAGAGCTCTCAGTCTGTCACTCCTTGCTATGTCTGGACCTGGTAAGTGTCCCCGTGTTGAGTCAAATTAAG\tAAFFFJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJ-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJFFJJFJJJJJJJJJJJJJJJJJJJJJJJ-FFJJJJJJJJJJJJJJJJJJJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1108:31061:43427\t0\t2\t41932024\t60\t1S149M1S\t*\t0\t0\tTTTGATATAGGACCACTCACAATAAACAGCGATGGAAAAGGTAAAATGCAAGTGACAACAGTTTAAAGGGCTTCCCAACACAACCAGACCGGGCTCCAGCGCCCCCCGCCACAAACGAGGTGGGACACCGAGGCCTTGCCCGGCAGCATCT\tAAFFFJJJJJJJFAFJJJJJJJJJJJJ<FJFJFJJJJJJJJFJJFJJJFJF7<JFJJJJJJFAFJJJFJ<F7AAAJJ7F--FJJJJFFJJA<JJ<<JFFJJFAAJFJJ-FJ<<JJJJJFA<JFJAFJJFJJ7A7AJ<A7FFFFJFJJJJJ-\tAS:i:-21\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:78C25A7G26A9\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2214:2808:28481\t16\t3\t128257712\t60\t151M\t*\t0\t0\tTTCAGAACTGCACCGATATGAATCAGCCTGTTGAACTTGGCAAGGAAGATACACAGTTAAAATCCAATGTAGATGCTGCTGCTGAGAATAGCTCAGCTCCAGCTCAAACTGTTCCACTATCTGCCGGTGGTGCAAAAGAACATAGTGCGGC\tJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2204:30685:34969\t16\t4\t151563684\t60\t72M301N48M105N30M1S\t*\t0\t0\tGACTACCTTTTCAAACTTCTGCTTATTGGTGATTCTGGTGTTGGGAAATCATGCTTGCTTCTCAGATTTGCAGACGATTCTTATTTGGACAGCTATATCAGCACAATTGGGGTTGATTTCAAAATTCGCACAGTGGAGCAAGATGGGAAGG\t-F<-FJFFFAJJJJFJFAJAFJFF<7FJJJJJJJFFJAJJJJFJFJJJJJJJJJJJFJJFJFJJFAFJJJJJJJAJFFJFF<-FJ7<<JJJJJFJAJJJJJJJFFJJJJJJJJJJJJJFJJJFFJJJJJJJJJJJ7JJJFF-FA<JFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2112:21369:11548\t16\t10\t116392848\t60\t151M\t*\t0\t0\tCTCCTCCTCTCCGCTCCGAATGGCCCGCACGAAGCAGACGGCGCGCAAGTCGACCGGCGGCAAGGCGCCGCGGAAGCAGCTGGCCACCAAGGCGGCGCGCAAGTCGGCCCCGGCCACGGGCGGCGTGAAGAAGCCGCACCGCTTCCGCCCC\tF7A7-JA-AAJJJJFAFAJJFJJJJJJJJJFJAJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJFJFFFAA\tAS:i:0\tZS:i:-10\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:9932:37589\t16\t9\t137122034\t60\t147M4S\t*\t0\t0\tGCCCTAGTTGTCTTGGATGTAATTCTGTTAATTTTGTTCAATGTTTGTCGAACTTGCCGTGTGACCCATACTATATCTGCTTTGGGATTTAGTAAGCCTGGATTGCAGTTTTTTTTATGTTCCTGGTGGAAATTGCAATTGTTTGTCCGCG\tA<J<<-JFFAAJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJFAAFAA\tAS:i:-8\tZS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:5T141\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1210:7811:13341\t16\t2\t227373319\t60\t151M\t*\t0\t0\tAAAGTGTTGAAGCTGCATTGATGGAGATCAACCGGATACTTGGATTCAAGCTGGTGCTGGAGTGCTTAGAACTAAGGTTCTGTTGCTACCTTGCCCTTGAGGATTATAGGGCTGCCTTATGTGACGTGCAGGCAATACTCACTCTTGCCCC\tJJJJJJJJJJFJFFJJJJJJJFJJ<JJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2227:23531:12796\t16\t10\t17615419\t60\t151M\t*\t0\t0\tCCATATGTATTCGGTAGTGTTTTCTGTTAACATGATAGGTGTCTGATGTCTAAATAATGGAACGAGTCCTTTTGTTGAGTTTGAGTTGTGGAACTTTCAAAGCGTGCGTTGCATGCTGGCGCTTGTGCCGGCAAACATGGCATACTTTCAT\tJAJJJJJJJJJJJJAJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ7JJJJJJJJJFAJJAFJFJJJJJJJJAJJJF<JJFFJFJAFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFF<AA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1121:6299:24366\t0\t10\t97215327\t60\t51M1426N38M\t*\t0\t0\tGTCGCTTTAATTGCAACGTCTTGATGTCCACATTATCAAGTAAAACTTGCCCTGGTGGGTGCTCGTCGCCGGCGACGGGCCCAGCGAAG\tAAFFFFFJFJJJJJJJJJJJFFJFJFJJJFFJJJJJJFJJFJFFJJJJFJJFJJJF<AAJJJJJJJJJJJFFJJJJFFJJJJJJJJJJJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:65G23\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2226:29447:11231\t16\t6\t113525866\t60\t14M76N84M122N53M\t*\t0\t0\tTGGAAAAGATGATGTTTGACCAGCGCCAGAAGCAGATGGGCCTTCCAACAAGTGATGAAATGCAAAAGCAAGAAATTCTCAAGAAGTTCATGTCTGAGCACCCGGAGATGGACTTCTCTAGAGCGAAATTAGATTGAGGATGTGTTTGGCC\t-77JAFA-JJJJF-FFJFFJJJFF-JF7-JFAFFAFJJF<FA7FJJJJFFJJF7FJJJJFFFJJJFFJAJJJJJFJA7JJFJJFJJJJJJJJJJJJJJJJJJJJFAFJJJJJJJJJFFFJJJJJJJJJJJJJJJFJJJJJJJJJJJFFAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2201:2595:39189\t0\t6\t31869156\t60\t151M\t*\t0\t0\tGCTCATCAGTGTTCCCGCCGGCCCTGTCGGCGAGATACCTTCTCCTTATGACGCCCCTCGCCATGGCAAAGACCCCGGTACCGCCGTAGATCGCCCACTCGTCAGACCCGGCCGTTATCACGCCATTGGTGACGAGCGTGGAGCCCTTGAG\tAAAFFAAJJFJJJJJJJJJJAFJJJJJ<AAFAJ<<FJFAFFFAJ7FAJFJJJ<AFJJFJJJJJJJFJJJJFFJF-AAAAJJFFFJJJJ<JFAAAAJJJFJFJJJJJJ<JJJF<-7FJAFJJJJFJ7--<--F<AF<JFAJFJ-)-7<AFJ-\tAS:i:-11\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:70C8T71\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2101:23825:23048\t0\t9\t25284769\t60\t10M2469N141M\t*\t0\t0\tTGAATTTCTCCTTGCCGCCCAAGAGGAAGATACCGGCGAGGCCGGCGGTGGCGACGCCGCCCCAAATGACGACTGTGGACGAGAGCAGCGACCAGTTGCGGTAGATCATTTTCGCCTTCCTCGGGTTCCTCTCCCGTCCCTTGCTTCGCAA\tAAFFFJJJ<AFJJJJJFJJJJJ<-<J<A7FJJJFJJJJ<7AJJJJJJJJFJJJAJFJJJJJJFFJJFJJJJFJJJJFAAJJ-FJJJFJJJJJJJJJFJJJJJJJFAFFJJJFJJJJAJFJJJJJJJJJ)FFFAJJJJJJJFJJJJJJ7A-A\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1211:7314:47682\t16\t2\t2271650\t60\t146M5S\t*\t0\t0\tGATGGATGCGGTTTCGGGCGGATTCGCGAGGAGGCTGTAGAGGTTAGGTTCCGGTGTCCCCCGGCCCGGCCGGTCGTGGGAGATGAGCTCCTCGTCCGCGGCCAGCATCGGGCAGCCGCCGCCGCCCGCGCCGCCGCCGGAAGAAGAGAAG\tFJJJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-15\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:27T118\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2118:11566:1279\t16\t4\t99330587\t60\t101M\t*\t0\t0\tAAGCCGATACCCTAACCCTCGCCCTTCTCGATGTCGGCCGCCACCACCGGCCCAGCCCCTTCAGTAGCCACTCCGATGGCGCCCCCTCCGTCCTACCCGGC\tJJJJJJJJJJJJFAAJJJAJJJJFF<JJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJF<<JJJFJJJJJJFJAAF<JJJJJJJJJFJJJJJJFJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:101\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2114:8643:15873\t16\t4\t69867282\t60\t33M448N103M112N15M\t*\t0\t0\tGCTGCGCAGGGTGGAGCCATATGTTGCGTACGGATACCCCAACCTGAAGAGCGTTAGGGAGTTAATCTACAATAGAGGTTACGGGAAGCTGAACAAGCAGAGGATCCCACTTGGCAACAACAGTGTCATAGAGGAGAGCCTCGGGAAGCAC\tJJJJJJJJJJJJJFFJFJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJF<-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF7-FFJJFF-JJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJFFFAA\tAS:i:-14\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:72G32T6A23G14\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:8176:22028\t0\t4\t236627019\t60\t4S147M\t*\t0\t0\tCCCCCCTCATTCAGGGGCTCCCACATCAACATATCATCAAGTCGACAGACCGTACAACGACTCATCAATCAACAATTCCGCATATAACCATGCCAGGGTACTACGGTGCAAATCTGCCCCCTTCTACAACTAAAATACAGGCCATCGATCT\tAAF<FJJJJJJJJJJJJJJJJJJJFJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJFJJJJJ<FJJJJJJJJJJJJJJJFFJJJJJJJJ-AJJJAFF<AA\tAS:i:-25\tZS:i:-27\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:43A65T5A25G5\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1205:12520:31136\t0\t7\t18156056\t60\t1S51M1207N99M\t*\t0\t0\tTGACTTCCTTGCTTCCTCCTTCCCTCCATAGTATCCATGATATTTCTCATACCCATCGACGGCCGACTTGACTTCCACCTTGTTGATCTTCCCTCCGAGGCCAGAAGCAAGATCCAGCGCTCCCGACTTGGACATCGTGTAGGCGGCCGGG\tA<-F<<7--AFJJJJJJJJJJJ<J7F<AFA<<A-<AF-<FJJJJJJFAJF-7-<<JAJAFJJJJ<AJJJJJJFJJJJFJ<FAAFFJ-JAJ<<777FF<F--777<A<<F-FA-FJJ<A<J-F--AJ7JJFF-7FAFJFFA--<JFFA<FAJ\tAS:i:-4\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:51T98\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1208:6055:25527\t16\t3\t165766800\t60\t151M\t*\t0\t0\tCACCCCCGGCAAGGGTATCCTTGCTGCTGATGAGTCCACTGGCACCATTGGCAAGCGCCTTTCCAGCATCAATGTCGAGAACGTTGAGGAGAACCGCCGTGCCCTCCGTGAGCTCCTATTCTGCTGCCCTGGTGCTCTCCAGTACATCAGC\tJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2117:24799:39471\t4\t*\t0\t0\t*\t*\t0\t0\tGCTCCCACAAGTCTAGGCTGATGATTGGAATGTCACCAAAAGCAAGTGGGGAGATAGCATTTCGTGTATGAACAACCGAAAGCTTTCTCTCATTTCTCTTCAAGACAAGCCAAACCCAACCAGACCCTAACAGCTGTAAGGCTGAGCGGAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJ<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1112:28919:12251\t4\t*\t0\t0\t*\t*\t0\t0\tGGTCCCTCTAAGAAGCTAACTACGGAGGGATGGCTCCGCATAGCTAGTTAGCAGGCTGAGGTCTCGTTCGTTAACGGAATTAACCAGACAAATCGCTCCACCAACTAAGAACGGCCATGCACCACCACCCATAGAATCAAGAAAGAGCTCT\tAAAFFFFFFJJJJJJJJJJJJJJJJJFJFJJJJJF7FFFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJ7FJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJ<JJJJJFJJJJJJJJJJJJJJJJAJJJJJJF<F7\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2206:5863:25615\t16\tscaffold_304\t263\t60\t95M131N56M\t*\t0\t0\tACGACCATTATCAATCACCCTGCTGAGATTGTGAATATGCTTGCCAAGAGCCTGCCTGGAAGCGCAACGTTCTTCCTCACGTTTGTTGCACTGAAGTTCTTGGTTGGCTATGGCCTTGAGCTCTCTCGCCTGGTTCCTCTCATCATTTTCC\t7JJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJJFJFJJJJJJAJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:86C2G61\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2222:7019:13974\t16\t1\t269102744\t60\t125M\t*\t0\t0\tACGAGCTGCCTGACGGACAGGTCATCACCATTGGTGCTGAGCGCTTCCGCTGCCCTGAGGTCCTCTTCCAGCCATCCTTCATTGGGATGGAAGCTGCTGGTATCCACGAGACCACCTACAACTCC\tJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:125\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2227:17695:11442\t4\t*\t0\t0\t*\t*\t0\t0\tTCTCCTTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCATAGCCCGCGTCAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJ<JJFAJ-FFFJJJFJJJJJ<<-)-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1207:7527:36903\t16\tPt\t57087\t1\t151M\t*\t0\t0\tGATGGACTTACCAGTCTTGATCGTTACAAAGGACGATGCTATCACATCGAGCCCGTTCCTGGGGACCCAGATCAATATATCTGTTATGTAGCTTATCCATTAGACCTATTTGAAGAGGGTTCTGTTACTAACATGTTTACTTCCATTGTGG\tJJJJJFJJJJJJJJJJJJJFJJJAJJFJJJJJJJJJJJJJJJJJJJAJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:5\n",
      "K00364:134:H2YYHBBXY:1:1207:7527:36903\t256\t1\t207559909\t1\t151M\t*\t0\t0\tCCACAATGGAAGTAAACATGTTAGTAACAGAACCCTCTTCAAATAGGTCTAATGGATAAGCTACATAACAGATATATTGATCTGGGTCCCCAGGAACGGGCTCGATGTGATAGCATCGTCCTTTGTAACGATCAAGACTGGTAAGTCCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJJJJJJJJJJJJJJJJJJFJJAJJJFJJJJJJJJJJJJJFJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:5\n",
      "K00364:134:H2YYHBBXY:1:1207:7527:36903\t256\t1\t203966915\t1\t151M\t*\t0\t0\tCCACAATGGAAGTAAACATGTTAGTAACAGAACCCTCTTCAAATAGGTCTAATGGATAAGCTACATAACAGATATATTGATCTGGGTCCCCAGGAACGGGCTCGATGTGATAGCATCGTCCTTTGTAACGATCAAGACTGGTAAGTCCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJJJJJJJJJJJJJJJJJJFJJAJJJFJJJJJJJJJJJJJFJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:5\n",
      "K00364:134:H2YYHBBXY:1:1207:7527:36903\t256\t2\t200756170\t1\t151M\t*\t0\t0\tCCACAATGGAAGTAAACATGTTAGTAACAGAACCCTCTTCAAATAGGTCTAATGGATAAGCTACATAACAGATATATTGATCTGGGTCCCCAGGAACGGGCTCGATGTGATAGCATCGTCCTTTGTAACGATCAAGACTGGTAAGTCCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJJJJJJJJJJJJJJJJJJFJJAJJJFJJJJJJJJJJJJJFJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:5\n",
      "K00364:134:H2YYHBBXY:1:1207:7527:36903\t256\t4\t156748057\t1\t151M\t*\t0\t0\tCCACAATGGAAGTAAACATGTTAGTAACAGAACCCTCTTCAAATAGGTCTAATGGATAAGCTACATAACAGATATATTGATCTGGGTCCCCAGGAACGGGCTCGATGTGATAGCATCGTCCTTTGTAACGATCAAGACTGGTAAGTCCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJJJJJJJJJJJJJJJJJJFJJAJJJFJJJJJJJJJJJJJFJJJJJ\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:5\n",
      "K00364:134:H2YYHBBXY:1:1215:23328:17544\t0\t1\t178504520\t60\t92M1D59M\t*\t0\t0\tGCCGTAGGTGCTGAAGGCGTTGTGGAGGGAGTGGTCGTCCGTGGCCCAGGCGAGGCCGCCGACGAAGCAGCGGTACTCAACATCCGACGCCGCATCTCGATCCGCTCACCCACGAAACCGAACCCCGGAGAACCTCACGAACCCGAACCCG\t<AFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJ<JJJJJFJJJ-FJJJAJJFJJAJJJJJ\tAS:i:-14\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:1\tNM:i:2\tMD:Z:92^C24C34\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2222:25337:35303\t0\t1\t229918881\t60\t151M\t*\t0\t0\tCATGGATTTATGGCCCTGAGGGAACTAACAGGCAGCTACTATGGGAACACAAAATGTGAAGTGAAACTTATGTTAATACTCAAGTCTGACAGACATGTGCAAGTACGGGCCAGCAATATATAGTCCAGACAGAGAGTTAATGTTTTTTTAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJFJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJAJJJJJJFJ<FFFAJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2224:22140:48860\t4\t*\t0\t0\t*\t*\t0\t0\tNAGCGACGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTACTAGGCATTCCTCGTTGAAGACCAACAATTGCAATGATCTATCCCCATCACGATGAAATTTCCCAAGATTACCCGGGCCTGTCGGCC\t#AAFFJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJFJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2204:12631:40789\t16\t8\t134065023\t60\t60M112N91M\t*\t0\t0\tAGGAAGTACAGTGTCTGGATAGGAGGGTCGATCCTAGCCTCGCTCAGCACTTTCCAACAGATGTGGATATCGAAGGAGGAGTACGACGAGTCTGGCCCGGCAATTGTTCACCGGAAGTGCTTCTAGGGTTTCACAAGTTCCTCGTCGTTAC\tFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2109:26189:22151\t16\t8\t86268751\t60\t151M\t*\t0\t0\tGGATGGGATAGTAAGTTCAGGACTCGATTTTATGTTTCACCAACTGTTTCAGACAGTTTTAGGAAAGTCAAAGATTTAATTAAGATCTTTCAAATATGTAAACTTATATTTCTCTATGGAAATTGTCTTCTTTCCATTGAGATGTTTGCCC\tJJJJJJJJJJJJJJJJJFFJFJJJ7FJJFFFJJJJJJJJJFJJJJJJJJJJJAJFJ-JJJJJJJJ<JJJJJJJJJJAJAJF7A<AJJFFJJJJJJFJFJFJJJJJJJJJJJJJJJJJJJFJJJJJJFJJFJJJ7JJJJJJJJJJJJFF<AA\tAS:i:-18\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:64G5G54C25\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2216:14864:12023\t16\t3\t220592626\t60\t19M437N50M84N82M\t*\t0\t0\tGGAGGAGAGGCTGGCCCAGGGCCCAAGAGAACCAACAGCACCTGTCGCAGCCCCAGCTCCATCTACTGGGGTGCCAAAGAAAGCAAAGAAGTGAAGTATGGAAACAAAGCTGTAGCTTTTTGGTACTATATAAGCATGACTTTGGAGCACC\tJJJJA<JJJFJJJJJJJJJFJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2112:18375:38820\t4\t*\t0\t0\t*\t*\t0\t0\tGATAACTTGACATCTTCGGCACACTGGACAGCAACGCGGAACATCCGGGCGGCAGCGCTCTTGGTGGCGCCACGCTTCTTACCCGAGATGTCCTCCTTGACGCAGTTGAGGAGCGCCAGGACGAGGAGTGGCCACGTGGTGGCTCCGGTAA\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJFFJJJFFJFJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1220:29477:18529\t4\t*\t0\t0\t*\t*\t0\t0\tTTCCAATTACCAGACACTAACGCGCCCGGTATTGTTCTTTATTGTCACTACCTCCCCGTGTCCGGCTTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCC\tAAAFFJJJJFJFJJ-JJAFFFFJJJJJJJJJFJJJJ-<FJJFJJJJJJJ-<-FJJF<FFFJA-7--7AAJJJ7JFJFJFJAJFFFJJJJJJJJJFFJJJJJFJ<JJJFJJFAJFJJ7AJFJJJ-A-7FFJFF-7<FAF<7)FAF-A<F7F<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2219:19421:42214\t4\t*\t0\t0\t*\t*\t0\t0\tGCGACTACATCCGCGCACCAACACAGAGCTCGGAGCTCCACCAAGAGATCTAGCTTGTGGCCCCAACACGCAACGTCTCCACTCTCCACAAGGTTCCTCGCAACTAAAGGTAATTCAAATTTCCAGCCCGAATCCGCAACTATTCACCACA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJAJJJJFJJFJ-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2124:5213:11337\t16\t6\t127557229\t60\t48M322N103M\t*\t0\t0\tCGCACTGCGCCCAAGTCTGACGATGTCTACCTCAAGCTCCTCGTCAAGCTCTACCGTTTCTTGGTCAGGAGGACCAAGAGCAATTTCAACGCTGTCATTCTCAAGAGGCTTTTCATGAGTAAAACCAACCGACCACCAATCTCCACGCGCC\tFJJJJJJJJJJAJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJ-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAF-JJJJJJJJJJJJJJFFFAA\tAS:i:-6\tZS:i:-11\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:145T5\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2223:10754:43304\t0\t5\t210918436\t60\t3S148M\t*\t0\t0\tGCTCGCCGAGGACGGGTAGGACGACGCGAGCGCCGTGAGCGCGATGGGCGACGACACCTTGACGGCCCCGTCGAGGCCGAGCCTGGCCAGCGCGGCGTGCACGTTGGCCATGGCCGGGACCAGCTGCGCCGTCACGTTCTTGGCCGACGCG\tAAAFFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJAJAFJJJJJJJJJJJAF)\tAS:i:-9\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:75C72\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2126:13809:31962\t16\t2\t219691628\t60\t149M2S\t*\t0\t0\tGCCGAGCTCCCGCGCCACGAGTCGTCGCCGTCCACCTCCTCCTCCTCCTCGTCGTCGTCGGCGGTGGAACCGCCACCACCGCAGGCCGGCGAGGAGAGCAGCCCCGACGACGGGAAGGAGCCGCCGGCCAACTGCTACAAGCTGTTCCCAG\t<JJF<JJJJJJJJJJFJJJJJJJJJJJJJJJJFAFFFJJJJFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1223:26971:43234\t0\t5\t39652244\t60\t133M\t*\t0\t0\tCTCGAAATAACAAATGAATAAACAAATAAATAGGATCCGAAAAAAGAGGAGGAAAACACCCAACACACACAGCAGAGAAAGAGTGCAGCTCTAGCTCCTCCACGGATCCACCGACATATCAGATGTCCCAAGG\tAAFFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:133\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1211:30147:29360\t16\t8\t39133656\t60\t7S144M\t*\t0\t0\tTTTCGGTAAGGAAACTGGGAAGTCTTGCTAGGTGATAATCCTTTTTGCTTTGTTTTGTTTTGTTGCAATGGTAGTAGTTTCTTGGAACAGAACACCTGTATCTCTTTTGGCGACGCTTAATTTGGGTTTAATTATATCATGTCGTTATGGG\t<A-77JFF<<A-A<-<AF<JJF7F<JJFJJAF-JJJJFFJJJJJJF<AFA7<JJJJJJJFJFF<-FFJJF7JA-AJFJFJJJJJFJFFA<JFJJJJJA-<<--JJ7FFAFFJAFAJJJJJJJJJJJAFJJAFJJJJJJJJJJJJJJFAFAA\tAS:i:-29\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:7A101G2G22T8\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1128:18619:44623\t4\t*\t0\t0\t*\t*\t0\t0\tGTCAGGATTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGACTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2112:22506:15487\t16\t10\t143741059\t60\t151M\t*\t0\t0\tCCACCGCATCGCAGAGAAGAGTAGAGGTTAGATTTAGGCTGAGGTAAGATGGTGTGCATCCGGCAGGCGACCATCGAAGACCTGCTGGCGATGCAGGCGTGCAACCTGATGTGCCTGCCGGAGAACTACCAGATGAAGTACTACTTCTACC\tJJJJJJJJFJJJJJJJJJJJJJJJJJJJ<JJJJAJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:17C133\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:22323:32367\t0\t2\t157844151\t60\t151M\t*\t0\t0\tATGTAGACTGATAACTCCGATAACTGTTGCATCTCATCTCCTGTGCTTATACTATTCATCAAAGGGGGAGAAAAGGGAGGGAGGCATCCGCTTTTCACACTACAACAAATTCTGTGTTGCGCACCTAAGGTACTTAGCCCTTGGAGATACC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-JJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJA\tAS:i:-18\tZS:i:-25\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:8C55T57G28\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2103:16650:7134\t16\t1\t273848727\t60\t151M\t*\t0\t0\tGCGCCGTGCTCGCCGCGGTGCACGAGATCCAGGACCGCCGTGACGCGGCGCGCGAGGTGGAACGCAGCCTCCTGGAGCTCCACCAAGTGTCCCTCGACATGGCCGTCATGGTGGAGACGCAGGGGGAGAAGCTCGACGACATCGAGAGCCA\t<JJJJJJJFJA<AJFFJJJJJJJJJJJ<JFAJJJJJJJJJJJFJ<FJJJJJJJAFJJJFJJJJJJJJJJJJJJJAJJFJJJJJJJJJJJFJJAJJJJFF7JJJJJFJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJFAJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:90T60\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2119:26179:13341\t0\t10\t3066699\t60\t151M\t*\t0\t0\tTTGATGATGGTGGCAAAGATTTCTGCCAGACGTGCACGTGCCCTGTCGCGGCGCTTATGTGCAGGGATTGGGAGGTATGGGAAGAGTACACTGATTGGCTGCATCCCGTTGTCAAGGTCGTGGAAGAGAGCAGAAACATCGTCGAACAGCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJAJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFF\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:86G64\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1210:27935:1683\t4\t*\t0\t0\t*\t*\t0\t0\tCGTCAAGTCTTGTAATATCAAAACACATTAAAATTGCACGTACATAAGGCTCACACAGGACCGGACCGGAGTAGTACTTCCATTATGGTAGCACAGCGGTACGTCTCCCTCCCTTAACCGCTACTTAAACAACAACAACAACAACAGAAAA\tA<AFFJJFJJJJJJJJJ-7JJJJJJJJ-7AJJF<JJJJJJ<FFJJJJJJFJJJJJF<-FAF<F-JA-<<JJFJFJJFFFJJJFJJJJJ<JJ<JJJJJJJJAJ<JAJJJJJJJFJJJJJFA<A<FF7<FFFJJJJJJJ-AAFJJFAAAJJFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2116:9749:28797\t0\t9\t129535167\t60\t151M\t*\t0\t0\tTCCACAAAACACATCGAAACGATAGGATGTCACATCAAAACCGACATGTAACACATACTCAGGTGATCCAACAGCATAATCCAGCACAGTTTAGTCCATCGGTGGCTACAGGTGCATAATGACAAATACGATGATCACAGATCAGCGACGC\tAAFFFJJJJJJJJJJJFJJFJJJJJJFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-11\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:91C11T47\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2120:8643:5077\t4\t*\t0\t0\t*\t*\t0\t0\tTGCTATTCCCGATATTTCTATCGATATATTCCCTATCGTATAGACGAGGGATGCCCTTCCTCTGCTTCACCTCCCGGACAGGTATGGAAATCGTAATCAATAGTGGAATGCAACGGAGGGATATGCTCCAACTACTCTAGGAGTGTATCCC\tAAFFFJJJJJJJJJJJJJFJF7<FJJFJJJJJJJJJJJJJJJJJJJJJJ<-FJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJ-FJJJJJJJJJJJJJJFJJF-FFJJJFJJJJJJJJJJJJJJ-AFFFFJFJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1121:12946:5534\t0\t1\t178504517\t60\t151M\t*\t0\t0\tCTCGCCGTAGGTGCTGAAGGCGTTGTGGAGGGAGTGGTCGTCCGTGGCCCAGGCGAGGCCGCCGACGAAGCAGCGGTACTCAACATCCGACGCCGCCATCTCGATCCGCTCACCCACGAACCCGAACCCCGGAGAACCTCACGAACCCGAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJ<FJJJJJJJAJJA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2128:15990:26881\t16\t8\t116061026\t60\t150M1S\t*\t0\t0\tAACGAAGAGGGAGTCCATGGTCAGGGGCCTGATCAGAGAGGTTGCTGGCTTTGCTCCTTATGAGAAGCGTATCACCGAGCTTCTGAAGGTTGGCAAGGACAAGCGCGCTCTGAAGCTTGCCAAGAGAAAGCTTGGAACTCACAAGAGGGCG\tJJJAJJJJJFA-F7JJJJJJJJJJJJJJJJJ7-JJJJJJJJFJJJJJJJJJ<JFAJJJJJJJJJJF<JJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJFFFAA\tAS:i:-5\tZS:i:-22\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:11T138\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2202:17229:47137\t0\tPt\t8072\t60\t1S132M4S\t*\t0\t0\tCCCGCTTTAGTCCACTCAGCCATCTCTCCCCGTTCCAAATCGAAAGGTTTTCATGATATGGCAGAGGCAAGAAATAACGATTGTCAAAAATCCTTCCTTTTTCTTTCATTTCAAAAGTGCATAAAAATTATATAGCC\tAAFFFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJ\tAS:i:-5\tZS:i:-9\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:132\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2110:28635:14045\t16\t4\t21160466\t60\t90M\t*\t0\t0\tCAGGCTCTGATGGAGGGTCTTCAGGTCCTTCCCTTGGAGGACGTTGTCTCTGAAGCTGACATCTTCGTGACCACCACTGGCAACAAGGAT\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:90\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1128:17919:19214\t0\t1\t32326504\t60\t151M\t*\t0\t0\tCGCCGTCGGAACCCTGCTCGAGTCGAAGACTCCAGAAAATTCTCTCGTGAACGGGGCAAGGAAGGAAAGCAGAACCGCGGGCGCCTCGGTCTCTCTGATCTGCGAGTTCGTACGGATGGGAAGAGAAGTGAATGTGCGGGTTGGTTCGCCT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJ<FJJJJJJJJ\tAS:i:-5\tZS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:4A146\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1203:6563:30310\t0\t2\t847525\t60\t151M\t*\t0\t0\tGCGTAGCATATCAACCTGAAGCACAGGTATTGGGCGCACATAGTGCATTTTTGGAAGCAAGGGCCTAGAATACAGGTACAACATGTTTACCAGAGATGTTGAGGAAAATCGAATTGCCAGGTGCAGTTCGCCCATCTTTTTGACCCCCGAC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2202:21998:31347\t0\t5\t56871119\t60\t151M\t*\t0\t0\tCTGGAACAGGTCGCTTGTAACAATTACCAAGAGTTCTTCAGGTTCACAGACAGGACAGCAAAATTAACTACGGCAACAGGACCAAACCACCACTTCTGAGATCCAGATGACACACACAACTCTAGTCGAACAGGGAGAATCCCATGTCGTC\tAA-FFFJ<AF<JFFJ7JJJJJJJJJJJFJFJJJ7FJJJ-FFFJFFFJJJJJFFJJJJJJJFFFJJFFFJ-FFA7FJFJJJFJFJJJJJFFJ<A-<FFJJJJ-AF7JJJJJJ7FFJJJJJJFJJJJJF<FJJJJJ)7<77F<FJAJJJ<AJ7\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1224:27448:46803\t0\t5\t60002960\t60\t148M3S\t*\t0\t0\tCCGCGGGCTCCGCGGCGGCGGCGACCCGCCGTACGACGCCAGGCTCTGCTCGCACACCGACGGGTACTGCGTCGTGCGGCACGACTTGCGGACGAAGTCGCTGGCTGCCGGCGACGCGGCGGACACGAGCCCCGCTGCGGCGGCGGCGGCG\tAA<FFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAFFAAJJJFJJJJJFJFJJJJJJJJJFJJFF7JJJJJJJFJJJJJFJJJJJJJFAJJJFJJJJJJJJJFJJJJJFFJJJJJJJJ7AFJFAJJJJJJJJJJJJJ-\tAS:i:-3\tZS:i:-14\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:148\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1101:22313:1578\t16\t6\t111423533\t60\t94M\t*\t0\t0\tGCGGAAACCGACCGCGCTTCCGTCCGCCACCGCCGATTGACAATGAAGCTCCCAGCCGCACTCCCCTGGATCTCCCCGCTGCAGTACCGAAGAC\tJJJJJJFJJJJJJJJJFFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJAJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:53G40\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1210:14336:27813\t0\t7\t171775241\t60\t134M179N12M\t*\t0\t0\tCGCTTATTTGGAGTACTCAAATTGTGTTGCCAATACAGGGGAAATGGAACAGCAACATCCTGAGGTGCTCGCTTCGACCTTCTATGGTGTTTGCCTCGGAGGTTAACTGCTTTAGCCTTGGAGCCCAATGATACCACAGGCAACAC\tAAFFFFFFFAFJJAJJJJJJJJJJJJFA<JJJJJJJJJFFJJJJJFJAFFJFJFJJJFJJJJJJJJFJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJFFFJAFFJJJFJFJFJJJJJJJFFJFJJJFJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:146\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1221:9435:7785\t0\t8\t9903351\t60\t66M1432N85M\t*\t0\t0\tCTTTCCGCATGTTGCATTTGACTCATCAATTGCCAGAATACCACGTCCAGGGGAAGCAACCGATTTCGCGGTGGAGACGAGCTCGTCGGAGTAGGAGCCTGCCGCGGCGCGGATCACCGTCGCGCGGCGCGCGGGCGCGGACCGGCGCGCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-JJJJFJJJJJJJJJJJJJJJJJ\tAS:i:-12\tZS:i:-15\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:6A122G21\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1217:17381:20568\t4\t*\t0\t0\t*\t*\t0\t0\tGGCCGCCTCGCCGGGGCGCACCGACAGGAAGTCGGTGCGCACGAGCCCGAGCGAGGACACGGAGCGCGCGGCCACATGGTAGAAGAGGCCGATGGAGTTGAGGAAGTAGCGCACGAGGTCCTCCTGCGTGAGCCAGCAGAAGTCGCCGGCG\tAAFFFJJJJJJJJJJJJJJJJJJJJFJFJJFJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJ<AFJ7F7AJFJJ7JFJ7FJJJJJJJFAJ<FJFJFJJJJJJJJJJJJJJJJFJJ<JJ--AA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2128:8846:2756\t16\t3\t32237774\t60\t89M91N62M\t*\t0\t0\tAGGAGCAACGCAACATGTTGTACAACCAAACCTACAATCTCGACCAAGTTGGCTTTGCTGCCGATGGCATCAAGGATGCTCAGCAGACTATGAATGCAATGAAGGCAGCAAACAAGGAGCTCAAGGGAATGATGAAAACTGTCAAAATTGA\tAJ<JJJJJJFAAJJJFJJJJFJJJJJJJJJFA7JJJJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:68C82\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2125:25043:12814\t0\t2\t9270533\t60\t151M\t*\t0\t0\tCGTGGCCTCCGGCTTCGCGGCCATGGTGGCGGCGGGCATGGAGGACGCGTCCAGGGCCTCCGGGAGCGCCGCGGCCTCGGGATTGAAGAAGGTCTGCATCTCCGGCGTCGCGTCGCTGCTTGCGAAGTCCTCCAGGCCGTGGGACTCCGAC\tAAFAFJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJAJJJJFJJFFJJAJJJFJJFJJJJJJF\tAS:i:-11\tZS:i:-17\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:81C61C7\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2116:3650:2545\t0\t5\t14490585\t60\t3S148M\t*\t0\t0\tCCCTGTATGGTAGCAACCTCTGGCATATCCGCAGGAGTCCAAAAATAGCTTTCTCCACGAGAGCACACGGCATAACTGTGGACTGAACTATGCTGGCAATATGCTCGTATACACCTTGCCAGAGGAGCACGATCCTGTCTCGATTATTAAG\tAAAFAFFFFFAFFJ<FA<JFFFJFAFAFJJJJJJ<<AAA<F7F-<<JJF-AAFA<AA-FJJF<AAF-AAFJ<AAFJ<FAF7JJ-AJJJFAAF<<J-7FJF--7F-<JJAFJFJ7<A77-7AJJJF---7-7A-FJJJAJ-<)-<--<-A-7\tAS:i:-9\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:27T120\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2214:26890:33703\t0\t2\t4306079\t60\t146M5S\t*\t0\t0\tAGCACGAAGCGGATGAAGAACTTGAGCAGGTCCAGGGGGAAGTAGAAGACGATGCTGTAGAGCCAGACCACACCTGCCCAGCCCCAGCCGATACCCTTGATCCTGGCGAAGCCCCAGTTGGCGTAGACAGCGAGGAAGGTGGCAACAAGTT\tAAFFFJJJJJJJJ<JJJJFJJJJJJFJJJJFJJJJFFJFJJFJJJF<FJJFJJJJJFJJJJJJFJJJJJJJJJJJJJJJJFAJJJFAJJJJJJJJJ7FJJJFFFFJJJJJJJJ<AJJ7FAJJFFJJJJJF7FJFJFJFJ<JAFJJ7JFAAF\tAS:i:-5\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:146\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2104:12966:43445\t16\t9\t85077715\t60\t79M77N72M\t*\t0\t0\tTGCTGGAGCACTTGCAGGAGTCAGCTCAACCCTGTGCACATATCCTTTGGAGTTGATTAAGACCAGATTGACGATAGAGAAAGATGTCTATAACAACTTCCTCCATGCTTTCGTCAAGATTTTACGAGAAGAAGGTCCATCAGAGCTCTAC\tAJJJJJJJFJFF<JJJJJJJJJJJJJJJJJAJFFJF7JJJJJFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJFJJFJJJJJJFJJJJJFAFFJJJJJJJFJFJJJJFJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1101:18568:1595\t0\t3\t147048214\t60\t151M\t*\t0\t0\tAACAAAACAAAATACATTCAACATATCCACTTCAGACACCAATTAGCAGAAATAAGTTCAGAACTTCAAAATACTTGGGGTACAATCAAAGTACACAAGCCAACACAGCAGATAATTCTGTCTAGGGTAGACCACACAAACTAGGGCGGTC\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJ-<77\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:11302:40104\t0\t8\t14071772\t60\t72M1421N79M\t*\t0\t0\tCCTTCTCAAAATAACCACGCTCATGCCCTCATCGTCGATTAATTCAAATGGGAAGTTGTCTGGAACAGGTCCCTGCTGGTCGCTGTCGAGGTACTCCTGCGACTCCCTGATCTCGACGTCGAGGGCCGCGGTAAGGTTCCCGGAGCTGGGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJ<AFFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJ-JJJJJJFJJJJJJJJJJFJJJJJJJJJJJJ7AAJJJJJJJJJJJJJJJJFJJJJJ-AFAJ-7FFJFFJJJJJFAJJJAA-\tAS:i:-17\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:30T39T30C24T4G19\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1201:19887:25755\t0\t2\t4371896\t60\t2S149M\t*\t0\t0\tCTTCGTCGTCGTCGTCCTTCTCCTCTTCCTCGTCCTCCCCCGAGCCGGTCTCCTCGGGCGAGGAGAACAGTAAAACGGAAGAAATGACCCTCTTCGCTCCGTAGACGAAACCGGAGAGCCAGCCGGTCTCCGTCGCCGCGCCGGGCCCCGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJFF\tAS:i:-8\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:107G41\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2123:8288:40367\t16\t5\t208245020\t60\t89M1D62M\t*\t0\t0\tCTGCGGGCGATCATCCGGTGCTTCACCTGCCCTGACAGATACTTCGAGAAAGTCATTCGACTGGTTCTTGGAGGCATGGGCACAGATGAGATGACCTCACCAGGGTCGTAACTACTCGCGCCGAGGTCGACCTGAAGCTGATAAAGGAGGC\tJJJJJJJJJJJJJJJJJJJJF7JJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJA-JJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-17\tXN:i:0\tXM:i:2\tXO:i:1\tXG:i:1\tNM:i:3\tMD:Z:64C18T5^G62\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1215:29386:25263\t4\t*\t0\t0\t*\t*\t0\t0\tGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAGGCTCCCTCTCCGGAATCGAACCCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGC\tA-AFF7AJ-FJFJFJJJJ<JJJJJJFF<-<FJFJJJJJJJJJJJJ<JJ-FFFJF--AFJ<FFFFJJJJJ7AAFJJJJA<7-FJJJ<AFAA<JFJFF-7FFJJJJJJFJJFFJJJJ<JFJFJJJ<F<<FJFJFA<JAJJJJJFF-7FJJF<<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1112:31883:9895\t16\t2\t7193302\t60\t151M\t*\t0\t0\tAGAAGTGTACGATGCAATGAGAAAACTCGCAGAGCAGCCCGAGGACTGTGTGGACGATGATGACGCAGAGGCTGGTGATGAACATAAAGCAGTAGATGACAATAGAGGACCATCACCAATTGTTGTGATTCCAGATGCTGTGCACAGGAAG\tAJF7-A-AF77<<-FFA-7<FJ<-7J<FAJJJJJJFFJJJF-<AFFJA<7--FF-AFJAA7JJJ<JFJJAFFFFFAFF<FJJJAF-FJJJJAFF<JJJJJJJFF-FFFJJJJJFJJJJJJJFJFJJJFF7AAF<7-AJJJJAJJJJFFAAA\tAS:i:-21\tZS:i:-24\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:39T23T54T16G15\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2110:11140:22837\t0\t6\t167630110\t60\t1S150M\t*\t0\t0\tGTCACGATAGGATATAGCATAGGATAGACTGCAGCCACGGCCGAACACTTTGATCGTACGAGCCGCGGAGGCTACAGCTCAGGTAACAAGTTACTACTTGCTTAGAAGCAACGCTTCCATGTACCATCTACGCTAGTGAAGCCTCGTGAGC\tAAFFFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJ-FJJJJJJJJJJFJJJJJFJFAJJJJ-<7FAFJFFJJJJJFFJJJJJ<-7FJJJJJJJJJJJFJJ-FJJJJJJJ-<FFFFJJ<JJJFJJJ7AF<\tAS:i:-21\tZS:i:-21\tXN:i:0\tXM:i:6\tXO:i:0\tXG:i:0\tNM:i:6\tMD:Z:56G25G20C17T8A15C3\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2218:8400:45520\t16\t5\t163150893\t60\t61M81N90M\t*\t0\t0\tTCCTGTCCACCCTGCCACCGCCGACGCCGCGCCGCCGCAGCAACTGTCACCGTCGCTATGGCCCGTATCAAGGTGCACGAGCTCCGGGGGAAGATCAAGACGGACCTGCAGGCGCAGCTCAAGGAGCTGAAGTCTGAGCTCTCCCTCCTCC\t-JFJAAJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:43T50G56\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1217:26088:33475\t16\t8\t144699125\t60\t151M\t*\t0\t0\tTAGTCGAGATTGTGAAGATGAAGCTCCAGGGTGTCGCTGACAAAACGGTGGTAAAGAAGCTGGTGCCCAACAGACAGAACTGCCTAGCCGAGGAGAGCTCCATATACATATCGGAGGGGAAGGTGAAGCAAGACCTGGTCGTACGGCTAAG\tJJFF7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAAJJJFJJJJJFJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJ<JAJFJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2205:6654:12673\t0\t10\t33751928\t60\t2S149M\t*\t0\t0\tGCGATGATTTCGAATAGGTTGTGCACACCCTCTGGCTCCTCCAGATTGTACTGCGAGATCATTCCGCAGGTAGGGATCCGACCATGGATACGCATGTTGGGCAGCACGGCTTCCAGTGTGCTGCCACCCACGTTCTCGAAGTAGATGTCAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJFFFFJ\tAS:i:-14\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:9A89A49\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1127:5162:6994\t16\t9\t139442314\t60\t151M\t*\t0\t0\tCAAGAAGGCATAGACTGACTGTCTGGAGTGATCTGATGTTTAGCAGGACCTTGATACCAGTATGCGGTGATGCGTAGGCTGTAGATGGGTCGAATGTGCTTGTGTGTGTATGTGCCCTGCTGCAAGTTGCCCTTTGCTATTTAAAATGGAA\tFJJJJJJFJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1213:5487:41106\t0\t6\t11913898\t60\t151M\t*\t0\t0\tTGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACAACCCGGCCAGTTAAGGCCAGGAGCGCATCGCCGGCAGAAGGGTCGAGCCGGTCGGTTCTCGCCGTGAGGCGGACCGGCC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2113:20811:34776\t16\t3\t223648864\t60\t151M\t*\t0\t0\tGTCCTTCCTGATGGCCAGGCTATTGCTGTTAAGCAACACAAGCTTGCTAGTTCCCAGGGTGATGTTGAGTTCTGCTCAGAGGTGGAAGTTCTTAGTTGTGCACAACATCGAAATGTTGTAATGCTGATAGGTTTTTGCGTTGAGGACAGAA\t<7JJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1106:17726:43779\t4\t*\t0\t0\t*\t*\t0\t0\tGCCTGTATGCAAAAAAAAAAAAATCTTAGATACAAAGGGTGAAATTGACTCATTAATAAGCTTTAATTTATACAAGAAACTTTTGAGAAACTCAATTCACTCTTTTTATTATTTTGATCCTTTCACATATGATCTCAAAATAGGCACTGGT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJFJJJJ-<JJAFA<-7A<J------FJAJJ--F-AJJFJJ--7-7A<<<-77<AJJJ----A<--F---A----77-FAA-7----7A<---7-77F<-FF------7-F-<FJ-----7))))-\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1116:13007:31417\t0\t5\t203603355\t60\t1S136M\t*\t0\t0\tGCTTCGATATGAAAGGCAAAAACTTCACCAGATCCTGATTGGAAAGGACCGGGAACACACACTGCTCTTGTTCGTCATCATATTTCTCCATTTAAGCACCGCCGGAGCCGTACTTCTTTCCGAAGAAGAGGATCTGG\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJFFFFJFJJ\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:136\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1116:17411:31804\t4\t*\t0\t0\t*\t*\t0\t0\tCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCAGAGCCCGCGTCAGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJFJJJJJJJ-F-AAFJJ7<))A<JJJJ<7A)<)\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2127:25438:41352\t16\t5\t6696209\t60\t151M\t*\t0\t0\tGGTTCGGTGACTCATGCTTCCAGAAGGACATCCTGAACCTGGTGTACATCTGCGAGTGCAACGGCCTGCCCGAGGTGGCCAACTACTGGAAGCAGGTGATCAGGATCAACGACTACCAGAAGAGCCGGTTCGTGAACCGCGTCGTGGCCTC\tJJFJJJJF7-JJJJJJFJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJFF-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-3\tZS:i:-16\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:9G141\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1223:19441:19144\t16\t5\t76242740\t60\t151M\t*\t0\t0\tCAGGGGCTCATCTGTACTACAACTGCCCCAGCGGGAACTACTTTGAGTACCAGGCATTTGCCATCGGTTCTCGTTCCCAAGCCGCCAAGACTTTCCTGGAGCGCAGGTTCCAGGGCTACAATGACTACACCCCGGAGCAGCTCATCAAGGA\tJJJJJJ<JJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:58C36T55\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1220:13301:30626\t4\t*\t0\t0\t*\t*\t0\t0\tACGACTTCTCCTTCCTCTAAATGATAAGGTTCAATGGACTTCTCGCGACGTCGGGGGCGGCGAACCGCCCCCGTCGCCGCGATCCGAACACTTCACCGGACCATTCAATCGGTAGGAGCGACGGGCGGTGTGTACAAAGGGCAGGGACGTA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<FAJFJJJJJFAFJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2123:18578:32595\t4\t*\t0\t0\t*\t*\t0\t0\tTGGTGGTGCCCTTCCGTCAATTCCTTTAAGTTTCAGCCTTGCGACCATACTCCCCCCGAAACCCAAAGACTTTGATTTCTCATAAGATGCCAGCGGGGTCCTATTAGTAACACCCGCTGATCC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-FJJJJJJJJJJJJJJJJJJJJJJJJJJ<<FJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1209:10866:25034\t0\t5\t170460393\t60\t73M2322N78M\t*\t0\t0\tCCAAGGTTGTTAATCGAGAAGTGCTGCAGCTGGGTCAGTGCACCATAATCAAAGTCCAGATTGTATGGGTAACCGAGGTGGTGCTTGGTGCGTTCGACGAGCGAGCGGCGGTAGAGCGCGAGAACGCTAGCCATGGTTGCCTCCCGCTCGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2110:4655:18669\t4\t*\t0\t0\t*\t*\t0\t0\tAGGTGAGAAGCATCTGGATGAGCATGTCGTTCCAGATGTCGATGGGTCCCGGGAGGCCCCAGTGGATGCAGTCGTTGTAGAGCTGCACCTTCTCGTTGGGCCAGAGCCCATAGCGGCTGGGGCGGCCGTCGCGCCGCATGAGCATCGGCGC\tAAFFFJJJJFJJJJJJJJ-7FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJ7JFJFJJJJJJJJJJJFFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJA7JJJ-FJJJJJJJJJJJFJJJJJJFF-7FJJJJJJAJJJJ<A)7<<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1124:23013:30222\t16\t1\t14384581\t60\t151M\t*\t0\t0\tTGAGCTGAGCGATGGATCGCGAGCTCAGCCCCAGGGGCAATGCTCCAGGGAGAGGGAGCGGCGGCGCGTCGTCATCGAAGCACAAGAACGGCGGCGGCGGCAAGAAGCCGATCAAGGTGCTTTACATCGCCAACCCCATGCGCGTCCAGAC\t<<AJA<AF7--FJJJF<<FF<FFJFJJJAAJJJJJJJF<JFJFFJFFJJJFJJFFFJJJFJJAJAJAF<FFFJJFJJFFJJFJJA<7-A7JFAJFJFF7JJJJJJAJJAAJF<JJJJJJJJJJJJJ<JJ<7FJJJFJ<<FJJJJJJFAFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1121:26129:36253\t16\t2\t183847163\t60\t151M\t*\t0\t0\tCTTCAAGGTGCTGAAGCAGGTGCACCCCGACATCGGCATCTCGTCCAAGGCCATGTCCATCATGAACTCCTTCATCAACGACATCTTCGAGAAGCTCGCGGCCGAGTCCGCCAAGCTGGCGCGCTATAACAAGAAGCCCACCGTCACCTCC\tJJJJJJJFJJJFJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJFJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2124:13595:18194\t0\t5\t1566305\t60\t144M\t*\t0\t0\tCAAAGATACTAAATCTTCAAATTTAAGGTACCAGCTGAAATCACAACATTGCATATAGTGACATGAATTCAAAACATAGCTAAGGCAGTGTCCAATTCTTGGTAAACTAGATGTAAAATCTACTGCTTGATATCAACCTTCTTC\tAAFFFJJJJJJJJJJJJJJ-<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFJJJJ\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:19G124\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1215:23389:23593\t0\t10\t13640270\t1\t1S38M156N82M112N27M3S\t*\t0\t0\tGTCCGGAGCTCCTAGTCAGTTCGCATTTGATATTCAAGCCTTGCTCGCAATGTTGCTGGACAGCCAGTCCAGGCCCTCGTACAGACCCTCGCCAGTTGTGGCACAAGTGCTCTGGATGTACCAGTGTCGCTGGCGCAGGGAGTGTAATAGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJ7F<JJJJJJJJJJFJJ-AJFJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJFFJJ-AJJF\tAS:i:-4\tZS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:147\tYT:Z:UU\tXS:A:-\tNH:i:3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:1215:23389:23593\t256\t9\t139157871\t1\t1S38M156N82M112N27M3S\t*\t0\t0\tGTCCGGAGCTCCTAGTCAGTTCGCATTTGATATTCAAGCCTTGCTCGCAATGTTGCTGGACAGCCAGTCCAGGCCCTCGTACAGACCCTCGCCAGTTGTGGCACAAGTGCTCTGGATGTACCAGTGTCGCTGGCGCAGGGAGTGTAATAGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJ7F<JJJJJJJJJJFJJ-AJFJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJFFJJ-AJJF\tAS:i:-4\tZS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:147\tYT:Z:UU\tXS:A:-\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1215:23389:23593\t256\t1\t84573233\t1\t1S38M156N82M112N27M3S\t*\t0\t0\tGTCCGGAGCTCCTAGTCAGTTCGCATTTGATATTCAAGCCTTGCTCGCAATGTTGCTGGACAGCCAGTCCAGGCCCTCGTACAGACCCTCGCCAGTTGTGGCACAAGTGCTCTGGATGTACCAGTGTCGCTGGCGCAGGGAGTGTAATAGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJ7F<JJJJJJJJJJFJJ-AJFJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJFFJJ-AJJF\tAS:i:-4\tZS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:147\tYT:Z:UU\tXS:A:-\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1103:3792:9473\t16\t4\t21160966\t60\t151M\t*\t0\t0\tGGTGTATGTGCTCCCCAAGCACCTTGATGAGAAGGTTGCTGCTCTCCACTTGGGCAAGCTTGGTGCCAAGCTGACCAAGCTCACCAAGTCTCAGGCCGACTACATCAGCGTGCCGATCGAGGGTCCCTACAAGCCTGCCCACTACCGGTAC\tF<7AAFFJJJJA<7A<-FFFAJAF<JF-FAFFFJJJAJJFJJFJJJJJJJJJJJJFJFAFAJJJJFFJJJJJJFAJJFJJF<JAFF<FAJJJJ7JJJJJFJJJJJF7JJJJJA7A<-FJJJFJJJJJJJJJJJFFJ<JJJJJJJFJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2103:10348:42706\t0\t3\t134745585\t60\t135M\t*\t0\t0\tCAAAGCACAGGTAACTCATAATCAGTATAACAAACAAAGCTTCAGTTCTCCATCACAAACATGGCGGACTTAAATAGCAGAGTTCTGAAGGGAACATCTAAATGACAGTAATTAAAATTGTGGCATCGCCTCTAC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:135\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1124:24109:40491\t0\t5\t9483300\t60\t117M93N34M\t*\t0\t0\tGTAGGACGAAAGCATGAAGTGGATCCTAGGGTAGGGCACCAGGTTGGTCTGGAACTCGTTCACATCAACGTTCAGAGCACCATCGAACCTCAGGGAAGCAGTCAGGGATGAGATGACCTGGGACACAAGCCTGTTGAGGTTGGTGTAGGTT\tAAFAAJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJAJJJFJJJJJJJJJJJJJJJJJJJJJJAJJ7JJFJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJFFFJFJJJJJFJJJJJJJAFFFFJJAFFJJJFA\tAS:i:-18\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:27C71C23G27\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1111:16386:35972\t16\t1\t52460719\t60\t34M1I116M\t*\t0\t0\tTGGTTAGACAGGTTCCGATTCTAACCATATTTTCTTGTCAGCATTTTGAATGCTAGGACAGACATGGTAACTCATCACAACCATAAATTGTTCCTATCACTGTAACGTTGAGCATTGCTCGTTTCTGGGAAGCAGATGGGCTGATGGCTAC\tJJJJ<JJJJJJJJJJFFJJJJJJFJJJJJJJJJJJJJJJFAJJJJJJJJJJJJJJF<JJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-14\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:1\tNM:i:2\tMD:Z:100T49\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1116:3346:47805\t0\t2\t1290076\t60\t81M134N46M\t*\t0\t0\tGCTAATCTTCCTCCATTGGCACTTGTGGTACGTCCCATTTGTCTTCCTCGATAGGCTTGGTAATGACGGAGATAGGTGACAAAGTATGCAAATTTGTATCTTTTTCACCCCTGTTGCAGCTCCCTCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-12\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:127\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2101:21460:34073\t0\t8\t4282262\t60\t151M\t*\t0\t0\tGAGCGATAAAACATTCGTGTTTGATAGAGTAGGATGGGCTGAGTCTTGCATAGTTAAATAATAGGGTAGTTTCTCCAGGCAGGCGGAGTTATGTAGAGGAACTGTTGTACCCGATAACTTACAATGGTGAGGAGCCTCTTGTCTTGCGCAT\tA-AFFAJ7AJFJ-J<JJA7<-FJ7JAJ<F-7FF7-FF<--7--JJFAJAA-FA-77-7<<AJF<FJJ7J7--A-7-77-J7AJ--7--A7<7F7<F7FFJJAF7-F<-F--7-7AA-7-AJAA<-7-FFJ77F)-))7)-A-7--A<<-7-\tAS:i:-11\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:38A36A48C10T15\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1104:29163:8277\t0\t5\t1566338\t60\t4S147M\t*\t0\t0\tGCCGGCTGAAATCACAACATTGCATATAGTGACATGAATTCAAAACATAGCTAAGGCAGTGTCCAATTCTTGGTAAAATAGATGTAAAATCTACTGCTTGATATCAACCTTCTTCTTCTGCCACAACCTGTCCAAGGAGCTGTCTGCGTCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJFAFJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFFFJJJJJJJJJJJFJJJJJJJ<\tAS:i:-10\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:73C73\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2120:28818:9543\t16\t7\t132217459\t60\t134M\t*\t0\t0\tTGGTGGTGGGGCCCGGTGTCAGTGGCGCGGTGCCCCATGCCTGCCGCTGCTGCAGCTTGTTGTGTCTTGTGTTGGTTGGTTCCTGTTCTTTCTGTCTATCTATTCTCATGCATGGCAGTTTGGGTATGGTATGG\tJJJJJJJF<-JJJF<JJJJJJJJJJJJJJJJJJJJJJJFJJJJ<JJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJAJJJJJFJJJJJJJJFJFJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:134\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2113:5345:30661\t16\t3\t12293750\t60\t151M\t*\t0\t0\tCACAATTAAAAGAAATGGCCCTTCGTGGAGGAGATGGTGGAAGATCACGTCGATGGGCATCTTCTGATGACTTTTATGGTAGTCAGGGATATGGTGCTAAAAGATCAACTGATAGTTTCAAGAACAGCAGCTTCGGCAACCAGGCCGGGGG\tFJJJJJJJJJJJFJJJJJJJJJJJJJJJJJF77JJJJJJJJJJFJJJJJJJA-JJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-9\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:32T39C78\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2212:9272:37783\t16\t9\t87871316\t60\t149M2S\t*\t0\t0\tACGGTGAACCCGGCCAAGGAGAAGAGCCCCTGGATGGATGAGATTGACAACTTCCCGGTGGATGTTCCAAAGATTTCAAAGGTTGGTGGCGCTTGAAGAACCATTCTTCTTATTATTAGGTGGCAAGTGGAGTTGGGTGATTGTCAGGGCC\t7JAJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFFFFAA\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1215:6745:10124\t0\t10\t27279411\t60\t16M80N135M\t*\t0\t0\tCTCCAGTGGAAAAGGTCTCTATGTTCTCCCCAGCTTTTCAAACAGTTTCTATGATATCGTTTGCTGCAGAGTTTGCAAGGAAGCATTTTGGCAGCTTTCGTGCTGCCCTCATTTTCTCCTGAAAAGCATAAACGGCACATAACCTTGACAG\tAAFFFJJJJJJJJJJJ<FJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJFFJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2122:4219:43374\t16\t1\t210269408\t60\t151M\t*\t0\t0\tATTACCAGCAACTTCAAGGAATCTAAAAGCAATGATGTTGGGAGAACTAATGCCTCATCTGATGGATGGCAGGATTTTGCCAGTTTTGATCAAGCTCAGAGAATTACTAAAGTTCATGTGGAAGGACATGTTGTAAAAGATCCTTCAGGGG\tFJJJJJJFJJFFAJJJJJJJJFF7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF-JJJJJFFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2126:8501:24489\t4\t*\t0\t0\t*\t*\t0\t0\tACGGGCGGTGTGTACAAAGGGCAGGGACGTAGTCAACGCGAGCTGATGACTCGCGCTTACTAGGCATTCCTCGTTGAAGACCAACAATTGCAATGATCTATCCCCATCACGATGAAATTTCCCAAGATTACCCGGGCCTGTCGGCCAAGGC\tAAFAFJJJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFFJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFFFJJJJJJJJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1206:1671:32736\t0\t8\t134745223\t60\t151M\t*\t0\t0\tCCCATGTTTCCCCGACTCGAACACCCATATTCCTGCGGGGGTCATTTTCTGCAAGAATGGGGCCAAAATGATCAGGTGCAATAGTCACAAAGATCTGGCCACTTGATGCGTTTGCTTTTCCAGCCCTCTTAGCCCTGTCAGTTGTAAAGGC\t<AFFF<A-AFFJFA<JJJJJJAJJJJJAJAFAFJAJFJJJJJJFJJAJJJF<JAFJJJFJFJJJJJFFJJJJJ7F7-FJJJFJFJJJJFJJFFJAJ<FJFFJ<JJJ-<--AA<FJF<JJF7-FFJFJJAFJJ7JAAJ))---<7----7AJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1119:24606:24542\t4\t*\t0\t0\t*\t*\t0\t0\tTTCCCCGACTGTCCCTATTAATCATTACTCCGATCCCGAAGGCCAACACAATAGGACCGGAATCCTATGATGTTATCCCATGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCAC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFJJJJJJJFJJJJJJJJJJJJJJJJJJFFJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFJJ<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2221:28919:46575\t0\t1\t20529017\t60\t1S134M2S\t*\t0\t0\tAGCCTTGCGGGAGATTGTGAAGCTGCCGAGCTTGGTGAGAGCCTTCTCCATCCTCCTTCCTCCCTCCGGAAGCGCAGCCGCTCCGCTCGGCTGGTTCTCTCTTCTCTGCGCCGCTCGGCCGCTCTTCTGGTGGTGGC\t-AFFAJJJJJJJJJFFJJJJJJJJJJJJJJJJJJ7FJJJJJJJJJJFJJJJJJFJJJJJJJJJJFJJJJ<JJJA-FAJJAJJJJJJFJJFJFJJJJJ-FJJJJJJJJJJJJJJJJJJFJJAA7FJFJFAAF<FJJJ-\tAS:i:-3\tZS:i:-5\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:134\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1219:1600:29026\t0\t3\t58906816\t60\t151M\t*\t0\t0\tCCCGATCTCGCGGCAGGTGTCGCGCACGATCTTCTCGTAGTCCACGGTCGCCTTGGTCGTGATCTCGCCGAACACCATCACCATGTTGGTCTTGGTGCAGGTCTCGCAGGCCACCTTGCTGTCGGGGTCCTGCGCGAGGCATGCGTCAAGC\tAAFFFJJFJJJJJJJJJJFAAJJJJJFFJ7FJFJJFJJFA<FJJJJFJJJJJF<AJ<JJFJJ-F-FAJJJJJJJJJJ7FJJJJJJFFJF-F<JAJJJFJJJAF<JJJJJJJJJJJAFJJFJAAJJJJ-FJJAJJJAFJJA<A<-FAAJJJ<\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2124:17848:35303\t16\t5\t199820096\t60\t108M\t*\t0\t0\tCTGACAGACGATGAGACTAGTGAAGATGACCTCCATGAAGATGAAGAGTGGGCTTCAGATGATGAGGAAACTGCCTCTAGTGACGAGAGCGATGAAGGTACTGAGTCC\tJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJF-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:47A60\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2221:9770:45818\t16\t6\t79358710\t60\t150M1S\t*\t0\t0\tACAACTACCTCAACTACCACGAGCGGCTGCTGAAGCTGGTGAAGCTGGGCGGCCTCATCGGCTACGACAACACGCTGTGGAACGGCTCCGTCGTGCTCCCCGACGACGCGCCCATGCGCAAGTACATCCGCTTCTACCGCGACTTCGTGCG\tAJJJJJ7JJJJFFFAAJJFJFAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2213:6319:19900\t4\t*\t0\t0\t*\t*\t0\t0\tACCCGGGTGCAAGAACTTGGCGTCCCCGCCACCAACACACCCGCCGCCTCCCCCGCGGGCCGCATTGCGCCGGCGGTGCGAGAAGAAGAAAGCGATGGACAGCCCGAGCACGGCCACCGTGAGCAGCGGCAGCAGTATGGCCGGGACGAGC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAFJJJFJFJJJFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2103:17949:44693\t4\t*\t0\t0\t*\t*\t0\t0\tGGCTTCGTTTATGGTTGAGACTAGGACGGTATCTGATCGTCTTCGAGCCCCCAACTTTCGTTCTTGATTAATGAAAACATCCTTGGCAAATGCTTTCGCAGTTGTTCGTCTTTCATAAATCCAAGAATTTCACCTCTGACTATGAAATACG\t<AF<FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJFA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2107:12276:12849\t16\t6\t10614407\t1\t151M\t*\t0\t0\tGGCTGGGCCGGCCGGTCCGCCTCACGGCGAGAACCGACCGGCTCGACCCTTCTGCCGGCGATGCGCTCCTGGCCTTAACTGGCCGGGTCGTGCCTCCGGCGCCGTTACTTTGAAGAAATTAGAGTGCTCAAAGCAAGCCATCGCTCTGGAT\tJJJFJJJJJJJJJJJJJJJFJJJJFJJFFFJJFJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2107:12276:12849\t272\t6\t20571898\t1\t151M\t*\t0\t0\tGGCTGGGCCGGCCGGTCCGCCTCACGGCGAGAACCGACCGGCTCGACCCTTCTGCCGGCGATGCGCTCCTGGCCTTAACTGGCCGGGTCGTGCCTCCGGCGCCGTTACTTTGAAGAAATTAGAGTGCTCAAAGCAAGCCATCGCTCTGGAT\tJJJFJJJJJJJJJJJJJJJFJJJJFJJFFFJJFJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tZS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2102:10937:3178\t4\t*\t0\t0\t*\t*\t0\t0\tCTAATTCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCAGAGCCCGC\tAAAFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ7-<<FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJ<<F77FFJJ<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2120:6380:21799\t16\tMt\t401353\t60\t4S146M1S\t*\t0\t0\tGAGCTCCAAAGTCGAGAGGGAAACAGTCCAGATCGTACGCTAAGGTCCCTAAGCAATCACTTAGTGGAAAAGGAAGTGATCGAGCGATGACAACCAGGAGTTGGGCTTGGAAGCAGCCATCCTTTGAAGGAAGCGTAATAGCTCACTGGTG\tA7)F-AF7AJJJFF7JJJJJJJJJJFAJJJJA<JF-FFJJJJJJJJJJJJJJJJJJJJJJFAFJJJJF-JJJJJAJFJJ<JJJJFJJJJFFJAJJJJF7A-JJJJJJFJJJJJJJJJJFAJFJJFJJFFAF<JJJJJFJJJJJJJF7FFAA\tAS:i:-18\tXN:i:0\tXM:i:3\tXO:i:0\tXG:i:0\tNM:i:3\tMD:Z:22C73G28A20\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2122:16122:30837\t16\t1\t5740991\t60\t151M\t*\t0\t0\tAAACATTTCTTTTCCTAACTCATTGGTTAAGTGTCGTTAATAAGCACCGTGCATTTTTTTCCAGGTGATGAACCTGCTGCTTGTACCTGTTTATTTATTCTTTGGATGATTCATTTGCTTGAGTTGCCAAACTGATCACATTGTAGCCGGT\tJFAJJJJFFJJJJJAF7<AFJ7A-AJF-FA<7FJFAAFAAA7AA7J<<FF7FJJJJFJA-JJJJJAJFJJFF<AJ<JFJFAFFJFFF<JF-FF-JF<J7JJFJJJJJJJJJF7-F-JJJFJA<J<FFFFJ7JJF-FFF7JJJJJFFF7FAA\tAS:i:-7\tZS:i:-25\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:113G24A12\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2212:28990:42231\t0\t7\t130003254\t60\t151M\t*\t0\t0\tCCCCTACCGCGCACCTCGCGGAGGCGCCCCTGCCCTGCGGCGCGAGCGCGGCCCGTAACAGACCCCGGCACGACGGGCTCGGGCGCAGCGCGGCCACCGACGACGGCCCCAGGGTCCTGCTCAGCGCCGGCGCCATGAGGTGTGGAATAGA\tAAFFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJ-AJJJJJJJAJJ<JJJJJJFJJJJFFJJJJJJJJFJJJJJJJJJJJJJJJJFFJFJJJJJJJJJ7\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2208:3995:41616\t0\t2\t111526735\t60\t151M\t*\t0\t0\tGACCACCTTATGCATACAGTATCCTTAACATGACACTGGATCGACAAAACTAGAGAACCCAAGTACAGTGGTACAGAGCGCAGATTAACCTCCTTAAACTTGGGCGGTGCCTTGGGTGAACTTGGGATCCTCGCACCTGTACTTACCATCG\tAAFFFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJFJJJFJJJFJFJJJJJJJJJJFJJJJJJJJJJJJJJJJJ7JJJJJJJJJJJFJ7AJJJJJJF<FFJJJJJJJJAJFFFJ<FJJJ-7<)\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1105:8836:11565\t0\t6\t6516916\t60\t151M\t*\t0\t0\tGGCTGGCTGAGCTCGTGGAACAGCACAAGCTCCGATTTCTGCCATTGGACCGGCGTGACATGCACCCGCCGCCGGCACCCTGGCAGAGTGGCGTCGCTGGACATATCATCGAGACAGCTGTCAGGCACCATCTCTCCGGCGATCGGCAACC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJ7JAJJJJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1128:30523:18757\t16\t2\t207295269\t60\t48M2I101M\t*\t0\t0\tTTATTTTGAGTAGATACTCTATGGTTCCACTCCATTGGATACTGCTGCTGTGTGTGTTTCCATTTTGTACTATGTAGTAAATTGTTCGTAGCCCCCTATTGGCCATGATTGTTCATATCATCCTTCTTTGGTTTGCAACGCTATTCGTCCA\t<--7JJJJAFF7-7F7JJJJFFJJFJJJFF7JFFFJA-FFJJJFFFJJJJJJJJJJJFFJJFF-JJJJJFJJJJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJF<FAA\tAS:i:-14\tZS:i:-28\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:2\tNM:i:3\tMD:Z:12T136\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2109:26666:41124\t0\t1\t95198066\t60\t151M\t*\t0\t0\tGTTCCCCTTCTCCGGTGCACGTTATACAACAGCACGGCGCATGTGTCTGTGTGCACGCACAGGCATGAGCATCCACCTACAATGCCGCGGCGGTGGCGGCGCCCTCAGCCTCCGGCGCCTTGCTCTTCTTGGCGGCGTCGAGGAACCCGAC\tAAA-AFA-FJJJAJJFFJFJJJJAFJJJJJFJJJJJJJJJJJJAFJJJJJJJJJJJJJFFJJJJJJJJ<JJJFJJJJJJJJJJFJJJJJJFFJJJJJFAJFJJJJJ<<AJAFJAF<JAJAFFJJJJFJJFAJJ))AAJJJAJ)7FJ<JFJ)\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1107:4868:12110\t16\t5\t65964081\t60\t151M\t*\t0\t0\tGGCGGACGGCGGCGGCGAATCCTTCGTGCTGCGCTCCGGCGTCCGCGCGGGCCTCAAGCGCGAATTCGCCTTCGCCATCGCCTCGCAGGCCGTCCTTTCGTCCTCATACGCACCCCTGGGCCGCACCCGCCGCTCATCCCGCATCCTCAAC\tJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2115:19522:8242\t16\t8\t77267100\t60\t151M\t*\t0\t0\tCGTCCGCGCAGTTCAACGTCCCGGATCTCCGCGTCGGCACCCTCGACTCGCTCCTCGCGCTCAGTGACGACCTCGTCAAGGTTCCCCGCACGATCTCGATCCCCTCTCCCCTCTCCAGCTGCCCCGGGGCACTAGCCTCTGATTCCGTGCC\tFJFJFJFJJJJJAF<FAFAAFFJFFJJJJJJJF77JFAFJJFFAAFJJJJJ<JJJJJJAJJJJJJJJJFAAAJJJJJJJJJJJJJJ7F--FJJJJJJJFFJJJJJJJJJJJFJJFFJJFJJFJJJJJJJJFJJJJJJJJJJJFAJFFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2218:31862:42601\t16\t6\t130529298\t60\t130M\t*\t0\t0\tATGCAGGTCAGTGTTGTCGGTTACAAGAAGGGACAGCTCAAGATGTTGTCGCATACGTATGACCAGTCTCTTGGCGGGAGGGACTTTGATGAGGCCCTGTTTAAGCACTTTGCAGCTAAATTAAAAGAGG\tAFJJJF<JF<JFJJJFAJJJJJJJFF-FJJF7FJJJJJJF<JJJJJJJJJFFA-JJAJFJJJJJJJJFFJJA<F-JJFJJAFFF-JJJJJJFJAJJJFJJF-JJ<FFJJJJJJJJFFJJJJJFAFF<AAA\tAS:i:-5\tZS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:122C7\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1208:24251:40561\t0\t2\t9018963\t60\t111M97N40M\t*\t0\t0\tGCGGATATTTCCGAATTTGGAGATGGGGATGCTTCTGAGCTAACCAAACCATCATTCAGTGCCTTGTCCTCGTCAAATGCAAACCAATTGGAATTTGTAAAGAGCGAACCACTTTCTTGGTCGTCTCCAAGGCGAAGGGAGGAGATTACAA\tAAAFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJJJJJFJFJJJJAJF7AFFJJFF\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1123:12337:47208\t16\t1\t288804248\t60\t151M\t*\t0\t0\tCATTGATGGCGACCGAGGTGGCTGAGACTCCCGCGCCGTTGGCGGAGGCGGTCCCCGAGACCCCCGCGGAGGCACCGGCGGCGCCCGCCGCCGAGGCGAAGCCTACCAAGGCCAAGAAGGCCTCTGCGCCCAAGAAGCGTGCCAACCCAAC\tFJJJJFJJJJJJJJJFJFAJJF<JJFFJAJFJJJJJJJAFJJJJJJJJJJJJJJJ<JJJJJJJJFJJJJJJJFJJJJJJJJJJJJJF<JJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAFAA\tAS:i:-5\tZS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:148G2\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2111:15260:10704\t4\t*\t0\t0\t*\t*\t0\t0\tCTCCGTCACCCGTCACCACCATGGTAGGCCCCTATCCTACCATCGAAAGTTGATAGGGCAGAAATTTGAATGATGCGTCGCCGGCACGAAGGCCGTGCGATCCGTCAAGTTATCATGAATCATCGGATCGGCGGGCAGAGCCCGCGTCAGC\tAAAFFJJJJJJJJJJJJJJJJJFJJJJ<FFJJAFFAFJJJFJJJFJJJJJJJJ<JJJJJJJJJJJJJFJJJJJJFFJJJFJJJJJJJJAJ7FFJJJFAJFJJJJFJFAFJJJJJJJFFJJJJJF<JJJJJFJ7JJ-A-))FFJJA))AJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1112:6583:7486\t0\t4\t233320739\t60\t3S56M214N90M2S\t*\t0\t0\tCCCTCTTCCTCTGTAGCTTCTTCATGTAGAACTCAAGCTCTTTACCCTCAAGGATGTACCCATCAGCTCGGCCACACTGTCCAGGGCGGGAAGAAATGCATGCCAGCAACCGTCCACTGCCAAATTGCTCCTCAATGTGTGCGTCAAGGAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJAJJJJJJJJFA\tAS:i:-11\tZS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:138G7\tYT:Z:UU\tXS:A:-\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2215:24627:5692\t16\t6\t145042565\t60\t40M78N110M1S\t*\t0\t0\tTTTAAGTTCGACCATGTTTTTGGCCCAGAGGATGACCAAGCCAACCTTTTTTATGCAGAGGCCGTGTTTTCAGAAACAGTGCCCGTTGTTGGGTCAGTGATGGATGGTTTCAACGTATGCATCTTTGCATATGGGCAAACAGGAACCGGGG\tJJF<AJJJFA<AFAF77JJJJFJJFA--FJJJJJJFJFFFJJJJFAJJJAJJJJJFJJJFJFJJJJJJJJFJJFAJJJJJJ<FJJJJJJJFJAJA<JJJFJJJFJFJJJJJJJJJF<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFA-\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1211:27123:18318\t4\t*\t0\t0\t*\t*\t0\t0\tTTAGGATTTAACTTAGAATAAAGCCTGACGGGGGTAACAGCACCCATCGATCCGCAGTCCATGCTGGTTGATGCATGGGTCTGCTGGAGCCGAGGATGAACACAGCACTAACAACTGACACAGAGCGACCTTCAATCAAAAACACA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJFJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2113:8653:22362\t4\t*\t0\t0\t*\t*\t0\t0\tCCTGTGGTAACTTTTCTGACACCTCTAGCTTCAAACTCCGAAGGTCTAAAGGATCGATAGGCCACGCTATCACGGTTCGTATTCGTACTGGAAATCAGAATCAAACGAGCTTTTACCCTTTTGTTCCACACGAGATTTCTGTTCTCGTTGA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ-FJJJJJJJJJJJJJJJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1109:4584:27619\t16\t9\t87871408\t60\t150M1S\t*\t0\t0\tTTGAAGAACCATTCTTCTTATTATTAGGTGGCAAGTGGAGTTGGGTGATTGTCAGGGGCTGGGCAGAGATTTATTCCATCGTTTCCACCGTTGGTTCTTGTATTCAGTTGTAGCATGGCATCTGTGAAGGAACTTGAGTAGATGATTCGGC\tJJJJJAJJJJJFAJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAFAA\tAS:i:-2\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1218:13555:38486\t0\t5\t63828890\t60\t43M279N108M\t*\t0\t0\tTCCATTGTCTCGAGGAGCTCCATCACTCGTCATCATCTTCATCCTCCTCACTACCCTCTTCATCATCCTCGTCATTGACCTCCGACTTGGACTTGTCAGACTCCTCTTCATCTTCCTCCTCTTCCTCCTTGGCAGGAGCCTTCTTAGCTGC\tAAFFFJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJJJJFJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJ<<JF7FJFJFFJJJJJJJFJJJFJJJJJJJJJJJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1205:22506:8981\t16\t1\t293225372\t60\t14M462N86M1183N51M\t*\t0\t0\tCTAGAAAACTAAGGGTTTCTAGGTATCGTAGTTTTGAGGAGGAATTTAGCTCTCCGTTTGTCTCTGAGATTCAGAGGTATTTCACCGATGAAGACTACAGTTACGCAACGAACTTCTATATTCTACTACGTGCTGTTGATCGATTGGCTGC\tJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:62C88\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1207:25114:21272\t16\t1\t266222095\t1\t56M84N95M\t*\t0\t0\tATGTTGTGCTCCAATACAACTGTACCAATACAATTCCTGAACAATTGCTTGAACAGGTTACTGTTTTTGTTGATGCTTCTGAGGCTGATGAATTTTTAGAAGTTGCTTCGAAGCCTTTAGAATCATTACCATATGATTCTCCTGGGCAGAC\tFJJJJJJJFJFJAJJJJJFJJJJJF<JJJJJJJJJJJJFJJJJJJJJJJJJF<JJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJFFFA<\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:60T90\tYT:Z:UU\tXS:A:+\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1207:25114:21272\t256\t5\t15365\t1\t95M84N56M\t*\t0\t0\tGTCTGCCCAGGAGAATCATATGGTAATGATTCTAAAGGCTTCGAAGCAACTTCTAAAAATTCATCAGCCTCAGAAGCATCAACAAAAACAGTAACCTGTTCAAGCAATTGTTCAGGAATTGTATTGGTACAGTTGTATTGGAGCACAACAT\t<AFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJ<FJJJJJJJJJJJJFJJJJJJJJJJJJ<FJJJJJFJJJJJAJFJFJJJJJJJF\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:90A60\tYT:Z:UU\tXS:A:-\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1207:25114:21272\t256\t2\t232773258\t1\t95M84N56M\t*\t0\t0\tGTCTGCCCAGGAGAATCATATGGTAATGATTCTAAAGGCTTCGAAGCAACTTCTAAAAATTCATCAGCCTCAGAAGCATCAACAAAAACAGTAACCTGTTCAAGCAATTGTTCAGGAATTGTATTGGTACAGTTGTATTGGAGCACAACAT\t<AFFFJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJ<FJJJJJJJJJJJJFJJJJJJJJJJJJ<FJJJJJFJJJJJAJFJFJJJJJJJF\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:90A60\tYT:Z:UU\tXS:A:-\tNH:i:3\n",
      "K00364:134:H2YYHBBXY:1:1114:18883:22397\t0\t1\t226899829\t1\t1S20M81N130M\t*\t0\t0\tCGTTGTGCCATCCACATAAACCTTGTATATGAGGATTCTCAGCAGACCAAGAGCCCCTGCGAGGCTGCAGTCCGTCCACTGCACCAGGAAGAGGAATATATGAGCTGCGGGACTGTACGACAGCCGCATCTGCAGGCACGCACCATCATAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJF\tAS:i:-1\tZS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:-\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1114:18883:22397\t256\t1\t226686672\t1\t1S20M81N130M\t*\t0\t0\tCGTTGTGCCATCCACATAAACCTTGTATATGAGGATTCTCAGCAGACCAAGAGCCCCTGCGAGGCTGCAGTCCGTCCACTGCACCAGGAAGAGGAATATATGAGCTGCGGGACTGTACGACAGCCGCATCTGCAGGCACGCACCATCATAG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJF\tAS:i:-1\tZS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tXS:A:-\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1124:5061:38468\t0\t1\t178504291\t60\t111M100N40M\t*\t0\t0\tGTCCAGCTCCTTGCCGTTCATGCCCTCGATGGCGTTCCGCATCGCCTCCTCCGTGGAGAAGGTGACGAAGCCGAAGCCGCGGGACCTCTGCGTCTCCCGATCGAGGATGATCTTGGACTCGAGGACCTCGCCGTAGGTGCTGAAGGCGTTG\tAAFFFJJJJJJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFJJ7AJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJAJJJJJJ<JJJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJFJAFJF\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2101:29437:27180\t16\t9\t131113128\t60\t48M5D91M393N12M\t*\t0\t0\tTAGGAGGCCGCGCGTCGGGCGTGACTTGGTAGAGTGAGAAACTCGGGGAGAGAAGATGATGTCAAGGGAGCGCAAGAAGGCGGCTGCTGCTCTCCAGGAGAAGCTGAAGATCCTCCGCTCCATCACTCATTCCCACGCTCTGAGCAACACA\t-JF-JJFJJJJFFF-<JFJJJJJJAFJJJJFJFJJJJJJJFJFFJF-JFJJJJJFFJJFJJAJJFA7FJAFAFJAJJJJJJJJJJJJJJJFF7JJJJJJJJJJJFFJFFFJJJJJJJJJJJJFJAFJJJJJJJJJJFJJJJJJJJJFFFAA\tAS:i:-26\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:5\tNM:i:6\tMD:Z:48^AGAGA39G63\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2126:14133:38504\t16\t7\t130911674\t60\t12M972N139M\t*\t0\t0\tCTACTAGCAAAGATCAGGAGCTGGGGATTGCCCGGCCACTCTGCTGAAAGTTTTGGAGAAGTTGTGGTGTTTCCCTGCCTATTAGAGTCTGCGCTTCTGTGTTCCTGATCAACTGCTACTACGCGATTGTTGTCTTCTCAATCTGTCTGGC\t-FAF-AAAJA<A---7FF-A-7FF-AFJFA<-7-JAF-JA-J<7AJJFAFJAJAFJJAF<JAA7-7FJAF7-77-A7-<-<J<JJ<FJFAJJJFF<<JJJF<7AJF-F<-<-F7A<-JJ--JJFJAFFJFFJFJFAFA7A7A7A<F-<FAA\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:33T117\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2228:16894:41774\t16\t9\t115084083\t60\t74M2I74M1S\t*\t0\t0\tGCAGCATGTTCATCGTCTCACCAGGACACAACACTTGGAGCAAGACAGTCATCGGTCGTTCCATCGTACATACTTATATTGACACCAGAAAGCTGATTGTGATCATCTAGTTTATTACAGACATGAGAATAGCCAAATCGTGTTCCCGTCC\tJJJJJJJJJJJJJJJ7JAJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJAJF7-JJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJFFAAA\tAS:i:-28\tXN:i:0\tXM:i:3\tXO:i:1\tXG:i:2\tNM:i:5\tMD:Z:52A10C60C23\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2114:8410:36745\t16\t3\t202505950\t60\t9M92N142M\t*\t0\t0\tTATTGCCCAGGATGCAGAGGACTCTCCGAAGATGGAGGAGGCTGCTCCTGTTGCTGAGAAACAAGGTGAGCAGAATGATACCCCAGCAGCAGATGAGAACAAAGATAACAAGGATGCTACTGCAAATGAGGAGGAAGAAAAGGAAGAGGAC\tJJJJF7JJJ<JJJAF7JJJFFJJJJFJJJJJJJFJJFJJAJ<-JFJFJJJJJJJJJJJFAJJJJJJJJJJJJJJFJJJJAJFFJJJFJJJJJJJJJJJJJFJJJFJJFJJJA-JJJFJJJJJJFFJJJJJJFFJJFFJJJJJJJA-FFAAA\tAS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:79G71\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2113:12905:18335\t16\t5\t175314226\t1\t151M\t*\t0\t0\tCTCCTCCCCGCGCGTGGCTGCCGTGGGCTCCTGGGCGGTCTTCCTCCTCCTCGCCGTCGCCGCGCCCGCGGCCGTCCGCCTCTTGGGCACCGCTCCGCCGCGGCCCTTCGACGGGCAGGTCCAGGTCTCGCTCACCCTGGCCGCGTCGCTC\tJAJFJJJJJJJJJJFJFA-JJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFF-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:18G63C68\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:2113:12905:18335\t272\t5\t175282891\t1\t151M\t*\t0\t0\tCTCCTCCCCGCGCGTGGCTGCCGTGGGCTCCTGGGCGGTCTTCCTCCTCCTCGCCGTCGCCGCGCCCGCGGCCGTCCGCCTCTTGGGCACCGCTCCGCCGCGGCCCTTCGACGGGCAGGTCCAGGTCTCGCTCACCCTGGCCGCGTCGCTC\tJAJFJJJJJJJJJJFJFA-JJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFF-JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tZS:i:-6\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:18G63C68\tYT:Z:UU\tNH:i:2\n",
      "K00364:134:H2YYHBBXY:1:1101:14854:1666\t0\t6\t136703478\t60\t151M\t*\t0\t0\tGGGGGGGATACGCGGAGAATGTCTCCACCACCATAGGCTTGGTGGGAATCATCTTCACCATACCAGCATCACCGTTCTTCAGGAACTTGGGCTCCTTCTCGAGCTCCTTGCCAGATCGCCTGTCGATCTTGGTAATGAGCTCAGCAAACTT\tAAFFFJJJJJJFJJJJFJJJJFJJJJJJJJJJJJJJJJFJAF<JJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFF<FFJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJFJF<JFJJFJAFAJJFAFJJ<FFFJJ<FAJJJJFFJJF-\tAS:i:-5\tZS:i:-5\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:4A146\tYT:Z:UU\tNH:i:1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "K00364:134:H2YYHBBXY:1:2124:8663:12497\t0\t8\t14593840\t60\t151M\t*\t0\t0\tGCGCCGAAGGTCCAGTAGTAGGCACAGTTCCTGAAGACGTTCGAGAGCGGAGACGTCGCGTGGCTGAAGCGGTGCACGAAGAACGTCTCCATGATCCGCTTGAAGTAGTGGAAGCACCAGTAGTACATGGCGTAGGTCTGGACAGGGTGCA\tAAFFFJJFJJJJJJJJJJJ-JFJJJJJJFJJJJJJJJJJ<FFJJJJJJJJJJJJJFJJJJJJJJJAJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJ<FJJAFJJJAJJJJAFFJ\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:50C77T22\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1214:10673:44535\t4\t*\t0\t0\t*\t*\t0\t0\tCCCCAACTTTCGTTCTTGATTAATGAAAACATCCTTGGCAAATGCTTTCGCAGTTGTTCGTCTTTCATAAATCCAAGAATTTCACCTCTGACTATGAAATACGAATGCCCCCGACTGTCCCTATTAATCATTACTCCGATCCCGAAGGCCA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1114:20111:13377\t4\t*\t0\t0\t*\t*\t0\t0\tCCCGGCTCGTTGACGGCGCCTCGTGGTGCGACAGGGTCCGGGCCGGACGGGGCTCTCACCCTCCCAGGCGTCCCTTTCCAGAGAACTTGGGCCCGGTCCGTCGCTGAGGACGCCTCTCCAGACTACAATTCGGGCGGCGAGGCCGCCCGAT\tAAFFFJJJJJJJJJJJJJJJJJJ-JJJ7JJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJJJJJFFJJJFFJJJJFJJJJJJJJJFJJJJJJJJJJJJFJJJJ-A<FFFFFJ7AJJJAJJJJJJJJJFFFFAAJFFFJJF\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2107:12276:45484\t16\t1\t66586671\t60\t151M\t*\t0\t0\tTTTGATTGGGTCTCCGAGTAGAGTGTGTGCTGAAGTTATTTGATCTACATGAAGTGTTGAGGTACTTAGTTTAATTTATGCTGCCAGGTACTGTAAAACTGAAAGATACTGGTTCTAGTTTTGAGTCTATGGAGAATACTGTTACACCGTC\tJJJJJFJJJJJJFAAJFJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1219:27001:13253\t16\t4\t150604160\t60\t151M\t*\t0\t0\tTAGAGGAGAATTTTCAGAAAAGGAAATCTACATGCCCGAGTGACCATGTAAATATTTACCATTGCAGTTCGCTGACCCATGAGTACCAAAAAAAAGAGATGATTCATATTGACCTGTAACATTGTAAGATCAACCTCAGGAATATTTGAAG\tJJJJJFAAJJJJJFJJJJJJJJJJJJJJJJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJAFFJJFJJAJJJJJJJJJJJJFJJJJJJJJJJJJJJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:72G78\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2124:9181:13957\t16\t2\t176889684\t60\t127M3D24M\t*\t0\t0\tCAAGGCCCCGCTACCCAAGGCGATCCTGAAGAGGAAGAAGGCGCAGTCCTACTGGTCATGGGCTGGCCCAGCTGCCATGGTGGCTGCTGTGCTAGTTGCCCTCCTCGCGGTGCTGGGATACTACCAGTACTGCCTCCCAGCGAATACCAGC\tAJJJAFJFJJJ<JJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJFJJJJJJJJJJJJJJJJJF-AJJJJJJJJJJJJAFFFAA\tAS:i:-29\tXN:i:0\tXM:i:3\tXO:i:1\tXG:i:3\tNM:i:6\tMD:Z:42T62T21^TAC4A19\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2124:6380:36218\t0\t10\t113848451\t60\t151M\t*\t0\t0\tCTCTGATCAGATCTCGAGCGTGGAAGCTTACAGCTGGATTATCAGGGAACTTCAAGCCCTGTGAGATCACATTTGAGAGTGTTTCCTCATTGCCAGGTCCCTTGAATGGTGTCTTGCCATAGAGTAATTCGTAAAGGAAGATTCCAAGAGT\tAAAFFJJJJJJJJJJJFJJJJAJF<FJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJAJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJFFJJJJJJJFJFJAJFJJJJJJJJJFJJJJJ7AFJJJFJ<AFFA-\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1206:20527:39981\t16\t1\t55391686\t60\t151M\t*\t0\t0\tGATCCTGGTCTACGATTACATGGCCCATGGTACTCTCCGTGAGCACCTGTACAAGACCCAGAAGCCACCTCTTACCTGGAGGCAGCGTTTGGATATCTGTATTGGTGCAGCTCGTGGGCTTCACTACCTTCACACTGGTGCAAAGCACACC\tJJJJJJJJJJJJJFFJJJJJJJJJJJ7JJJJJ<<FJJJJJJAJAJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJFFF<A\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:151\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2208:19806:7082\t16\t8\t139412007\t60\t137M\t*\t0\t0\tAGAAAAGCGTGCTCTTTTTTCATCCTTCGTGCGTGGTGCGCGAGCGTGCGCCTCCCTTACCATCTCGCGATCCCAATTCGATCCGCTCGCCAGCTCTCCCCATCAGACCTGCCCTCCCCGGCCGGGCGATTCGCCGC\tJJFAFAJJJFFAAJJJJJJJFJFJJJFJF-FJJJJJJJJJJJJJJJJJJJJ7FJJJFJJJFFFJFJJJJJJJJJJJJJJFAJJJJJJJJJJJJJJJJJJJJJAFFJJJJF<FJJJJJJJJJJJJJJJJFFJJFFFAA\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:137\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1201:22049:33440\t0\t5\t170405522\t60\t151M\t*\t0\t0\tTCGGTCTTGAAGTCCTGCGCGATCTCACGGACGAGGCGCTGGAAGGGCAACTTGCGGATGAGCAGCTCCGTGCTCTTCTGGTACTTGCGGATCTCCCGGAGCGCGACGGTGCCGGGGCGGAAGCGGTGGGGCTTCTTCACGCCGCCGGTGG\tAAAFFJJJFJJFJJJJJJJJJ<FJJJJJFJJJJJ<JFJJJJFJAJAFJJFJJJ7JJJAJJJJJF<A<AJJJJF<JJFFJJJJJJJJJJJJJFJJJJJJFJJJFJJFJJFAJFFJJ7FJFJFFFFFFJJJJ7JJJJJJFJJFFJJJJJJJJJ\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:86A2A61\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1203:21887:36816\t0\t2\t49686280\t60\t72M160N53M117N26M\t*\t0\t0\tCTTCACGTCGTTCTCCGGCACGTTCGCGTCGGCGCCGTTGCCTCGTGCCGTGCCTCCGCCTGCTTTACGGACGTTCTCGTCAGGGACAATGGCGTATCGTTCTGACGGCGACACCCAAACTTTGGCAGACCTGCGAACGGGGCTGCCGTGG\tAA<FFJJJFFJJJJFJJJJFJFJJJJJJJJFJJJJJJJJJJJJJJJJJJFJJJJJJFJJJJJJJJAAJJFFJJJFJ<JJJJJ<JFAJJAJJJJJFJJJFJFJJJJJJJJJJJJJAAJJJJJJJJJJJJJJJJ<FFAFJJJ7JJJJJJJJ-<\tAS:i:-12\tZS:i:-27\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:87T49T13\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2112:2402:30239\t0\t1\t220254762\t60\t2S149M\t*\t0\t0\tCTTCACATTTTGGAGTCTAGACTGTTTAGTTCCACCCAAATTTCAATGCACATGGTTCAACAATGTCCCAAGTTGACACAACTTATTGATTGATTAAACTGATAAAACTCAAGCCATGTCAATACTGACCACTGATCCAGAACAAACAAGG\tAAAFFJJFJJJJJFFFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJ7JJJJJJJJJJJJJJFJJJJJJJFJJJJJJJJJFJJJJJJFFFJJAAJJJJJFJJJJJAFAAFFJAJ7J-FFJFFFJJJJJJJJJJJJJJJ<\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:117A4G26\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2109:23257:41792\t16\t1\t149968606\t60\t2S54M400N53M426N42M\t*\t0\t0\tCTTGGGCGTTCTCGGCGGCGCCATCCTCGCCCACGCCGCCTACGCCACCATCCAGTACCGGGCGGTGCTCAAGATCACGGAGGAGGAGTTCTCGCGGCCGCCCATGGATGTGATGATGGAGTTGCTCCTAGGGTTGGCTTTGTGCATGTGG\tAJFJJJJJJJJFJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJFFJJAJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:149\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2103:9181:18247\t0\t7\t95932238\t60\t1S23M1256N43M87N84M\t*\t0\t0\tGGTCCATCTTTACTCCAACAAAACCTTAGATGCTGCTCGGGTACATGAACACTCTGACCTTGCGCCCCATGGACTCAGGCGGGAGGTTGGACTTGAACTTGGCACGGACGACGCCCGAGTTTCCGTGCGGACGGGTGACCTTGCCCCAGAT\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJJJJFJJJJJJJFJ-JJJJJJJJJJJJJJ\tAS:i:-7\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:91A58\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2104:4594:31857\t0\t6\t6903539\t60\t4S99M90N26M\t*\t0\t0\tGCAGCCTTGGCAGCACCGGTGCTGCTGGGAATGATGTTAAAGCTGGCAGCCCTGCCACCTCTCCAGTCCTTGGCTGAGGGTCCATCAACAGTCTTCTGGGTGGCAGTGATGGCATGAACAGTTGTCATC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ\tAS:i:-4\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:125\tYT:Z:UU\tXS:A:-\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2213:30472:10721\t4\t*\t0\t0\t*\t*\t0\t0\tCTATGATGTTATCCCATGCTAATGTATCCAGAGCGATGGCTTGCTTTGAGCACTCTAATTTCTTCAAAGTAACGGCGCCGGAGGCACGACCCGGCCAGTTAAGGCCAGGAGCGCATCGCCGGCAGAAGGGTCGAGCCGGTCGGTTCTCGCC\t-AFFFJJJJJJJJFJJFJJJ-JFJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJFJJJJJJJJJJJFAJJF-FJJJJJJFAJFJJ<JJFJJJJFJJFA-AA<FJJJJFF7FJJJ<<FJJJFJFFJA-F<FFJJ))FFAF-JJAA7FAJAFJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2204:27417:42143\t16\t4\t139468554\t60\t150M1S\t*\t0\t0\tCAGCTCTCCGGCTGCCTCCCCCACGAGCTCGGCATGCTCACCAAGGCGGCCGTGATTGACGCCGGCATGAACCAGCTCACCGGCCCAATCCCGTCATCTTTCTCGTGCCTAATCAGCGTCGAGCAGCTCAACCTGGCCGGAAACCGCCTCA\tA<<))FAAJJJJJ<-)JJJJJJFJ<-A7JJF<JFJJ7FFJJFFFFFJ<JJJJJJJJJJ7JJJJJFJAJ<JJF<AFAJFJFJJFFJJFFJJFJJFFFJFF7JJJJJJJJJJJFJJJJJJJJFJJJFJJJJJJJJJFJJJFJJJJJJJFFF<A\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1112:25682:14344\t16\t4\t145389838\t60\t5S47M182N76M402N22M1S\t*\t0\t0\tCATCTGTGCAAAAGATTCAGGCACTAACACCAAATATTGGAGTTGTCTATAGTGGGATGGGTCCAGATTTTCGTGTTCTGGTGAGAAAAAGTCGAAAGCAGGCACAGCAGTATTATCGGTTGTACAAGGAGCATATACCTGTAACACAGCA\tFFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFFFAA\tAS:i:-9\tZS:i:-13\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:145\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1103:22567:47172\t0\t8\t1371445\t60\t151M\t*\t0\t0\tCAATGAAGGTGGAGGCCATGGAGAGGCCACGCGGCGGGATATCACAAACACTGGACTTCACATTGTTGGGGATCCACTCCACGAAGTAGGACGAGTTCTTGTTCTGGACATTGATCATTTGCTCGTCCACCTCCTTGGTGCTCATCTTTCC\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJFJJJJFJJJJJJJJJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAJJAJJJJJJJJJJJJFJFJJJJ\tAS:i:-6\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:28G122\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1113:4533:6853\t0\t4\t141629364\t60\t151M\t*\t0\t0\tACCCGGTCACTCCTTAATCCGGGGCCAGCGTCCGAGCTGTGACGGGTTGACGACGCCGGGGATGCGGGCCCCGCCCGTCATAAGCAGGCGCGAAACGTCCGCGTAGCGCCTGCTAGGCTCCGGAGCGGGGGTTGGGGCGGGGGAGGGGGCG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ<JJJJJJFJJJJJJ<JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJ\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:22A91C36\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2106:31527:1472\t16\t2\t221211802\t60\t15M85N135M1S\t*\t0\t0\tGTGATTGCTAACATGGACGCGACCGCCAATGACGTGCCCAGTGAGTTCGAGGTCCAAGGCTACCCAACCATGTACTTCGTGACTCCCAGCGGGAAGGTCACCTCCTACGACAGTGGCAGGACGGCTGATGACATCGTCGACTTCATCAACC\t7<JJJAJ7A<--FA7)-))A-)FF<FAAJA-JJJFJAFFAAA7AAA7-7F-7FJJJJJJJ<JJJJJAJJFFJAFFJFFJA7A7-77FJAFJ<JFFF7-FJJFFJAJF<<F77-<7-AJFJJJA-<-FFJJJJF7-A<<FJA-FAA-FFAAA\tAS:i:-7\tZS:i:-22\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:11G113G24\tYT:Z:UU\tXS:A:+\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2216:8806:46891\t4\t*\t0\t0\t*\t*\t0\t0\tGTCGTCTGCAAAGGATTCAGCACGCCGCCCGTTGGGAAGGGAGCTTCGAGGCGGCCCGCCGCGGCGCGTCGGCCGGGCGGGCTGAGCCAATGGCACGGGCCCTTGGGGCGCGAACGCCCTAACGTGGGTCGGGGCGGGCGGCGAGCAGAGG\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJFJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJFFJJJJJJJJJFJJJ<FJAJJJJ\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:1221:29498:18071\t4\t*\t0\t0\t*\t*\t0\t0\tGTCCAACTACGAGCTTTTTAACTGCAACAACTTAAATATACGCTATTGGAGCTGGCATTACCGCGGCTGCTGGCACCAGACTTGCCCTCCAATGGATCCTCGTTAAGGGATTTAGATTGTACTCATTCCAATTACCAGACACTAACGCGCC\tAAFAFJJJJJFJJJJJJJJFJJJJJJJF-FFFJJ<F<-F<FJFJJ<-FFJJJJJF--<F-JFFJ-AJFAJAJF--77<JJJFAJ7JJAA--<FJJFAAAFJFFFAJJJJ-F<F-AFJFFF--7F-7AFJFFF<---FAAJ7-7A-A--7AA\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2101:32025:39471\t0\t5\t19459767\t60\t137M\t*\t0\t0\tCTCTCAATGATCTTTTGGCGGCCAGGGAACTTGAACTTAGCGCGACGCAGAGCTTCGCTGGCATGGGCAGCATTGTTGTCCTTGCATCGCACGGAAAGGAGGACCTGACCAATGTCCACCCTAGCACAGGTGCCCTG\t<AAFFFJFJFFJJFJJFJJJJAAJ<JJ7FJFAFAFJJJFF7FFFFJJ7FJJJF<FFJAJJJJAFAJJA7<AAJJAJFJJFFAFJJJFJJJJJAJJFJJJJJJA7A<JFF7FJJJJF7<F7<AFFFAA<FJJ<)))7A\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:137\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2108:8643:3635\t16\t3\t181560599\t60\t151M\t*\t0\t0\tGCACCAAGGCCGTTACCAAGTTTACTTCCTCTTAGATCGCCGCCGTACGGAGCGACGGCTCTGTTTCGTTTCCTCGTACTTTTAGGCCAGTGCTAATAGGTTGCTCGAGGAACAATGTAGTGTGGAAATTTCTAATATATTCCAAATTGTG\tJJJJJJJJJJJJJJJJJAJJJJJFJJJJJJJJJJJJJJJJJJFFA-JJJJJFJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-3\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:45C105\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2209:17817:37607\t16\t1\t296349088\t60\t151M\t*\t0\t0\tCAAAGCTTTGAGAATCATGTAGTCATGCATTAGATTCACATGCACCAAATTTAAGCTAACTGGAAAGTGGGATAATATTTTTTTATCTCGTGCAAATAACTATTAAACTGCTCAAGTTTCTTTATTTGGTAATTAATTAAATTTTTCAACC\tAJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJ<JJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFFAAA\tAS:i:-12\tXN:i:0\tXM:i:2\tXO:i:0\tXG:i:0\tNM:i:2\tMD:Z:73C32G44\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2227:21663:30837\t16\t5\t3946891\t60\t150M1S\t*\t0\t0\tCTGGTTTAGATCACGCTACTGAGTTTATGGACTTCCCATATTTCACTAATAGTGAAGACCTTGATATCCTGAATAGTGAAAATCTTTTGAATTCTCCAAGTGAGGGAATCCAGGAGGATTTAGACGATACTGCATTAGTTGTTCATGGGGC\t7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJAJJJJJJJJJJJJJJJ7JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJFFFAA\tAS:i:-1\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:150\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1113:21095:33721\t0\t9\t12329150\t60\t86M3I62M\t*\t0\t0\tGACAGTAGGTCTTAGTTCAACAAATACACTCATCAAAAGGTTTTACTGCTACTTATAAAGTAGACACAAGATAATAGGTATATAGAAATAAGTTATGAGAACCTTAACCAAAGTCAAGCAGTCGACGAAGGATCTTTCAAGCAGGTTATTG\t-AFFFJJJJJJJJJJFFJFJJJJJJJJJJJJJJJJAJJJJFJJFJJJJJJJJJJJJFJJJJJJJJFFJJ<JJJAJJJJJJJJFJJJJJJJJJFFJJAJJJJJJJJJJJJJJJJJJJJJJJJ<J<-AJJJJJAAFFJFJJJJJJJJJJFJJF\tAS:i:-20\tXN:i:0\tXM:i:1\tXO:i:1\tXG:i:3\tNM:i:4\tMD:Z:132G15\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:1212:32045:4620\t4\t*\t0\t0\t*\t*\t0\t0\tGCTGGAATTACCGCGGCTGCTGGCACCAGACTTGCCCTCCAATGGTTCCTCGTTAAGGGATTTAGCTTGTACTCATTCCAATTACCAGACACTAACGCGCCCGGTATTGTTATTTATTGTCACTCCCTCCCCGTGTCAGGATTGGGTAATT\tAAAFAFAAJJJJFJAJ-FJ<AJ<FJJAAFJJJFF-FFJJJJ7FJF-<<-AAAJ-F-FAJ7FJJJA-F-FAAFJJ-AAA<AFJ<FFAFF-7AA7-7AJAFJFJF<J7AFAAA7-FF7AJFJJ7-7-7<7FFFF<AFJJA<F-<FAJAF7--<\tYT:Z:UU\n",
      "K00364:134:H2YYHBBXY:1:2222:26311:48139\t0\t9\t122485595\t60\t96M\t*\t0\t0\tGGACCTCCAATCCAAAGGAGAGGAGAGAACAATCCCCTCCAAGGAAACGCAACGCAGTGAACGCAAATCAAATGGACGGGGTGAACGGAGGTGGAA\tAAFFFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJFFAJJJ\tAS:i:0\tXN:i:0\tXM:i:0\tXO:i:0\tXG:i:0\tNM:i:0\tMD:Z:96\tYT:Z:UU\tNH:i:1\n",
      "K00364:134:H2YYHBBXY:1:2102:15919:33440\t16\t5\t938669\t60\t136M\t*\t0\t0\tCCCAACATCCACACTGTGCTCCTCCCCAAGAAGGTGGCCGAGAAGGCGGCCAAGGAGCCCAAGAAGGCCGCTAAGTCCCCCAAGAAGGCGTAGAGTGATCTCCTGTCTTAGAACAATATTACCATTAGCAATCGTG\tJJFFFJFAJF<FFFA(ERR): hisat2-align died with signal 2 (INT) \n",
      "(alignment) "
     ]
    }
   ],
   "source": [
    "# use HiSAT2 to align your reads to the genome\n",
    "# HiSAT2 is a fast, splicing aware aligner that can accept cDNA inputs\n",
    "\n",
    "# lets try running HiSAT2 on just one file\n",
    "# but we will cancel it after just a few seconds\n",
    "# use control+c to terminate the file after you have seen its output\n",
    "# how do we determine the options to use?\n",
    "\n",
    "hisat2 --min-intronlen 60 --max-intronlen 50000 \\\n",
    "    -p 4 -x ./maize_genome/ZmaysAGPv3.30.hisat2 \\\n",
    "    -U ./trimmed_reads/WT10_S8.fastq.gz\n",
    "    \n",
    "# this will output a SAM file directly to the command line"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# use HiSAT2 to align your reads to the genome\n",
    "# HiSAT2 is a fast, splicing aware aligner that can accept cDNA inputs\n",
    "\n",
    "mkdir ./aligned_reads # make destination folder\n",
    "mkdir ./HiSAT2_report # just for summary stats\n",
    "\n",
    "#min intron 60 max intron 50000 from Chen et al. 2018\n",
    "#running in unstranded mode - input files with -U\n",
    "#output to command line then pipe to samtools view\n",
    "#and capture stdout with > to make a BAM file\n",
    "#capture stderr with 2> to make a report file\n",
    "find ./trimmed_reads -name \"*.fastq.gz\" | while read f; do\n",
    "\tSTEM=$(basename \"${f}\" .fastq.gz)\n",
    "\thisat2 --min-intronlen 60 --max-intronlen 50000 \\\n",
    "\t-p 4 -x ./maize_genome/ZmaysAGPv3.30.hisat2 \\\n",
    "\t-U ./trimmed_reads/\"${STEM}\".fastq.gz \\\n",
    "    2> ./HiSAT2_report/\"${STEM}\"_summary.txt \\\n",
    "    | samtools view -bS - \\\n",
    "    > ./aligned_reads/\"${STEM}\".bam\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) total 574M\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 WT10_S8.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:07 WT13_S12.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 WT18_S4.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:08 WT24_S10.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:06 WT25_S7.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 WT38_S5.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 WT41_S6.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 WT55_S9.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:08 WT58_S3.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 WT78_S2.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:06 WT93_S11.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:10 WT95_S13.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:07 WT97_S1.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 dbl19_S46.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 dbl28_S48.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:10 dbl31_S41.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 dbl50_S43.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:10 dbl54_S44.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 dbl56_S50.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 dbl57_S52.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 dbl65_S42.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 dbl66_S49.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:10 dbl70_S47.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 dbl88_S51.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:07 dbl89_S40.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 dbl9_S45.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:09 lgn17_S35.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 lgn21_S36.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:07 lgn28_S37.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 lgn33_S38.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 lgn37_S33.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:10 lgn39_S27.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 lgn3_S39.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 lgn42_S34.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 lgn46_S30.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 lgn59_S28.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:06 lgn72_S31.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:09 lgn77_S29.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:09 lgn8_S32.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:06 nod11_S24.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 nod14_S18.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:07 nod16_S15.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:09 nod2_S26.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:06 nod30_S17.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 nod36_S21.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:10 nod43_S22.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:10 nod47_S23.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:07 nod48_S16.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:10 nod60_S19.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:08 nod61_S25.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 11M Oct 16 00:08 nod62_S14.bam\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 12M Oct 16 00:08 nod6_S20.bam\n",
      "(alignment) total 208K\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 WT10_S8_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 WT13_S12_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 WT18_S4_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 WT24_S10_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 WT25_S7_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 WT38_S5_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 WT41_S6_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 WT55_S9_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 WT58_S3_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 WT78_S2_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 WT93_S11_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 WT95_S13_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 WT97_S1_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 dbl19_S46_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 dbl28_S48_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 dbl31_S41_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 dbl50_S43_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 dbl54_S44_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 dbl56_S50_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 dbl57_S52_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 dbl65_S42_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 dbl66_S49_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 dbl70_S47_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 dbl88_S51_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 dbl89_S40_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 dbl9_S45_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 lgn17_S35_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 lgn21_S36_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 lgn28_S37_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 lgn33_S38_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 lgn37_S33_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 lgn39_S27_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 lgn3_S39_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 lgn42_S34_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 lgn46_S30_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 lgn59_S28_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 lgn72_S31_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 lgn77_S29_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 lgn8_S32_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 nod11_S24_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 nod14_S18_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 nod16_S15_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:09 nod2_S26_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:06 nod30_S17_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 nod36_S21_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 nod43_S22_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 nod47_S23_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:07 nod48_S16_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:10 nod60_S19_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 nod61_S25_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 nod62_S14_summary.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 207 Oct 16 00:08 nod6_S20_summary.txt\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#check on your alignments and reports\n",
    "ls -lh ./aligned_reads \n",
    "ls -lh ./HiSAT2_report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) 99844 reads; of these:\n",
      "  99844 (100.00%) were unpaired; of these:\n",
      "    23468 (23.50%) aligned 0 times\n",
      "    71910 (72.02%) aligned exactly 1 time\n",
      "    4466 (4.47%) aligned >1 times\n",
      "76.50% overall alignment rate\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# what does the report look like?\n",
    "# we can use cat to lump the text together and output it to the command line\n",
    "\n",
    "cat ./HiSAT2_report/WT10_S8_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) 0/52 -  0.00% of files done   0.000/573.035 MB -  0.00% of file sizes done\n",
      "Processing lgn46_S30.bam - 11.339 MB   2019-10-16.00:12:05\n",
      "1/52 -  1.92% of files done   11.339/573.035 MB -  1.98% of file sizes done\n",
      "Processing dbl54_S44.bam - 10.911 MB   2019-10-16.00:12:07\n",
      "2/52 -  3.85% of files done   22.251/573.035 MB -  3.88% of file sizes done\n",
      "Processing dbl19_S46.bam - 11.164 MB   2019-10-16.00:12:09\n",
      "3/52 -  5.77% of files done   33.414/573.035 MB -  5.83% of file sizes done\n",
      "Processing dbl50_S43.bam - 11.369 MB   2019-10-16.00:12:11\n",
      "4/52 -  7.69% of files done   44.783/573.035 MB -  7.82% of file sizes done\n",
      "Processing lgn39_S27.bam - 11.058 MB   2019-10-16.00:12:14\n",
      "5/52 -  9.62% of files done   55.841/573.035 MB -  9.74% of file sizes done\n",
      "Processing lgn28_S37.bam - 10.736 MB   2019-10-16.00:12:15\n",
      "6/52 - 11.54% of files done   66.577/573.035 MB - 11.62% of file sizes done\n",
      "Processing WT95_S13.bam - 10.384 MB   2019-10-16.00:12:18\n",
      "7/52 - 13.46% of files done   76.960/573.035 MB - 13.43% of file sizes done\n",
      "Processing lgn33_S38.bam - 11.043 MB   2019-10-16.00:12:18\n",
      "8/52 - 15.38% of files done   88.003/573.035 MB - 15.36% of file sizes done\n",
      "Processing WT41_S6.bam - 11.075 MB   2019-10-16.00:12:21\n",
      "9/52 - 17.31% of files done   99.078/573.035 MB - 17.29% of file sizes done\n",
      "Processing dbl65_S42.bam - 11.058 MB   2019-10-16.00:12:23\n",
      "10/52 - 19.23% of files done   110.136/573.035 MB - 19.22% of file sizes done\n",
      "Processing WT18_S4.bam - 11.027 MB   2019-10-16.00:12:24\n",
      "11/52 - 21.15% of files done   121.163/573.035 MB - 21.14% of file sizes done\n",
      "Processing dbl9_S45.bam - 11.217 MB   2019-10-16.00:12:25\n",
      "12/52 - 23.08% of files done   132.380/573.035 MB - 23.10% of file sizes done\n",
      "Processing lgn37_S33.bam - 11.237 MB   2019-10-16.00:12:28\n",
      "13/52 - 25.00% of files done   143.617/573.035 MB - 25.06% of file sizes done\n",
      "Processing dbl57_S52.bam - 11.129 MB   2019-10-16.00:12:28\n",
      "14/52 - 26.92% of files done   154.746/573.035 MB - 27.00% of file sizes done\n",
      "Processing WT93_S11.bam - 10.945 MB   2019-10-16.00:12:30\n",
      "15/52 - 28.85% of files done   165.691/573.035 MB - 28.91% of file sizes done\n",
      "Processing WT97_S1.bam - 10.918 MB   2019-10-16.00:12:32\n",
      "16/52 - 30.77% of files done   176.609/573.035 MB - 30.82% of file sizes done\n",
      "Processing nod62_S14.bam - 10.920 MB   2019-10-16.00:12:33\n",
      "17/52 - 32.69% of files done   187.529/573.035 MB - 32.73% of file sizes done\n",
      "Processing WT24_S10.bam - 10.475 MB   2019-10-16.00:12:35\n",
      "18/52 - 34.62% of files done   198.004/573.035 MB - 34.55% of file sizes done\n",
      "Processing WT13_S12.bam - 10.976 MB   2019-10-16.00:12:37\n",
      "19/52 - 36.54% of files done   208.980/573.035 MB - 36.47% of file sizes done\n",
      "Processing dbl70_S47.bam - 11.185 MB   2019-10-16.00:12:38\n",
      "20/52 - 38.46% of files done   220.165/573.035 MB - 38.42% of file sizes done\n",
      "Processing dbl89_S40.bam - 10.737 MB   2019-10-16.00:12:39\n",
      "21/52 - 40.38% of files done   230.902/573.035 MB - 40.29% of file sizes done\n",
      "Processing lgn77_S29.bam - 11.101 MB   2019-10-16.00:12:41\n",
      "22/52 - 42.31% of files done   242.003/573.035 MB - 42.23% of file sizes done\n",
      "Processing nod14_S18.bam - 11.011 MB   2019-10-16.00:12:42\n",
      "23/52 - 44.23% of files done   253.013/573.035 MB - 44.15% of file sizes done\n",
      "Processing dbl28_S48.bam - 11.155 MB   2019-10-16.00:12:42\n",
      "24/52 - 46.15% of files done   264.169/573.035 MB - 46.10% of file sizes done\n",
      "Processing WT78_S2.bam - 11.166 MB   2019-10-16.00:12:43\n",
      "25/52 - 48.08% of files done   275.335/573.035 MB - 48.05% of file sizes done\n",
      "Processing dbl66_S49.bam - 11.158 MB   2019-10-16.00:12:44\n",
      "26/52 - 50.00% of files done   286.492/573.035 MB - 50.00% of file sizes done\n",
      "Processing WT55_S9.bam - 11.113 MB   2019-10-16.00:12:45\n",
      "27/52 - 51.92% of files done   297.605/573.035 MB - 51.93% of file sizes done\n",
      "Processing WT10_S8.bam - 11.149 MB   2019-10-16.00:12:46\n",
      "28/52 - 53.85% of files done   308.754/573.035 MB - 53.88% of file sizes done\n",
      "Processing nod16_S15.bam - 11.255 MB   2019-10-16.00:12:47\n",
      "29/52 - 55.77% of files done   320.009/573.035 MB - 55.84% of file sizes done\n",
      "Processing nod43_S22.bam - 11.238 MB   2019-10-16.00:12:48\n",
      "30/52 - 57.69% of files done   331.247/573.035 MB - 57.81% of file sizes done\n",
      "Processing nod61_S25.bam - 10.830 MB   2019-10-16.00:12:48\n",
      "31/52 - 59.62% of files done   342.076/573.035 MB - 59.70% of file sizes done\n",
      "Processing WT25_S7.bam - 10.911 MB   2019-10-16.00:12:49\n",
      "32/52 - 61.54% of files done   352.987/573.035 MB - 61.60% of file sizes done\n",
      "Processing nod48_S16.bam - 10.942 MB   2019-10-16.00:12:50\n",
      "33/52 - 63.46% of files done   363.929/573.035 MB - 63.51% of file sizes done\n",
      "Processing dbl31_S41.bam - 10.977 MB   2019-10-16.00:12:51\n",
      "34/52 - 65.38% of files done   374.906/573.035 MB - 65.42% of file sizes done\n",
      "Processing nod6_S20.bam - 11.174 MB   2019-10-16.00:12:52\n",
      "35/52 - 67.31% of files done   386.081/573.035 MB - 67.37% of file sizes done\n",
      "Processing lgn21_S36.bam - 11.078 MB   2019-10-16.00:12:53\n",
      "36/52 - 69.23% of files done   397.158/573.035 MB - 69.31% of file sizes done\n",
      "Processing nod11_S24.bam - 10.831 MB   2019-10-16.00:12:54\n",
      "37/52 - 71.15% of files done   407.990/573.035 MB - 71.20% of file sizes done\n",
      "Processing nod36_S21.bam - 11.184 MB   2019-10-16.00:12:55\n",
      "38/52 - 73.08% of files done   419.174/573.035 MB - 73.15% of file sizes done\n",
      "Processing dbl88_S51.bam - 11.214 MB   2019-10-16.00:12:56\n",
      "39/52 - 75.00% of files done   430.388/573.035 MB - 75.11% of file sizes done\n",
      "Processing dbl56_S50.bam - 11.126 MB   2019-10-16.00:12:57\n",
      "40/52 - 76.92% of files done   441.514/573.035 MB - 77.05% of file sizes done\n",
      "Processing lgn59_S28.bam - 11.045 MB   2019-10-16.00:12:58\n",
      "41/52 - 78.85% of files done   452.559/573.035 MB - 78.98% of file sizes done\n",
      "Processing lgn8_S32.bam - 10.792 MB   2019-10-16.00:12:59\n",
      "42/52 - 80.77% of files done   463.350/573.035 MB - 80.86% of file sizes done\n",
      "Processing nod47_S23.bam - 11.049 MB   2019-10-16.00:13:00\n",
      "43/52 - 82.69% of files done   474.399/573.035 MB - 82.79% of file sizes done\n",
      "Processing lgn42_S34.bam - 11.154 MB   2019-10-16.00:13:01\n",
      "44/52 - 84.62% of files done   485.554/573.035 MB - 84.73% of file sizes done\n",
      "Processing lgn72_S31.bam - 11.044 MB   2019-10-16.00:13:02\n",
      "45/52 - 86.54% of files done   496.598/573.035 MB - 86.66% of file sizes done\n",
      "Processing lgn3_S39.bam - 11.345 MB   2019-10-16.00:13:03\n",
      "46/52 - 88.46% of files done   507.943/573.035 MB - 88.64% of file sizes done\n",
      "Processing WT38_S5.bam - 11.206 MB   2019-10-16.00:13:04\n",
      "47/52 - 90.38% of files done   519.149/573.035 MB - 90.60% of file sizes done\n",
      "Processing nod2_S26.bam - 10.549 MB   2019-10-16.00:13:04\n",
      "48/52 - 92.31% of files done   529.698/573.035 MB - 92.44% of file sizes done\n",
      "Processing WT58_S3.bam - 10.732 MB   2019-10-16.00:13:05\n",
      "49/52 - 94.23% of files done   540.430/573.035 MB - 94.31% of file sizes done\n",
      "Processing nod60_S19.bam - 10.921 MB   2019-10-16.00:13:06\n",
      "50/52 - 96.15% of files done   551.351/573.035 MB - 96.22% of file sizes done\n",
      "Processing lgn17_S35.bam - 10.823 MB   2019-10-16.00:13:07\n",
      "51/52 - 98.08% of files done   562.173/573.035 MB - 98.10% of file sizes done\n",
      "Processing nod30_S17.bam - 10.861 MB   2019-10-16.00:13:08\n",
      "(alignment) 50/52 - 96.15% of files done   0.010/0.010 MB - 96.15% of file sizes done\n",
      "Processing dbl28_S48_summary.txt - 0.000 MB   2019-10-16.00:13:35\n",
      "52/52 - 100.00% of files done   0.010/0.010 MB - 100.00% of file sizes done\n",
      "Processing dbl66_S49_summary.txt - 0.000 MB   2019-10-16.00:13:36\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#save your work\n",
    "icd \n",
    "icd ./CSHL_Cereal_Genomics_2019/finished_analysis\n",
    "iput -rKPf -N 4 ./aligned_reads\n",
    "iput -rKPf -N 4 ./HiSAT2_report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) (alignment) \n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT10_S8.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT10_S8.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT10_S8.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT10_S8.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107622\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64554 (60.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT10_S8 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT13_S12.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT13_S12.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mWT13_S12.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT13_S12.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107446\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m60447 (56.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT13_S1 \u001b[0m ||\n",
      "|| 2.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT18_S4.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT18_S4.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT18_S4.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT18_S4.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107629\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62796 (58.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT18_S4 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT24_S10.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT24_S10.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mWT24_S10.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT24_S10.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108344\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m47368 (43.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT24_S1 \u001b[0m ||\n",
      "|| 0.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||                           \u001b[32mS\u001b[36m WT25_S7.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT25_S7.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT25_S7.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT25_S7.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107706\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62656 (58.2%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT25_S7 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT38_S5.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT38_S5.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT38_S5.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT38_S5.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107314\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64030 (59.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT38_S5 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT41_S6.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT41_S6.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT41_S6.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT41_S6.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107247\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m66058 (61.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT41_S6 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT55_S9.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT55_S9.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT55_S9.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT55_S9.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106315\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m67058 (63.1%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT55_S9 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT58_S3.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT58_S3.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT58_S3.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT58_S3.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107939\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62245 (57.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT58_S3 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT78_S2.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT78_S2.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT78_S2.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT78_S2.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107162\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m67515 (63.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT78_S2 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT93_S11.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT93_S11.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mWT93_S11.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT93_S11.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109661\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m55339 (50.5%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT93_S1 \u001b[0m ||\n",
      "|| 1.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT95_S13.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT95_S13.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mWT95_S13.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT95_S13.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108706\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m49571 (45.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT95_S1 \u001b[0m ||\n",
      "|| 3.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m WT97_S1.bam\u001b[0m \u001b[0m                                   ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mWT97_S1.counts.txt\u001b[0m \u001b[0m                              ||\n",
      "||                 Summary : \u001b[36mWT97_S1.counts.txt.summary\u001b[0m \u001b[0m                      ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file WT97_S1.bam... \u001b[0m                                           ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107422\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m55293 (51.5%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/WT97_S1 \u001b[0m ||\n",
      "|| .counts.txt.summary\" \u001b[0m                                                      ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl19_S46.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl19_S46.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl19_S46.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl19_S46.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108792\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64194 (59.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl19_S \u001b[0m ||\n",
      "|| 46.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl28_S48.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl28_S48.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl28_S48.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl28_S48.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106677\u001b[0m \u001b[0m                                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Successfully assigned alignments : \u001b[36m66577 (62.4%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl28_S \u001b[0m ||\n",
      "|| 48.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl31_S41.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl31_S41.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl31_S41.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl31_S41.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108900\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m59067 (54.2%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl31_S \u001b[0m ||\n",
      "|| 41.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl50_S43.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl50_S43.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl50_S43.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl50_S43.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108675\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m68928 (63.4%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl50_S \u001b[0m ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|| 43.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl54_S44.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl54_S44.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl54_S44.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl54_S44.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108958\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m59770 (54.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl54_S \u001b[0m ||\n",
      "|| 44.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl56_S50.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl56_S50.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl56_S50.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl56_S50.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108063\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64037 (59.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl56_S \u001b[0m ||\n",
      "|| 50.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl57_S52.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl57_S52.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl57_S52.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl57_S52.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107742\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m65745 (61.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl57_S \u001b[0m ||\n",
      "|| 52.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl65_S42.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl65_S42.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl65_S42.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl65_S42.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109260\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m60729 (55.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl65_S \u001b[0m ||\n",
      "|| 42.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl66_S49.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl66_S49.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl66_S49.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl66_S49.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109167\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m61973 (56.8%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl66_S \u001b[0m ||\n",
      "|| 49.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl70_S47.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl70_S47.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl70_S47.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl70_S47.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108439\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64619 (59.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl70_S \u001b[0m ||\n",
      "|| 47.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl88_S51.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl88_S51.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl88_S51.counts.txt.summary\u001b[0m \u001b[0m                    ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl88_S51.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108097\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m65386 (60.5%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl88_S \u001b[0m ||\n",
      "|| 51.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl89_S40.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl89_S40.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mdbl89_S40.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl89_S40.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108799\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m59107 (54.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl89_S \u001b[0m ||\n",
      "|| 40.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m dbl9_S45.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mdbl9_S45.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mdbl9_S45.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file dbl9_S45.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108691\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64043 (58.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/dbl9_S4 \u001b[0m ||\n",
      "|| 5.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn17_S35.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn17_S35.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn17_S35.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn17_S35.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108175\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62322 (57.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn17_S \u001b[0m ||\n",
      "|| 35.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn21_S36.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn21_S36.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn21_S36.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn21_S36.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107054\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m66732 (62.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn21_S \u001b[0m ||\n",
      "|| 36.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn28_S37.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn28_S37.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn28_S37.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn28_S37.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107941\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m57255 (53.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn28_S \u001b[0m ||\n",
      "|| 37.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn33_S38.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn33_S38.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn33_S38.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn33_S38.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107063\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62870 (58.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn33_S \u001b[0m ||\n",
      "|| 38.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn37_S33.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn37_S33.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn37_S33.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn37_S33.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107283\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m65917 (61.4%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn37_S \u001b[0m ||\n",
      "|| 33.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn39_S27.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn39_S27.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn39_S27.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn39_S27.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107775\u001b[0m \u001b[0m                                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Successfully assigned alignments : \u001b[36m64529 (59.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn39_S \u001b[0m ||\n",
      "|| 27.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn3_S39.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn3_S39.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mlgn3_S39.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn3_S39.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109495\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m65642 (59.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn3_S3 \u001b[0m ||\n",
      "|| 9.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn42_S34.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn42_S34.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn42_S34.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn42_S34.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107450\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m66651 (62.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn42_S \u001b[0m ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|| 34.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn46_S30.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn46_S30.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn46_S30.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn46_S30.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106876\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m69562 (65.1%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn46_S \u001b[0m ||\n",
      "|| 30.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn59_S28.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn59_S28.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn59_S28.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn59_S28.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106708\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m66304 (62.1%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn59_S \u001b[0m ||\n",
      "|| 28.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn72_S31.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn72_S31.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn72_S31.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn72_S31.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106484\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64723 (60.8%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn72_S \u001b[0m ||\n",
      "|| 31.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn77_S29.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn77_S29.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mlgn77_S29.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn77_S29.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106719\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m65841 (61.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn77_S \u001b[0m ||\n",
      "|| 29.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m lgn8_S32.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mlgn8_S32.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mlgn8_S32.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file lgn8_S32.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107659\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m60478 (56.2%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/lgn8_S3 \u001b[0m ||\n",
      "|| 2.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod11_S24.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod11_S24.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod11_S24.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod11_S24.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109957\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m51626 (47.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod11_S \u001b[0m ||\n",
      "|| 24.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod14_S18.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod14_S18.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod14_S18.counts.txt.summary\u001b[0m \u001b[0m                    ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod14_S18.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107365\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62933 (58.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod14_S \u001b[0m ||\n",
      "|| 18.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod16_S15.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod16_S15.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod16_S15.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod16_S15.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107393\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m69018 (64.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod16_S \u001b[0m ||\n",
      "|| 15.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod2_S26.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod2_S26.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mnod2_S26.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod2_S26.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107589\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m53866 (50.1%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod2_S2 \u001b[0m ||\n",
      "|| 6.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod30_S17.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod30_S17.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod30_S17.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod30_S17.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107698\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m59889 (55.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod30_S \u001b[0m ||\n",
      "|| 17.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod36_S21.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod36_S21.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod36_S21.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod36_S21.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108180\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m62950 (58.2%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod36_S \u001b[0m ||\n",
      "|| 21.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod43_S22.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod43_S22.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod43_S22.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod43_S22.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m108386\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64893 (59.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod43_S \u001b[0m ||\n",
      "|| 22.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod47_S23.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod47_S23.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod47_S23.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod47_S23.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107463\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m63248 (58.9%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod47_S \u001b[0m ||\n",
      "|| 23.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod48_S16.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod48_S16.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod48_S16.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod48_S16.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107698\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m63495 (59.0%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod48_S \u001b[0m ||\n",
      "|| 16.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod60_S19.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod60_S19.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod60_S19.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod60_S19.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107334\u001b[0m \u001b[0m                                              ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Successfully assigned alignments : \u001b[36m64233 (59.8%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod60_S \u001b[0m ||\n",
      "|| 19.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod61_S25.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod61_S25.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod61_S25.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod61_S25.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m109647\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m53304 (48.6%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod61_S \u001b[0m ||\n",
      "|| 25.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod62_S14.bam\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod62_S14.counts.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mnod62_S14.counts.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod62_S14.bam... \u001b[0m                                         ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m107169\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m64633 (60.3%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.01 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod62_S \u001b[0m ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|| 14.counts.txt.summary\" \u001b[0m                                                    ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.4\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mS\u001b[36m nod6_S20.bam\u001b[0m \u001b[0m                                  ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mnod6_S20.counts.txt\u001b[0m \u001b[0m                             ||\n",
      "||                 Summary : \u001b[36mnod6_S20.counts.txt.summary\u001b[0m \u001b[0m                     ||\n",
      "||              Annotation : \u001b[36mZea_mays.AGPv3.30.repaired.gtf (GTF)\u001b[0m \u001b[0m            ||\n",
      "||      Dir for temp files : \u001b[36m./featureCounts\u001b[0m \u001b[0m                                 ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m4\u001b[0m \u001b[0m                                               ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file Zea_mays.AGPv3.30.repaired.gtf \u001b[0m... \u001b[0m                   ||\n",
      "||    Features : \u001b[36m330419\u001b[0m \u001b[0m                                                      ||\n",
      "||    Meta-features : \u001b[36m39625\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m132\u001b[0m \u001b[0m                                              ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file nod6_S20.bam... \u001b[0m                                          ||\n",
      "||    Single-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m106755\u001b[0m \u001b[0m                                              ||\n",
      "||    Successfully assigned alignments : \u001b[36m66936 (62.7%)\u001b[0m \u001b[0m                       ||\n",
      "||    Running time : \u001b[36m0.00 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"./featureCounts/nod6_S2 \u001b[0m ||\n",
      "|| 0.counts.txt.summary\" \u001b[0m                                                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\============================================================================//\n",
      "\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "## use featureCounts to get expression values\n",
    "# count by EXON and\n",
    "# summarize by GENE\n",
    "# this is an UNION-EXON approach\n",
    "\n",
    "#command template\n",
    "#featureCounts -p -a annotation.gtf -t exon -g gene id\n",
    "#-o counts.txt mapping results PE.bam\n",
    "\n",
    "mkdir ./featureCounts\n",
    "\n",
    "for f in ./aligned_reads/*.bam; do\n",
    "  STEM=$(basename \"${f}\" .bam)\n",
    "  featureCounts -T 4 \\\n",
    "      -a ./maize_genome/Zea_mays.AGPv3.30.repaired.gtf \\\n",
    "      -t exon -g gene_id \\\n",
    "      -o ./featureCounts/${STEM}.counts.txt \\\n",
    "      ./aligned_reads/${STEM}.bam\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) total 430M\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT10_S8.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT10_S8.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT13_S12.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 WT13_S12.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT18_S4.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT18_S4.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT24_S10.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 WT24_S10.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT25_S7.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT25_S7.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT38_S5.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT38_S5.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT41_S6.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT41_S6.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT55_S9.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT55_S9.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT58_S3.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT58_S3.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT78_S2.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT78_S2.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT93_S11.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 WT93_S11.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT95_S13.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 WT95_S13.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 WT97_S1.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  336 Oct 16 00:15 WT97_S1.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl19_S46.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl19_S46.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl28_S48.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl28_S48.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl31_S41.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl31_S41.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl50_S43.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl50_S43.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl54_S44.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl54_S44.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl56_S50.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl56_S50.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl57_S52.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl57_S52.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl65_S42.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl65_S42.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl66_S49.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl66_S49.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl70_S47.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl70_S47.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl88_S51.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl88_S51.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl89_S40.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 dbl89_S40.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 dbl9_S45.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 dbl9_S45.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn17_S35.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn17_S35.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn21_S36.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn21_S36.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn28_S37.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn28_S37.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn33_S38.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn33_S38.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn37_S33.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn37_S33.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn39_S27.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn39_S27.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn3_S39.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 lgn3_S39.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn42_S34.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn42_S34.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn46_S30.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn46_S30.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn59_S28.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn59_S28.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn72_S31.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn72_S31.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn77_S29.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 lgn77_S29.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 lgn8_S32.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:15 lgn8_S32.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 nod11_S24.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 nod11_S24.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:15 nod14_S18.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:15 nod14_S18.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod16_S15.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod16_S15.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod2_S26.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:16 nod2_S26.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod30_S17.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod30_S17.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod36_S21.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod36_S21.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod43_S22.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod43_S22.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod47_S23.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod47_S23.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod48_S16.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod48_S16.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod60_S19.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod60_S19.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod61_S25.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod61_S25.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod62_S14.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  338 Oct 16 00:16 nod62_S14.counts.txt.summary\n",
      "-rw-r--r-- 1 sal269 iplant-everyone 8.3M Oct 16 00:16 nod6_S20.counts.txt\n",
      "-rw-r--r-- 1 sal269 iplant-everyone  337 Oct 16 00:16 nod6_S20.counts.txt.summary\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#check on your count files\n",
    "ls -lh ./featureCounts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) # Program:featureCounts v1.6.4; Command:\"featureCounts\" \"-T\" \"4\" \"-a\" \"./maize_genome/Zea_mays.AGPv3.30.repaired.gtf\" \"-t\" \"exon\" \"-g\" \"gene_id\" \"-o\" \"./featureCounts/WT10_S8.counts.txt\" \"./aligned_reads/WT10_S8.bam\" \n",
      "Geneid\tChr\tStart\tEnd\tStrand\tLength\t./aligned_reads/WT10_S8.bam\n",
      "gene:GRMZM2G059865\t1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1\t4854;4857;4857;5342;5342;5342;5857;6108;6108;6108;6362;6362;6639;6639;6918;6918;7594;7594;9193;9193\t5188;5188;5188;5975;5975;5407;5975;6265;6355;6265;6517;6517;6797;6797;7120;7120;7903;7903;9652;9652\t-;-;-;-;-;-;-;-;-;-;-;-;-;-;-;-;-;-;-;-\t2505\t18\n",
      "gene:GRMZM5G888250\t1\t9882\t10387\t-\t506\t0\n",
      "gene:GRMZM2G093344\t1;1;1;1\t109519;109759;110769;111466\t109675;109935;111142;111769\t-;-;-;-\t1012\t0\n",
      "gene:GRMZM2G093399\t1;1;1;1\t136307;136719;137205;138552\t136634;137013;137290;138929\t+;+;+;+\t1087\t0\n",
      "gene:GRMZM5G809743\t1\t144361\t144657\t+\t297\t0\n",
      "gene:GRMZM5G833153\t1\t144957\t145646\t+\t690\t3\n",
      "gene:GRMZM2G306216\t1;1;1;1\t146265;146374;146697;147423\t146272;146634;146754;147500\t-;-;-;-\t405\t0\n",
      "gene:AC177838.2_FG015\t1\t161143\t161925\t+\t783\t0\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#lets browse one of the count files\n",
    "head ./featureCounts/WT10_S8.counts.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What does the summary file .summary contain???  \n",
    "???"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) 0/104 -  0.00% of files done   0.000/429.105 MB -  0.00% of file sizes done\n",
      "Processing lgn46_S30.counts.txt - 8.252 MB   2019-10-16.00:18:13\n",
      "1/104 -  0.96% of files done   8.252/429.105 MB -  1.92% of file sizes done\n",
      "Processing lgn3_S39.counts.txt - 8.252 MB   2019-10-16.00:18:14\n",
      "2/104 -  1.92% of files done   16.504/429.105 MB -  3.85% of file sizes done\n",
      "Processing dbl9_S45.counts.txt - 8.252 MB   2019-10-16.00:18:16\n",
      "3/104 -  2.88% of files done   24.755/429.105 MB -  5.77% of file sizes done\n",
      "Processing WT97_S1.counts.txt - 8.251 MB   2019-10-16.00:18:18\n",
      "4/104 -  3.85% of files done   33.007/429.105 MB -  7.69% of file sizes done\n",
      "Processing dbl70_S47.counts.txt - 8.252 MB   2019-10-16.00:18:20\n",
      "5/104 -  4.81% of files done   41.259/429.105 MB -  9.62% of file sizes done\n",
      "Processing WT55_S9.counts.txt - 8.252 MB   2019-10-16.00:18:21\n",
      "6/104 -  5.77% of files done   49.510/429.105 MB - 11.54% of file sizes done\n",
      "Processing lgn77_S29.counts.txt - 8.252 MB   2019-10-16.00:18:23\n",
      "7/104 -  6.73% of files done   57.762/429.105 MB - 13.46% of file sizes done\n",
      "Processing WT18_S4.counts.txt - 8.252 MB   2019-10-16.00:18:25\n",
      "8/104 -  7.69% of files done   66.014/429.105 MB - 15.38% of file sizes done\n",
      "Processing WT93_S11.counts.txt - 8.251 MB   2019-10-16.00:18:25\n",
      "9/104 -  8.65% of files done   74.265/429.105 MB - 17.31% of file sizes done\n",
      "Processing lgn33_S38.counts.txt - 8.252 MB   2019-10-16.00:18:27\n",
      "10/104 -  9.62% of files done   82.517/429.105 MB - 19.23% of file sizes done\n",
      "Processing WT13_S12.counts.txt - 8.252 MB   2019-10-16.00:18:29\n",
      "11/104 - 10.58% of files done   90.769/429.105 MB - 21.15% of file sizes done\n",
      "Processing lgn8_S32.counts.txt - 8.252 MB   2019-10-16.00:18:32\n",
      "12/104 - 11.54% of files done   99.020/429.105 MB - 23.08% of file sizes done\n",
      "Processing nod11_S24.counts.txt - 8.251 MB   2019-10-16.00:18:34\n",
      "13/104 - 12.50% of files done   107.272/429.105 MB - 25.00% of file sizes done\n",
      "Processing dbl89_S40.counts.txt - 8.252 MB   2019-10-16.00:18:38\n",
      "14/104 - 13.46% of files done   115.523/429.105 MB - 26.92% of file sizes done\n",
      "Processing WT58_S3.counts.txt - 8.252 MB   2019-10-16.00:18:39\n",
      "15/104 - 14.42% of files done   123.775/429.105 MB - 28.84% of file sizes done\n",
      "Processing nod62_S14.counts.txt - 8.252 MB   2019-10-16.00:18:39\n",
      "16/104 - 15.38% of files done   132.027/429.105 MB - 30.77% of file sizes done\n",
      "Processing WT78_S2.counts.txt - 8.252 MB   2019-10-16.00:18:40\n",
      "17/104 - 16.35% of files done   140.279/429.105 MB - 32.69% of file sizes done\n",
      "Processing lgn59_S28.counts.txt - 8.252 MB   2019-10-16.00:18:41\n",
      "18/104 - 17.31% of files done   148.530/429.105 MB - 34.61% of file sizes done\n",
      "Processing lgn28_S37.counts.txt - 8.252 MB   2019-10-16.00:18:41\n",
      "19/104 - 18.27% of files done   156.782/429.105 MB - 36.54% of file sizes done\n",
      "Processing dbl50_S43.counts.txt - 8.252 MB   2019-10-16.00:18:42\n",
      "20/104 - 19.23% of files done   165.034/429.105 MB - 38.46% of file sizes done\n",
      "Processing nod2_S26.counts.txt - 8.251 MB   2019-10-16.00:18:43\n",
      "21/104 - 20.19% of files done   173.285/429.105 MB - 40.38% of file sizes done\n",
      "Processing WT25_S7.counts.txt - 8.252 MB   2019-10-16.00:18:43\n",
      "22/104 - 21.15% of files done   181.537/429.105 MB - 42.31% of file sizes done\n",
      "Processing WT41_S6.counts.txt - 8.252 MB   2019-10-16.00:18:44\n",
      "23/104 - 22.12% of files done   189.789/429.105 MB - 44.23% of file sizes done\n",
      "Processing lgn39_S27.counts.txt - 8.252 MB   2019-10-16.00:18:48\n",
      "24/104 - 23.08% of files done   198.040/429.105 MB - 46.15% of file sizes done\n",
      "Processing nod48_S16.counts.txt - 8.252 MB   2019-10-16.00:18:49\n",
      "25/104 - 24.04% of files done   206.292/429.105 MB - 48.08% of file sizes done\n",
      "Processing lgn17_S35.counts.txt - 8.252 MB   2019-10-16.00:18:50\n",
      "26/104 - 25.00% of files done   214.544/429.105 MB - 50.00% of file sizes done\n",
      "Processing lgn37_S33.counts.txt - 8.252 MB   2019-10-16.00:18:50\n",
      "27/104 - 25.96% of files done   222.795/429.105 MB - 51.92% of file sizes done\n",
      "Processing WT24_S10.counts.txt - 8.251 MB   2019-10-16.00:18:51\n",
      "28/104 - 26.92% of files done   231.047/429.105 MB - 53.84% of file sizes done\n",
      "Processing dbl88_S51.counts.txt - 8.252 MB   2019-10-16.00:18:52\n",
      "29/104 - 27.88% of files done   239.298/429.105 MB - 55.77% of file sizes done\n",
      "Processing nod6_S20.counts.txt - 8.252 MB   2019-10-16.00:18:52\n",
      "30/104 - 28.85% of files done   247.550/429.105 MB - 57.69% of file sizes done\n",
      "Processing dbl31_S41.counts.txt - 8.252 MB   2019-10-16.00:18:53\n",
      "31/104 - 29.81% of files done   255.802/429.105 MB - 59.61% of file sizes done\n",
      "Processing dbl19_S46.counts.txt - 8.252 MB   2019-10-16.00:18:53\n",
      "32/104 - 30.77% of files done   264.054/429.105 MB - 61.54% of file sizes done\n",
      "Processing nod61_S25.counts.txt - 8.251 MB   2019-10-16.00:18:54\n",
      "33/104 - 31.73% of files done   272.305/429.105 MB - 63.46% of file sizes done\n",
      "Processing WT95_S13.counts.txt - 8.251 MB   2019-10-16.00:18:57\n",
      "34/104 - 32.69% of files done   280.556/429.105 MB - 65.38% of file sizes done\n",
      "Processing WT10_S8.counts.txt - 8.252 MB   2019-10-16.00:18:57\n",
      "35/104 - 33.65% of files done   288.808/429.105 MB - 67.30% of file sizes done\n",
      "Processing nod43_S22.counts.txt - 8.252 MB   2019-10-16.00:18:58\n",
      "36/104 - 34.62% of files done   297.060/429.105 MB - 69.23% of file sizes done\n",
      "Processing dbl66_S49.counts.txt - 8.252 MB   2019-10-16.00:18:58\n",
      "37/104 - 35.58% of files done   305.312/429.105 MB - 71.15% of file sizes done\n",
      "Processing dbl28_S48.counts.txt - 8.252 MB   2019-10-16.00:18:59\n",
      "38/104 - 36.54% of files done   313.564/429.105 MB - 73.07% of file sizes done\n",
      "Processing dbl57_S52.counts.txt - 8.252 MB   2019-10-16.00:19:00\n",
      "39/104 - 37.50% of files done   321.815/429.105 MB - 75.00% of file sizes done\n",
      "Processing WT38_S5.counts.txt - 8.252 MB   2019-10-16.00:19:00\n",
      "40/104 - 38.46% of files done   330.067/429.105 MB - 76.92% of file sizes done\n",
      "Processing dbl54_S44.counts.txt - 8.252 MB   2019-10-16.00:19:01\n",
      "41/104 - 39.42% of files done   338.319/429.105 MB - 78.84% of file sizes done\n",
      "Processing dbl65_S42.counts.txt - 8.252 MB   2019-10-16.00:19:02\n",
      "42/104 - 40.38% of files done   346.570/429.105 MB - 80.77% of file sizes done\n",
      "Processing lgn21_S36.counts.txt - 8.252 MB   2019-10-16.00:19:02\n",
      "43/104 - 41.35% of files done   354.822/429.105 MB - 82.69% of file sizes done\n",
      "Processing lgn72_S31.counts.txt - 8.252 MB   2019-10-16.00:19:03\n",
      "44/104 - 42.31% of files done   363.074/429.105 MB - 84.61% of file sizes done\n",
      "Processing nod14_S18.counts.txt - 8.252 MB   2019-10-16.00:19:04\n",
      "45/104 - 43.27% of files done   371.326/429.105 MB - 86.54% of file sizes done\n",
      "Processing nod30_S17.counts.txt - 8.252 MB   2019-10-16.00:19:04\n",
      "46/104 - 44.23% of files done   379.577/429.105 MB - 88.46% of file sizes done\n",
      "Processing nod16_S15.counts.txt - 8.252 MB   2019-10-16.00:19:05\n",
      "47/104 - 45.19% of files done   387.829/429.105 MB - 90.38% of file sizes done\n",
      "Processing dbl56_S50.counts.txt - 8.252 MB   2019-10-16.00:19:06\n",
      "48/104 - 46.15% of files done   396.081/429.105 MB - 92.30% of file sizes done\n",
      "Processing nod36_S21.counts.txt - 8.252 MB   2019-10-16.00:19:06\n",
      "49/104 - 47.12% of files done   404.333/429.105 MB - 94.23% of file sizes done\n",
      "Processing lgn42_S34.counts.txt - 8.252 MB   2019-10-16.00:19:08\n",
      "50/104 - 48.08% of files done   412.584/429.105 MB - 96.15% of file sizes done\n",
      "Processing nod60_S19.counts.txt - 8.252 MB   2019-10-16.00:19:09\n",
      "51/104 - 49.04% of files done   420.836/429.105 MB - 98.07% of file sizes done\n",
      "Processing nod47_S23.counts.txt - 8.252 MB   2019-10-16.00:19:09\n",
      "102/104 - 98.08% of files done   429.104/429.105 MB - 100.00% of file sizes done\n",
      "Processing dbl66_S49.counts.txt.summary - 8.252 MB   2019-10-16.00:19:34\n",
      "104/104 - 100.00% of files done   429.105/429.105 MB - 100.00% of file sizes done\n",
      "Processing dbl19_S46.counts.txt.summary - 8.252 MB   2019-10-16.00:19:36\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#save your work\n",
    "icd \n",
    "icd ./CSHL_Cereal_Genomics_2019/finished_analysis\n",
    "iput -rKPf -N 4 ./featureCounts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (alignment) (alignment) (alignment) (alignment) (alignment) Collecting package metadata (current_repodata.json): done\n",
      "Solving environment: done\n",
      "\n",
      "\n",
      "==> WARNING: A newer version of conda exists. <==\n",
      "  current version: 4.7.10\n",
      "  latest version: 4.7.12\n",
      "\n",
      "Please update conda by running\n",
      "\n",
      "    $ conda update -n base -c defaults conda\n",
      "\n",
      "\n",
      "\n",
      "## Package Plan ##\n",
      "\n",
      "  environment location: /home/sal269/miniconda3/envs/DEG\n",
      "\n",
      "  added / updated specs:\n",
      "    - bioconductor-deseq2\n",
      "    - python=3.6\n",
      "    - r-base\n",
      "    - r-essentials\n",
      "    - r-ggplot2\n",
      "    - r-pheatmap\n",
      "    - r-randomforest\n",
      "    - r-venndiagram\n",
      "\n",
      "\n",
      "The following packages will be downloaded:\n",
      "\n",
      "    package                    |            build\n",
      "    ---------------------------|-----------------\n",
      "    _r-mutex-1.0.1             |      anacondar_1           3 KB  conda-forge\n",
      "    attrs-19.3.0               |             py_0          35 KB  conda-forge\n",
      "    backcall-0.1.0             |             py_0          13 KB  conda-forge\n",
      "    binutils_impl_linux-64-2.31.1|       h7fc9f1b_2        16.4 MB  conda-forge\n",
      "    binutils_linux-64-2.31.1   |      h6176602_12          10 KB  conda-forge\n",
      "    bioconductor-annotate-1.62.0|            r36_1         2.1 MB  bioconda\n",
      "    bioconductor-annotationdbi-1.46.0|            r36_1         4.9 MB  bioconda\n",
      "    bioconductor-biobase-2.44.0|    r36h516909a_1         2.3 MB  bioconda\n",
      "    bioconductor-biocgenerics-0.30.0|            r36_1         683 KB  bioconda\n",
      "    bioconductor-biocparallel-1.18.0|    r36he1b5a44_1         1.1 MB  bioconda\n",
      "    bioconductor-delayedarray-0.10.0|    r36h516909a_1         2.2 MB  bioconda\n",
      "    bioconductor-deseq2-1.24.0 |    r36he1b5a44_1         2.5 MB  bioconda\n",
      "    bioconductor-genefilter-1.66.0|    r36hc99cbb1_1         1.7 MB  bioconda\n",
      "    bioconductor-geneplotter-1.62.0|            r36_1         1.5 MB  bioconda\n",
      "    bioconductor-genomeinfodb-1.20.0|            r36_1         3.8 MB  bioconda\n",
      "    bioconductor-genomeinfodbdata-1.2.1|            r36_1           6 KB  bioconda\n",
      "    bioconductor-genomicranges-1.36.0|    r36h516909a_1         2.1 MB  bioconda\n",
      "    bioconductor-iranges-2.18.2|    r36h516909a_0         2.3 MB  bioconda\n",
      "    bioconductor-s4vectors-0.22.0|    r36h516909a_1         2.0 MB  bioconda\n",
      "    bioconductor-summarizedexperiment-1.14.0|            r36_1         2.5 MB  bioconda\n",
      "    bioconductor-xvector-0.24.0|    r36h516909a_1         721 KB  bioconda\n",
      "    bioconductor-zlibbioc-1.30.0|    r36h516909a_1         112 KB  bioconda\n",
      "    bleach-3.1.0               |             py_0         110 KB  conda-forge\n",
      "    bwidget-1.9.13             |                0         119 KB  conda-forge\n",
      "    cairo-1.16.0               |    hfb77d84_1002         1.5 MB  conda-forge\n",
      "    decorator-4.4.0            |             py_0          11 KB  conda-forge\n",
      "    defusedxml-0.6.0           |             py_0          22 KB  conda-forge\n",
      "    entrypoints-0.3            |        py36_1000          12 KB  conda-forge\n",
      "    fontconfig-2.13.1          |    h86ecdb6_1001         340 KB  conda-forge\n",
      "    freetype-2.10.0            |       he983fc9_1         884 KB  conda-forge\n",
      "    fribidi-1.0.5              |    h516909a_1002         112 KB  conda-forge\n",
      "    gcc_impl_linux-64-7.3.0    |       habb00fd_2        70.5 MB  conda-forge\n",
      "    gcc_linux-64-7.3.0         |      h553295d_12          11 KB  conda-forge\n",
      "    gettext-0.19.8.1           |    hc5be6a0_1002         3.6 MB  conda-forge\n",
      "    gfortran_impl_linux-64-7.3.0|       hdf63c60_2         9.4 MB  conda-forge\n",
      "    gfortran_linux-64-7.3.0    |      h553295d_12          11 KB  conda-forge\n",
      "    glib-2.58.3                |    h6f030ca_1002         3.3 MB  conda-forge\n",
      "    graphite2-1.3.13           |    hf484d3e_1000         109 KB  conda-forge\n",
      "    gsl-2.5                    |       h294904e_1         2.7 MB  conda-forge\n",
      "    gxx_impl_linux-64-7.3.0    |       hdf63c60_2        19.1 MB  conda-forge\n",
      "    gxx_linux-64-7.3.0         |      h553295d_12          11 KB  conda-forge\n",
      "    harfbuzz-2.4.0             |       h9f30f68_3         1.5 MB  conda-forge\n",
      "    icu-64.2                   |       he1b5a44_1        12.6 MB  conda-forge\n",
      "    importlib_metadata-0.23    |           py36_0          39 KB  conda-forge\n",
      "    ipykernel-5.1.2            |   py36h5ca1d4c_0         159 KB  conda-forge\n",
      "    ipython-7.8.0              |   py36h5ca1d4c_0         1.1 MB  conda-forge\n",
      "    ipython_genutils-0.2.0     |             py_1          21 KB  conda-forge\n",
      "    jedi-0.15.1                |           py36_0         717 KB  conda-forge\n",
      "    jinja2-2.10.3              |             py_0          91 KB  conda-forge\n",
      "    jpeg-9c                    |    h14c3975_1001         251 KB  conda-forge\n",
      "    jsonschema-3.1.1           |           py36_0          88 KB  conda-forge\n",
      "    jupyter_client-5.3.3       |           py36_1         132 KB  conda-forge\n",
      "    jupyter_core-4.5.0         |             py_0          44 KB  conda-forge\n",
      "    libblas-3.8.0              |      14_openblas          10 KB  conda-forge\n",
      "    libcblas-3.8.0             |      14_openblas          10 KB  conda-forge\n",
      "    libgfortran-ng-7.3.0       |       hdf63c60_2         1.7 MB  conda-forge\n",
      "    libiconv-1.15              |    h516909a_1005         2.0 MB  conda-forge\n",
      "    liblapack-3.8.0            |      14_openblas          10 KB  conda-forge\n",
      "    libopenblas-0.3.7          |       h6e990d7_2         7.6 MB  conda-forge\n",
      "    libpng-1.6.37              |       hed695b0_0         343 KB  conda-forge\n",
      "    libsodium-1.0.17           |       h516909a_0         330 KB  conda-forge\n",
      "    libtiff-4.0.10             |    h57b8799_1003         587 KB  conda-forge\n",
      "    libuuid-2.32.1             |    h14c3975_1000          26 KB  conda-forge\n",
      "    libxcb-1.13                |    h14c3975_1002         396 KB  conda-forge\n",
      "    libxml2-2.9.9              |       hee79883_5         1.3 MB  conda-forge\n",
      "    lz4-c-1.8.3                |    he1b5a44_1001         187 KB  conda-forge\n",
      "    make-4.2.1                 |    h14c3975_2004         458 KB  conda-forge\n",
      "    markupsafe-1.1.1           |   py36h14c3975_0          26 KB  conda-forge\n",
      "    mistune-0.8.4              |py36h14c3975_1000          53 KB  conda-forge\n",
      "    more-itertools-7.2.0       |             py_0          48 KB  conda-forge\n",
      "    nbconvert-5.6.0            |           py36_1         465 KB  conda-forge\n",
      "    nbformat-4.4.0             |             py_1          91 KB  conda-forge\n",
      "    notebook-6.0.1             |           py36_0         6.0 MB  conda-forge\n",
      "    pandoc-2.7.3               |                0        22.4 MB  conda-forge\n",
      "    pandocfilters-1.4.2        |             py_1           9 KB  conda-forge\n",
      "    pango-1.42.4               |       ha030887_1         517 KB  conda-forge\n",
      "    parso-0.5.1                |             py_0          65 KB  conda-forge\n",
      "    pcre-8.43                  |       he1b5a44_0         257 KB  conda-forge\n",
      "    pexpect-4.7.0              |           py36_0          78 KB  conda-forge\n",
      "    pickleshare-0.7.5          |        py36_1000          12 KB  conda-forge\n",
      "    pixman-0.38.0              |    h516909a_1003         594 KB  conda-forge\n",
      "    prometheus_client-0.7.1    |             py_0          38 KB  conda-forge\n",
      "    prompt_toolkit-2.0.10      |             py_0         221 KB  conda-forge\n",
      "    pthread-stubs-0.4          |    h14c3975_1001           5 KB  conda-forge\n",
      "    ptyprocess-0.6.0           |          py_1001          15 KB  conda-forge\n",
      "    pygments-2.4.2             |             py_0         661 KB  conda-forge\n",
      "    pyrsistent-0.15.4          |   py36h516909a_0          89 KB  conda-forge\n",
      "    python-dateutil-2.8.0      |             py_0         219 KB  conda-forge\n",
      "    pyzmq-18.1.0               |   py36h1768529_0         516 KB  conda-forge\n",
      "    r-acepack-1.4.1            | r36h9bbef5b_1004          70 KB  conda-forge\n",
      "    r-askpass-1.1              |    r36hcdcec82_1          27 KB  conda-forge\n",
      "    r-assertthat-0.2.1         |    r36h6115d3f_1          70 KB  conda-forge\n",
      "    r-backports-1.1.5          |    r36hcdcec82_0          68 KB  conda-forge\n",
      "    r-base-3.6.1               |       hba50c9b_4        23.4 MB  conda-forge\n",
      "    r-base64enc-0.1_3          | r36hcdcec82_1003          43 KB  conda-forge\n",
      "    r-bh-1.69.0_1              |    r36h6115d3f_1        10.6 MB  conda-forge\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    r-bit-1.1_14               |    r36hcdcec82_1         247 KB  conda-forge\n",
      "    r-bit64-0.9_7              | r36hcdcec82_1001         482 KB  conda-forge\n",
      "    r-bitops-1.0_6             | r36hcdcec82_1003          39 KB  conda-forge\n",
      "    r-blob-1.2.0               |            r36_1          48 KB  conda-forge\n",
      "    r-boot-1.3_23              |    r36h6115d3f_2         632 KB  conda-forge\n",
      "    r-broom-0.5.2              |    r36h6115d3f_1         1.9 MB  conda-forge\n",
      "    r-callr-3.3.2              |    r36h6115d3f_0         332 KB  conda-forge\n",
      "    r-caret-6.0_84             |    r36hcdcec82_1         6.0 MB  conda-forge\n",
      "    r-cellranger-1.1.0         | r36h6115d3f_1002         107 KB  conda-forge\n",
      "    r-checkmate-1.9.4          |    r36hcdcec82_1         647 KB  conda-forge\n",
      "    r-class-7.3_15             | r36hcdcec82_1001         106 KB  conda-forge\n",
      "    r-cli-1.1.0                |    r36h6115d3f_2         181 KB  conda-forge\n",
      "    r-clipr-0.7.0              |    r36h6115d3f_0          64 KB  conda-forge\n",
      "    r-cluster-2.1.0            |    r36h9bbef5b_2         554 KB  conda-forge\n",
      "    r-codetools-0.2_16         | r36h6115d3f_1001         105 KB  conda-forge\n",
      "    r-colorspace-1.4_1         |    r36hcdcec82_1         2.4 MB  conda-forge\n",
      "    r-crayon-1.3.4             | r36h6115d3f_1002         746 KB  conda-forge\n",
      "    r-curl-4.2                 |    r36hcdcec82_0         692 KB  conda-forge\n",
      "    r-data.table-1.12.4        |    r36hcdcec82_0         1.8 MB  conda-forge\n",
      "    r-dbi-1.0.0                | r36h6115d3f_1002         890 KB  conda-forge\n",
      "    r-dbplyr-1.4.2             |    r36h6115d3f_1         602 KB  conda-forge\n",
      "    r-digest-0.6.21            |    r36h0357c0b_0         189 KB  conda-forge\n",
      "    r-dplyr-0.8.3              |    r36h0357c0b_3         1.9 MB  conda-forge\n",
      "    r-ellipsis-0.3.0           |    r36hcdcec82_0          47 KB  conda-forge\n",
      "    r-essentials-3.6           |         r36_2001           4 KB  conda-forge\n",
      "    r-evaluate-0.14            |    r36h6115d3f_1          80 KB  conda-forge\n",
      "    r-fansi-0.4.0              | r36hcdcec82_1001         193 KB  conda-forge\n",
      "    r-fastmap-1.0.1            |    r36h0357c0b_0          55 KB  conda-forge\n",
      "    r-forcats-0.4.0            |    r36h6115d3f_1         365 KB  conda-forge\n",
      "    r-foreach-1.4.7            |    r36h6115d3f_0         401 KB  conda-forge\n",
      "    r-foreign-0.8_72           |    r36hcdcec82_0         267 KB  conda-forge\n",
      "    r-formatr-1.7              |    r36h6115d3f_1         166 KB  conda-forge\n",
      "    r-formula-1.2_3            | r36h6115d3f_1002         193 KB  conda-forge\n",
      "    r-fs-1.3.1                 |    r36h0357c0b_1         535 KB  conda-forge\n",
      "    r-futile.logger-1.4.3      | r36h6115d3f_1002         104 KB  conda-forge\n",
      "    r-futile.options-1.0.1     | r36h6115d3f_1001          24 KB  conda-forge\n",
      "    r-generics-0.0.2           | r36h6115d3f_1002          77 KB  conda-forge\n",
      "    r-ggplot2-3.2.1            |    r36h6115d3f_0         3.9 MB  conda-forge\n",
      "    r-gistr-0.4.2              | r36h6115d3f_1002         1.8 MB  conda-forge\n",
      "    r-glmnet-2.0_18            |    r36h9bbef5b_2         1.4 MB  conda-forge\n",
      "    r-glue-1.3.1               |    r36hcdcec82_1         165 KB  conda-forge\n",
      "    r-gower-0.2.1              |    r36hcdcec82_1         175 KB  conda-forge\n",
      "    r-gridextra-2.3            | r36h6115d3f_1002         1.1 MB  conda-forge\n",
      "    r-gtable-0.3.0             |    r36h6115d3f_2         423 KB  conda-forge\n",
      "    r-haven-2.1.1              |    r36hde08347_3         335 KB  conda-forge\n",
      "    r-hexbin-1.27.3            |    r36h9bbef5b_2         965 KB  conda-forge\n",
      "    r-highr-0.8                |    r36h6115d3f_1          47 KB  conda-forge\n",
      "    r-hmisc-4.2_0              |    r36h9bbef5b_2         3.0 MB  conda-forge\n",
      "    r-hms-0.5.1                |    r36h6115d3f_0         123 KB  conda-forge\n",
      "    r-htmltable-1.13.2         |    r36h6115d3f_0         330 KB  conda-forge\n",
      "    r-htmltools-0.4.0          |    r36h0357c0b_0         215 KB  conda-forge\n",
      "    r-htmlwidgets-1.5.1        |    r36h6115d3f_0         679 KB  conda-forge\n",
      "    r-httpuv-1.5.2             |    r36h0357c0b_1         842 KB  conda-forge\n",
      "    r-httr-1.4.1               |    r36h6115d3f_1         489 KB  conda-forge\n",
      "    r-ipred-0.9_9              |    r36hcdcec82_1         394 KB  conda-forge\n",
      "    r-irdisplay-0.7            |         r36_1001          35 KB  conda-forge\n",
      "    r-irkernel-1.0.2           |    r36h6115d3f_2         155 KB  conda-forge\n",
      "    r-iterators-1.0.12         |    r36h6115d3f_0         334 KB  conda-forge\n",
      "    r-jsonlite-1.6             | r36hcdcec82_1001         971 KB  conda-forge\n",
      "    r-kernsmooth-2.23_15       | r36h9bbef5b_1004         103 KB  conda-forge\n",
      "    r-knitr-1.25               |    r36h6115d3f_0         1.3 MB  conda-forge\n",
      "    r-labeling-0.3             | r36h6115d3f_1002          66 KB  conda-forge\n",
      "    r-lambda.r-1.2.4           |    r36h6115d3f_0         119 KB  conda-forge\n",
      "    r-later-1.0.0              |    r36h0357c0b_0         140 KB  conda-forge\n",
      "    r-lattice-0.20_38          | r36hcdcec82_1002         1.1 MB  conda-forge\n",
      "    r-latticeextra-0.6_28      | r36h6115d3f_1002         2.1 MB  conda-forge\n",
      "    r-lava-1.6.6               |    r36h6115d3f_0         2.1 MB  conda-forge\n",
      "    r-lazyeval-0.2.2           |    r36hcdcec82_1         166 KB  conda-forge\n",
      "    r-lifecycle-0.1.0          |    r36h6115d3f_0          95 KB  conda-forge\n",
      "    r-locfit-1.5_9.1           | r36h516909a_1004         556 KB  conda-forge\n",
      "    r-lubridate-1.7.4          | r36h0357c0b_1002         1.1 MB  conda-forge\n",
      "    r-magrittr-1.5             | r36h6115d3f_1002         167 KB  conda-forge\n",
      "    r-maps-3.3.0               | r36hcdcec82_1003         3.4 MB  conda-forge\n",
      "    r-markdown-1.1             |    r36hcdcec82_0         143 KB  conda-forge\n",
      "    r-mass-7.3_51.4            |    r36hcdcec82_1         1.1 MB  conda-forge\n",
      "    r-matrix-1.2_17            |    r36hcdcec82_1         3.8 MB  conda-forge\n",
      "    r-matrixstats-0.55.0       |    r36hcdcec82_0         912 KB  conda-forge\n",
      "    r-memoise-1.1.0            | r36h6115d3f_1003          42 KB  conda-forge\n",
      "    r-mgcv-1.8_29              |    r36hcdcec82_0         2.7 MB  conda-forge\n",
      "    r-mime-0.7                 |    r36hcdcec82_1          51 KB  conda-forge\n",
      "    r-modelmetrics-1.2.2       |    r36h0357c0b_1         162 KB  conda-forge\n",
      "    r-modelr-0.1.5             |    r36h6115d3f_0         217 KB  conda-forge\n",
      "    r-munsell-0.5.0            | r36h6115d3f_1002         245 KB  conda-forge\n",
      "    r-nlme-3.1_141             |    r36h9bbef5b_1         2.3 MB  conda-forge\n",
      "    r-nnet-7.3_12              | r36hcdcec82_1003         131 KB  conda-forge\n",
      "    r-numderiv-2016.8_1.1      |    r36h6115d3f_1         125 KB  conda-forge\n",
      "    r-openssl-1.4.1            |    r36h9c8475f_0         1.2 MB  conda-forge\n",
      "    r-pbdzmq-0.3_3             | r36h559a7a4_1002         380 KB  conda-forge\n",
      "    r-pheatmap-1.0.12          |    r36h6115d3f_1          88 KB  conda-forge\n",
      "    r-pillar-1.4.2             |    r36h6115d3f_2         194 KB  conda-forge\n",
      "    r-pkgconfig-2.0.3          |    r36h6115d3f_0          24 KB  conda-forge\n",
      "    r-plogr-0.2.0              | r36h6115d3f_1002          18 KB  conda-forge\n",
      "    r-plyr-1.8.4               | r36h0357c0b_1003         819 KB  conda-forge\n",
      "    r-prettyunits-1.0.2        | r36h6115d3f_1002          35 KB  conda-forge\n",
      "    r-processx-3.4.1           |    r36hcdcec82_0         253 KB  conda-forge\n",
      "    r-prodlim-2018.04.18       | r36h0357c0b_1003         414 KB  conda-forge\n",
      "    r-progress-1.2.2           |    r36h6115d3f_1          89 KB  conda-forge\n",
      "    r-promises-1.1.0           |    r36h0357c0b_0         1.2 MB  conda-forge\n",
      "    r-pryr-0.1.4               | r36h0357c0b_1003         237 KB  conda-forge\n",
      "    r-ps-1.3.0                 | r36hcdcec82_1001         222 KB  conda-forge\n",
      "    r-purrr-0.3.2              |    r36hcdcec82_1         403 KB  conda-forge\n",
      "    r-quantmod-0.4_15          |    r36h6115d3f_1         989 KB  conda-forge\n",
      "    r-r6-2.4.0                 |    r36h6115d3f_2          63 KB  conda-forge\n",
      "    r-randomforest-4.6_14      | r36h9bbef5b_1002         235 KB  conda-forge\n",
      "    r-rbokeh-0.5.0             | r36h6115d3f_1002         1.4 MB  conda-forge\n",
      "    r-rcolorbrewer-1.1_2       | r36h6115d3f_1002          59 KB  conda-forge\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    r-rcpp-1.0.2               |    r36h0357c0b_0         3.3 MB  conda-forge\n",
      "    r-rcpparmadillo-0.9.700.2.0|    r36h0357c0b_0         1.2 MB  conda-forge\n",
      "    r-rcurl-1.95_4.12          |    r36hcdcec82_1         920 KB  conda-forge\n",
      "    r-readr-1.3.1              | r36h0357c0b_1002         827 KB  conda-forge\n",
      "    r-readxl-1.3.1             |    r36h0357c0b_2         849 KB  conda-forge\n",
      "    r-recipes-0.1.7            |    r36h6115d3f_0         1.5 MB  conda-forge\n",
      "    r-recommended-3.6          |         r36_1003           3 KB  conda-forge\n",
      "    r-rematch-1.0.1            | r36h6115d3f_1002          18 KB  conda-forge\n",
      "    r-repr-1.0.1               |    r36h6115d3f_1         143 KB  conda-forge\n",
      "    r-reprex-0.3.0             |    r36h6115d3f_1         428 KB  conda-forge\n",
      "    r-reshape2-1.4.3           | r36h0357c0b_1004         129 KB  conda-forge\n",
      "    r-rlang-0.4.0              |    r36hcdcec82_1        1020 KB  conda-forge\n",
      "    r-rmarkdown-1.16           |    r36h6115d3f_0         3.0 MB  conda-forge\n",
      "    r-rpart-4.1_15             |    r36hcdcec82_1         733 KB  conda-forge\n",
      "    r-rsqlite-2.1.2            |    r36h0357c0b_1         1.0 MB  conda-forge\n",
      "    r-rstudioapi-0.10          |    r36h6115d3f_2         222 KB  conda-forge\n",
      "    r-rvest-0.3.4              |    r36h6115d3f_1         922 KB  conda-forge\n",
      "    r-scales-1.0.0             | r36h0357c0b_1002         582 KB  conda-forge\n",
      "    r-selectr-0.4_1            | r36h6115d3f_1001         476 KB  conda-forge\n",
      "    r-shiny-1.4.0              |    r36h6115d3f_0         4.3 MB  conda-forge\n",
      "    r-snow-0.4_3               | r36h6115d3f_1001         124 KB  conda-forge\n",
      "    r-sourcetools-0.1.7        | r36he1b5a44_1001          51 KB  conda-forge\n",
      "    r-spatial-7.3_11           | r36hcdcec82_1003         154 KB  conda-forge\n",
      "    r-squarem-2017.10_1        | r36h6115d3f_1002         301 KB  conda-forge\n",
      "    r-stringi-1.4.3            |    r36h0e574ca_3         778 KB  conda-forge\n",
      "    r-stringr-1.4.0            |    r36h6115d3f_1         209 KB  conda-forge\n",
      "    r-survival-2.44_1.1        |    r36hcdcec82_1         4.8 MB  conda-forge\n",
      "    r-sys-3.3                  |    r36hcdcec82_0          48 KB  conda-forge\n",
      "    r-tibble-2.1.3             |    r36hcdcec82_1         315 KB  conda-forge\n",
      "    r-tidyr-1.0.0              |    r36h0357c0b_0         757 KB  conda-forge\n",
      "    r-tidyselect-0.2.5         | r36h0357c0b_1001         138 KB  conda-forge\n",
      "    r-tidyverse-1.2.1          | r36h6115d3f_1002          98 KB  conda-forge\n",
      "    r-timedate-3043.102        | r36h6115d3f_1001         1.6 MB  conda-forge\n",
      "    r-tinytex-0.16             |    r36h6115d3f_0         103 KB  conda-forge\n",
      "    r-ttr-0.23_5               |    r36hcdcec82_0         516 KB  conda-forge\n",
      "    r-utf8-1.1.4               | r36hcdcec82_1001         159 KB  conda-forge\n",
      "    r-uuid-0.1_2               | r36hcdcec82_1002          21 KB  conda-forge\n",
      "    r-vctrs-0.2.0              |    r36hcdcec82_1         764 KB  conda-forge\n",
      "    r-venndiagram-1.6.20       | r36h6115d3f_1001         266 KB  conda-forge\n",
      "    r-viridis-0.5.1            | r36h6115d3f_1003         1.8 MB  conda-forge\n",
      "    r-viridislite-0.3.0        | r36h6115d3f_1002          64 KB  conda-forge\n",
      "    r-whisker-0.4              |    r36h6115d3f_0          80 KB  conda-forge\n",
      "    r-withr-2.1.2              | r36h6115d3f_1001         175 KB  conda-forge\n",
      "    r-xfun-0.10                |    r36h6115d3f_0         184 KB  conda-forge\n",
      "    r-xml-3.98_1.20            |    r36hcdcec82_1         2.0 MB  conda-forge\n",
      "    r-xml2-1.2.2               |    r36h0357c0b_0         359 KB  conda-forge\n",
      "    r-xtable-1.8_4             |    r36h6115d3f_2         699 KB  conda-forge\n",
      "    r-xts-0.11_2               |    r36hcdcec82_1         912 KB  conda-forge\n",
      "    r-yaml-2.2.0               | r36hcdcec82_1002         113 KB  conda-forge\n",
      "    r-zeallot-0.1.0            | r36h6115d3f_1001          62 KB  conda-forge\n",
      "    r-zoo-1.8_6                |    r36hcdcec82_1         1.1 MB  conda-forge\n",
      "    sed-4.7                    |    h1bed415_1000         245 KB  conda-forge\n",
      "    send2trash-1.5.0           |             py_0          12 KB  conda-forge\n",
      "    six-1.12.0                 |        py36_1000          22 KB  conda-forge\n",
      "    terminado-0.8.2            |           py36_0          23 KB  conda-forge\n",
      "    testpath-0.4.2             |          py_1001          85 KB  conda-forge\n",
      "    tktable-2.10               |       h555a92e_3          89 KB  conda-forge\n",
      "    tornado-6.0.3              |   py36h516909a_0         636 KB  conda-forge\n",
      "    traitlets-4.3.3            |           py36_0         133 KB  conda-forge\n",
      "    wcwidth-0.1.7              |             py_1          17 KB  conda-forge\n",
      "    webencodings-0.5.1         |             py_1          12 KB  conda-forge\n",
      "    xorg-kbproto-1.0.7         |    h14c3975_1002          26 KB  conda-forge\n",
      "    xorg-libice-1.0.10         |       h516909a_0          57 KB  conda-forge\n",
      "    xorg-libsm-1.2.3           |    h84519dc_1000          25 KB  conda-forge\n",
      "    xorg-libx11-1.6.9          |       h516909a_0         918 KB  conda-forge\n",
      "    xorg-libxau-1.0.9          |       h14c3975_0          13 KB  conda-forge\n",
      "    xorg-libxdmcp-1.1.3        |       h516909a_0          18 KB  conda-forge\n",
      "    xorg-libxext-1.3.4         |       h516909a_0          51 KB  conda-forge\n",
      "    xorg-libxrender-0.9.10     |    h516909a_1002          31 KB  conda-forge\n",
      "    xorg-renderproto-0.11.1    |    h14c3975_1002           8 KB  conda-forge\n",
      "    xorg-xextproto-7.3.0       |    h14c3975_1002          27 KB  conda-forge\n",
      "    xorg-xproto-7.0.31         |    h14c3975_1007          72 KB  conda-forge\n",
      "    zeromq-4.3.2               |       he1b5a44_2         668 KB  conda-forge\n",
      "    zipp-0.6.0                 |             py_0           7 KB  conda-forge\n",
      "    zstd-1.4.0                 |       h3b9ef0a_0         928 KB  conda-forge\n",
      "    ------------------------------------------------------------\n",
      "                                           Total:       376.2 MB\n",
      "\n",
      "The following NEW packages will be INSTALLED:\n",
      "\n",
      "  _libgcc_mutex      pkgs/main/linux-64::_libgcc_mutex-0.1-main\n",
      "  _r-mutex           conda-forge/noarch::_r-mutex-1.0.1-anacondar_1\n",
      "  attrs              conda-forge/noarch::attrs-19.3.0-py_0\n",
      "  backcall           conda-forge/noarch::backcall-0.1.0-py_0\n",
      "  binutils_impl_lin~ conda-forge/linux-64::binutils_impl_linux-64-2.31.1-h7fc9f1b_2\n",
      "  binutils_linux-64  conda-forge/linux-64::binutils_linux-64-2.31.1-h6176602_12\n",
      "  bioconductor-anno~ bioconda/noarch::bioconductor-annotate-1.62.0-r36_1\n",
      "  bioconductor-anno~ bioconda/noarch::bioconductor-annotationdbi-1.46.0-r36_1\n",
      "  bioconductor-biob~ bioconda/linux-64::bioconductor-biobase-2.44.0-r36h516909a_1\n",
      "  bioconductor-bioc~ bioconda/noarch::bioconductor-biocgenerics-0.30.0-r36_1\n",
      "  bioconductor-bioc~ bioconda/linux-64::bioconductor-biocparallel-1.18.0-r36he1b5a44_1\n",
      "  bioconductor-dela~ bioconda/linux-64::bioconductor-delayedarray-0.10.0-r36h516909a_1\n",
      "  bioconductor-dese~ bioconda/linux-64::bioconductor-deseq2-1.24.0-r36he1b5a44_1\n",
      "  bioconductor-gene~ bioconda/linux-64::bioconductor-genefilter-1.66.0-r36hc99cbb1_1\n",
      "  bioconductor-gene~ bioconda/noarch::bioconductor-geneplotter-1.62.0-r36_1\n",
      "  bioconductor-geno~ bioconda/noarch::bioconductor-genomeinfodb-1.20.0-r36_1\n",
      "  bioconductor-geno~ bioconda/noarch::bioconductor-genomeinfodbdata-1.2.1-r36_1\n",
      "  bioconductor-geno~ bioconda/linux-64::bioconductor-genomicranges-1.36.0-r36h516909a_1\n",
      "  bioconductor-iran~ bioconda/linux-64::bioconductor-iranges-2.18.2-r36h516909a_0\n",
      "  bioconductor-s4ve~ bioconda/linux-64::bioconductor-s4vectors-0.22.0-r36h516909a_1\n",
      "  bioconductor-summ~ bioconda/noarch::bioconductor-summarizedexperiment-1.14.0-r36_1\n",
      "  bioconductor-xvec~ bioconda/linux-64::bioconductor-xvector-0.24.0-r36h516909a_1\n",
      "  bioconductor-zlib~ bioconda/linux-64::bioconductor-zlibbioc-1.30.0-r36h516909a_1\n",
      "  bleach             conda-forge/noarch::bleach-3.1.0-py_0\n",
      "  bwidget            conda-forge/linux-64::bwidget-1.9.13-0\n",
      "  bzip2              conda-forge/linux-64::bzip2-1.0.8-h516909a_1\n",
      "  ca-certificates    conda-forge/linux-64::ca-certificates-2019.9.11-hecc5488_0\n",
      "  cairo              conda-forge/linux-64::cairo-1.16.0-hfb77d84_1002\n",
      "  certifi            conda-forge/linux-64::certifi-2019.9.11-py36_0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  curl               conda-forge/linux-64::curl-7.65.3-hf8cf82a_0\n",
      "  decorator          conda-forge/noarch::decorator-4.4.0-py_0\n",
      "  defusedxml         conda-forge/noarch::defusedxml-0.6.0-py_0\n",
      "  entrypoints        conda-forge/linux-64::entrypoints-0.3-py36_1000\n",
      "  fontconfig         conda-forge/linux-64::fontconfig-2.13.1-h86ecdb6_1001\n",
      "  freetype           conda-forge/linux-64::freetype-2.10.0-he983fc9_1\n",
      "  fribidi            conda-forge/linux-64::fribidi-1.0.5-h516909a_1002\n",
      "  gcc_impl_linux-64  conda-forge/linux-64::gcc_impl_linux-64-7.3.0-habb00fd_2\n",
      "  gcc_linux-64       conda-forge/linux-64::gcc_linux-64-7.3.0-h553295d_12\n",
      "  gettext            conda-forge/linux-64::gettext-0.19.8.1-hc5be6a0_1002\n",
      "  gfortran_impl_lin~ conda-forge/linux-64::gfortran_impl_linux-64-7.3.0-hdf63c60_2\n",
      "  gfortran_linux-64  conda-forge/linux-64::gfortran_linux-64-7.3.0-h553295d_12\n",
      "  glib               conda-forge/linux-64::glib-2.58.3-h6f030ca_1002\n",
      "  graphite2          conda-forge/linux-64::graphite2-1.3.13-hf484d3e_1000\n",
      "  gsl                conda-forge/linux-64::gsl-2.5-h294904e_1\n",
      "  gxx_impl_linux-64  conda-forge/linux-64::gxx_impl_linux-64-7.3.0-hdf63c60_2\n",
      "  gxx_linux-64       conda-forge/linux-64::gxx_linux-64-7.3.0-h553295d_12\n",
      "  harfbuzz           conda-forge/linux-64::harfbuzz-2.4.0-h9f30f68_3\n",
      "  icu                conda-forge/linux-64::icu-64.2-he1b5a44_1\n",
      "  importlib_metadata conda-forge/linux-64::importlib_metadata-0.23-py36_0\n",
      "  ipykernel          conda-forge/linux-64::ipykernel-5.1.2-py36h5ca1d4c_0\n",
      "  ipython            conda-forge/linux-64::ipython-7.8.0-py36h5ca1d4c_0\n",
      "  ipython_genutils   conda-forge/noarch::ipython_genutils-0.2.0-py_1\n",
      "  jedi               conda-forge/linux-64::jedi-0.15.1-py36_0\n",
      "  jinja2             conda-forge/noarch::jinja2-2.10.3-py_0\n",
      "  jpeg               conda-forge/linux-64::jpeg-9c-h14c3975_1001\n",
      "  jsonschema         conda-forge/linux-64::jsonschema-3.1.1-py36_0\n",
      "  jupyter_client     conda-forge/linux-64::jupyter_client-5.3.3-py36_1\n",
      "  jupyter_core       conda-forge/noarch::jupyter_core-4.5.0-py_0\n",
      "  krb5               conda-forge/linux-64::krb5-1.16.3-h05b26f9_1001\n",
      "  libblas            conda-forge/linux-64::libblas-3.8.0-14_openblas\n",
      "  libcblas           conda-forge/linux-64::libcblas-3.8.0-14_openblas\n",
      "  libcurl            conda-forge/linux-64::libcurl-7.65.3-hda55be3_0\n",
      "  libedit            conda-forge/linux-64::libedit-3.1.20170329-hf8c457e_1001\n",
      "  libffi             conda-forge/linux-64::libffi-3.2.1-he1b5a44_1006\n",
      "  libgcc-ng          pkgs/main/linux-64::libgcc-ng-9.1.0-hdf63c60_0\n",
      "  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-7.3.0-hdf63c60_2\n",
      "  libiconv           conda-forge/linux-64::libiconv-1.15-h516909a_1005\n",
      "  liblapack          conda-forge/linux-64::liblapack-3.8.0-14_openblas\n",
      "  libopenblas        conda-forge/linux-64::libopenblas-0.3.7-h6e990d7_2\n",
      "  libpng             conda-forge/linux-64::libpng-1.6.37-hed695b0_0\n",
      "  libsodium          conda-forge/linux-64::libsodium-1.0.17-h516909a_0\n",
      "  libssh2            conda-forge/linux-64::libssh2-1.8.2-h22169c7_2\n",
      "  libstdcxx-ng       pkgs/main/linux-64::libstdcxx-ng-9.1.0-hdf63c60_0\n",
      "  libtiff            conda-forge/linux-64::libtiff-4.0.10-h57b8799_1003\n",
      "  libuuid            conda-forge/linux-64::libuuid-2.32.1-h14c3975_1000\n",
      "  libxcb             conda-forge/linux-64::libxcb-1.13-h14c3975_1002\n",
      "  libxml2            conda-forge/linux-64::libxml2-2.9.9-hee79883_5\n",
      "  lz4-c              conda-forge/linux-64::lz4-c-1.8.3-he1b5a44_1001\n",
      "  make               conda-forge/linux-64::make-4.2.1-h14c3975_2004\n",
      "  markupsafe         conda-forge/linux-64::markupsafe-1.1.1-py36h14c3975_0\n",
      "  mistune            conda-forge/linux-64::mistune-0.8.4-py36h14c3975_1000\n",
      "  more-itertools     conda-forge/noarch::more-itertools-7.2.0-py_0\n",
      "  nbconvert          conda-forge/linux-64::nbconvert-5.6.0-py36_1\n",
      "  nbformat           conda-forge/noarch::nbformat-4.4.0-py_1\n",
      "  ncurses            conda-forge/linux-64::ncurses-6.1-hf484d3e_1002\n",
      "  notebook           conda-forge/linux-64::notebook-6.0.1-py36_0\n",
      "  openssl            conda-forge/linux-64::openssl-1.1.1c-h516909a_0\n",
      "  pandoc             conda-forge/linux-64::pandoc-2.7.3-0\n",
      "  pandocfilters      conda-forge/noarch::pandocfilters-1.4.2-py_1\n",
      "  pango              conda-forge/linux-64::pango-1.42.4-ha030887_1\n",
      "  parso              conda-forge/noarch::parso-0.5.1-py_0\n",
      "  pcre               conda-forge/linux-64::pcre-8.43-he1b5a44_0\n",
      "  pexpect            conda-forge/linux-64::pexpect-4.7.0-py36_0\n",
      "  pickleshare        conda-forge/linux-64::pickleshare-0.7.5-py36_1000\n",
      "  pip                conda-forge/linux-64::pip-19.3-py36_0\n",
      "  pixman             conda-forge/linux-64::pixman-0.38.0-h516909a_1003\n",
      "  prometheus_client  conda-forge/noarch::prometheus_client-0.7.1-py_0\n",
      "  prompt_toolkit     conda-forge/noarch::prompt_toolkit-2.0.10-py_0\n",
      "  pthread-stubs      conda-forge/linux-64::pthread-stubs-0.4-h14c3975_1001\n",
      "  ptyprocess         conda-forge/noarch::ptyprocess-0.6.0-py_1001\n",
      "  pygments           conda-forge/noarch::pygments-2.4.2-py_0\n",
      "  pyrsistent         conda-forge/linux-64::pyrsistent-0.15.4-py36h516909a_0\n",
      "  python             conda-forge/linux-64::python-3.6.7-h357f687_1005\n",
      "  python-dateutil    conda-forge/noarch::python-dateutil-2.8.0-py_0\n",
      "  pyzmq              conda-forge/linux-64::pyzmq-18.1.0-py36h1768529_0\n",
      "  r-acepack          conda-forge/linux-64::r-acepack-1.4.1-r36h9bbef5b_1004\n",
      "  r-askpass          conda-forge/linux-64::r-askpass-1.1-r36hcdcec82_1\n",
      "  r-assertthat       conda-forge/noarch::r-assertthat-0.2.1-r36h6115d3f_1\n",
      "  r-backports        conda-forge/linux-64::r-backports-1.1.5-r36hcdcec82_0\n",
      "  r-base             conda-forge/linux-64::r-base-3.6.1-hba50c9b_4\n",
      "  r-base64enc        conda-forge/linux-64::r-base64enc-0.1_3-r36hcdcec82_1003\n",
      "  r-bh               conda-forge/noarch::r-bh-1.69.0_1-r36h6115d3f_1\n",
      "  r-bit              conda-forge/linux-64::r-bit-1.1_14-r36hcdcec82_1\n",
      "  r-bit64            conda-forge/linux-64::r-bit64-0.9_7-r36hcdcec82_1001\n",
      "  r-bitops           conda-forge/linux-64::r-bitops-1.0_6-r36hcdcec82_1003\n",
      "  r-blob             conda-forge/noarch::r-blob-1.2.0-r36_1\n",
      "  r-boot             conda-forge/noarch::r-boot-1.3_23-r36h6115d3f_2\n",
      "  r-broom            conda-forge/noarch::r-broom-0.5.2-r36h6115d3f_1\n",
      "  r-callr            conda-forge/noarch::r-callr-3.3.2-r36h6115d3f_0\n",
      "  r-caret            conda-forge/linux-64::r-caret-6.0_84-r36hcdcec82_1\n",
      "  r-cellranger       conda-forge/noarch::r-cellranger-1.1.0-r36h6115d3f_1002\n",
      "  r-checkmate        conda-forge/linux-64::r-checkmate-1.9.4-r36hcdcec82_1\n",
      "  r-class            conda-forge/linux-64::r-class-7.3_15-r36hcdcec82_1001\n",
      "  r-cli              conda-forge/noarch::r-cli-1.1.0-r36h6115d3f_2\n",
      "  r-clipr            conda-forge/noarch::r-clipr-0.7.0-r36h6115d3f_0\n",
      "  r-cluster          conda-forge/linux-64::r-cluster-2.1.0-r36h9bbef5b_2\n",
      "  r-codetools        conda-forge/noarch::r-codetools-0.2_16-r36h6115d3f_1001\n",
      "  r-colorspace       conda-forge/linux-64::r-colorspace-1.4_1-r36hcdcec82_1\n",
      "  r-crayon           conda-forge/noarch::r-crayon-1.3.4-r36h6115d3f_1002\n",
      "  r-curl             conda-forge/linux-64::r-curl-4.2-r36hcdcec82_0\n",
      "  r-data.table       conda-forge/linux-64::r-data.table-1.12.4-r36hcdcec82_0\n",
      "  r-dbi              conda-forge/noarch::r-dbi-1.0.0-r36h6115d3f_1002\n",
      "  r-dbplyr           conda-forge/noarch::r-dbplyr-1.4.2-r36h6115d3f_1\n",
      "  r-digest           conda-forge/linux-64::r-digest-0.6.21-r36h0357c0b_0\n",
      "  r-dplyr            conda-forge/linux-64::r-dplyr-0.8.3-r36h0357c0b_3\n",
      "  r-ellipsis         conda-forge/linux-64::r-ellipsis-0.3.0-r36hcdcec82_0\n",
      "  r-essentials       conda-forge/noarch::r-essentials-3.6-r36_2001\n",
      "  r-evaluate         conda-forge/noarch::r-evaluate-0.14-r36h6115d3f_1\n",
      "  r-fansi            conda-forge/linux-64::r-fansi-0.4.0-r36hcdcec82_1001\n",
      "  r-fastmap          conda-forge/linux-64::r-fastmap-1.0.1-r36h0357c0b_0\n",
      "  r-forcats          conda-forge/noarch::r-forcats-0.4.0-r36h6115d3f_1\n",
      "  r-foreach          conda-forge/noarch::r-foreach-1.4.7-r36h6115d3f_0\n",
      "  r-foreign          conda-forge/linux-64::r-foreign-0.8_72-r36hcdcec82_0\n",
      "  r-formatr          conda-forge/noarch::r-formatr-1.7-r36h6115d3f_1\n",
      "  r-formula          conda-forge/noarch::r-formula-1.2_3-r36h6115d3f_1002\n",
      "  r-fs               conda-forge/linux-64::r-fs-1.3.1-r36h0357c0b_1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  r-futile.logger    conda-forge/noarch::r-futile.logger-1.4.3-r36h6115d3f_1002\n",
      "  r-futile.options   conda-forge/noarch::r-futile.options-1.0.1-r36h6115d3f_1001\n",
      "  r-generics         conda-forge/noarch::r-generics-0.0.2-r36h6115d3f_1002\n",
      "  r-ggplot2          conda-forge/noarch::r-ggplot2-3.2.1-r36h6115d3f_0\n",
      "  r-gistr            conda-forge/noarch::r-gistr-0.4.2-r36h6115d3f_1002\n",
      "  r-glmnet           conda-forge/linux-64::r-glmnet-2.0_18-r36h9bbef5b_2\n",
      "  r-glue             conda-forge/linux-64::r-glue-1.3.1-r36hcdcec82_1\n",
      "  r-gower            conda-forge/linux-64::r-gower-0.2.1-r36hcdcec82_1\n",
      "  r-gridextra        conda-forge/noarch::r-gridextra-2.3-r36h6115d3f_1002\n",
      "  r-gtable           conda-forge/noarch::r-gtable-0.3.0-r36h6115d3f_2\n",
      "  r-haven            conda-forge/linux-64::r-haven-2.1.1-r36hde08347_3\n",
      "  r-hexbin           conda-forge/linux-64::r-hexbin-1.27.3-r36h9bbef5b_2\n",
      "  r-highr            conda-forge/noarch::r-highr-0.8-r36h6115d3f_1\n",
      "  r-hmisc            conda-forge/linux-64::r-hmisc-4.2_0-r36h9bbef5b_2\n",
      "  r-hms              conda-forge/noarch::r-hms-0.5.1-r36h6115d3f_0\n",
      "  r-htmltable        conda-forge/noarch::r-htmltable-1.13.2-r36h6115d3f_0\n",
      "  r-htmltools        conda-forge/linux-64::r-htmltools-0.4.0-r36h0357c0b_0\n",
      "  r-htmlwidgets      conda-forge/noarch::r-htmlwidgets-1.5.1-r36h6115d3f_0\n",
      "  r-httpuv           conda-forge/linux-64::r-httpuv-1.5.2-r36h0357c0b_1\n",
      "  r-httr             conda-forge/noarch::r-httr-1.4.1-r36h6115d3f_1\n",
      "  r-ipred            conda-forge/linux-64::r-ipred-0.9_9-r36hcdcec82_1\n",
      "  r-irdisplay        conda-forge/noarch::r-irdisplay-0.7-r36_1001\n",
      "  r-irkernel         conda-forge/noarch::r-irkernel-1.0.2-r36h6115d3f_2\n",
      "  r-iterators        conda-forge/noarch::r-iterators-1.0.12-r36h6115d3f_0\n",
      "  r-jsonlite         conda-forge/linux-64::r-jsonlite-1.6-r36hcdcec82_1001\n",
      "  r-kernsmooth       conda-forge/linux-64::r-kernsmooth-2.23_15-r36h9bbef5b_1004\n",
      "  r-knitr            conda-forge/noarch::r-knitr-1.25-r36h6115d3f_0\n",
      "  r-labeling         conda-forge/noarch::r-labeling-0.3-r36h6115d3f_1002\n",
      "  r-lambda.r         conda-forge/noarch::r-lambda.r-1.2.4-r36h6115d3f_0\n",
      "  r-later            conda-forge/linux-64::r-later-1.0.0-r36h0357c0b_0\n",
      "  r-lattice          conda-forge/linux-64::r-lattice-0.20_38-r36hcdcec82_1002\n",
      "  r-latticeextra     conda-forge/noarch::r-latticeextra-0.6_28-r36h6115d3f_1002\n",
      "  r-lava             conda-forge/noarch::r-lava-1.6.6-r36h6115d3f_0\n",
      "  r-lazyeval         conda-forge/linux-64::r-lazyeval-0.2.2-r36hcdcec82_1\n",
      "  r-lifecycle        conda-forge/noarch::r-lifecycle-0.1.0-r36h6115d3f_0\n",
      "  r-locfit           conda-forge/linux-64::r-locfit-1.5_9.1-r36h516909a_1004\n",
      "  r-lubridate        conda-forge/linux-64::r-lubridate-1.7.4-r36h0357c0b_1002\n",
      "  r-magrittr         conda-forge/noarch::r-magrittr-1.5-r36h6115d3f_1002\n",
      "  r-maps             conda-forge/linux-64::r-maps-3.3.0-r36hcdcec82_1003\n",
      "  r-markdown         conda-forge/linux-64::r-markdown-1.1-r36hcdcec82_0\n",
      "  r-mass             conda-forge/linux-64::r-mass-7.3_51.4-r36hcdcec82_1\n",
      "  r-matrix           conda-forge/linux-64::r-matrix-1.2_17-r36hcdcec82_1\n",
      "  r-matrixstats      conda-forge/linux-64::r-matrixstats-0.55.0-r36hcdcec82_0\n",
      "  r-memoise          conda-forge/noarch::r-memoise-1.1.0-r36h6115d3f_1003\n",
      "  r-mgcv             conda-forge/linux-64::r-mgcv-1.8_29-r36hcdcec82_0\n",
      "  r-mime             conda-forge/linux-64::r-mime-0.7-r36hcdcec82_1\n",
      "  r-modelmetrics     conda-forge/linux-64::r-modelmetrics-1.2.2-r36h0357c0b_1\n",
      "  r-modelr           conda-forge/noarch::r-modelr-0.1.5-r36h6115d3f_0\n",
      "  r-munsell          conda-forge/noarch::r-munsell-0.5.0-r36h6115d3f_1002\n",
      "  r-nlme             conda-forge/linux-64::r-nlme-3.1_141-r36h9bbef5b_1\n",
      "  r-nnet             conda-forge/linux-64::r-nnet-7.3_12-r36hcdcec82_1003\n",
      "  r-numderiv         conda-forge/noarch::r-numderiv-2016.8_1.1-r36h6115d3f_1\n",
      "  r-openssl          conda-forge/linux-64::r-openssl-1.4.1-r36h9c8475f_0\n",
      "  r-pbdzmq           conda-forge/linux-64::r-pbdzmq-0.3_3-r36h559a7a4_1002\n",
      "  r-pheatmap         conda-forge/noarch::r-pheatmap-1.0.12-r36h6115d3f_1\n",
      "  r-pillar           conda-forge/noarch::r-pillar-1.4.2-r36h6115d3f_2\n",
      "  r-pkgconfig        conda-forge/noarch::r-pkgconfig-2.0.3-r36h6115d3f_0\n",
      "  r-plogr            conda-forge/noarch::r-plogr-0.2.0-r36h6115d3f_1002\n",
      "  r-plyr             conda-forge/linux-64::r-plyr-1.8.4-r36h0357c0b_1003\n",
      "  r-prettyunits      conda-forge/noarch::r-prettyunits-1.0.2-r36h6115d3f_1002\n",
      "  r-processx         conda-forge/linux-64::r-processx-3.4.1-r36hcdcec82_0\n",
      "  r-prodlim          conda-forge/linux-64::r-prodlim-2018.04.18-r36h0357c0b_1003\n",
      "  r-progress         conda-forge/noarch::r-progress-1.2.2-r36h6115d3f_1\n",
      "  r-promises         conda-forge/linux-64::r-promises-1.1.0-r36h0357c0b_0\n",
      "  r-pryr             conda-forge/linux-64::r-pryr-0.1.4-r36h0357c0b_1003\n",
      "  r-ps               conda-forge/linux-64::r-ps-1.3.0-r36hcdcec82_1001\n",
      "  r-purrr            conda-forge/linux-64::r-purrr-0.3.2-r36hcdcec82_1\n",
      "  r-quantmod         conda-forge/noarch::r-quantmod-0.4_15-r36h6115d3f_1\n",
      "  r-r6               conda-forge/noarch::r-r6-2.4.0-r36h6115d3f_2\n",
      "  r-randomforest     conda-forge/linux-64::r-randomforest-4.6_14-r36h9bbef5b_1002\n",
      "  r-rbokeh           conda-forge/noarch::r-rbokeh-0.5.0-r36h6115d3f_1002\n",
      "  r-rcolorbrewer     conda-forge/noarch::r-rcolorbrewer-1.1_2-r36h6115d3f_1002\n",
      "  r-rcpp             conda-forge/linux-64::r-rcpp-1.0.2-r36h0357c0b_0\n",
      "  r-rcpparmadillo    conda-forge/linux-64::r-rcpparmadillo-0.9.700.2.0-r36h0357c0b_0\n",
      "  r-rcurl            conda-forge/linux-64::r-rcurl-1.95_4.12-r36hcdcec82_1\n",
      "  r-readr            conda-forge/linux-64::r-readr-1.3.1-r36h0357c0b_1002\n",
      "  r-readxl           conda-forge/linux-64::r-readxl-1.3.1-r36h0357c0b_2\n",
      "  r-recipes          conda-forge/noarch::r-recipes-0.1.7-r36h6115d3f_0\n",
      "  r-recommended      conda-forge/noarch::r-recommended-3.6-r36_1003\n",
      "  r-rematch          conda-forge/noarch::r-rematch-1.0.1-r36h6115d3f_1002\n",
      "  r-repr             conda-forge/noarch::r-repr-1.0.1-r36h6115d3f_1\n",
      "  r-reprex           conda-forge/noarch::r-reprex-0.3.0-r36h6115d3f_1\n",
      "  r-reshape2         conda-forge/linux-64::r-reshape2-1.4.3-r36h0357c0b_1004\n",
      "  r-rlang            conda-forge/linux-64::r-rlang-0.4.0-r36hcdcec82_1\n",
      "  r-rmarkdown        conda-forge/noarch::r-rmarkdown-1.16-r36h6115d3f_0\n",
      "  r-rpart            conda-forge/linux-64::r-rpart-4.1_15-r36hcdcec82_1\n",
      "  r-rsqlite          conda-forge/linux-64::r-rsqlite-2.1.2-r36h0357c0b_1\n",
      "  r-rstudioapi       conda-forge/noarch::r-rstudioapi-0.10-r36h6115d3f_2\n",
      "  r-rvest            conda-forge/noarch::r-rvest-0.3.4-r36h6115d3f_1\n",
      "  r-scales           conda-forge/linux-64::r-scales-1.0.0-r36h0357c0b_1002\n",
      "  r-selectr          conda-forge/noarch::r-selectr-0.4_1-r36h6115d3f_1001\n",
      "  r-shiny            conda-forge/noarch::r-shiny-1.4.0-r36h6115d3f_0\n",
      "  r-snow             conda-forge/noarch::r-snow-0.4_3-r36h6115d3f_1001\n",
      "  r-sourcetools      conda-forge/linux-64::r-sourcetools-0.1.7-r36he1b5a44_1001\n",
      "  r-spatial          conda-forge/linux-64::r-spatial-7.3_11-r36hcdcec82_1003\n",
      "  r-squarem          conda-forge/noarch::r-squarem-2017.10_1-r36h6115d3f_1002\n",
      "  r-stringi          conda-forge/linux-64::r-stringi-1.4.3-r36h0e574ca_3\n",
      "  r-stringr          conda-forge/noarch::r-stringr-1.4.0-r36h6115d3f_1\n",
      "  r-survival         conda-forge/linux-64::r-survival-2.44_1.1-r36hcdcec82_1\n",
      "  r-sys              conda-forge/linux-64::r-sys-3.3-r36hcdcec82_0\n",
      "  r-tibble           conda-forge/linux-64::r-tibble-2.1.3-r36hcdcec82_1\n",
      "  r-tidyr            conda-forge/linux-64::r-tidyr-1.0.0-r36h0357c0b_0\n",
      "  r-tidyselect       conda-forge/linux-64::r-tidyselect-0.2.5-r36h0357c0b_1001\n",
      "  r-tidyverse        conda-forge/noarch::r-tidyverse-1.2.1-r36h6115d3f_1002\n",
      "  r-timedate         conda-forge/noarch::r-timedate-3043.102-r36h6115d3f_1001\n",
      "  r-tinytex          conda-forge/noarch::r-tinytex-0.16-r36h6115d3f_0\n",
      "  r-ttr              conda-forge/linux-64::r-ttr-0.23_5-r36hcdcec82_0\n",
      "  r-utf8             conda-forge/linux-64::r-utf8-1.1.4-r36hcdcec82_1001\n",
      "  r-uuid             conda-forge/linux-64::r-uuid-0.1_2-r36hcdcec82_1002\n",
      "  r-vctrs            conda-forge/linux-64::r-vctrs-0.2.0-r36hcdcec82_1\n",
      "  r-venndiagram      conda-forge/noarch::r-venndiagram-1.6.20-r36h6115d3f_1001\n",
      "  r-viridis          conda-forge/noarch::r-viridis-0.5.1-r36h6115d3f_1003\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  r-viridislite      conda-forge/noarch::r-viridislite-0.3.0-r36h6115d3f_1002\n",
      "  r-whisker          conda-forge/noarch::r-whisker-0.4-r36h6115d3f_0\n",
      "  r-withr            conda-forge/noarch::r-withr-2.1.2-r36h6115d3f_1001\n",
      "  r-xfun             conda-forge/noarch::r-xfun-0.10-r36h6115d3f_0\n",
      "  r-xml              conda-forge/linux-64::r-xml-3.98_1.20-r36hcdcec82_1\n",
      "  r-xml2             conda-forge/linux-64::r-xml2-1.2.2-r36h0357c0b_0\n",
      "  r-xtable           conda-forge/noarch::r-xtable-1.8_4-r36h6115d3f_2\n",
      "  r-xts              conda-forge/linux-64::r-xts-0.11_2-r36hcdcec82_1\n",
      "  r-yaml             conda-forge/linux-64::r-yaml-2.2.0-r36hcdcec82_1002\n",
      "  r-zeallot          conda-forge/noarch::r-zeallot-0.1.0-r36h6115d3f_1001\n",
      "  r-zoo              conda-forge/linux-64::r-zoo-1.8_6-r36hcdcec82_1\n",
      "  readline           conda-forge/linux-64::readline-8.0-hf8c457e_0\n",
      "  sed                conda-forge/linux-64::sed-4.7-h1bed415_1000\n",
      "  send2trash         conda-forge/noarch::send2trash-1.5.0-py_0\n",
      "  setuptools         conda-forge/linux-64::setuptools-41.4.0-py36_0\n",
      "  six                conda-forge/linux-64::six-1.12.0-py36_1000\n",
      "  sqlite             conda-forge/linux-64::sqlite-3.30.1-hcee41ef_0\n",
      "  terminado          conda-forge/linux-64::terminado-0.8.2-py36_0\n",
      "  testpath           conda-forge/noarch::testpath-0.4.2-py_1001\n",
      "  tk                 conda-forge/linux-64::tk-8.6.9-hed695b0_1003\n",
      "  tktable            conda-forge/linux-64::tktable-2.10-h555a92e_3\n",
      "  tornado            conda-forge/linux-64::tornado-6.0.3-py36h516909a_0\n",
      "  traitlets          conda-forge/linux-64::traitlets-4.3.3-py36_0\n",
      "  wcwidth            conda-forge/noarch::wcwidth-0.1.7-py_1\n",
      "  webencodings       conda-forge/noarch::webencodings-0.5.1-py_1\n",
      "  wheel              conda-forge/linux-64::wheel-0.33.6-py36_0\n",
      "  xorg-kbproto       conda-forge/linux-64::xorg-kbproto-1.0.7-h14c3975_1002\n",
      "  xorg-libice        conda-forge/linux-64::xorg-libice-1.0.10-h516909a_0\n",
      "  xorg-libsm         conda-forge/linux-64::xorg-libsm-1.2.3-h84519dc_1000\n",
      "  xorg-libx11        conda-forge/linux-64::xorg-libx11-1.6.9-h516909a_0\n",
      "  xorg-libxau        conda-forge/linux-64::xorg-libxau-1.0.9-h14c3975_0\n",
      "  xorg-libxdmcp      conda-forge/linux-64::xorg-libxdmcp-1.1.3-h516909a_0\n",
      "  xorg-libxext       conda-forge/linux-64::xorg-libxext-1.3.4-h516909a_0\n",
      "  xorg-libxrender    conda-forge/linux-64::xorg-libxrender-0.9.10-h516909a_1002\n",
      "  xorg-renderproto   conda-forge/linux-64::xorg-renderproto-0.11.1-h14c3975_1002\n",
      "  xorg-xextproto     conda-forge/linux-64::xorg-xextproto-7.3.0-h14c3975_1002\n",
      "  xorg-xproto        conda-forge/linux-64::xorg-xproto-7.0.31-h14c3975_1007\n",
      "  xz                 conda-forge/linux-64::xz-5.2.4-h14c3975_1001\n",
      "  zeromq             conda-forge/linux-64::zeromq-4.3.2-he1b5a44_2\n",
      "  zipp               conda-forge/noarch::zipp-0.6.0-py_0\n",
      "  zlib               conda-forge/linux-64::zlib-1.2.11-h516909a_1006\n",
      "  zstd               conda-forge/linux-64::zstd-1.4.0-h3b9ef0a_0\n",
      "\n",
      "\n",
      "\n",
      "Downloading and Extracting Packages\n",
      "gettext-0.19.8.1     | 3.6 MB    | ##################################### | 100% \n",
      "r-base-3.6.1         | 23.4 MB   | ##################################### | 100% \n",
      "r-snow-0.4_3         | 124 KB    | ##################################### | 100% \n",
      "r-htmlwidgets-1.5.1  | 679 KB    | ##################################### | 100% \n",
      "r-ps-1.3.0           | 222 KB    | ##################################### | 100% \n",
      "bioconductor-genefil | 1.7 MB    | ##################################### | 100% \n",
      "zeromq-4.3.2         | 668 KB    | ##################################### | 100% \n",
      "r-vctrs-0.2.0        | 764 KB    | ##################################### | 100% \n",
      "r-hexbin-1.27.3      | 965 KB    | ##################################### | 100% \n",
      "r-askpass-1.1        | 27 KB     | ##################################### | 100% \n",
      "libiconv-1.15        | 2.0 MB    | ##################################### | 100% \n",
      "r-rstudioapi-0.10    | 222 KB    | ##################################### | 100% \n",
      "r-viridis-0.5.1      | 1.8 MB    | ##################################### | 100% \n",
      "ipykernel-5.1.2      | 159 KB    | ##################################### | 100% \n",
      "r-gridextra-2.3      | 1.1 MB    | ##################################### | 100% \n",
      "r-formula-1.2_3      | 193 KB    | ##################################### | 100% \n",
      "libgfortran-ng-7.3.0 | 1.7 MB    | ##################################### | 100% \n",
      "markupsafe-1.1.1     | 26 KB     | ##################################### | 100% \n",
      "r-futile.options-1.0 | 24 KB     | ##################################### | 100% \n",
      "ptyprocess-0.6.0     | 15 KB     | ##################################### | 100% \n",
      "r-modelr-0.1.5       | 217 KB    | ##################################### | 100% \n",
      "binutils_impl_linux- | 16.4 MB   | ##################################### | 100% \n",
      "pickleshare-0.7.5    | 12 KB     | ##################################### | 100% \n",
      "liblapack-3.8.0      | 10 KB     | ##################################### | 100% \n",
      "_r-mutex-1.0.1       | 3 KB      | ##################################### | 100% \n",
      "zipp-0.6.0           | 7 KB      | ##################################### | 100% \n",
      "r-whisker-0.4        | 80 KB     | ##################################### | 100% \n",
      "r-fs-1.3.1           | 535 KB    | ##################################### | 100% \n",
      "libblas-3.8.0        | 10 KB     | ##################################### | 100% \n",
      "gcc_impl_linux-64-7. | 70.5 MB   | ##################################### | 100% \n",
      "r-clipr-0.7.0        | 64 KB     | ##################################### | 100% \n",
      "r-bh-1.69.0_1        | 10.6 MB   | ##################################### | 100% \n",
      "r-bit-1.1_14         | 247 KB    | ##################################### | 100% \n",
      "xorg-libxau-1.0.9    | 13 KB     | ##################################### | 100% \n",
      "r-stringi-1.4.3      | 778 KB    | ##################################### | 100% \n",
      "r-plyr-1.8.4         | 819 KB    | ##################################### | 100% \n",
      "bioconductor-iranges | 2.3 MB    | ##################################### | 100% \n",
      "r-tibble-2.1.3       | 315 KB    | ##################################### | 100% \n",
      "bioconductor-genomei | 3.8 MB    | ##################################### | 100% \n",
      "r-rcolorbrewer-1.1_2 | 59 KB     | ##################################### | 100% \n",
      "libpng-1.6.37        | 343 KB    | ##################################### | 100% \n",
      "r-rcpp-1.0.2         | 3.3 MB    | ##################################### | 100% \n",
      "r-rematch-1.0.1      | 18 KB     | ##################################### | 100% \n",
      "r-lifecycle-0.1.0    | 95 KB     | ##################################### | 100% \n",
      "r-mgcv-1.8_29        | 2.7 MB    | ##################################### | 100% \n",
      "r-viridislite-0.3.0  | 64 KB     | ##################################### | 100% \n",
      "bioconductor-xvector | 721 KB    | ##################################### | 100% \n",
      "r-fansi-0.4.0        | 193 KB    | ##################################### | 100% \n",
      "r-curl-4.2           | 692 KB    | ##################################### | 100% \n",
      "r-later-1.0.0        | 140 KB    | ##################################### | 100% \n",
      "sed-4.7              | 245 KB    | ##################################### | 100% \n",
      "bioconductor-annotat | 4.9 MB    | ##################################### | 100% \n",
      "r-munsell-0.5.0      | 245 KB    | ##################################### | 100% \n",
      "r-magrittr-1.5       | 167 KB    | ##################################### | 100% \n",
      "pandocfilters-1.4.2  | 9 KB      | ##################################### | 100% \n",
      "r-spatial-7.3_11     | 154 KB    | ##################################### | 100% \n",
      "r-cellranger-1.1.0   | 107 KB    | ##################################### | 100% \n",
      "xorg-xproto-7.0.31   | 72 KB     | ##################################### | 100% \n",
      "r-prettyunits-1.0.2  | 35 KB     | ##################################### | 100% \n",
      "gxx_linux-64-7.3.0   | 11 KB     | ##################################### | 100% \n",
      "r-digest-0.6.21      | 189 KB    | ##################################### | 100% \n",
      "r-reprex-0.3.0       | 428 KB    | ##################################### | 100% \n",
      "r-rbokeh-0.5.0       | 1.4 MB    | ##################################### | 100% \n",
      "pandoc-2.7.3         | 22.4 MB   | ##################################### | 100% \n",
      "r-data.table-1.12.4  | 1.8 MB    | ##################################### | 100% \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "r-foreach-1.4.7      | 401 KB    | ##################################### | 100% \n",
      "r-boot-1.3_23        | 632 KB    | ##################################### | 100% \n",
      "r-survival-2.44_1.1  | 4.8 MB    | ##################################### | 100% \n",
      "graphite2-1.3.13     | 109 KB    | ##################################### | 100% \n",
      "r-ttr-0.23_5         | 516 KB    | ##################################### | 100% \n",
      "r-rlang-0.4.0        | 1020 KB   | ##################################### | 100% \n",
      "bioconductor-geneplo | 1.5 MB    | ##################################### | 100% \n",
      "r-knitr-1.25         | 1.3 MB    | ##################################### | 100% \n",
      "r-essentials-3.6     | 4 KB      | ##################################### | 100% \n",
      "r-rvest-0.3.4        | 922 KB    | ##################################### | 100% \n",
      "r-mime-0.7           | 51 KB     | ##################################### | 100% \n",
      "backcall-0.1.0       | 13 KB     | ##################################### | 100% \n",
      "r-squarem-2017.10_1  | 301 KB    | ##################################### | 100% \n",
      "traitlets-4.3.3      | 133 KB    | ##################################### | 100% \n",
      "r-irdisplay-0.7      | 35 KB     | ##################################### | 100% \n",
      "lz4-c-1.8.3          | 187 KB    | ##################################### | 100% \n",
      "r-pkgconfig-2.0.3    | 24 KB     | ##################################### | 100% \n",
      "r-randomforest-4.6_1 | 235 KB    | ##################################### | 100% \n",
      "make-4.2.1           | 458 KB    | ##################################### | 100% \n",
      "xorg-kbproto-1.0.7   | 26 KB     | ##################################### | 100% \n",
      "r-uuid-0.1_2         | 21 KB     | ##################################### | 100% \n",
      "r-htmltools-0.4.0    | 215 KB    | ##################################### | 100% \n",
      "r-matrix-1.2_17      | 3.8 MB    | ##################################### | 100% \n",
      "r-sourcetools-0.1.7  | 51 KB     | ##################################### | 100% \n",
      "r-forcats-0.4.0      | 365 KB    | ##################################### | 100% \n",
      "xorg-libx11-1.6.9    | 918 KB    | ##################################### | 100% \n",
      "r-rpart-4.1_15       | 733 KB    | ##################################### | 100% \n",
      "r-withr-2.1.2        | 175 KB    | ##################################### | 100% \n",
      "r-readr-1.3.1        | 827 KB    | ##################################### | 100% \n",
      "r-maps-3.3.0         | 3.4 MB    | ##################################### | 100% \n",
      "r-xml-3.98_1.20      | 2.0 MB    | ##################################### | 100% \n",
      "r-plogr-0.2.0        | 18 KB     | ##################################### | 100% \n",
      "r-backports-1.1.5    | 68 KB     | ##################################### | 100% \n",
      "r-latticeextra-0.6_2 | 2.1 MB    | ##################################### | 100% \n",
      "r-pryr-0.1.4         | 237 KB    | ##################################### | 100% \n",
      "r-httpuv-1.5.2       | 842 KB    | ##################################### | 100% \n",
      "r-modelmetrics-1.2.2 | 162 KB    | ##################################### | 100% \n",
      "libuuid-2.32.1       | 26 KB     | ##################################### | 100% \n",
      "xorg-libxdmcp-1.1.3  | 18 KB     | ##################################### | 100% \n",
      "bioconductor-biocpar | 1.1 MB    | ##################################### | 100% \n",
      "r-highr-0.8          | 47 KB     | ##################################### | 100% \n",
      "r-ellipsis-0.3.0     | 47 KB     | ##################################### | 100% \n",
      "bioconductor-genomei | 6 KB      | ##################################### | 100% \n",
      "bioconductor-summari | 2.5 MB    | ##################################### | 100% \n",
      "bioconductor-biobase | 2.3 MB    | ##################################### | 100% \n",
      "libxcb-1.13          | 396 KB    | ##################################### | 100% \n",
      "tornado-6.0.3        | 636 KB    | ##################################### | 100% \n",
      "jupyter_client-5.3.3 | 132 KB    | ##################################### | 100% \n",
      "r-kernsmooth-2.23_15 | 103 KB    | ##################################### | 100% \n",
      "notebook-6.0.1       | 6.0 MB    | ##################################### | 100% \n",
      "r-xts-0.11_2         | 912 KB    | ##################################### | 100% \n",
      "pexpect-4.7.0        | 78 KB     | ##################################### | 100% \n",
      "r-dplyr-0.8.3        | 1.9 MB    | ##################################### | 100% \n",
      "r-lattice-0.20_38    | 1.1 MB    | ##################################### | 100% \n",
      "send2trash-1.5.0     | 12 KB     | ##################################### | 100% \n",
      "r-gistr-0.4.2        | 1.8 MB    | ##################################### | 100% \n",
      "bioconductor-delayed | 2.2 MB    | ##################################### | 100% \n",
      "r-httr-1.4.1         | 489 KB    | ##################################### | 100% \n",
      "r-selectr-0.4_1      | 476 KB    | ##################################### | 100% \n",
      "r-tidyverse-1.2.1    | 98 KB     | ##################################### | 100% \n",
      "r-recipes-0.1.7      | 1.5 MB    | ##################################### | 100% \n",
      "bioconductor-zlibbio | 112 KB    | ##################################### | 100% \n",
      "xorg-xextproto-7.3.0 | 27 KB     | ##################################### | 100% \n",
      "r-futile.logger-1.4. | 104 KB    | ##################################### | 100% \n",
      "pango-1.42.4         | 517 KB    | ##################################### | 100% \n",
      "r-memoise-1.1.0      | 42 KB     | ##################################### | 100% \n",
      "xorg-libxext-1.3.4   | 51 KB     | ##################################### | 100% \n",
      "r-zeallot-0.1.0      | 62 KB     | ##################################### | 100% \n",
      "bioconductor-genomic | 2.1 MB    | ##################################### | 100% \n",
      "r-venndiagram-1.6.20 | 266 KB    | ##################################### | 100% \n",
      "r-repr-1.0.1         | 143 KB    | ##################################### | 100% \n",
      "libtiff-4.0.10       | 587 KB    | ##################################### | 100% \n",
      "r-fastmap-1.0.1      | 55 KB     | ##################################### | 100% \n",
      "attrs-19.3.0         | 35 KB     | ##################################### | 100% \n",
      "r-reshape2-1.4.3     | 129 KB    | ##################################### | 100% \n",
      "fribidi-1.0.5        | 112 KB    | ##################################### | 100% \n",
      "glib-2.58.3          | 3.3 MB    | ##################################### | 100% \n",
      "r-lambda.r-1.2.4     | 119 KB    | ##################################### | 100% \n",
      "r-iterators-1.0.12   | 334 KB    | ##################################### | 100% \n",
      "r-labeling-0.3       | 66 KB     | ##################################### | 100% \n",
      "jpeg-9c              | 251 KB    | ##################################### | 100% \n",
      "importlib_metadata-0 | 39 KB     | ##################################### | 100% \n",
      "libopenblas-0.3.7    | 7.6 MB    | ##################################### | 100% \n",
      "r-nlme-3.1_141       | 2.3 MB    | ##################################### | 100% \n",
      "jupyter_core-4.5.0   | 44 KB     | ##################################### | 100% \n",
      "bioconductor-biocgen | 683 KB    | ##################################### | 100% \n",
      "r-processx-3.4.1     | 253 KB    | ##################################### | 100% \n",
      "parso-0.5.1          | 65 KB     | ##################################### | 100% \n",
      "r-callr-3.3.2        | 332 KB    | ##################################### | 100% \n",
      "binutils_linux-64-2. | 10 KB     | ##################################### | 100% \n",
      "gfortran_linux-64-7. | 11 KB     | ##################################### | 100% \n",
      "terminado-0.8.2      | 23 KB     | ##################################### | 100% \n",
      "mistune-0.8.4        | 53 KB     | ##################################### | 100% \n",
      "r-lazyeval-0.2.2     | 166 KB    | ##################################### | 100% \n",
      "r-class-7.3_15       | 106 KB    | ##################################### | 100% \n",
      "pcre-8.43            | 257 KB    | ##################################### | 100% \n",
      "r-base64enc-0.1_3    | 43 KB     | ##################################### | 100% \n",
      "pyrsistent-0.15.4    | 89 KB     | ##################################### | 100% \n",
      "fontconfig-2.13.1    | 340 KB    | ##################################### | 100% \n",
      "r-rsqlite-2.1.2      | 1.0 MB    | ##################################### | 100% \n",
      "r-glmnet-2.0_18      | 1.4 MB    | ##################################### | 100% \n",
      "r-tidyr-1.0.0        | 757 KB    | ##################################### | 100% \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "r-promises-1.1.0     | 1.2 MB    | ##################################### | 100% \n",
      "libsodium-1.0.17     | 330 KB    | ##################################### | 100% \n",
      "r-readxl-1.3.1       | 849 KB    | ##################################### | 100% \n",
      "r-hms-0.5.1          | 123 KB    | ##################################### | 100% \n",
      "r-recommended-3.6    | 3 KB      | ##################################### | 100% \n",
      "r-crayon-1.3.4       | 746 KB    | ##################################### | 100% \n",
      "r-assertthat-0.2.1   | 70 KB     | ##################################### | 100% \n",
      "ipython-7.8.0        | 1.1 MB    | ##################################### | 100% \n",
      "ipython_genutils-0.2 | 21 KB     | ##################################### | 100% \n",
      "r-rmarkdown-1.16     | 3.0 MB    | ##################################### | 100% \n",
      "wcwidth-0.1.7        | 17 KB     | ##################################### | 100% \n",
      "r-haven-2.1.1        | 335 KB    | ##################################### | 100% \n",
      "pthread-stubs-0.4    | 5 KB      | ##################################### | 100% \n",
      "xorg-libice-1.0.10   | 57 KB     | ##################################### | 100% \n",
      "entrypoints-0.3      | 12 KB     | ##################################### | 100% \n",
      "r-foreign-0.8_72     | 267 KB    | ##################################### | 100% \n",
      "r-glue-1.3.1         | 165 KB    | ##################################### | 100% \n",
      "r-matrixstats-0.55.0 | 912 KB    | ##################################### | 100% \n",
      "r-stringr-1.4.0      | 209 KB    | ##################################### | 100% \n",
      "r-tinytex-0.16       | 103 KB    | ##################################### | 100% \n",
      "r-scales-1.0.0       | 582 KB    | ##################################### | 100% \n",
      "r-r6-2.4.0           | 63 KB     | ##################################### | 100% \n",
      "r-tidyselect-0.2.5   | 138 KB    | ##################################### | 100% \n",
      "r-pheatmap-1.0.12    | 88 KB     | ##################################### | 100% \n",
      "gxx_impl_linux-64-7. | 19.1 MB   | ##################################### | 100% \n",
      "defusedxml-0.6.0     | 22 KB     | ##################################### | 100% \n",
      "r-numderiv-2016.8_1. | 125 KB    | ##################################### | 100% \n",
      "r-nnet-7.3_12        | 131 KB    | ##################################### | 100% \n",
      "r-rcpparmadillo-0.9. | 1.2 MB    | ##################################### | 100% \n",
      "r-irkernel-1.0.2     | 155 KB    | ##################################### | 100% \n",
      "r-gower-0.2.1        | 175 KB    | ##################################### | 100% \n",
      "r-xfun-0.10          | 184 KB    | ##################################### | 100% \n",
      "jsonschema-3.1.1     | 88 KB     | ##################################### | 100% \n",
      "xorg-libxrender-0.9. | 31 KB     | ##################################### | 100% \n",
      "r-purrr-0.3.2        | 403 KB    | ##################################### | 100% \n",
      "r-dbplyr-1.4.2       | 602 KB    | ##################################### | 100% \n",
      "r-gtable-0.3.0       | 423 KB    | ##################################### | 100% \n",
      "r-cli-1.1.0          | 181 KB    | ##################################### | 100% \n",
      "prompt_toolkit-2.0.1 | 221 KB    | ##################################### | 100% \n",
      "harfbuzz-2.4.0       | 1.5 MB    | ##################################### | 100% \n",
      "webencodings-0.5.1   | 12 KB     | ##################################### | 100% \n",
      "r-rcurl-1.95_4.12    | 920 KB    | ##################################### | 100% \n",
      "xorg-libsm-1.2.3     | 25 KB     | ##################################### | 100% \n",
      "jinja2-2.10.3        | 91 KB     | ##################################### | 100% \n",
      "r-bit64-0.9_7        | 482 KB    | ##################################### | 100% \n",
      "r-lubridate-1.7.4    | 1.1 MB    | ##################################### | 100% \n",
      "r-dbi-1.0.0          | 890 KB    | ##################################### | 100% \n",
      "r-pbdzmq-0.3_3       | 380 KB    | ##################################### | 100% \n",
      "r-utf8-1.1.4         | 159 KB    | ##################################### | 100% \n",
      "r-evaluate-0.14      | 80 KB     | ##################################### | 100% \n",
      "r-broom-0.5.2        | 1.9 MB    | ##################################### | 100% \n",
      "r-zoo-1.8_6          | 1.1 MB    | ##################################### | 100% \n",
      "pyzmq-18.1.0         | 516 KB    | ##################################### | 100% \n",
      "gcc_linux-64-7.3.0   | 11 KB     | ##################################### | 100% \n",
      "r-lava-1.6.6         | 2.1 MB    | ##################################### | 100% \n",
      "nbconvert-5.6.0      | 465 KB    | ##################################### | 100% \n",
      "r-hmisc-4.2_0        | 3.0 MB    | ##################################### | 100% \n",
      "r-shiny-1.4.0        | 4.3 MB    | ##################################### | 100% \n",
      "prometheus_client-0. | 38 KB     | ##################################### | 100% \n",
      "python-dateutil-2.8. | 219 KB    | ##################################### | 100% \n",
      "r-cluster-2.1.0      | 554 KB    | ##################################### | 100% \n",
      "r-codetools-0.2_16   | 105 KB    | ##################################### | 100% \n",
      "bioconductor-s4vecto | 2.0 MB    | ##################################### | 100% \n",
      "r-jsonlite-1.6       | 971 KB    | ##################################### | 100% \n",
      "libxml2-2.9.9        | 1.3 MB    | ##################################### | 100% \n",
      "r-timedate-3043.102  | 1.6 MB    | ##################################### | 100% \n",
      "pygments-2.4.2       | 661 KB    | ##################################### | 100% \n",
      "bwidget-1.9.13       | 119 KB    | ##################################### | 100% \n",
      "r-htmltable-1.13.2   | 330 KB    | ##################################### | 100% \n",
      "r-quantmod-0.4_15    | 989 KB    | ##################################### | 100% \n",
      "gfortran_impl_linux- | 9.4 MB    | ##################################### | 100% \n",
      "r-xml2-1.2.2         | 359 KB    | ##################################### | 100% \n",
      "cairo-1.16.0         | 1.5 MB    | ##################################### | 100% \n",
      "nbformat-4.4.0       | 91 KB     | ##################################### | 100% \n",
      "r-yaml-2.2.0         | 113 KB    | ##################################### | 100% \n",
      "r-ipred-0.9_9        | 394 KB    | ##################################### | 100% \n",
      "r-blob-1.2.0         | 48 KB     | ##################################### | 100% \n",
      "freetype-2.10.0      | 884 KB    | ##################################### | 100% \n",
      "r-locfit-1.5_9.1     | 556 KB    | ##################################### | 100% \n",
      "bioconductor-annotat | 2.1 MB    | ##################################### | 100% \n",
      "tktable-2.10         | 89 KB     | ##################################### | 100% \n",
      "r-mass-7.3_51.4      | 1.1 MB    | ##################################### | 100% \n",
      "six-1.12.0           | 22 KB     | ##################################### | 100% \n",
      "jedi-0.15.1          | 717 KB    | ##################################### | 100% \n",
      "r-xtable-1.8_4       | 699 KB    | ##################################### | 100% \n",
      "r-checkmate-1.9.4    | 647 KB    | ##################################### | 100% \n",
      "more-itertools-7.2.0 | 48 KB     | ##################################### | 100% \n",
      "bleach-3.1.0         | 110 KB    | ##################################### | 100% \n",
      "r-formatr-1.7        | 166 KB    | ##################################### | 100% \n",
      "r-bitops-1.0_6       | 39 KB     | ##################################### | 100% \n",
      "xorg-renderproto-0.1 | 8 KB      | ##################################### | 100% \n",
      "r-sys-3.3            | 48 KB     | ##################################### | 100% \n",
      "r-ggplot2-3.2.1      | 3.9 MB    | ##################################### | 100% \n",
      "icu-64.2             | 12.6 MB   | ##################################### | 100% \n",
      "r-progress-1.2.2     | 89 KB     | ##################################### | 100% \n",
      "gsl-2.5              | 2.7 MB    | ##################################### | 100% \n",
      "bioconductor-deseq2- | 2.5 MB    | ##################################### | 100% \n",
      "r-caret-6.0_84       | 6.0 MB    | ##################################### | 100% \n",
      "r-openssl-1.4.1      | 1.2 MB    | ##################################### | 100% \n",
      "testpath-0.4.2       | 85 KB     | ##################################### | 100% \n",
      "r-prodlim-2018.04.18 | 414 KB    | ##################################### | 100% \n",
      "r-pillar-1.4.2       | 194 KB    | ##################################### | 100% \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pixman-0.38.0        | 594 KB    | ##################################### | 100% \n",
      "zstd-1.4.0           | 928 KB    | ##################################### | 100% \n",
      "decorator-4.4.0      | 11 KB     | ##################################### | 100% \n",
      "r-generics-0.0.2     | 77 KB     | ##################################### | 100% \n",
      "libcblas-3.8.0       | 10 KB     | ##################################### | 100% \n",
      "r-markdown-1.1       | 143 KB    | ##################################### | 100% \n",
      "r-colorspace-1.4_1   | 2.4 MB    | ##################################### | 100% \n",
      "r-acepack-1.4.1      | 70 KB     | ##################################### | 100% \n",
      "Preparing transaction: done\n",
      "Verifying transaction: done\n",
      "Executing transaction: done\n",
      "#\n",
      "# To activate this environment, use\n",
      "#\n",
      "#     $ conda activate DEG\n",
      "#\n",
      "# To deactivate an active environment, use\n",
      "#\n",
      "#     $ conda deactivate\n",
      "\n",
      "(alignment) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "## all done with alignment and counting!\n",
    "# good job!\n",
    "\n",
    "# lets prepare an R environment using conda to analyze our data\n",
    "# and discuss visualization\n",
    "\n",
    "conda create -y --name DEG python=3.6 r-essentials r-base r-ggplot2 bioconductor-deseq2 r-pheatmap r-venndiagram r-randomforest"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "IF you prefer, the DEG analysis can be completed on your own computer  \n",
    "We will use the following packages in R  \n",
    "ggplot2, deseq2, pheatmap, venndiagram, and randomforest  \n",
    "DESeq2 can be retrieved from bioconductor"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# login to your instance using the web desktop\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(alignment) (DEG) "
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# switch conda environments\n",
    "conda activate DEG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# open a command line\n",
    "# run R with these options to open a GUI\n",
    "R -g Tk &"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
